<!DOCTYPE html>
<!-- saved from url=(0067)https://www.sciencedirect.com/science/article/pii/S1525157822001337 -->
<html lang="en-US" class="" data-sd-ui-what-input="mouse"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
      <meta name="citation_pii" content="S1525157822001337">
<meta name="citation_issn" content="1525-1578">
<meta name="citation_volume" content="24">
<meta name="citation_lastpage" content="877">
<meta name="citation_issue" content="8">
<meta name="citation_publisher" content="Elsevier">
<meta name="citation_firstpage" content="867">
<meta name="citation_fulltext_world_readable" content="">
<meta name="citation_journal_title" content="The Journal of Molecular Diagnostics">
<meta name="citation_type" content="JOUR">
<meta name="citation_doi" content="10.1016/j.jmoldx.2022.04.007">
<meta name="dc.identifier" content="10.1016/j.jmoldx.2022.04.007">
<meta name="citation_article_type" content="Full-length article">
<meta property="og:description" content="Detection of serum embryonic miRNAs miR-371a-3p and miR-372-3p has been proposed to aid in diagnosis, prognosis, and management of patients with testi…">
<meta property="og:image" content="https://ars.els-cdn.com/content/image/1-s2.0-S1525157821X00190-cov150h.gif">
<meta name="citation_title" content="Analytical Validation and Performance Characteristics of Molecular Serum Biomarkers, miR-371a-3p and miR-372-3p, for Male Germ Cell Tumors, in a Clinical Laboratory Setting">
<meta property="og:title" content="Analytical Validation and Performance Characteristics of Molecular Serum Biomarkers, miR-371a-3p and miR-372-3p, for Male Germ Cell Tumors, in a Clinical Laboratory Setting">
<meta name="citation_publication_date" content="2022/08/01">
<meta name="citation_online_date" content="2022/08/04">
<meta name="robots" content="INDEX,FOLLOW,NOARCHIVE,NOCACHE,NOODP,NOYDIR">
      <title>Analytical Validation and Performance Characteristics of Molecular Serum Biomarkers, miR-371a-3p and miR-372-3p, for Male Germ Cell Tumors, in a Clinical Laboratory Setting - ScienceDirect</title>
      <link rel="canonical" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337">
      <meta name="tdm-reservation" content="1">
      <meta name="tdm-policy" content="https://www.elsevier.com/tdm/tdmrep-policy.json">
      <meta property="og:type" content="article">
      <meta name="viewport" content="initial-scale=1">
      <meta name="SDTech" content="Proudly brought to you by the SD Technology team">
      <script async="" src="./ye2022analytical_files/pendo.js"></script><script type="text/javascript">(function newRelicBrowserProSPA() {
  ;
  window.NREUM || (NREUM = {});
  NREUM.init = {
    privacy: {
      cookies_enabled: false
    },
    ajax: {
      deny_list: ["bam-cell.nr-data.net"]
    }
  };
  ;
  NREUM.loader_config = {
    accountID: "2128461",
    trustKey: "2038175",
    agentID: "1118783207",
    licenseKey: "7ac4127487",
    applicationID: "814813181"
  };
  ;
  NREUM.info = {
    beacon: "bam.nr-data.net",
    errorBeacon: "bam.nr-data.net",
    licenseKey: "7ac4127487",
    applicationID: "814813181",
    sa: 1
  };
  ; /*! For license information please see nr-loader-spa-1.238.0.min.js.LICENSE.txt */
  (() => {
    "use strict";

    var e,
      t,
      r = {
        5763: (e, t, r) => {
          r.d(t, {
            P_: () => f,
            Mt: () => p,
            C5: () => s,
            DL: () => v,
            OP: () => T,
            lF: () => D,
            Yu: () => y,
            Dg: () => h,
            CX: () => c,
            GE: () => b,
            sU: () => _
          });
          var n = r(8632),
            i = r(9567);
          const o = {
              beacon: n.ce.beacon,
              errorBeacon: n.ce.errorBeacon,
              licenseKey: void 0,
              applicationID: void 0,
              sa: void 0,
              queueTime: void 0,
              applicationTime: void 0,
              ttGuid: void 0,
              user: void 0,
              account: void 0,
              product: void 0,
              extra: void 0,
              jsAttributes: {},
              userAttributes: void 0,
              atts: void 0,
              transactionName: void 0,
              tNamePlain: void 0
            },
            a = {};
          function s(e) {
            if (!e) throw new Error("All info objects require an agent identifier!");
            if (!a[e]) throw new Error("Info for ".concat(e, " was never set"));
            return a[e];
          }
          function c(e, t) {
            if (!e) throw new Error("All info objects require an agent identifier!");
            a[e] = (0, i.D)(t, o), (0, n.Qy)(e, a[e], "info");
          }
          var u = r(7056);
          const d = () => {
              const e = {
                blockSelector: "[data-nr-block]",
                maskInputOptions: {
                  password: !0
                }
              };
              return {
                allow_bfcache: !0,
                privacy: {
                  cookies_enabled: !0
                },
                ajax: {
                  deny_list: void 0,
                  block_internal: !0,
                  enabled: !0,
                  harvestTimeSeconds: 10
                },
                distributed_tracing: {
                  enabled: void 0,
                  exclude_newrelic_header: void 0,
                  cors_use_newrelic_header: void 0,
                  cors_use_tracecontext_headers: void 0,
                  allowed_origins: void 0
                },
                session: {
                  domain: void 0,
                  expiresMs: u.oD,
                  inactiveMs: u.Hb
                },
                ssl: void 0,
                obfuscate: void 0,
                jserrors: {
                  enabled: !0,
                  harvestTimeSeconds: 10
                },
                metrics: {
                  enabled: !0
                },
                page_action: {
                  enabled: !0,
                  harvestTimeSeconds: 30
                },
                page_view_event: {
                  enabled: !0
                },
                page_view_timing: {
                  enabled: !0,
                  harvestTimeSeconds: 30,
                  long_task: !1
                },
                session_trace: {
                  enabled: !0,
                  harvestTimeSeconds: 10
                },
                harvest: {
                  tooManyRequestsDelay: 60
                },
                session_replay: {
                  enabled: !1,
                  harvestTimeSeconds: 60,
                  sampleRate: .1,
                  errorSampleRate: .1,
                  maskTextSelector: "*",
                  maskAllInputs: !0,
                  get blockClass() {
                    return "nr-block";
                  },
                  get ignoreClass() {
                    return "nr-ignore";
                  },
                  get maskTextClass() {
                    return "nr-mask";
                  },
                  get blockSelector() {
                    return e.blockSelector;
                  },
                  set blockSelector(t) {
                    e.blockSelector += ",".concat(t);
                  },
                  get maskInputOptions() {
                    return e.maskInputOptions;
                  },
                  set maskInputOptions(t) {
                    e.maskInputOptions = {
                      ...t,
                      password: !0
                    };
                  }
                },
                spa: {
                  enabled: !0,
                  harvestTimeSeconds: 10
                }
              };
            },
            l = {};
          function f(e) {
            if (!e) throw new Error("All configuration objects require an agent identifier!");
            if (!l[e]) throw new Error("Configuration for ".concat(e, " was never set"));
            return l[e];
          }
          function h(e, t) {
            if (!e) throw new Error("All configuration objects require an agent identifier!");
            l[e] = (0, i.D)(t, d()), (0, n.Qy)(e, l[e], "config");
          }
          function p(e, t) {
            if (!e) throw new Error("All configuration objects require an agent identifier!");
            var r = f(e);
            if (r) {
              for (var n = t.split("."), i = 0; i < n.length - 1; i++) if ("object" != typeof (r = r[n[i]])) return;
              r = r[n[n.length - 1]];
            }
            return r;
          }
          const g = {
              accountID: void 0,
              trustKey: void 0,
              agentID: void 0,
              licenseKey: void 0,
              applicationID: void 0,
              xpid: void 0
            },
            m = {};
          function v(e) {
            if (!e) throw new Error("All loader-config objects require an agent identifier!");
            if (!m[e]) throw new Error("LoaderConfig for ".concat(e, " was never set"));
            return m[e];
          }
          function b(e, t) {
            if (!e) throw new Error("All loader-config objects require an agent identifier!");
            m[e] = (0, i.D)(t, g), (0, n.Qy)(e, m[e], "loader_config");
          }
          const y = (0, n.mF)().o;
          var w = r(385),
            A = r(6818);
          const x = {
              buildEnv: A.Re,
              bytesSent: {},
              queryBytesSent: {},
              customTransaction: void 0,
              disabled: !1,
              distMethod: A.gF,
              isolatedBacklog: !1,
              loaderType: void 0,
              maxBytes: 3e4,
              offset: Math.floor(w._A?.performance?.timeOrigin || w._A?.performance?.timing?.navigationStart || Date.now()),
              onerror: void 0,
              origin: "" + w._A.location,
              ptid: void 0,
              releaseIds: {},
              session: void 0,
              xhrWrappable: "function" == typeof w._A.XMLHttpRequest?.prototype?.addEventListener,
              version: A.q4,
              denyList: void 0
            },
            E = {};
          function T(e) {
            if (!e) throw new Error("All runtime objects require an agent identifier!");
            if (!E[e]) throw new Error("Runtime for ".concat(e, " was never set"));
            return E[e];
          }
          function _(e, t) {
            if (!e) throw new Error("All runtime objects require an agent identifier!");
            E[e] = (0, i.D)(t, x), (0, n.Qy)(e, E[e], "runtime");
          }
          function D(e) {
            return function (e) {
              try {
                const t = s(e);
                return !!t.licenseKey && !!t.errorBeacon && !!t.applicationID;
              } catch (e) {
                return !1;
              }
            }(e);
          }
        },
        9567: (e, t, r) => {
          r.d(t, {
            D: () => i
          });
          var n = r(50);
          function i(e, t) {
            try {
              if (!e || "object" != typeof e) return (0, n.Z)("Setting a Configurable requires an object as input");
              if (!t || "object" != typeof t) return (0, n.Z)("Setting a Configurable requires a model to set its initial properties");
              const r = Object.create(Object.getPrototypeOf(t), Object.getOwnPropertyDescriptors(t)),
                o = 0 === Object.keys(r).length ? e : r;
              for (let a in o) if (void 0 !== e[a]) try {
                "object" == typeof e[a] && "object" == typeof t[a] ? r[a] = i(e[a], t[a]) : r[a] = e[a];
              } catch (e) {
                (0, n.Z)("An error occurred while setting a property of a Configurable", e);
              }
              return r;
            } catch (e) {
              (0, n.Z)("An error occured while setting a Configurable", e);
            }
          }
        },
        6818: (e, t, r) => {
          r.d(t, {
            Re: () => i,
            gF: () => o,
            q4: () => n
          });
          const n = "1.238.0",
            i = "PROD",
            o = "CDN";
        },
        385: (e, t, r) => {
          r.d(t, {
            FN: () => a,
            IF: () => u,
            Nk: () => l,
            Tt: () => s,
            _A: () => o,
            il: () => n,
            pL: () => c,
            v6: () => i,
            w1: () => d
          });
          const n = "undefined" != typeof window && !!window.document,
            i = "undefined" != typeof WorkerGlobalScope && ("undefined" != typeof self && self instanceof WorkerGlobalScope && self.navigator instanceof WorkerNavigator || "undefined" != typeof globalThis && globalThis instanceof WorkerGlobalScope && globalThis.navigator instanceof WorkerNavigator),
            o = n ? window : "undefined" != typeof WorkerGlobalScope && ("undefined" != typeof self && self instanceof WorkerGlobalScope && self || "undefined" != typeof globalThis && globalThis instanceof WorkerGlobalScope && globalThis),
            a = "" + o?.location,
            s = /iPad|iPhone|iPod/.test(navigator.userAgent),
            c = s && "undefined" == typeof SharedWorker,
            u = (() => {
              const e = navigator.userAgent.match(/Firefox[/\s](\d+\.\d+)/);
              return Array.isArray(e) && e.length >= 2 ? +e[1] : 0;
            })(),
            d = Boolean(n && window.document.documentMode),
            l = !!navigator.sendBeacon;
        },
        1117: (e, t, r) => {
          r.d(t, {
            w: () => o
          });
          var n = r(50);
          const i = {
            agentIdentifier: "",
            ee: void 0
          };
          class o {
            constructor(e) {
              try {
                if ("object" != typeof e) return (0, n.Z)("shared context requires an object as input");
                this.sharedContext = {}, Object.assign(this.sharedContext, i), Object.entries(e).forEach(e => {
                  let [t, r] = e;
                  Object.keys(i).includes(t) && (this.sharedContext[t] = r);
                });
              } catch (e) {
                (0, n.Z)("An error occured while setting SharedContext", e);
              }
            }
          }
        },
        8e3: (e, t, r) => {
          r.d(t, {
            L: () => d,
            R: () => c
          });
          var n = r(8325),
            i = r(1284),
            o = r(4322),
            a = r(3325);
          const s = {};
          function c(e, t) {
            const r = {
              staged: !1,
              priority: a.p[t] || 0
            };
            u(e), s[e].get(t) || s[e].set(t, r);
          }
          function u(e) {
            e && (s[e] || (s[e] = new Map()));
          }
          function d() {
            let e = arguments.length > 0 && void 0 !== arguments[0] ? arguments[0] : "",
              t = arguments.length > 1 && void 0 !== arguments[1] ? arguments[1] : "feature";
            if (u(e), !e || !s[e].get(t)) return a(t);
            s[e].get(t).staged = !0;
            const r = [...s[e]];
            function a(t) {
              const r = e ? n.ee.get(e) : n.ee,
                a = o.X.handlers;
              if (r.backlog && a) {
                var s = r.backlog[t],
                  c = a[t];
                if (c) {
                  for (var u = 0; s && u < s.length; ++u) l(s[u], c);
                  (0, i.D)(c, function (e, t) {
                    (0, i.D)(t, function (t, r) {
                      r[0].on(e, r[1]);
                    });
                  });
                }
                delete a[t], r.backlog[t] = null, r.emit("drain-" + t, []);
              }
            }
            r.every(e => {
              let [t, r] = e;
              return r.staged;
            }) && (r.sort((e, t) => e[1].priority - t[1].priority), r.forEach(e => {
              let [t] = e;
              a(t);
            }));
          }
          function l(e, t) {
            var r = e[1];
            (0, i.D)(t[r], function (t, r) {
              var n = e[0];
              if (r[0] === n) {
                var i = r[1],
                  o = e[3],
                  a = e[2];
                i.apply(o, a);
              }
            });
          }
        },
        8325: (e, t, r) => {
          r.d(t, {
            A: () => c,
            ee: () => u
          });
          var n = r(8632),
            i = r(2210),
            o = r(5763);
          class a {
            constructor(e) {
              this.contextId = e;
            }
          }
          var s = r(3117);
          const c = "nr@context:".concat(s.a),
            u = function e(t, r) {
              var n = {},
                s = {},
                d = {},
                f = !1;
              try {
                f = 16 === r.length && (0, o.OP)(r).isolatedBacklog;
              } catch (e) {}
              var h = {
                on: g,
                addEventListener: g,
                removeEventListener: function (e, t) {
                  var r = n[e];
                  if (!r) return;
                  for (var i = 0; i < r.length; i++) r[i] === t && r.splice(i, 1);
                },
                emit: function (e, r, n, i, o) {
                  !1 !== o && (o = !0);
                  if (u.aborted && !i) return;
                  t && o && t.emit(e, r, n);
                  for (var a = p(n), c = m(e), d = c.length, l = 0; l < d; l++) c[l].apply(a, r);
                  var f = b()[s[e]];
                  f && f.push([h, e, r, a]);
                  return a;
                },
                get: v,
                listeners: m,
                context: p,
                buffer: function (e, t) {
                  const r = b();
                  if (t = t || "feature", h.aborted) return;
                  Object.entries(e || {}).forEach(e => {
                    let [n, i] = e;
                    s[i] = t, t in r || (r[t] = []);
                  });
                },
                abort: l,
                aborted: !1,
                isBuffering: function (e) {
                  return !!b()[s[e]];
                },
                debugId: r,
                backlog: f ? {} : t && "object" == typeof t.backlog ? t.backlog : {}
              };
              return h;
              function p(e) {
                return e && e instanceof a ? e : e ? (0, i.X)(e, c, () => new a(c)) : new a(c);
              }
              function g(e, t) {
                n[e] = m(e).concat(t);
              }
              function m(e) {
                return n[e] || [];
              }
              function v(t) {
                return d[t] = d[t] || e(h, t);
              }
              function b() {
                return h.backlog;
              }
            }(void 0, "globalEE"),
            d = (0, n.fP)();
          function l() {
            u.aborted = !0, u.backlog = {};
          }
          d.ee || (d.ee = u);
        },
        5546: (e, t, r) => {
          r.d(t, {
            E: () => n,
            p: () => i
          });
          var n = r(8325).ee.get("handle");
          function i(e, t, r, i, o) {
            o ? (o.buffer([e], i), o.emit(e, t, r)) : (n.buffer([e], i), n.emit(e, t, r));
          }
        },
        4322: (e, t, r) => {
          r.d(t, {
            X: () => o
          });
          var n = r(5546);
          o.on = a;
          var i = o.handlers = {};
          function o(e, t, r, o) {
            a(o || n.E, i, e, t, r);
          }
          function a(e, t, r, i, o) {
            o || (o = "feature"), e || (e = n.E);
            var a = t[o] = t[o] || {};
            (a[r] = a[r] || []).push([e, i]);
          }
        },
        3239: (e, t, r) => {
          r.d(t, {
            bP: () => s,
            iz: () => c,
            m$: () => a
          });
          var n = r(385);
          let i = !1,
            o = !1;
          try {
            const e = {
              get passive() {
                return i = !0, !1;
              },
              get signal() {
                return o = !0, !1;
              }
            };
            n._A.addEventListener("test", null, e), n._A.removeEventListener("test", null, e);
          } catch (e) {}
          function a(e, t) {
            return i || o ? {
              capture: !!e,
              passive: i,
              signal: t
            } : !!e;
          }
          function s(e, t) {
            let r = arguments.length > 2 && void 0 !== arguments[2] && arguments[2],
              n = arguments.length > 3 ? arguments[3] : void 0;
            window.addEventListener(e, t, a(r, n));
          }
          function c(e, t) {
            let r = arguments.length > 2 && void 0 !== arguments[2] && arguments[2],
              n = arguments.length > 3 ? arguments[3] : void 0;
            document.addEventListener(e, t, a(r, n));
          }
        },
        3117: (e, t, r) => {
          r.d(t, {
            a: () => n
          });
          const n = (0, r(4402).Rl)();
        },
        4402: (e, t, r) => {
          r.d(t, {
            Ht: () => u,
            M: () => c,
            Rl: () => a,
            ky: () => s
          });
          var n = r(385);
          const i = "xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx";
          function o(e, t) {
            return e ? 15 & e[t] : 16 * Math.random() | 0;
          }
          function a() {
            const e = n._A?.crypto || n._A?.msCrypto;
            let t,
              r = 0;
            return e && e.getRandomValues && (t = e.getRandomValues(new Uint8Array(31))), i.split("").map(e => "x" === e ? o(t, ++r).toString(16) : "y" === e ? (3 & o() | 8).toString(16) : e).join("");
          }
          function s(e) {
            const t = n._A?.crypto || n._A?.msCrypto;
            let r,
              i = 0;
            t && t.getRandomValues && (r = t.getRandomValues(new Uint8Array(31)));
            const a = [];
            for (var s = 0; s < e; s++) a.push(o(r, ++i).toString(16));
            return a.join("");
          }
          function c() {
            return s(16);
          }
          function u() {
            return s(32);
          }
        },
        7056: (e, t, r) => {
          r.d(t, {
            Bq: () => n,
            Hb: () => o,
            oD: () => i
          });
          const n = "NRBA",
            i = 144e5,
            o = 18e5;
        },
        7894: (e, t, r) => {
          function n() {
            return Math.round(performance.now());
          }
          r.d(t, {
            z: () => n
          });
        },
        7243: (e, t, r) => {
          r.d(t, {
            e: () => o
          });
          var n = r(385),
            i = {};
          function o(e) {
            if (e in i) return i[e];
            if (0 === (e || "").indexOf("data:")) return {
              protocol: "data"
            };
            let t;
            var r = n._A?.location,
              o = {};
            if (n.il) t = document.createElement("a"), t.href = e;else try {
              t = new URL(e, r.href);
            } catch (e) {
              return o;
            }
            o.port = t.port;
            var a = t.href.split("://");
            !o.port && a[1] && (o.port = a[1].split("/")[0].split("@").pop().split(":")[1]), o.port && "0" !== o.port || (o.port = "https" === a[0] ? "443" : "80"), o.hostname = t.hostname || r.hostname, o.pathname = t.pathname, o.protocol = a[0], "/" !== o.pathname.charAt(0) && (o.pathname = "/" + o.pathname);
            var s = !t.protocol || ":" === t.protocol || t.protocol === r.protocol,
              c = t.hostname === r.hostname && t.port === r.port;
            return o.sameOrigin = s && (!t.hostname || c), "/" === o.pathname && (i[e] = o), o;
          }
        },
        50: (e, t, r) => {
          function n(e, t) {
            "function" == typeof console.warn && (console.warn("New Relic: ".concat(e)), t && console.warn(t));
          }
          r.d(t, {
            Z: () => n
          });
        },
        2587: (e, t, r) => {
          r.d(t, {
            N: () => c,
            T: () => u
          });
          var n = r(8325),
            i = r(5546),
            o = r(8e3),
            a = r(3325);
          const s = {
            stn: [a.D.sessionTrace],
            err: [a.D.jserrors, a.D.metrics],
            ins: [a.D.pageAction],
            spa: [a.D.spa],
            sr: [a.D.sessionReplay, a.D.sessionTrace]
          };
          function c(e, t) {
            const r = n.ee.get(t);
            e && "object" == typeof e && (Object.entries(e).forEach(e => {
              let [t, n] = e;
              void 0 === u[t] && (s[t] ? s[t].forEach(e => {
                n ? (0, i.p)("feat-" + t, [], void 0, e, r) : (0, i.p)("block-" + t, [], void 0, e, r), (0, i.p)("rumresp-" + t, [Boolean(n)], void 0, e, r);
              }) : n && (0, i.p)("feat-" + t, [], void 0, void 0, r), u[t] = Boolean(n));
            }), Object.keys(s).forEach(e => {
              void 0 === u[e] && (s[e]?.forEach(t => (0, i.p)("rumresp-" + e, [!1], void 0, t, r)), u[e] = !1);
            }), (0, o.L)(t, a.D.pageViewEvent));
          }
          const u = {};
        },
        2210: (e, t, r) => {
          r.d(t, {
            X: () => i
          });
          var n = Object.prototype.hasOwnProperty;
          function i(e, t, r) {
            if (n.call(e, t)) return e[t];
            var i = r();
            if (Object.defineProperty && Object.keys) try {
              return Object.defineProperty(e, t, {
                value: i,
                writable: !0,
                enumerable: !1
              }), i;
            } catch (e) {}
            return e[t] = i, i;
          }
        },
        1284: (e, t, r) => {
          r.d(t, {
            D: () => n
          });
          const n = (e, t) => Object.entries(e || {}).map(e => {
            let [r, n] = e;
            return t(r, n);
          });
        },
        4351: (e, t, r) => {
          r.d(t, {
            P: () => o
          });
          var n = r(8325);
          const i = () => {
            const e = new WeakSet();
            return (t, r) => {
              if ("object" == typeof r && null !== r) {
                if (e.has(r)) return;
                e.add(r);
              }
              return r;
            };
          };
          function o(e) {
            try {
              return JSON.stringify(e, i());
            } catch (e) {
              try {
                n.ee.emit("internal-error", [e]);
              } catch (e) {}
            }
          }
        },
        3960: (e, t, r) => {
          r.d(t, {
            K: () => a,
            b: () => o
          });
          var n = r(3239);
          function i() {
            return "undefined" == typeof document || "complete" === document.readyState;
          }
          function o(e, t) {
            if (i()) return e();
            (0, n.bP)("load", e, t);
          }
          function a(e) {
            if (i()) return e();
            (0, n.iz)("DOMContentLoaded", e);
          }
        },
        8632: (e, t, r) => {
          r.d(t, {
            EZ: () => u,
            Qy: () => c,
            ce: () => o,
            fP: () => a,
            gG: () => d,
            mF: () => s
          });
          var n = r(7894),
            i = r(385);
          const o = {
            beacon: "bam.nr-data.net",
            errorBeacon: "bam.nr-data.net"
          };
          function a() {
            return i._A.NREUM || (i._A.NREUM = {}), void 0 === i._A.newrelic && (i._A.newrelic = i._A.NREUM), i._A.NREUM;
          }
          function s() {
            let e = a();
            return e.o || (e.o = {
              ST: i._A.setTimeout,
              SI: i._A.setImmediate,
              CT: i._A.clearTimeout,
              XHR: i._A.XMLHttpRequest,
              REQ: i._A.Request,
              EV: i._A.Event,
              PR: i._A.Promise,
              MO: i._A.MutationObserver,
              FETCH: i._A.fetch
            }), e;
          }
          function c(e, t, r) {
            let i = a();
            const o = i.initializedAgents || {},
              s = o[e] || {};
            return Object.keys(s).length || (s.initializedAt = {
              ms: (0, n.z)(),
              date: new Date()
            }), i.initializedAgents = {
              ...o,
              [e]: {
                ...s,
                [r]: t
              }
            }, i;
          }
          function u(e, t) {
            a()[e] = t;
          }
          function d() {
            return function () {
              let e = a();
              const t = e.info || {};
              e.info = {
                beacon: o.beacon,
                errorBeacon: o.errorBeacon,
                ...t
              };
            }(), function () {
              let e = a();
              const t = e.init || {};
              e.init = {
                ...t
              };
            }(), s(), function () {
              let e = a();
              const t = e.loader_config || {};
              e.loader_config = {
                ...t
              };
            }(), a();
          }
        },
        7956: (e, t, r) => {
          r.d(t, {
            N: () => i
          });
          var n = r(3239);
          function i(e) {
            let t = arguments.length > 1 && void 0 !== arguments[1] && arguments[1],
              r = arguments.length > 2 ? arguments[2] : void 0,
              i = arguments.length > 3 ? arguments[3] : void 0;
            return void (0, n.iz)("visibilitychange", function () {
              if (t) return void ("hidden" == document.visibilityState && e());
              e(document.visibilityState);
            }, r, i);
          }
        },
        1214: (e, t, r) => {
          r.d(t, {
            em: () => b,
            u5: () => j,
            QU: () => O,
            _L: () => I,
            Gm: () => H,
            Lg: () => L,
            BV: () => G,
            Kf: () => K
          });
          var n = r(8325),
            i = r(3117);
          const o = "nr@original:".concat(i.a);
          var a = Object.prototype.hasOwnProperty,
            s = !1;
          function c(e, t) {
            return e || (e = n.ee), r.inPlace = function (e, t, n, i, o) {
              n || (n = "");
              const a = "-" === n.charAt(0);
              for (let s = 0; s < t.length; s++) {
                const c = t[s],
                  u = e[c];
                d(u) || (e[c] = r(u, a ? c + n : n, i, c, o));
              }
            }, r.flag = o, r;
            function r(t, r, n, s, c) {
              return d(t) ? t : (r || (r = ""), nrWrapper[o] = t, function (e, t, r) {
                if (Object.defineProperty && Object.keys) try {
                  return Object.keys(e).forEach(function (r) {
                    Object.defineProperty(t, r, {
                      get: function () {
                        return e[r];
                      },
                      set: function (t) {
                        return e[r] = t, t;
                      }
                    });
                  }), t;
                } catch (e) {
                  u([e], r);
                }
                for (var n in e) a.call(e, n) && (t[n] = e[n]);
              }(t, nrWrapper, e), nrWrapper);
              function nrWrapper() {
                var o, a, d, l;
                try {
                  a = this, o = [...arguments], d = "function" == typeof n ? n(o, a) : n || {};
                } catch (t) {
                  u([t, "", [o, a, s], d], e);
                }
                i(r + "start", [o, a, s], d, c);
                try {
                  return l = t.apply(a, o);
                } catch (e) {
                  throw i(r + "err", [o, a, e], d, c), e;
                } finally {
                  i(r + "end", [o, a, l], d, c);
                }
              }
            }
            function i(r, n, i, o) {
              if (!s || t) {
                var a = s;
                s = !0;
                try {
                  e.emit(r, n, i, t, o);
                } catch (t) {
                  u([t, r, n, i], e);
                }
                s = a;
              }
            }
          }
          function u(e, t) {
            t || (t = n.ee);
            try {
              t.emit("internal-error", e);
            } catch (e) {}
          }
          function d(e) {
            return !(e && e instanceof Function && e.apply && !e[o]);
          }
          var l = r(2210),
            f = r(385);
          const h = {},
            p = f._A.XMLHttpRequest,
            g = "addEventListener",
            m = "removeEventListener",
            v = "nr@wrapped:".concat(n.A);
          function b(e) {
            var t = function (e) {
              return (e || n.ee).get("events");
            }(e);
            if (h[t.debugId]++) return t;
            h[t.debugId] = 1;
            var r = c(t, !0);
            function i(e) {
              r.inPlace(e, [g, m], "-", o);
            }
            function o(e, t) {
              return e[1];
            }
            return "getPrototypeOf" in Object && (f.il && y(document, i), y(f._A, i), y(p.prototype, i)), t.on(g + "-start", function (e, t) {
              var n = e[1];
              if (null !== n && ("function" == typeof n || "object" == typeof n)) {
                var i = (0, l.X)(n, v, function () {
                  var e = {
                    object: function () {
                      if ("function" != typeof n.handleEvent) return;
                      return n.handleEvent.apply(n, arguments);
                    },
                    function: n
                  }[typeof n];
                  return e ? r(e, "fn-", null, e.name || "anonymous") : n;
                });
                this.wrapped = e[1] = i;
              }
            }), t.on(m + "-start", function (e) {
              e[1] = this.wrapped || e[1];
            }), t;
          }
          function y(e, t) {
            let r = e;
            for (; "object" == typeof r && !Object.prototype.hasOwnProperty.call(r, g);) r = Object.getPrototypeOf(r);
            for (var n = arguments.length, i = new Array(n > 2 ? n - 2 : 0), o = 2; o < n; o++) i[o - 2] = arguments[o];
            r && t(r, ...i);
          }
          var w = "fetch-",
            A = w + "body-",
            x = ["arrayBuffer", "blob", "json", "text", "formData"],
            E = f._A.Request,
            T = f._A.Response,
            _ = "prototype";
          const D = {};
          function j(e) {
            const t = function (e) {
              return (e || n.ee).get("fetch");
            }(e);
            if (!(E && T && f._A.fetch)) return t;
            if (D[t.debugId]++) return t;
            function r(e, r, i) {
              var o = e[r];
              "function" == typeof o && (e[r] = function () {
                var e,
                  r = [...arguments],
                  a = {};
                t.emit(i + "before-start", [r], a), a[n.A] && a[n.A].dt && (e = a[n.A].dt);
                var s = o.apply(this, r);
                return t.emit(i + "start", [r, e], s), s.then(function (e) {
                  return t.emit(i + "end", [null, e], s), e;
                }, function (e) {
                  throw t.emit(i + "end", [e], s), e;
                });
              });
            }
            return D[t.debugId] = 1, x.forEach(e => {
              r(E[_], e, A), r(T[_], e, A);
            }), r(f._A, "fetch", w), t.on(w + "end", function (e, r) {
              var n = this;
              if (r) {
                var i = r.headers.get("content-length");
                null !== i && (n.rxSize = i), t.emit(w + "done", [null, r], n);
              } else t.emit(w + "done", [e], n);
            }), t;
          }
          const C = {},
            N = ["pushState", "replaceState"];
          function O(e) {
            const t = function (e) {
              return (e || n.ee).get("history");
            }(e);
            return !f.il || C[t.debugId]++ || (C[t.debugId] = 1, c(t).inPlace(window.history, N, "-")), t;
          }
          var S = r(3239);
          const P = {},
            R = ["appendChild", "insertBefore", "replaceChild"];
          function I(e) {
            const t = function (e) {
              return (e || n.ee).get("jsonp");
            }(e);
            if (!f.il || P[t.debugId]) return t;
            P[t.debugId] = !0;
            var r = c(t),
              i = /[?&](?:callback|cb)=([^&#]+)/,
              o = /(.*)\.([^.]+)/,
              a = /^(\w+)(\.|$)(.*)$/;
            function s(e, t) {
              if (!e) return t;
              const r = e.match(a),
                n = r[1];
              return s(r[3], t[n]);
            }
            return r.inPlace(Node.prototype, R, "dom-"), t.on("dom-start", function (e) {
              !function (e) {
                if (!e || "string" != typeof e.nodeName || "script" !== e.nodeName.toLowerCase()) return;
                if ("function" != typeof e.addEventListener) return;
                var n = (a = e.src, c = a.match(i), c ? c[1] : null);
                var a, c;
                if (!n) return;
                var u = function (e) {
                  var t = e.match(o);
                  if (t && t.length >= 3) return {
                    key: t[2],
                    parent: s(t[1], window)
                  };
                  return {
                    key: e,
                    parent: window
                  };
                }(n);
                if ("function" != typeof u.parent[u.key]) return;
                var d = {};
                function l() {
                  t.emit("jsonp-end", [], d), e.removeEventListener("load", l, (0, S.m$)(!1)), e.removeEventListener("error", f, (0, S.m$)(!1));
                }
                function f() {
                  t.emit("jsonp-error", [], d), t.emit("jsonp-end", [], d), e.removeEventListener("load", l, (0, S.m$)(!1)), e.removeEventListener("error", f, (0, S.m$)(!1));
                }
                r.inPlace(u.parent, [u.key], "cb-", d), e.addEventListener("load", l, (0, S.m$)(!1)), e.addEventListener("error", f, (0, S.m$)(!1)), t.emit("new-jsonp", [e.src], d);
              }(e[0]);
            }), t;
          }
          const k = {};
          function H(e) {
            const t = function (e) {
              return (e || n.ee).get("mutation");
            }(e);
            if (!f.il || k[t.debugId]) return t;
            k[t.debugId] = !0;
            var r = c(t),
              i = f._A.MutationObserver;
            return i && (window.MutationObserver = function (e) {
              return this instanceof i ? new i(r(e, "fn-")) : i.apply(this, arguments);
            }, MutationObserver.prototype = i.prototype), t;
          }
          const z = {};
          function L(e) {
            const t = function (e) {
              return (e || n.ee).get("promise");
            }(e);
            if (z[t.debugId]) return t;
            z[t.debugId] = !0;
            var r = t.context,
              i = c(t),
              a = f._A.Promise;
            return a && function () {
              function e(r) {
                var n = t.context(),
                  o = i(r, "executor-", n, null, !1);
                const s = Reflect.construct(a, [o], e);
                return t.context(s).getCtx = function () {
                  return n;
                }, s;
              }
              f._A.Promise = e, Object.defineProperty(e, "name", {
                value: "Promise"
              }), e.toString = function () {
                return a.toString();
              }, Object.setPrototypeOf(e, a), ["all", "race"].forEach(function (r) {
                const n = a[r];
                e[r] = function (e) {
                  let i = !1;
                  [...(e || [])].forEach(e => {
                    this.resolve(e).then(a("all" === r), a(!1));
                  });
                  const o = n.apply(this, arguments);
                  return o;
                  function a(e) {
                    return function () {
                      t.emit("propagate", [null, !i], o, !1, !1), i = i || !e;
                    };
                  }
                };
              }), ["resolve", "reject"].forEach(function (r) {
                const n = a[r];
                e[r] = function (e) {
                  const r = n.apply(this, arguments);
                  return e !== r && t.emit("propagate", [e, !0], r, !1, !1), r;
                };
              }), e.prototype = a.prototype;
              const n = a.prototype.then;
              a.prototype.then = function () {
                var e = this,
                  o = r(e);
                o.promise = e;
                for (var a = arguments.length, s = new Array(a), c = 0; c < a; c++) s[c] = arguments[c];
                s[0] = i(s[0], "cb-", o, null, !1), s[1] = i(s[1], "cb-", o, null, !1);
                const u = n.apply(this, s);
                return o.nextPromise = u, t.emit("propagate", [e, !0], u, !1, !1), u;
              }, a.prototype.then[o] = n, t.on("executor-start", function (e) {
                e[0] = i(e[0], "resolve-", this, null, !1), e[1] = i(e[1], "resolve-", this, null, !1);
              }), t.on("executor-err", function (e, t, r) {
                e[1](r);
              }), t.on("cb-end", function (e, r, n) {
                t.emit("propagate", [n, !0], this.nextPromise, !1, !1);
              }), t.on("propagate", function (e, r, n) {
                this.getCtx && !r || (this.getCtx = function () {
                  if (e instanceof Promise) var r = t.context(e);
                  return r && r.getCtx ? r.getCtx() : this;
                });
              });
            }(), t;
          }
          const M = {},
            B = "setTimeout",
            F = "setInterval",
            U = "clearTimeout",
            q = "-start",
            Z = "-",
            V = [B, "setImmediate", F, U, "clearImmediate"];
          function G(e) {
            const t = function (e) {
              return (e || n.ee).get("timer");
            }(e);
            if (M[t.debugId]++) return t;
            M[t.debugId] = 1;
            var r = c(t);
            return r.inPlace(f._A, V.slice(0, 2), B + Z), r.inPlace(f._A, V.slice(2, 3), F + Z), r.inPlace(f._A, V.slice(3), U + Z), t.on(F + q, function (e, t, n) {
              e[0] = r(e[0], "fn-", null, n);
            }), t.on(B + q, function (e, t, n) {
              this.method = n, this.timerDuration = isNaN(e[1]) ? 0 : +e[1], e[0] = r(e[0], "fn-", this, n);
            }), t;
          }
          var W = r(50);
          const X = {},
            Q = ["open", "send"];
          function K(e) {
            var t = e || n.ee;
            const r = function (e) {
              return (e || n.ee).get("xhr");
            }(t);
            if (X[r.debugId]++) return r;
            X[r.debugId] = 1, b(t);
            var i = c(r),
              o = f._A.XMLHttpRequest,
              a = f._A.MutationObserver,
              s = f._A.Promise,
              u = f._A.setInterval,
              d = "readystatechange",
              l = ["onload", "onerror", "onabort", "onloadstart", "onloadend", "onprogress", "ontimeout"],
              h = [],
              p = f._A.XMLHttpRequest = function (e) {
                const t = new o(e),
                  n = r.context(t);
                try {
                  r.emit("new-xhr", [t], n), t.addEventListener(d, (a = n, function () {
                    var e = this;
                    e.readyState > 3 && !a.resolved && (a.resolved = !0, r.emit("xhr-resolved", [], e)), i.inPlace(e, l, "fn-", A);
                  }), (0, S.m$)(!1));
                } catch (e) {
                  (0, W.Z)("An error occurred while intercepting XHR", e);
                  try {
                    r.emit("internal-error", [e]);
                  } catch (e) {}
                }
                var a;
                return t;
              };
            function g(e, t) {
              i.inPlace(t, ["onreadystatechange"], "fn-", A);
            }
            if (function (e, t) {
              for (var r in e) t[r] = e[r];
            }(o, p), p.prototype = o.prototype, i.inPlace(p.prototype, Q, "-xhr-", A), r.on("send-xhr-start", function (e, t) {
              g(e, t), function (e) {
                h.push(e), a && (m ? m.then(w) : u ? u(w) : (v = -v, y.data = v));
              }(t);
            }), r.on("open-xhr-start", g), a) {
              var m = s && s.resolve();
              if (!u && !s) {
                var v = 1,
                  y = document.createTextNode(v);
                new a(w).observe(y, {
                  characterData: !0
                });
              }
            } else t.on("fn-end", function (e) {
              e[0] && e[0].type === d || w();
            });
            function w() {
              for (var e = 0; e < h.length; e++) g(0, h[e]);
              h.length && (h = []);
            }
            function A(e, t) {
              return t;
            }
            return r;
          }
        },
        7825: (e, t, r) => {
          r.d(t, {
            t: () => n
          });
          const n = r(3325).D.ajax;
        },
        6660: (e, t, r) => {
          r.d(t, {
            t: () => n
          });
          const n = r(3325).D.jserrors;
        },
        3081: (e, t, r) => {
          r.d(t, {
            gF: () => o,
            mY: () => i,
            t9: () => n,
            vz: () => s,
            xS: () => a
          });
          const n = r(3325).D.metrics,
            i = "sm",
            o = "cm",
            a = "storeSupportabilityMetrics",
            s = "storeEventMetrics";
        },
        4649: (e, t, r) => {
          r.d(t, {
            t: () => n
          });
          const n = r(3325).D.pageAction;
        },
        7633: (e, t, r) => {
          r.d(t, {
            Dz: () => i,
            OJ: () => a,
            qw: () => o,
            t9: () => n
          });
          const n = r(3325).D.pageViewEvent,
            i = "firstbyte",
            o = "domcontent",
            a = "windowload";
        },
        9251: (e, t, r) => {
          r.d(t, {
            t: () => n
          });
          const n = r(3325).D.pageViewTiming;
        },
        3614: (e, t, r) => {
          r.d(t, {
            BST_RESOURCE: () => i,
            END: () => s,
            FEATURE_NAME: () => n,
            FN_END: () => u,
            FN_START: () => c,
            PUSH_STATE: () => d,
            RESOURCE: () => o,
            START: () => a
          });
          const n = r(3325).D.sessionTrace,
            i = "bstResource",
            o = "resource",
            a = "-start",
            s = "-end",
            c = "fn" + a,
            u = "fn" + s,
            d = "pushState";
        },
        7836: (e, t, r) => {
          r.d(t, {
            BODY: () => x,
            CB_END: () => E,
            CB_START: () => u,
            END: () => A,
            FEATURE_NAME: () => i,
            FETCH: () => _,
            FETCH_BODY: () => v,
            FETCH_DONE: () => m,
            FETCH_START: () => g,
            FN_END: () => c,
            FN_START: () => s,
            INTERACTION: () => f,
            INTERACTION_API: () => d,
            INTERACTION_EVENTS: () => o,
            JSONP_END: () => b,
            JSONP_NODE: () => p,
            JS_TIME: () => T,
            MAX_TIMER_BUDGET: () => a,
            REMAINING: () => l,
            SPA_NODE: () => h,
            START: () => w,
            originalSetTimeout: () => y
          });
          var n = r(5763);
          const i = r(3325).D.spa,
            o = ["click", "submit", "keypress", "keydown", "keyup", "change"],
            a = 999,
            s = "fn-start",
            c = "fn-end",
            u = "cb-start",
            d = "api-ixn-",
            l = "remaining",
            f = "interaction",
            h = "spaNode",
            p = "jsonpNode",
            g = "fetch-start",
            m = "fetch-done",
            v = "fetch-body-",
            b = "jsonp-end",
            y = n.Yu.ST,
            w = "-start",
            A = "-end",
            x = "-body",
            E = "cb" + A,
            T = "jsTime",
            _ = "fetch";
        },
        5938: (e, t, r) => {
          r.d(t, {
            W: () => o
          });
          var n = r(5763),
            i = r(8325);
          class o {
            constructor(e, t, r) {
              this.agentIdentifier = e, this.aggregator = t, this.ee = i.ee.get(e, (0, n.OP)(this.agentIdentifier).isolatedBacklog), this.featureName = r, this.blocked = !1;
            }
          }
        },
        9144: (e, t, r) => {
          r.d(t, {
            j: () => m
          });
          var n = r(3325),
            i = r(5763),
            o = r(5546),
            a = r(8325),
            s = r(7894),
            c = r(8e3),
            u = r(3960),
            d = r(385),
            l = r(50),
            f = r(3081),
            h = r(8632);
          function p() {
            const e = (0, h.gG)();
            ["setErrorHandler", "finished", "addToTrace", "inlineHit", "addRelease", "addPageAction", "setCurrentRouteName", "setPageViewName", "setCustomAttribute", "interaction", "noticeError", "setUserId", "setApplicationVersion"].forEach(t => {
              e[t] = function () {
                for (var r = arguments.length, n = new Array(r), i = 0; i < r; i++) n[i] = arguments[i];
                return function (t) {
                  for (var r = arguments.length, n = new Array(r > 1 ? r - 1 : 0), i = 1; i < r; i++) n[i - 1] = arguments[i];
                  let o = [];
                  return Object.values(e.initializedAgents).forEach(e => {
                    e.exposed && e.api[t] && o.push(e.api[t](...n));
                  }), o.length > 1 ? o : o[0];
                }(t, ...n);
              };
            });
          }
          var g = r(2587);
          function m(e) {
            let t = arguments.length > 1 && void 0 !== arguments[1] ? arguments[1] : {},
              m = arguments.length > 2 ? arguments[2] : void 0,
              v = arguments.length > 3 ? arguments[3] : void 0,
              {
                init: b,
                info: y,
                loader_config: w,
                runtime: A = {
                  loaderType: m
                },
                exposed: x = !0
              } = t;
            const E = (0, h.gG)();
            y || (b = E.init, y = E.info, w = E.loader_config), (0, i.Dg)(e, b || {}), (0, i.GE)(e, w || {}), y.jsAttributes ??= {}, d.v6 && (y.jsAttributes.isWorker = !0), (0, i.CX)(e, y);
            const T = (0, i.P_)(e);
            A.denyList = [...(T.ajax?.deny_list || []), ...(T.ajax?.block_internal ? [y.beacon, y.errorBeacon] : [])], (0, i.sU)(e, A), p();
            const _ = function (e, t) {
              t || (0, c.R)(e, "api");
              const h = {};
              var p = a.ee.get(e),
                g = p.get("tracer"),
                m = "api-",
                v = m + "ixn-";
              function b(t, r, n, o) {
                const a = (0, i.C5)(e);
                return null === r ? delete a.jsAttributes[t] : (0, i.CX)(e, {
                  ...a,
                  jsAttributes: {
                    ...a.jsAttributes,
                    [t]: r
                  }
                }), A(m, n, !0, o || null === r ? "session" : void 0)(t, r);
              }
              function y() {}
              ["setErrorHandler", "finished", "addToTrace", "inlineHit", "addRelease"].forEach(e => h[e] = A(m, e, !0, "api")), h.addPageAction = A(m, "addPageAction", !0, n.D.pageAction), h.setCurrentRouteName = A(m, "routeName", !0, n.D.spa), h.setPageViewName = function (t, r) {
                if ("string" == typeof t) return "/" !== t.charAt(0) && (t = "/" + t), (0, i.OP)(e).customTransaction = (r || "http://custom.transaction") + t, A(m, "setPageViewName", !0)();
              }, h.setCustomAttribute = function (e, t) {
                let r = arguments.length > 2 && void 0 !== arguments[2] && arguments[2];
                if ("string" == typeof e) {
                  if (["string", "number"].includes(typeof t) || null === t) return b(e, t, "setCustomAttribute", r);
                  (0, l.Z)("Failed to execute setCustomAttribute.\nNon-null value must be a string or number type, but a type of <".concat(typeof t, "> was provided."));
                } else (0, l.Z)("Failed to execute setCustomAttribute.\nName must be a string type, but a type of <".concat(typeof e, "> was provided."));
              }, h.setUserId = function (e) {
                if ("string" == typeof e || null === e) return b("enduser.id", e, "setUserId", !0);
                (0, l.Z)("Failed to execute setUserId.\nNon-null value must be a string type, but a type of <".concat(typeof e, "> was provided."));
              }, h.setApplicationVersion = function (e) {
                if ("string" == typeof e || null === e) return b("application.version", e, "setApplicationVersion", !1);
                (0, l.Z)("Failed to execute setApplicationVersion. Expected <String | null>, but got <".concat(typeof e, ">."));
              }, h.interaction = function () {
                return new y().get();
              };
              var w = y.prototype = {
                createTracer: function (e, t) {
                  var r = {},
                    i = this,
                    a = "function" == typeof t;
                  return (0, o.p)(v + "tracer", [(0, s.z)(), e, r], i, n.D.spa, p), function () {
                    if (g.emit((a ? "" : "no-") + "fn-start", [(0, s.z)(), i, a], r), a) try {
                      return t.apply(this, arguments);
                    } catch (e) {
                      throw g.emit("fn-err", [arguments, this, e], r), e;
                    } finally {
                      g.emit("fn-end", [(0, s.z)()], r);
                    }
                  };
                }
              };
              function A(e, t, r, i) {
                return function () {
                  return (0, o.p)(f.xS, ["API/" + t + "/called"], void 0, n.D.metrics, p), i && (0, o.p)(e + t, [(0, s.z)(), ...arguments], r ? null : this, i, p), r ? void 0 : this;
                };
              }
              function x() {
                r.e(111).then(r.bind(r, 7438)).then(t => {
                  let {
                    setAPI: r
                  } = t;
                  r(e), (0, c.L)(e, "api");
                }).catch(() => (0, l.Z)("Downloading runtime APIs failed..."));
              }
              return ["actionText", "setName", "setAttribute", "save", "ignore", "onEnd", "getContext", "end", "get"].forEach(e => {
                w[e] = A(v, e, void 0, n.D.spa);
              }), h.noticeError = function (e, t) {
                "string" == typeof e && (e = new Error(e)), (0, o.p)(f.xS, ["API/noticeError/called"], void 0, n.D.metrics, p), (0, o.p)("err", [e, (0, s.z)(), !1, t], void 0, n.D.jserrors, p);
              }, d.il ? (0, u.b)(() => x(), !0) : x(), h;
            }(e, v);
            return (0, h.Qy)(e, _, "api"), (0, h.Qy)(e, x, "exposed"), (0, h.EZ)("activatedFeatures", g.T), _;
          }
        },
        3325: (e, t, r) => {
          r.d(t, {
            D: () => n,
            p: () => i
          });
          const n = {
              ajax: "ajax",
              jserrors: "jserrors",
              metrics: "metrics",
              pageAction: "page_action",
              pageViewEvent: "page_view_event",
              pageViewTiming: "page_view_timing",
              sessionReplay: "session_replay",
              sessionTrace: "session_trace",
              spa: "spa"
            },
            i = {
              [n.pageViewEvent]: 1,
              [n.pageViewTiming]: 2,
              [n.metrics]: 3,
              [n.jserrors]: 4,
              [n.ajax]: 5,
              [n.sessionTrace]: 6,
              [n.pageAction]: 7,
              [n.spa]: 8,
              [n.sessionReplay]: 9
            };
        }
      },
      n = {};
    function i(e) {
      var t = n[e];
      if (void 0 !== t) return t.exports;
      var o = n[e] = {
        exports: {}
      };
      return r[e](o, o.exports, i), o.exports;
    }
    i.m = r, i.d = (e, t) => {
      for (var r in t) i.o(t, r) && !i.o(e, r) && Object.defineProperty(e, r, {
        enumerable: !0,
        get: t[r]
      });
    }, i.f = {}, i.e = e => Promise.all(Object.keys(i.f).reduce((t, r) => (i.f[r](e, t), t), [])), i.u = e => "nr-spa.1097a448-1.238.0.min.js", i.o = (e, t) => Object.prototype.hasOwnProperty.call(e, t), e = {}, t = "NRBA-1.238.0.PROD:", i.l = (r, n, o, a) => {
      if (e[r]) e[r].push(n);else {
        var s, c;
        if (void 0 !== o) for (var u = document.getElementsByTagName("script"), d = 0; d < u.length; d++) {
          var l = u[d];
          if (l.getAttribute("src") == r || l.getAttribute("data-webpack") == t + o) {
            s = l;
            break;
          }
        }
        s || (c = !0, (s = document.createElement("script")).charset = "utf-8", s.timeout = 120, i.nc && s.setAttribute("nonce", i.nc), s.setAttribute("data-webpack", t + o), s.src = r), e[r] = [n];
        var f = (t, n) => {
            s.onerror = s.onload = null, clearTimeout(h);
            var i = e[r];
            if (delete e[r], s.parentNode && s.parentNode.removeChild(s), i && i.forEach(e => e(n)), t) return t(n);
          },
          h = setTimeout(f.bind(null, void 0, {
            type: "timeout",
            target: s
          }), 12e4);
        s.onerror = f.bind(null, s.onerror), s.onload = f.bind(null, s.onload), c && document.head.appendChild(s);
      }
    }, i.r = e => {
      "undefined" != typeof Symbol && Symbol.toStringTag && Object.defineProperty(e, Symbol.toStringTag, {
        value: "Module"
      }), Object.defineProperty(e, "__esModule", {
        value: !0
      });
    }, i.p = "https://js-agent.newrelic.com/", (() => {
      var e = {
        801: 0,
        92: 0
      };
      i.f.j = (t, r) => {
        var n = i.o(e, t) ? e[t] : void 0;
        if (0 !== n) if (n) r.push(n[2]);else {
          var o = new Promise((r, i) => n = e[t] = [r, i]);
          r.push(n[2] = o);
          var a = i.p + i.u(t),
            s = new Error();
          i.l(a, r => {
            if (i.o(e, t) && (0 !== (n = e[t]) && (e[t] = void 0), n)) {
              var o = r && ("load" === r.type ? "missing" : r.type),
                a = r && r.target && r.target.src;
              s.message = "Loading chunk " + t + " failed.\n(" + o + ": " + a + ")", s.name = "ChunkLoadError", s.type = o, s.request = a, n[1](s);
            }
          }, "chunk-" + t, t);
        }
      };
      var t = (t, r) => {
          var n,
            o,
            [a, s, c] = r,
            u = 0;
          if (a.some(t => 0 !== e[t])) {
            for (n in s) i.o(s, n) && (i.m[n] = s[n]);
            if (c) c(i);
          }
          for (t && t(r); u < a.length; u++) o = a[u], i.o(e, o) && e[o] && e[o][0](), e[o] = 0;
        },
        r = self["webpackChunk:NRBA-1.238.0.PROD"] = self["webpackChunk:NRBA-1.238.0.PROD"] || [];
      r.forEach(t.bind(null, 0)), r.push = t.bind(null, r.push.bind(r));
    })(), (() => {
      var e = i(50);
      class t {
        addPageAction(t, r) {
          (0, e.Z)("Call to agent api addPageAction failed. The session trace feature is not currently initialized.");
        }
        setPageViewName(t, r) {
          (0, e.Z)("Call to agent api setPageViewName failed. The page view feature is not currently initialized.");
        }
        setCustomAttribute(t, r, n) {
          (0, e.Z)("Call to agent api setCustomAttribute failed. The js errors feature is not currently initialized.");
        }
        noticeError(t, r) {
          (0, e.Z)("Call to agent api noticeError failed. The js errors feature is not currently initialized.");
        }
        setUserId(t) {
          (0, e.Z)("Call to agent api setUserId failed. The js errors feature is not currently initialized.");
        }
        setApplicationVersion(t) {
          (0, e.Z)("Call to agent api setApplicationVersion failed. The agent is not currently initialized.");
        }
        setErrorHandler(t) {
          (0, e.Z)("Call to agent api setErrorHandler failed. The js errors feature is not currently initialized.");
        }
        finished(t) {
          (0, e.Z)("Call to agent api finished failed. The page action feature is not currently initialized.");
        }
        addRelease(t, r) {
          (0, e.Z)("Call to agent api addRelease failed. The agent is not currently initialized.");
        }
      }
      var r = i(3325),
        n = i(5763);
      const o = Object.values(r.D);
      function a(e) {
        const t = {};
        return o.forEach(r => {
          t[r] = function (e, t) {
            return !1 !== (0, n.Mt)(t, "".concat(e, ".enabled"));
          }(r, e);
        }), t;
      }
      var s = i(9144);
      var c = i(5546),
        u = i(385),
        d = i(8e3),
        l = i(5938),
        f = i(3960);
      class h extends l.W {
        constructor(e, t, r) {
          let n = !(arguments.length > 3 && void 0 !== arguments[3]) || arguments[3];
          super(e, t, r), this.auto = n, this.abortHandler, this.featAggregate, this.onAggregateImported, n && (0, d.R)(e, r);
        }
        importAggregator() {
          let t = arguments.length > 0 && void 0 !== arguments[0] ? arguments[0] : {};
          if (this.featAggregate || !this.auto) return;
          const r = u.il && !0 === (0, n.Mt)(this.agentIdentifier, "privacy.cookies_enabled");
          let o;
          this.onAggregateImported = new Promise(e => {
            o = e;
          });
          const a = async () => {
            let n;
            try {
              if (r) {
                const {
                  setupAgentSession: e
                } = await i.e(111).then(i.bind(i, 3228));
                n = e(this.agentIdentifier);
              }
            } catch (t) {
              (0, e.Z)("A problem occurred when starting up session manager. This page will not start or extend any session.", t);
            }
            try {
              if (!this.shouldImportAgg(this.featureName, n)) return (0, d.L)(this.agentIdentifier, this.featureName), void o(!1);
              const {
                  lazyFeatureLoader: e
                } = await i.e(111).then(i.bind(i, 8582)),
                {
                  Aggregate: r
                } = await e(this.featureName, "aggregate");
              this.featAggregate = new r(this.agentIdentifier, this.aggregator, t), o(!0);
            } catch (t) {
              (0, e.Z)("Downloading and initializing ".concat(this.featureName, " failed..."), t), this.abortHandler?.(), o(!1);
            }
          };
          u.il ? (0, f.b)(() => a(), !0) : a();
        }
        shouldImportAgg(e, t) {
          return e !== r.D.sessionReplay || !!n.Yu.MO && !1 !== (0, n.Mt)(this.agentIdentifier, "session_trace.enabled") && (!!t?.isNew || !!t?.state.sessionReplay);
        }
      }
      var p = i(7633),
        g = i(7894);
      class m extends h {
        static featureName = p.t9;
        constructor(e, t) {
          let i = !(arguments.length > 2 && void 0 !== arguments[2]) || arguments[2];
          if (super(e, t, p.t9, i), ("undefined" == typeof PerformanceNavigationTiming || u.Tt) && "undefined" != typeof PerformanceTiming) {
            const t = (0, n.OP)(e);
            t[p.Dz] = Math.max(Date.now() - t.offset, 0), (0, f.K)(() => t[p.qw] = Math.max((0, g.z)() - t[p.Dz], 0)), (0, f.b)(() => {
              const e = (0, g.z)();
              t[p.OJ] = Math.max(e - t[p.Dz], 0), (0, c.p)("timing", ["load", e], void 0, r.D.pageViewTiming, this.ee);
            });
          }
          this.importAggregator();
        }
      }
      var v = i(1117),
        b = i(1284);
      class y extends v.w {
        constructor(e) {
          super(e), this.aggregatedData = {};
        }
        store(e, t, r, n, i) {
          var o = this.getBucket(e, t, r, i);
          return o.metrics = function (e, t) {
            t || (t = {
              count: 0
            });
            return t.count += 1, (0, b.D)(e, function (e, r) {
              t[e] = w(r, t[e]);
            }), t;
          }(n, o.metrics), o;
        }
        merge(e, t, r, n, i) {
          var o = this.getBucket(e, t, n, i);
          if (o.metrics) {
            var a = o.metrics;
            a.count += r.count, (0, b.D)(r, function (e, t) {
              if ("count" !== e) {
                var n = a[e],
                  i = r[e];
                i && !i.c ? a[e] = w(i.t, n) : a[e] = function (e, t) {
                  if (!t) return e;
                  t.c || (t = A(t.t));
                  return t.min = Math.min(e.min, t.min), t.max = Math.max(e.max, t.max), t.t += e.t, t.sos += e.sos, t.c += e.c, t;
                }(i, a[e]);
              }
            });
          } else o.metrics = r;
        }
        storeMetric(e, t, r, n) {
          var i = this.getBucket(e, t, r);
          return i.stats = w(n, i.stats), i;
        }
        getBucket(e, t, r, n) {
          this.aggregatedData[e] || (this.aggregatedData[e] = {});
          var i = this.aggregatedData[e][t];
          return i || (i = this.aggregatedData[e][t] = {
            params: r || {}
          }, n && (i.custom = n)), i;
        }
        get(e, t) {
          return t ? this.aggregatedData[e] && this.aggregatedData[e][t] : this.aggregatedData[e];
        }
        take(e) {
          for (var t = {}, r = "", n = !1, i = 0; i < e.length; i++) t[r = e[i]] = x(this.aggregatedData[r]), t[r].length && (n = !0), delete this.aggregatedData[r];
          return n ? t : null;
        }
      }
      function w(e, t) {
        return null == e ? function (e) {
          e ? e.c++ : e = {
            c: 1
          };
          return e;
        }(t) : t ? (t.c || (t = A(t.t)), t.c += 1, t.t += e, t.sos += e * e, e > t.max && (t.max = e), e < t.min && (t.min = e), t) : {
          t: e
        };
      }
      function A(e) {
        return {
          t: e,
          min: e,
          max: e,
          sos: e * e,
          c: 1
        };
      }
      function x(e) {
        return "object" != typeof e ? [] : (0, b.D)(e, E);
      }
      function E(e, t) {
        return t;
      }
      var T = i(8632),
        _ = i(4402),
        D = i(4351);
      var j = i(7956),
        C = i(3239),
        N = i(9251);
      class O extends h {
        static featureName = N.t;
        constructor(e, t) {
          let r = !(arguments.length > 2 && void 0 !== arguments[2]) || arguments[2];
          super(e, t, N.t, r), u.il && ((0, n.OP)(e).initHidden = Boolean("hidden" === document.visibilityState), (0, j.N)(() => (0, c.p)("docHidden", [(0, g.z)()], void 0, N.t, this.ee), !0), (0, C.bP)("pagehide", () => (0, c.p)("winPagehide", [(0, g.z)()], void 0, N.t, this.ee)), this.importAggregator());
        }
      }
      var S = i(3081);
      class P extends h {
        static featureName = S.t9;
        constructor(e, t) {
          let r = !(arguments.length > 2 && void 0 !== arguments[2]) || arguments[2];
          super(e, t, S.t9, r), this.importAggregator();
        }
      }
      var R = i(6660);
      class I {
        constructor(e, t, r, n) {
          this.name = "UncaughtError", this.message = e, this.sourceURL = t, this.line = r, this.column = n;
        }
      }
      class k extends h {
        static featureName = R.t;
        #e = new Set();
        constructor(e, t) {
          let n = !(arguments.length > 2 && void 0 !== arguments[2]) || arguments[2];
          super(e, t, R.t, n);
          try {
            this.removeOnAbort = new AbortController();
          } catch (e) {}
          this.ee.on("fn-err", (e, t, n) => {
            this.abortHandler && !this.#e.has(n) && (this.#e.add(n), (0, c.p)("err", [this.#t(n), (0, g.z)()], void 0, r.D.jserrors, this.ee));
          }), this.ee.on("internal-error", e => {
            this.abortHandler && (0, c.p)("ierr", [this.#t(e), (0, g.z)(), !0], void 0, r.D.jserrors, this.ee);
          }), u._A.addEventListener("unhandledrejection", e => {
            this.abortHandler && (0, c.p)("err", [this.#r(e), (0, g.z)(), !1, {
              unhandledPromiseRejection: 1
            }], void 0, r.D.jserrors, this.ee);
          }, (0, C.m$)(!1, this.removeOnAbort?.signal)), u._A.addEventListener("error", e => {
            this.abortHandler && (this.#e.has(e.error) ? this.#e.delete(e.error) : (0, c.p)("err", [this.#n(e), (0, g.z)()], void 0, r.D.jserrors, this.ee));
          }, (0, C.m$)(!1, this.removeOnAbort?.signal)), this.abortHandler = this.#i, this.importAggregator();
        }
        #i() {
          this.removeOnAbort?.abort(), this.#e.clear(), this.abortHandler = void 0;
        }
        #t(e) {
          return e instanceof Error ? e : void 0 !== e?.message ? new I(e.message, e.filename || e.sourceURL, e.lineno || e.line, e.colno || e.col) : new I("string" == typeof e ? e : (0, D.P)(e));
        }
        #r(e) {
          let t = "Unhandled Promise Rejection: ";
          if (e?.reason instanceof Error) try {
            return e.reason.message = t + e.reason.message, e.reason;
          } catch (t) {
            return e.reason;
          }
          if (void 0 === e.reason) return new I(t);
          const r = this.#t(e.reason);
          return r.message = t + r.message, r;
        }
        #n(e) {
          return e.error instanceof Error ? e.error : new I(e.message, e.filename, e.lineno, e.colno);
        }
      }
      var H = i(2210);
      let z = 1;
      const L = "nr@id";
      function M(e) {
        const t = typeof e;
        return !e || "object" !== t && "function" !== t ? -1 : e === u._A ? 0 : (0, H.X)(e, L, function () {
          return z++;
        });
      }
      function B(e) {
        if ("string" == typeof e && e.length) return e.length;
        if ("object" == typeof e) {
          if ("undefined" != typeof ArrayBuffer && e instanceof ArrayBuffer && e.byteLength) return e.byteLength;
          if ("undefined" != typeof Blob && e instanceof Blob && e.size) return e.size;
          if (!("undefined" != typeof FormData && e instanceof FormData)) try {
            return (0, D.P)(e).length;
          } catch (e) {
            return;
          }
        }
      }
      var F = i(1214),
        U = i(7243);
      class q {
        constructor(e) {
          this.agentIdentifier = e;
        }
        generateTracePayload(e) {
          if (!this.shouldGenerateTrace(e)) return null;
          var t = (0, n.DL)(this.agentIdentifier);
          if (!t) return null;
          var r = (t.accountID || "").toString() || null,
            i = (t.agentID || "").toString() || null,
            o = (t.trustKey || "").toString() || null;
          if (!r || !i) return null;
          var a = (0, _.M)(),
            s = (0, _.Ht)(),
            c = Date.now(),
            u = {
              spanId: a,
              traceId: s,
              timestamp: c
            };
          return (e.sameOrigin || this.isAllowedOrigin(e) && this.useTraceContextHeadersForCors()) && (u.traceContextParentHeader = this.generateTraceContextParentHeader(a, s), u.traceContextStateHeader = this.generateTraceContextStateHeader(a, c, r, i, o)), (e.sameOrigin && !this.excludeNewrelicHeader() || !e.sameOrigin && this.isAllowedOrigin(e) && this.useNewrelicHeaderForCors()) && (u.newrelicHeader = this.generateTraceHeader(a, s, c, r, i, o)), u;
        }
        generateTraceContextParentHeader(e, t) {
          return "00-" + t + "-" + e + "-01";
        }
        generateTraceContextStateHeader(e, t, r, n, i) {
          return i + "@nr=0-1-" + r + "-" + n + "-" + e + "----" + t;
        }
        generateTraceHeader(e, t, r, n, i, o) {
          if (!("function" == typeof u._A?.btoa)) return null;
          var a = {
            v: [0, 1],
            d: {
              ty: "Browser",
              ac: n,
              ap: i,
              id: e,
              tr: t,
              ti: r
            }
          };
          return o && n !== o && (a.d.tk = o), btoa((0, D.P)(a));
        }
        shouldGenerateTrace(e) {
          return this.isDtEnabled() && this.isAllowedOrigin(e);
        }
        isAllowedOrigin(e) {
          var t = !1,
            r = {};
          if ((0, n.Mt)(this.agentIdentifier, "distributed_tracing") && (r = (0, n.P_)(this.agentIdentifier).distributed_tracing), e.sameOrigin) t = !0;else if (r.allowed_origins instanceof Array) for (var i = 0; i < r.allowed_origins.length; i++) {
            var o = (0, U.e)(r.allowed_origins[i]);
            if (e.hostname === o.hostname && e.protocol === o.protocol && e.port === o.port) {
              t = !0;
              break;
            }
          }
          return t;
        }
        isDtEnabled() {
          var e = (0, n.Mt)(this.agentIdentifier, "distributed_tracing");
          return !!e && !!e.enabled;
        }
        excludeNewrelicHeader() {
          var e = (0, n.Mt)(this.agentIdentifier, "distributed_tracing");
          return !!e && !!e.exclude_newrelic_header;
        }
        useNewrelicHeaderForCors() {
          var e = (0, n.Mt)(this.agentIdentifier, "distributed_tracing");
          return !!e && !1 !== e.cors_use_newrelic_header;
        }
        useTraceContextHeadersForCors() {
          var e = (0, n.Mt)(this.agentIdentifier, "distributed_tracing");
          return !!e && !!e.cors_use_tracecontext_headers;
        }
      }
      var Z = i(7825),
        V = ["load", "error", "abort", "timeout"],
        G = V.length,
        W = n.Yu.REQ,
        X = n.Yu.XHR;
      class Q extends h {
        static featureName = Z.t;
        constructor(e, t) {
          let i = !(arguments.length > 2 && void 0 !== arguments[2]) || arguments[2];
          super(e, t, Z.t, i), (0, n.OP)(e).xhrWrappable && (this.dt = new q(e), this.handler = (e, t, r, n) => (0, c.p)(e, t, r, n, this.ee), (0, F.u5)(this.ee), (0, F.Kf)(this.ee), function (e, t, i, o) {
            function a(e) {
              var t = this;
              t.totalCbs = 0, t.called = 0, t.cbTime = 0, t.end = E, t.ended = !1, t.xhrGuids = {}, t.lastSize = null, t.loadCaptureCalled = !1, t.params = this.params || {}, t.metrics = this.metrics || {}, e.addEventListener("load", function (r) {
                _(t, e);
              }, (0, C.m$)(!1)), u.IF || e.addEventListener("progress", function (e) {
                t.lastSize = e.loaded;
              }, (0, C.m$)(!1));
            }
            function s(e) {
              this.params = {
                method: e[0]
              }, T(this, e[1]), this.metrics = {};
            }
            function c(t, r) {
              var i = (0, n.DL)(e);
              i.xpid && this.sameOrigin && r.setRequestHeader("X-NewRelic-ID", i.xpid);
              var a = o.generateTracePayload(this.parsedOrigin);
              if (a) {
                var s = !1;
                a.newrelicHeader && (r.setRequestHeader("newrelic", a.newrelicHeader), s = !0), a.traceContextParentHeader && (r.setRequestHeader("traceparent", a.traceContextParentHeader), a.traceContextStateHeader && r.setRequestHeader("tracestate", a.traceContextStateHeader), s = !0), s && (this.dt = a);
              }
            }
            function d(e, r) {
              var n = this.metrics,
                i = e[0],
                o = this;
              if (n && i) {
                var a = B(i);
                a && (n.txSize = a);
              }
              this.startTime = (0, g.z)(), this.listener = function (e) {
                try {
                  "abort" !== e.type || o.loadCaptureCalled || (o.params.aborted = !0), ("load" !== e.type || o.called === o.totalCbs && (o.onloadCalled || "function" != typeof r.onload) && "function" == typeof o.end) && o.end(r);
                } catch (e) {
                  try {
                    t.emit("internal-error", [e]);
                  } catch (e) {}
                }
              };
              for (var s = 0; s < G; s++) r.addEventListener(V[s], this.listener, (0, C.m$)(!1));
            }
            function l(e, t, r) {
              this.cbTime += e, t ? this.onloadCalled = !0 : this.called += 1, this.called !== this.totalCbs || !this.onloadCalled && "function" == typeof r.onload || "function" != typeof this.end || this.end(r);
            }
            function f(e, t) {
              var r = "" + M(e) + !!t;
              this.xhrGuids && !this.xhrGuids[r] && (this.xhrGuids[r] = !0, this.totalCbs += 1);
            }
            function h(e, t) {
              var r = "" + M(e) + !!t;
              this.xhrGuids && this.xhrGuids[r] && (delete this.xhrGuids[r], this.totalCbs -= 1);
            }
            function p() {
              this.endTime = (0, g.z)();
            }
            function m(e, r) {
              r instanceof X && "load" === e[0] && t.emit("xhr-load-added", [e[1], e[2]], r);
            }
            function v(e, r) {
              r instanceof X && "load" === e[0] && t.emit("xhr-load-removed", [e[1], e[2]], r);
            }
            function b(e, t, r) {
              t instanceof X && ("onload" === r && (this.onload = !0), ("load" === (e[0] && e[0].type) || this.onload) && (this.xhrCbStart = (0, g.z)()));
            }
            function y(e, r) {
              this.xhrCbStart && t.emit("xhr-cb-time", [(0, g.z)() - this.xhrCbStart, this.onload, r], r);
            }
            function w(e) {
              var t,
                r = e[1] || {};
              if ("string" == typeof e[0] ? 0 === (t = e[0]).length && u.il && (t = "" + u._A.location.href) : e[0] && e[0].url ? t = e[0].url : u._A?.URL && e[0] && e[0] instanceof URL ? t = e[0].href : "function" == typeof e[0].toString && (t = e[0].toString()), "string" == typeof t && 0 !== t.length) {
                t && (this.parsedOrigin = (0, U.e)(t), this.sameOrigin = this.parsedOrigin.sameOrigin);
                var n = o.generateTracePayload(this.parsedOrigin);
                if (n && (n.newrelicHeader || n.traceContextParentHeader)) if (e[0] && e[0].headers) s(e[0].headers, n) && (this.dt = n);else {
                  var i = {};
                  for (var a in r) i[a] = r[a];
                  i.headers = new Headers(r.headers || {}), s(i.headers, n) && (this.dt = n), e.length > 1 ? e[1] = i : e.push(i);
                }
              }
              function s(e, t) {
                var r = !1;
                return t.newrelicHeader && (e.set("newrelic", t.newrelicHeader), r = !0), t.traceContextParentHeader && (e.set("traceparent", t.traceContextParentHeader), t.traceContextStateHeader && e.set("tracestate", t.traceContextStateHeader), r = !0), r;
              }
            }
            function A(e, t) {
              this.params = {}, this.metrics = {}, this.startTime = (0, g.z)(), this.dt = t, e.length >= 1 && (this.target = e[0]), e.length >= 2 && (this.opts = e[1]);
              var r,
                n = this.opts || {},
                i = this.target;
              "string" == typeof i ? r = i : "object" == typeof i && i instanceof W ? r = i.url : u._A?.URL && "object" == typeof i && i instanceof URL && (r = i.href), T(this, r);
              var o = ("" + (i && i instanceof W && i.method || n.method || "GET")).toUpperCase();
              this.params.method = o, this.txSize = B(n.body) || 0;
            }
            function x(e, t) {
              var n;
              this.endTime = (0, g.z)(), this.params || (this.params = {}), this.params.status = t ? t.status : 0, "string" == typeof this.rxSize && this.rxSize.length > 0 && (n = +this.rxSize);
              var o = {
                txSize: this.txSize,
                rxSize: n,
                duration: (0, g.z)() - this.startTime
              };
              i("xhr", [this.params, o, this.startTime, this.endTime, "fetch"], this, r.D.ajax);
            }
            function E(e) {
              var t = this.params,
                n = this.metrics;
              if (!this.ended) {
                this.ended = !0;
                for (var o = 0; o < G; o++) e.removeEventListener(V[o], this.listener, !1);
                t.aborted || (n.duration = (0, g.z)() - this.startTime, this.loadCaptureCalled || 4 !== e.readyState ? null == t.status && (t.status = 0) : _(this, e), n.cbTime = this.cbTime, i("xhr", [t, n, this.startTime, this.endTime, "xhr"], this, r.D.ajax));
              }
            }
            function T(e, t) {
              var r = (0, U.e)(t),
                n = e.params;
              n.hostname = r.hostname, n.port = r.port, n.protocol = r.protocol, n.host = r.hostname + ":" + r.port, n.pathname = r.pathname, e.parsedOrigin = r, e.sameOrigin = r.sameOrigin;
            }
            function _(e, t) {
              e.params.status = t.status;
              var r = function (e, t) {
                var r = e.responseType;
                return "json" === r && null !== t ? t : "arraybuffer" === r || "blob" === r || "json" === r ? B(e.response) : "text" === r || "" === r || void 0 === r ? B(e.responseText) : void 0;
              }(t, e.lastSize);
              if (r && (e.metrics.rxSize = r), e.sameOrigin) {
                var n = t.getResponseHeader("X-NewRelic-App-Data");
                n && (e.params.cat = n.split(", ").pop());
              }
              e.loadCaptureCalled = !0;
            }
            t.on("new-xhr", a), t.on("open-xhr-start", s), t.on("open-xhr-end", c), t.on("send-xhr-start", d), t.on("xhr-cb-time", l), t.on("xhr-load-added", f), t.on("xhr-load-removed", h), t.on("xhr-resolved", p), t.on("addEventListener-end", m), t.on("removeEventListener-end", v), t.on("fn-end", y), t.on("fetch-before-start", w), t.on("fetch-start", A), t.on("fn-start", b), t.on("fetch-done", x);
          }(e, this.ee, this.handler, this.dt), this.importAggregator());
        }
      }
      var K = i(3614);
      const {
        BST_RESOURCE: Y,
        RESOURCE: J,
        START: ee,
        END: te,
        FEATURE_NAME: re,
        FN_END: ne,
        FN_START: ie,
        PUSH_STATE: oe
      } = K;
      var ae = i(7836);
      const {
        FEATURE_NAME: se,
        START: ce,
        END: ue,
        BODY: de,
        CB_END: le,
        JS_TIME: fe,
        FETCH: he,
        FN_START: pe,
        CB_START: ge,
        FN_END: me
      } = ae;
      var ve = i(4649);
      class be extends h {
        static featureName = ve.t;
        constructor(e, t) {
          let r = !(arguments.length > 2 && void 0 !== arguments[2]) || arguments[2];
          super(e, t, ve.t, r), this.importAggregator();
        }
      }
      new class extends t {
        constructor(t) {
          let r = arguments.length > 1 && void 0 !== arguments[1] ? arguments[1] : (0, _.ky)(16);
          super(), u._A ? (this.agentIdentifier = r, this.sharedAggregator = new y({
            agentIdentifier: this.agentIdentifier
          }), this.features = {}, this.desiredFeatures = new Set(t.features || []), this.desiredFeatures.add(m), Object.assign(this, (0, s.j)(this.agentIdentifier, t, t.loaderType || "agent")), this.start()) : (0, e.Z)("Failed to initial the agent. Could not determine the runtime environment.");
        }
        get config() {
          return {
            info: (0, n.C5)(this.agentIdentifier),
            init: (0, n.P_)(this.agentIdentifier),
            loader_config: (0, n.DL)(this.agentIdentifier),
            runtime: (0, n.OP)(this.agentIdentifier)
          };
        }
        start() {
          const t = "features";
          try {
            const n = a(this.agentIdentifier),
              i = [...this.desiredFeatures];
            i.sort((e, t) => r.p[e.featureName] - r.p[t.featureName]), i.forEach(t => {
              if (n[t.featureName] || t.featureName === r.D.pageViewEvent) {
                const i = function (e) {
                  switch (e) {
                    case r.D.ajax:
                      return [r.D.jserrors];
                    case r.D.sessionTrace:
                      return [r.D.ajax, r.D.pageViewEvent];
                    case r.D.sessionReplay:
                      return [r.D.sessionTrace];
                    case r.D.pageViewTiming:
                      return [r.D.pageViewEvent];
                    default:
                      return [];
                  }
                }(t.featureName);
                i.every(e => n[e]) || (0, e.Z)("".concat(t.featureName, " is enabled but one or more dependent features has been disabled (").concat((0, D.P)(i), "). This may cause unintended consequences or missing data...")), this.features[t.featureName] = new t(this.agentIdentifier, this.sharedAggregator);
              }
            }), (0, T.Qy)(this.agentIdentifier, this.features, t);
          } catch (r) {
            (0, e.Z)("Failed to initialize all enabled instrument classes (agent aborted) -", r);
            for (const e in this.features) this.features[e].abortHandler?.();
            const n = (0, T.fP)();
            return delete n.initializedAgents[this.agentIdentifier]?.api, delete n.initializedAgents[this.agentIdentifier]?.[t], delete this.sharedAggregator, n.ee?.abort(), delete n.ee?.get(this.agentIdentifier), !1;
          }
        }
        addToTrace(t) {
          (0, e.Z)("Call to agent api addToTrace failed. The page action feature is not currently initialized.");
        }
        setCurrentRouteName(t) {
          (0, e.Z)("Call to agent api setCurrentRouteName failed. The spa feature is not currently initialized.");
        }
        interaction() {
          (0, e.Z)("Call to agent api interaction failed. The spa feature is not currently initialized.");
        }
      }({
        features: [Q, m, O, class extends h {
          static featureName = re;
          constructor(e, t) {
            if (super(e, t, re, !(arguments.length > 2 && void 0 !== arguments[2]) || arguments[2]), !u.il) return;
            const n = this.ee;
            let i;
            (0, F.QU)(n), this.eventsEE = (0, F.em)(n), this.eventsEE.on(ie, function (e, t) {
              this.bstStart = (0, g.z)();
            }), this.eventsEE.on(ne, function (e, t) {
              (0, c.p)("bst", [e[0], t, this.bstStart, (0, g.z)()], void 0, r.D.sessionTrace, n);
            }), n.on(oe + ee, function (e) {
              this.time = (0, g.z)(), this.startPath = location.pathname + location.hash;
            }), n.on(oe + te, function (e) {
              (0, c.p)("bstHist", [location.pathname + location.hash, this.startPath, this.time], void 0, r.D.sessionTrace, n);
            });
            try {
              i = new PerformanceObserver(e => {
                const t = e.getEntries();
                (0, c.p)(Y, [t], void 0, r.D.sessionTrace, n);
              }), i.observe({
                type: J,
                buffered: !0
              });
            } catch (e) {}
            this.importAggregator({
              resourceObserver: i
            });
          }
        }, P, be, k, class extends h {
          static featureName = se;
          constructor(e, t) {
            if (super(e, t, se, !(arguments.length > 2 && void 0 !== arguments[2]) || arguments[2]), !u.il) return;
            if (!(0, n.OP)(e).xhrWrappable) return;
            try {
              this.removeOnAbort = new AbortController();
            } catch (e) {}
            let r,
              i = 0;
            const o = this.ee.get("tracer"),
              a = (0, F._L)(this.ee),
              s = (0, F.Lg)(this.ee),
              c = (0, F.BV)(this.ee),
              d = (0, F.Kf)(this.ee),
              l = this.ee.get("events"),
              f = (0, F.u5)(this.ee),
              h = (0, F.QU)(this.ee),
              p = (0, F.Gm)(this.ee);
            function m(e, t) {
              h.emit("newURL", ["" + window.location, t]);
            }
            function v() {
              i++, r = window.location.hash, this[pe] = (0, g.z)();
            }
            function b() {
              i--, window.location.hash !== r && m(0, !0);
              var e = (0, g.z)();
              this[fe] = ~~this[fe] + e - this[pe], this[me] = e;
            }
            function y(e, t) {
              e.on(t, function () {
                this[t] = (0, g.z)();
              });
            }
            this.ee.on(pe, v), s.on(ge, v), a.on(ge, v), this.ee.on(me, b), s.on(le, b), a.on(le, b), this.ee.buffer([pe, me, "xhr-resolved"], this.featureName), l.buffer([pe], this.featureName), c.buffer(["setTimeout" + ue, "clearTimeout" + ce, pe], this.featureName), d.buffer([pe, "new-xhr", "send-xhr" + ce], this.featureName), f.buffer([he + ce, he + "-done", he + de + ce, he + de + ue], this.featureName), h.buffer(["newURL"], this.featureName), p.buffer([pe], this.featureName), s.buffer(["propagate", ge, le, "executor-err", "resolve" + ce], this.featureName), o.buffer([pe, "no-" + pe], this.featureName), a.buffer(["new-jsonp", "cb-start", "jsonp-error", "jsonp-end"], this.featureName), y(f, he + ce), y(f, he + "-done"), y(a, "new-jsonp"), y(a, "jsonp-end"), y(a, "cb-start"), h.on("pushState-end", m), h.on("replaceState-end", m), window.addEventListener("hashchange", m, (0, C.m$)(!0, this.removeOnAbort?.signal)), window.addEventListener("load", m, (0, C.m$)(!0, this.removeOnAbort?.signal)), window.addEventListener("popstate", function () {
              m(0, i > 1);
            }, (0, C.m$)(!0, this.removeOnAbort?.signal)), this.abortHandler = this.#i, this.importAggregator();
          }
          #i() {
            this.removeOnAbort?.abort(), this.abortHandler = void 0;
          }
        }],
        loaderType: "spa"
      });
    })();
  })();
})()</script>
      <link rel="shortcut icon" href="https://sdfestaticassets-eu-west-1.sciencedirectassets.com/shared-assets/103/images/favSD.ico" type="image/x-icon">
      <link rel="icon" href="https://sdfestaticassets-eu-west-1.sciencedirectassets.com/shared-assets/103/images/favSD.ico" type="image/x-icon">
      <link rel="stylesheet" href="./ye2022analytical_files/arp.css">
      <link href="https://cdn.pendo.io/" rel="dns-prefetch">
      <link href="https://cdn.pendo.io/" rel="preconnect" crossorigin="anonymous">
      <link rel="dns-prefetch" href="https://smetrics.elsevier.com/">
      <script async="" id="reading-assistant-script-tag" src="./ye2022analytical_files/S1525157822001337" type="text/javascript"></script>
      <script type="text/javascript">
        var targetServerState = JSON.stringify({"4D6368F454EC41940A4C98A6@AdobeOrg":{"sdid":{"supplementalDataIDCurrent":"759419E65EAE8C41-3EBEB50C19990EA7","supplementalDataIDCurrentConsumed":{"payload:target-global-mbox":true},"supplementalDataIDLastConsumed":{}}}});
        window.appData = window.appData || [];
        window.pageTargeting = {"region":"eu-west-1","platform":"sdtech","entitled":true,"crawler":"","journal":"The Journal of Molecular Diagnostics","auth":"AE"};
        window.arp = {
          config: {"adobeSuite":"elsevier-sd-prod","arsUrl":"https://ars.els-cdn.com","recommendationsFeedback":{"enabled":true,"url":"https://feedback.recs.d.elsevier.com/raw/events","timeout":60000},"googleMapsApiKey":"AIzaSyCBYU6I6lrbEU6wQXUEIte3NwGtm3jwHQc","mediaBaseUrl":"https://ars.els-cdn.com/content/image/","strictMode":false,"seamlessAccess":{"enableSeamlessAccess":true,"scriptUrl":"https://unpkg.com/@theidentityselector/thiss-ds@1.0.13/dist/thiss-ds.js","persistenceUrl":"https://service.seamlessaccess.org/ps/","persistenceContext":"seamlessaccess.org","scienceDirectUrl":"https://www.sciencedirect.com","shibAuthUrl":"https://auth.elsevier.com/ShibAuth/institutionLogin"},"reaxys":{"apiUrl":"https://reaxys-sdlc.reaxys.com","origin":"sciencedirect","queryBuilderHostPath":"https://www.reaxys.com/reaxys/secured/hopinto.do","url":"https://www.reaxys.com"},"oneTrustCookie":{"enabled":true},"ssrn":{"url":"https://papers.ssrn.com","path":"/sol3/papers.cfm"},"plumX":"https://api.plu.mx/widget/elsevier/artifact","assetRoute":"https://sdfestaticassets-eu-west-1.sciencedirectassets.com/prod/5f37568fb8c8e6bc810d6b0e6e66d8f143fc0e6e"},
          subscriptions: [],
          subscribe: function(cb) {
            var self = this;
            var i = this.subscriptions.push(cb) - 1;
            return function unsubscribe() {
              self.subscriptions.splice(i, 1);
            }
          },
        };
        window.addEventListener('beforeprint', () => pendo.onGuideDismissed());
      </script>
    <script data-cfasync="false" src="./ye2022analytical_files/otSDKStub.js" data-document-language="true" data-domain-script="865ea198-88cc-4e41-8952-1df75d554d02"></script><script src="./ye2022analytical_files/thiss-ds.js" async=""></script><script src="./ye2022analytical_files/otBannerSdk.js" async="" type="text/javascript"></script><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style><style type="text/css">.MJX_Assistive_MathML {position: absolute!important; top: 0; left: 0; clip: rect(1px, 1px, 1px, 1px); padding: 1px 0 0 0!important; border: 0!important; height: 1px!important; width: 1px!important; overflow: hidden!important; display: block!important; -webkit-touch-callout: none; -webkit-user-select: none; -khtml-user-select: none; -moz-user-select: none; -ms-user-select: none; user-select: none}
.MJX_Assistive_MathML.MJX_Assistive_MathML_Block {width: 100%!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1em; bottom: 1.5em; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MJXp-script {font-size: .8em}
.MJXp-right {-webkit-transform-origin: right; -moz-transform-origin: right; -ms-transform-origin: right; -o-transform-origin: right; transform-origin: right}
.MJXp-bold {font-weight: bold}
.MJXp-italic {font-style: italic}
.MJXp-scr {font-family: MathJax_Script,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-frak {font-family: MathJax_Fraktur,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-sf {font-family: MathJax_SansSerif,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-cal {font-family: MathJax_Caligraphic,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-mono {font-family: MathJax_Typewriter,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-largeop {font-size: 150%}
.MJXp-largeop.MJXp-int {vertical-align: -.2em}
.MJXp-math {display: inline-block; line-height: 1.2; text-indent: 0; font-family: 'Times New Roman',Times,STIXGeneral,serif; white-space: nowrap; border-collapse: collapse}
.MJXp-display {display: block; text-align: center; margin: 1em 0}
.MJXp-math span {display: inline-block}
.MJXp-box {display: block!important; text-align: center}
.MJXp-box:after {content: " "}
.MJXp-rule {display: block!important; margin-top: .1em}
.MJXp-char {display: block!important}
.MJXp-mo {margin: 0 .15em}
.MJXp-mfrac {margin: 0 .125em; vertical-align: .25em}
.MJXp-denom {display: inline-table!important; width: 100%}
.MJXp-denom > * {display: table-row!important}
.MJXp-surd {vertical-align: top}
.MJXp-surd > * {display: block!important}
.MJXp-script-box > *  {display: table!important; height: 50%}
.MJXp-script-box > * > * {display: table-cell!important; vertical-align: top}
.MJXp-script-box > *:last-child > * {vertical-align: bottom}
.MJXp-script-box > * > * > * {display: block!important}
.MJXp-mphantom {visibility: hidden}
.MJXp-munderover, .MJXp-munder {display: inline-table!important}
.MJXp-over {display: inline-block!important; text-align: center}
.MJXp-over > * {display: block!important}
.MJXp-munderover > *, .MJXp-munder > * {display: table-row!important}
.MJXp-mtable {vertical-align: .25em; margin: 0 .125em}
.MJXp-mtable > * {display: inline-table!important; vertical-align: middle}
.MJXp-mtr {display: table-row!important}
.MJXp-mtd {display: table-cell!important; text-align: center; padding: .5em 0 0 .5em}
.MJXp-mtr > .MJXp-mtd:first-child {padding-left: 0}
.MJXp-mtr:first-child > .MJXp-mtd {padding-top: 0}
.MJXp-mlabeledtr {display: table-row!important}
.MJXp-mlabeledtr > .MJXp-mtd:first-child {padding-left: 0}
.MJXp-mlabeledtr:first-child > .MJXp-mtd {padding-top: 0}
.MJXp-merror {background-color: #FFFF88; color: #CC0000; border: 1px solid #CC0000; padding: 1px 3px; font-style: normal; font-size: 90%}
.MJXp-scale0 {-webkit-transform: scaleX(.0); -moz-transform: scaleX(.0); -ms-transform: scaleX(.0); -o-transform: scaleX(.0); transform: scaleX(.0)}
.MJXp-scale1 {-webkit-transform: scaleX(.1); -moz-transform: scaleX(.1); -ms-transform: scaleX(.1); -o-transform: scaleX(.1); transform: scaleX(.1)}
.MJXp-scale2 {-webkit-transform: scaleX(.2); -moz-transform: scaleX(.2); -ms-transform: scaleX(.2); -o-transform: scaleX(.2); transform: scaleX(.2)}
.MJXp-scale3 {-webkit-transform: scaleX(.3); -moz-transform: scaleX(.3); -ms-transform: scaleX(.3); -o-transform: scaleX(.3); transform: scaleX(.3)}
.MJXp-scale4 {-webkit-transform: scaleX(.4); -moz-transform: scaleX(.4); -ms-transform: scaleX(.4); -o-transform: scaleX(.4); transform: scaleX(.4)}
.MJXp-scale5 {-webkit-transform: scaleX(.5); -moz-transform: scaleX(.5); -ms-transform: scaleX(.5); -o-transform: scaleX(.5); transform: scaleX(.5)}
.MJXp-scale6 {-webkit-transform: scaleX(.6); -moz-transform: scaleX(.6); -ms-transform: scaleX(.6); -o-transform: scaleX(.6); transform: scaleX(.6)}
.MJXp-scale7 {-webkit-transform: scaleX(.7); -moz-transform: scaleX(.7); -ms-transform: scaleX(.7); -o-transform: scaleX(.7); transform: scaleX(.7)}
.MJXp-scale8 {-webkit-transform: scaleX(.8); -moz-transform: scaleX(.8); -ms-transform: scaleX(.8); -o-transform: scaleX(.8); transform: scaleX(.8)}
.MJXp-scale9 {-webkit-transform: scaleX(.9); -moz-transform: scaleX(.9); -ms-transform: scaleX(.9); -o-transform: scaleX(.9); transform: scaleX(.9)}
</style><style type="text/css">.MathJax_SVG_Display {text-align: center; margin: 1em 0em; position: relative; display: block!important; text-indent: 0; max-width: none; max-height: none; min-width: 0; min-height: 0; width: 100%}
.MathJax_SVG .MJX-monospace {font-family: monospace}
.MathJax_SVG .MJX-sans-serif {font-family: sans-serif}
#MathJax_SVG_Tooltip {background-color: InfoBackground; color: InfoText; border: 1px solid black; box-shadow: 2px 2px 5px #AAAAAA; -webkit-box-shadow: 2px 2px 5px #AAAAAA; -moz-box-shadow: 2px 2px 5px #AAAAAA; -khtml-box-shadow: 2px 2px 5px #AAAAAA; padding: 3px 4px; z-index: 401; position: absolute; left: 0; top: 0; width: auto; height: auto; display: none}
.MathJax_SVG {display: inline; font-style: normal; font-weight: normal; line-height: normal; font-size: 100%; font-size-adjust: none; text-indent: 0; text-align: left; text-transform: none; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; direction: ltr; max-width: none; max-height: none; min-width: 0; min-height: 0; border: 0; padding: 0; margin: 0}
.MathJax_SVG * {transition: none; -webkit-transition: none; -moz-transition: none; -ms-transition: none; -o-transition: none}
.MathJax_SVG > div {display: inline-block}
.mjx-svg-href {fill: blue; stroke: blue}
.MathJax_SVG_Processing {visibility: hidden; position: absolute; top: 0; left: 0; width: 0; height: 0; overflow: hidden; display: block!important}
.MathJax_SVG_Processed {display: none!important}
.MathJax_SVG_test {font-style: normal; font-weight: normal; font-size: 100%; font-size-adjust: none; text-indent: 0; text-transform: none; letter-spacing: normal; word-spacing: normal; overflow: hidden; height: 1px}
.MathJax_SVG_test.mjx-test-display {display: table!important}
.MathJax_SVG_test.mjx-test-inline {display: inline!important; margin-right: -1px}
.MathJax_SVG_test.mjx-test-default {display: block!important; clear: both}
.MathJax_SVG_ex_box {display: inline-block!important; position: absolute; overflow: hidden; min-height: 0; max-height: none; padding: 0; border: 0; margin: 0; width: 1px; height: 60ex}
.mjx-test-inline .MathJax_SVG_left_box {display: inline-block; width: 0; float: left}
.mjx-test-inline .MathJax_SVG_right_box {display: inline-block; width: 0; float: right}
.mjx-test-display .MathJax_SVG_right_box {display: table-cell!important; width: 10000em!important; min-width: 0; max-width: none; padding: 0; border: 0; margin: 0}
</style></head>
    <body data-sd-ui-layer-boundary="true"><div style="visibility: hidden; overflow: hidden; position: absolute; top: 0px; height: 1px; width: auto; padding: 0px; border: 0px; margin: 0px; text-align: left; text-indent: 0px; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal;"><div id="MathJax_SVG_Hidden"><br></br></div><svg><defs id="MathJax_SVG_glyphs"><path stroke-width="1" id="MJMAIN-52" d="M130 622Q123 629 119 631T103 634T60 637H27V683H202H236H300Q376 683 417 677T500 648Q595 600 609 517Q610 512 610 501Q610 468 594 439T556 392T511 361T472 343L456 338Q459 335 467 332Q497 316 516 298T545 254T559 211T568 155T578 94Q588 46 602 31T640 16H645Q660 16 674 32T692 87Q692 98 696 101T712 105T728 103T732 90Q732 59 716 27T672 -16Q656 -22 630 -22Q481 -16 458 90Q456 101 456 163T449 246Q430 304 373 320L363 322L297 323H231V192L232 61Q238 51 249 49T301 46H334V0H323Q302 3 181 3Q59 3 38 0H27V46H60Q102 47 111 49T130 61V622ZM491 499V509Q491 527 490 539T481 570T462 601T424 623T362 636Q360 636 340 636T304 637H283Q238 637 234 628Q231 624 231 492V360H289Q390 360 434 378T489 456Q491 467 491 499Z"></path><path stroke-width="1" id="MJMAIN-65" d="M28 218Q28 273 48 318T98 391T163 433T229 448Q282 448 320 430T378 380T406 316T415 245Q415 238 408 231H126V216Q126 68 226 36Q246 30 270 30Q312 30 342 62Q359 79 369 104L379 128Q382 131 395 131H398Q415 131 415 121Q415 117 412 108Q393 53 349 21T250 -11Q155 -11 92 58T28 218ZM333 275Q322 403 238 411H236Q228 411 220 410T195 402T166 381T143 340T127 274V267H333V275Z"></path><path stroke-width="1" id="MJMAIN-6C" d="M42 46H56Q95 46 103 60V68Q103 77 103 91T103 124T104 167T104 217T104 272T104 329Q104 366 104 407T104 482T104 542T103 586T103 603Q100 622 89 628T44 637H26V660Q26 683 28 683L38 684Q48 685 67 686T104 688Q121 689 141 690T171 693T182 694H185V379Q185 62 186 60Q190 52 198 49Q219 46 247 46H263V0H255L232 1Q209 2 183 2T145 3T107 3T57 1L34 0H26V46H42Z"></path><path stroke-width="1" id="MJMAIN-61" d="M137 305T115 305T78 320T63 359Q63 394 97 421T218 448Q291 448 336 416T396 340Q401 326 401 309T402 194V124Q402 76 407 58T428 40Q443 40 448 56T453 109V145H493V106Q492 66 490 59Q481 29 455 12T400 -6T353 12T329 54V58L327 55Q325 52 322 49T314 40T302 29T287 17T269 6T247 -2T221 -8T190 -11Q130 -11 82 20T34 107Q34 128 41 147T68 188T116 225T194 253T304 268H318V290Q318 324 312 340Q290 411 215 411Q197 411 181 410T156 406T148 403Q170 388 170 359Q170 334 154 320ZM126 106Q126 75 150 51T209 26Q247 26 276 49T315 109Q317 116 318 175Q318 233 317 233Q309 233 296 232T251 223T193 203T147 166T126 106Z"></path><path stroke-width="1" id="MJMAIN-74" d="M27 422Q80 426 109 478T141 600V615H181V431H316V385H181V241Q182 116 182 100T189 68Q203 29 238 29Q282 29 292 100Q293 108 293 146V181H333V146V134Q333 57 291 17Q264 -10 221 -10Q187 -10 162 2T124 33T105 68T98 100Q97 107 97 248V385H18V422H27Z"></path><path stroke-width="1" id="MJMAIN-69" d="M69 609Q69 637 87 653T131 669Q154 667 171 652T188 609Q188 579 171 564T129 549Q104 549 87 564T69 609ZM247 0Q232 3 143 3Q132 3 106 3T56 1L34 0H26V46H42Q70 46 91 49Q100 53 102 60T104 102V205V293Q104 345 102 359T88 378Q74 385 41 385H30V408Q30 431 32 431L42 432Q52 433 70 434T106 436Q123 437 142 438T171 441T182 442H185V62Q190 52 197 50T232 46H255V0H247Z"></path><path stroke-width="1" id="MJMAIN-76" d="M338 431Q344 429 422 429Q479 429 503 431H508V385H497Q439 381 423 345Q421 341 356 172T288 -2Q283 -11 263 -11Q244 -11 239 -2Q99 359 98 364Q93 378 82 381T43 385H19V431H25L33 430Q41 430 53 430T79 430T104 429T122 428Q217 428 232 431H240V385H226Q187 384 184 370Q184 366 235 234L286 102L377 341V349Q377 363 367 372T349 383T335 385H331V431H338Z"></path><path stroke-width="1" id="MJMAIN-45" d="M128 619Q121 626 117 628T101 631T58 634H25V680H597V676Q599 670 611 560T625 444V440H585V444Q584 447 582 465Q578 500 570 526T553 571T528 601T498 619T457 629T411 633T353 634Q266 634 251 633T233 622Q233 622 233 621Q232 619 232 497V376H286Q359 378 377 385Q413 401 416 469Q416 471 416 473V493H456V213H416V233Q415 268 408 288T383 317T349 328T297 330Q290 330 286 330H232V196V114Q232 57 237 52Q243 47 289 47H340H391Q428 47 452 50T505 62T552 92T584 146Q594 172 599 200T607 247T612 270V273H652V270Q651 267 632 137T610 3V0H25V46H58Q100 47 109 49T128 61V619Z"></path><path stroke-width="1" id="MJMAIN-78" d="M201 0Q189 3 102 3Q26 3 17 0H11V46H25Q48 47 67 52T96 61T121 78T139 96T160 122T180 150L226 210L168 288Q159 301 149 315T133 336T122 351T113 363T107 370T100 376T94 379T88 381T80 383Q74 383 44 385H16V431H23Q59 429 126 429Q219 429 229 431H237V385Q201 381 201 369Q201 367 211 353T239 315T268 274L272 270L297 304Q329 345 329 358Q329 364 327 369T322 376T317 380T310 384L307 385H302V431H309Q324 428 408 428Q487 428 493 431H499V385H492Q443 385 411 368Q394 360 377 341T312 257L296 236L358 151Q424 61 429 57T446 50Q464 46 499 46H516V0H510H502Q494 1 482 1T457 2T432 2T414 3Q403 3 377 3T327 1L304 0H295V46H298Q309 46 320 51T331 63Q331 65 291 120L250 175Q249 174 219 133T185 88Q181 83 181 74Q181 63 188 55T206 46Q208 46 208 23V0H201Z"></path><path stroke-width="1" id="MJMAIN-70" d="M36 -148H50Q89 -148 97 -134V-126Q97 -119 97 -107T97 -77T98 -38T98 6T98 55T98 106Q98 140 98 177T98 243T98 296T97 335T97 351Q94 370 83 376T38 385H20V408Q20 431 22 431L32 432Q42 433 61 434T98 436Q115 437 135 438T165 441T176 442H179V416L180 390L188 397Q247 441 326 441Q407 441 464 377T522 216Q522 115 457 52T310 -11Q242 -11 190 33L182 40V-45V-101Q182 -128 184 -134T195 -145Q216 -148 244 -148H260V-194H252L228 -193Q205 -192 178 -192T140 -191Q37 -191 28 -194H20V-148H36ZM424 218Q424 292 390 347T305 402Q234 402 182 337V98Q222 26 294 26Q345 26 384 80T424 218Z"></path><path stroke-width="1" id="MJMAIN-72" d="M36 46H50Q89 46 97 60V68Q97 77 97 91T98 122T98 161T98 203Q98 234 98 269T98 328L97 351Q94 370 83 376T38 385H20V408Q20 431 22 431L32 432Q42 433 60 434T96 436Q112 437 131 438T160 441T171 442H174V373Q213 441 271 441H277Q322 441 343 419T364 373Q364 352 351 337T313 322Q288 322 276 338T263 372Q263 381 265 388T270 400T273 405Q271 407 250 401Q234 393 226 386Q179 341 179 207V154Q179 141 179 127T179 101T180 81T180 66V61Q181 59 183 57T188 54T193 51T200 49T207 48T216 47T225 47T235 46T245 46H276V0H267Q249 3 140 3Q37 3 28 0H20V46H36Z"></path><path stroke-width="1" id="MJMAIN-73" d="M295 316Q295 356 268 385T190 414Q154 414 128 401Q98 382 98 349Q97 344 98 336T114 312T157 287Q175 282 201 278T245 269T277 256Q294 248 310 236T342 195T359 133Q359 71 321 31T198 -10H190Q138 -10 94 26L86 19L77 10Q71 4 65 -1L54 -11H46H42Q39 -11 33 -5V74V132Q33 153 35 157T45 162H54Q66 162 70 158T75 146T82 119T101 77Q136 26 198 26Q295 26 295 104Q295 133 277 151Q257 175 194 187T111 210Q75 227 54 256T33 318Q33 357 50 384T93 424T143 442T187 447H198Q238 447 268 432L283 424L292 431Q302 440 314 448H322H326Q329 448 335 442V310L329 304H301Q295 310 295 316Z"></path><path stroke-width="1" id="MJMAIN-6F" d="M28 214Q28 309 93 378T250 448Q340 448 405 380T471 215Q471 120 407 55T250 -10Q153 -10 91 57T28 214ZM250 30Q372 30 372 193V225V250Q372 272 371 288T364 326T348 362T317 390T268 410Q263 411 252 411Q222 411 195 399Q152 377 139 338T126 246V226Q126 130 145 91Q177 30 250 30Z"></path><path stroke-width="1" id="MJMAIN-6E" d="M41 46H55Q94 46 102 60V68Q102 77 102 91T102 122T103 161T103 203Q103 234 103 269T102 328V351Q99 370 88 376T43 385H25V408Q25 431 27 431L37 432Q47 433 65 434T102 436Q119 437 138 438T167 441T178 442H181V402Q181 364 182 364T187 369T199 384T218 402T247 421T285 437Q305 442 336 442Q450 438 463 329Q464 322 464 190V104Q464 66 466 59T477 49Q498 46 526 46H542V0H534L510 1Q487 2 460 2T422 3Q319 3 310 0H302V46H318Q379 46 379 62Q380 64 380 200Q379 335 378 343Q372 371 358 385T334 402T308 404Q263 404 229 370Q202 343 195 315T187 232V168V108Q187 78 188 68T191 55T200 49Q221 46 249 46H265V0H257L234 1Q210 2 183 2T145 3Q42 3 33 0H25V46H41Z"></path><path stroke-width="1" id="MJMAIN-4C" d="M128 622Q121 629 117 631T101 634T58 637H25V683H36Q48 680 182 680Q324 680 348 683H360V637H333Q273 637 258 635T233 622L232 342V129Q232 57 237 52Q243 47 313 47Q384 47 410 53Q470 70 498 110T536 221Q536 226 537 238T540 261T542 272T562 273H582V268Q580 265 568 137T554 5V0H25V46H58Q100 47 109 49T128 61V622Z"></path><path stroke-width="1" id="MJMAIN-3D" d="M56 347Q56 360 70 367H707Q722 359 722 347Q722 336 708 328L390 327H72Q56 332 56 347ZM56 153Q56 168 72 173H708Q722 163 722 153Q722 140 707 133H70Q56 140 56 153Z"></path><path stroke-width="1" id="MJMAIN-32" d="M109 429Q82 429 66 447T50 491Q50 562 103 614T235 666Q326 666 387 610T449 465Q449 422 429 383T381 315T301 241Q265 210 201 149L142 93L218 92Q375 92 385 97Q392 99 409 186V189H449V186Q448 183 436 95T421 3V0H50V19V31Q50 38 56 46T86 81Q115 113 136 137Q145 147 170 174T204 211T233 244T261 278T284 308T305 340T320 369T333 401T340 431T343 464Q343 527 309 573T212 619Q179 619 154 602T119 569T109 550Q109 549 114 549Q132 549 151 535T170 489Q170 464 154 447T109 429Z"></path><path stroke-width="1" id="MJMAINI-394" d="M409 0H145Q117 0 103 0T81 1T72 3T70 6Q70 9 310 354T557 707Q559 711 565 713Q570 716 587 716Q608 716 613 710Q616 704 683 358Q752 9 750 6Q750 2 747 2Q745 0 409 0ZM581 342L531 597Q530 597 352 342T174 85T403 84T632 86Q632 87 581 342Z"></path><path stroke-width="1" id="MJMAIN-43" d="M56 342Q56 428 89 500T174 615T283 681T391 705Q394 705 400 705T408 704Q499 704 569 636L582 624L612 663Q639 700 643 704Q644 704 647 704T653 705H657Q660 705 666 699V419L660 413H626Q620 419 619 430Q610 512 571 572T476 651Q457 658 426 658Q322 658 252 588Q173 509 173 342Q173 221 211 151Q232 111 263 84T328 45T384 29T428 24Q517 24 571 93T626 244Q626 251 632 257H660L666 251V236Q661 133 590 56T403 -21Q262 -21 159 83T56 342Z"></path></defs></svg></div><div id="MathJax_Message" style="display: none;"></div>
      <script type="text/javascript">
        window.__PRELOADED_STATE__ = {"abstracts":{"content":[{"$$":[{"$$":[{"$$":[{"#name":"__text__","_":"Detection of serum embryonic miRNAs miR-371a-3p and miR-372-3p has been proposed to aid in diagnosis, prognosis, and management of patients with testicular germ cell tumors (GCTs). This study describes the analytical validation and performance of a laboratory-developed test to detect these miRNA targets by stem loop real-time quantitative RT-PCR (RT-qPCR) in serum from patients with GCTs. The assay was standardized using an exogenous spike-in control of nonhuman miRNA from "},{"#name":"italic","_":"Caenorhabditis elegans"},{"#name":"__text__","_":" (cel-miR-39-3p) to assess extraction efficiency, and an endogenous housekeeping miRNA, miR-30b-5p, to control for miRNA normalization. miRNA results were expressed as relative expression level, using the comparative threshold cycle method (2"},{"$":{"loc":"post"},"#name":"sup","_":"ΔΔC"},{"$":{"loc":"post"},"#name":"sup","_":"T"},{"#name":"__text__","_":"). Analytical sensitivity of miR-371a-3p and miR-372-3p was 12.5 and 1.25 copies/μL, respectively. Clinical accuracy was evaluated using GCT patients with ("},{"#name":"italic","_":"n"},{"#name":"__text__","_":" = 34) and without ("},{"#name":"italic","_":"n"},{"#name":"__text__","_":" = 17) active disease. Positive/negative cutoffs and indeterminate findings were established on the basis of results from healthy volunteers ("},{"#name":"italic","_":"n"},{"#name":"__text__","_":" = 25) and assay precision. miR-371a-3p and miR-372-3p exhibited a sensitivity of 81.8% and 87.5%, respectively, and a specificity of 100% and 94%, respectively, and an area under the receiver operating characteristic curve of 0.93 and 0.95, respectively. Taken together, RT-qPCR testing for serum miR-371a-3p and miR-372-3p represents a robust, sensitive, and specific clinical assay to aid in the clinical management of patients with GCT."}],"$":{"view":"all","id":"abspara0010"},"#name":"simple-para"}],"$":{"view":"all","id":"abssec0010"},"#name":"abstract-sec"}],"$":{"view":"all","id":"abs0010","lang":"en","class":"author"},"#name":"abstract"}],"floats":[],"footnotes":[],"attachments":[]},"accessbarConfig":{"fallback":false,"id":"accessbar","version":"0.0.1","analytics":{"location":"accessbar","eventName":"ctaImpression"},"label":{},"ariaLabel":{"accessbar":"Download options and search","componentsList":"PDF Options"},"banners":[{"id":"BannerSsrn"}],"components":[{"target":"_blank","analytics":[{"ids":["accessbar:fta:single-article"],"eventName":"ctaClick"}],"label":"View&nbsp;**PDF**","ariaLabel":"View PDF. Opens in a new window.","id":"ViewPDF"},{"analytics":[{"ids":["accessbar:fta:full-issue"],"eventName":"ctaClick"}],"label":"Download full issue","id":"DownloadFullIssue"},{"target":"_blank","analytics":[{"ids":["accessbar:article:chorus-am"],"eventName":"ctaClick"}],"label":"View Open Manuscript","href":"/science/article/am/pii/S1525157822001337","id":"OpenManuscript"},{"id":"ViewClinicalKeyFullText","analytics":[{"eventName":"ctaClick","ids":["accessbar:fta:clinical-key"]}],"label":"View full text","rel":"noopener noreferrer","target":"_blank","href":"https://www.clinicalkey.com/#!/content/journal/1-s2.0-S1525157822001337"}],"search":{"inputPlaceHolder":"Search ScienceDirect","ariaLabel":{"input":"Search ScienceDirect","submit":"Submit search"},"formAction":"/search#submit","analytics":[{"ids":["accessbar:search"],"eventName":"searchStart"}],"id":"QuickSearch"}},"adobeTarget":{"sd:genai-question-and-answer":{}},"article":{"analyticsMetadata":{"accountId":"246711","accountName":"ClinicalKey Global Guest Users","loginStatus":"logged in","userId":"17847196","isLoggedIn":true},"cid":"280295","content-family":"serial","copyright-line":"© 2022 Association for Molecular Pathology and American Society for Investigative Pathology. Published by Elsevier Inc.","cover-date-years":["2022"],"cover-date-start":"2022-08-01","cover-date-text":"August 2022","document-subtype":"fla","document-type":"article","entitledToken":"424D1BF382E04DC9A7E1DC1AD7C6E4986AB4BED363D7D9B37D3D2096411BE1249A3B9B360375DA58","genAiToken":"eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJhdWQiOiJnZW5BaUFwcHMiLCJwaWkiOiJTMTUyNTE1NzgyMjAwMTMzNyIsInN1YiI6IjI0NjcxMSIsImlzcyI6ImFycCIsInNlc3Npb25JZCI6ImRmZjkyMDIwOWRlZDEwNGI3MDJhODYyODIxYmVjNWI4MGIyZWd4cnFiIiwiZXhwIjoxNzM3MzIwNTE1LCJpYXQiOjE3MzczMTg3MTUsInZlcnNpb24iOjEsImp0aSI6IjgzZDhjZTUzLTA4MTEtNDUyOC1iMGFlLTcyMzNmNjExMzczOCJ9.UxG42J79IppMbh3xsJY6144cjMDICJwe9fIO34n3iEI","eid":"1-s2.0-S1525157822001337","doi":"10.1016/j.jmoldx.2022.04.007","first-fp":"867","hub-eid":"1-s2.0-S1525157821X00190","issuePii":"S1525157821X00190","iss-first":"8","item-weight":"FULL-TEXT","language":"en","last-lp":"877","last-author":{"#name":"last-author","$":{"xmlns:ce":true,"xmlns:dm":true,"xmlns:sb":true},"$$":[{"#name":"author","$":{"id":"au11","author-id":"S1525157822001337-8e3052c5108e70480481ad606c608f6a"},"$$":[{"#name":"given-name","_":"Ellinor I."},{"#name":"surname","_":"Peerschke"},{"#name":"cross-ref","$":{"refid":"aff1","id":"crosref0085"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"∗"}]},{"#name":"cross-ref","$":{"refid":"cor1","id":"crosref0090"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"∗"}]},{"#name":"encoded-e-address","__encoded":"JTdCJTIyJTIzbmFtZSUyMiUzQSUyMmUtYWRkcmVzcyUyMiUyQyUyMiUyNCUyMiUzQSU3QiUyMnhtbG5zJTNBeGxpbmslMjIlM0F0cnVlJTJDJTIydHlwZSUyMiUzQSUyMmVtYWlsJTIyJTJDJTIyaHJlZiUyMiUzQSUyMm1haWx0byUzQXBlZXJzY2hlJTQwbXNrY2Mub3JnJTIyJTJDJTIyaWQlMjIlM0ElMjJlYWRkMDAxNSUyMiU3RCUyQyUyMl8lMjIlM0ElMjJwZWVyc2NoZSU0MG1za2NjLm9yZyUyMiU3RA=="}]}]},"normalized-first-auth-initial":"F","normalized-first-auth-surname":"YE","open-research":{"#name":"open-research","$":{"xmlns:xocs":true},"$$":[{"#name":"or-embargo-opening-date","_":"2023-08-04T00:00:00.000Z"},{"#name":"or-agreements","$$":[{"#name":"or-agreement","$$":[{"#name":"or-agreement-id","_":"https://vtw.elsevier.com/content/oragreement/10299"},{"#name":"or-agreement-name","_":"NR_NIHGRN"},{"#name":"or-agreement-terms","_":"publishAcceptedManuscriptIndexable"}]}]},{"#name":"or-agreement-license","_":"http://www.elsevier.com/open-access/userlicense/1.0/"}]},"pages":[{"last-page":"877","first-page":"867"}],"pii":"S1525157822001337","self-archiving":{"#name":"self-archiving","$":{"xmlns:xocs":true},"$$":[{"#name":"sa-start-date","_":"2023-08-04T00:00:00.000Z"},{"#name":"sa-user-license","_":"http://creativecommons.org/licenses/by-nc-nd/4.0/"}]},"srctitle":"The Journal of Molecular Diagnostics","timestamp":"2023-08-01T00:33:58.266778Z","title":{"content":[{"#name":"article-footnote","$":{"id":"aep-article-footnote-id7"},"$$":[{"#name":"note-para","$":{"id":"ntpara0010","view":"all"},"$$":[{"#name":"__text__","_":"Supported in part by the "},{"#name":"grant-sponsor","$":{"xmlns:xlink":true,"id":"gs6","sponsor-id":"https://doi.org/10.13039/100017507","type":"simple","role":"http://www.elsevier.com/xml/linking-roles/grant-sponsor"},"_":"Department of Laboratory Medicine"},{"#name":"__text__","_":" (E.I.P.), the Kravis Center for Molecular Oncology at "},{"#name":"grant-sponsor","$":{"xmlns:xlink":true,"id":"gs2","sponsor-id":"https://doi.org/10.13039/100007052","type":"simple","role":"http://www.elsevier.com/xml/linking-roles/grant-sponsor"},"_":"Memorial Sloan Kettering Cancer Center"},{"#name":"__text__","_":" (D.B.S.), "},{"#name":"grant-sponsor","$":{"xmlns:xlink":true,"id":"gs7","sponsor-id":"https://doi.org/10.13039/100017050","type":"simple","role":"http://www.elsevier.com/xml/linking-roles/grant-sponsor"},"_":"Cycle for Survival"},{"#name":"__text__","_":" (D.R.F.), and the "},{"#name":"grant-sponsor","$":{"xmlns:xlink":true,"id":"gs4","sponsor-id":"https://doi.org/10.13039/100000002","type":"simple","role":"http://www.elsevier.com/xml/linking-roles/grant-sponsor"},"_":"NIH"},{"#name":"__text__","_":"/"},{"#name":"grant-sponsor","$":{"xmlns:xlink":true,"id":"gs9","type":"simple","role":"http://www.elsevier.com/xml/linking-roles/grant-sponsor"},"_":"National Cancer Institute Cancer Center"},{"#name":"__text__","_":" Support grant "},{"#name":"grant-number","$":{"refid":"gs9"},"_":"P30 CA008748"},{"#name":"__text__","_":"."}]}]},{"#name":"article-footnote","$":{"id":"aep-article-footnote-id8"},"$$":[{"#name":"note-para","$":{"id":"ntpara0015","view":"all"},"_":"F.Y. and D.R.F. are co–first authors and contributed equally to this work."}]},{"#name":"article-footnote","$":{"id":"aep-article-footnote-id9"},"$$":[{"#name":"note-para","$":{"id":"ntpara0020","view":"all"},"$$":[{"#name":"__text__","_":"Disclosures: D.B.S. has consulted for/received honoraria from Pfizer, Loxo/Lilly Oncology, Vividion Therapeutics, FORE Therapeutics, Scorpion Therapeutics, and BridgeBio. S.A.F. receives research funding from "},{"#name":"grant-sponsor","$":{"xmlns:xlink":true,"id":"gs10","sponsor-id":"https://doi.org/10.13039/100004325","type":"simple","role":"http://www.elsevier.com/xml/linking-roles/grant-sponsor"},"_":"AstraZeneca"},{"#name":"__text__","_":" and "},{"#name":"grant-sponsor","$":{"xmlns:xlink":true,"id":"gs8","type":"simple","role":"http://www.elsevier.com/xml/linking-roles/grant-sponsor"},"_":"Genetech"},{"#name":"__text__","_":"/"},{"#name":"grant-sponsor","$":{"xmlns:xlink":true,"id":"gs11","sponsor-id":"https://doi.org/10.13039/100004337","type":"simple","role":"http://www.elsevier.com/xml/linking-roles/grant-sponsor"},"_":"Roche"},{"#name":"__text__","_":", has consulted for/received honoraria from Merck, and has stock/equity ownership in Allogene Therapeutics, Urogen, Kronos Bio, Neogene Therapeutics, IconOVir, and Vida Ventures. The other authors declare no conflicts of interest."}]}]},{"#name":"dochead","$":{"id":"dhead0010"},"$$":[{"#name":"textfn","_":"Regular article"}]},{"#name":"title","$":{"id":"title0010"},"_":"Analytical Validation and Performance Characteristics of Molecular Serum Biomarkers, miR-371a-3p and miR-372-3p, for Male Germ Cell Tumors, in a Clinical Laboratory Setting"}],"floats":[],"footnotes":[{"#name":"article-footnote","$":{"id":"aep-article-footnote-id7"},"$$":[{"#name":"note-para","$":{"id":"ntpara0010","view":"all"},"$$":[{"#name":"__text__","_":"Supported in part by the "},{"#name":"grant-sponsor","$":{"xmlns:xlink":true,"id":"gs6","sponsor-id":"https://doi.org/10.13039/100017507","type":"simple","role":"http://www.elsevier.com/xml/linking-roles/grant-sponsor"},"_":"Department of Laboratory Medicine"},{"#name":"__text__","_":" (E.I.P.), the Kravis Center for Molecular Oncology at "},{"#name":"grant-sponsor","$":{"xmlns:xlink":true,"id":"gs2","sponsor-id":"https://doi.org/10.13039/100007052","type":"simple","role":"http://www.elsevier.com/xml/linking-roles/grant-sponsor"},"_":"Memorial Sloan Kettering Cancer Center"},{"#name":"__text__","_":" (D.B.S.), "},{"#name":"grant-sponsor","$":{"xmlns:xlink":true,"id":"gs7","sponsor-id":"https://doi.org/10.13039/100017050","type":"simple","role":"http://www.elsevier.com/xml/linking-roles/grant-sponsor"},"_":"Cycle for Survival"},{"#name":"__text__","_":" (D.R.F.), and the "},{"#name":"grant-sponsor","$":{"xmlns:xlink":true,"id":"gs4","sponsor-id":"https://doi.org/10.13039/100000002","type":"simple","role":"http://www.elsevier.com/xml/linking-roles/grant-sponsor"},"_":"NIH"},{"#name":"__text__","_":"/"},{"#name":"grant-sponsor","$":{"xmlns:xlink":true,"id":"gs9","type":"simple","role":"http://www.elsevier.com/xml/linking-roles/grant-sponsor"},"_":"National Cancer Institute Cancer Center"},{"#name":"__text__","_":" Support grant "},{"#name":"grant-number","$":{"refid":"gs9"},"_":"P30 CA008748"},{"#name":"__text__","_":"."}]}]},{"#name":"article-footnote","$":{"id":"aep-article-footnote-id8"},"$$":[{"#name":"note-para","$":{"id":"ntpara0015","view":"all"},"_":"F.Y. and D.R.F. are co–first authors and contributed equally to this work."}]},{"#name":"article-footnote","$":{"id":"aep-article-footnote-id9"},"$$":[{"#name":"note-para","$":{"id":"ntpara0020","view":"all"},"$$":[{"#name":"__text__","_":"Disclosures: D.B.S. has consulted for/received honoraria from Pfizer, Loxo/Lilly Oncology, Vividion Therapeutics, FORE Therapeutics, Scorpion Therapeutics, and BridgeBio. S.A.F. receives research funding from "},{"#name":"grant-sponsor","$":{"xmlns:xlink":true,"id":"gs10","sponsor-id":"https://doi.org/10.13039/100004325","type":"simple","role":"http://www.elsevier.com/xml/linking-roles/grant-sponsor"},"_":"AstraZeneca"},{"#name":"__text__","_":" and "},{"#name":"grant-sponsor","$":{"xmlns:xlink":true,"id":"gs8","type":"simple","role":"http://www.elsevier.com/xml/linking-roles/grant-sponsor"},"_":"Genetech"},{"#name":"__text__","_":"/"},{"#name":"grant-sponsor","$":{"xmlns:xlink":true,"id":"gs11","sponsor-id":"https://doi.org/10.13039/100004337","type":"simple","role":"http://www.elsevier.com/xml/linking-roles/grant-sponsor"},"_":"Roche"},{"#name":"__text__","_":", has consulted for/received honoraria from Merck, and has stock/equity ownership in Allogene Therapeutics, Urogen, Kronos Bio, Neogene Therapeutics, IconOVir, and Vida Ventures. The other authors declare no conflicts of interest."}]}]}],"attachments":[]},"vol-first":"24","vol-iss-suppl-text":"Volume 24, Issue 8","userSettings":{"forceAbstract":false,"creditCardPurchaseAllowed":true,"blockFullTextForAnonymousAccess":false,"disableWholeIssueDownload":false,"preventTransactionalAccess":false,"preventDocumentDelivery":true},"contentType":"JL","crossmark":true,"document-references":36,"freeHtmlGiven":false,"userProfile":{"departmentName":"ScienceDirect Guests","webUserId":"17847196","accountName":"ClinicalKey Global Guest Users","departmentId":"291352","hasMultipleOrganizations":false,"accountNumber":"C000228598","userName":"José Andrés Castillo Rivas","supportUserData":"OUjuK9K7N5XLymooN4~iYJzc6mDq2m3HVgiU6CpJ5pv2PqbC1G8StBEM3ffoF4FCTI_QPShve3l07_bfdZ8xQkTaYsO9rD8Mj_6a_FCSvBokwW4uPVPJuHvJguXB9Tkn~HULFTMIoz4niUj_FDpgJOdD4wNFdF_xSaFD7_USdB_qaRLHbKYq5AbkZEoEXI6lWgE41_t4G_ltTu81dEOqxTEo89ZLlsi5h4UkN2erlX5fOIjXJs7b6VE7TKkgtbOzIAOM4KPcOAUpvEdFIkngXdLJnimhbLOTjlOm~FuwLT_D73CQZF43sLcJcYBqyEs3AQ5Og6MgVopqL~ejA0_XTBACQYzNAxutk9SFVmogOobxG4opoTNzn2i1BjI6j1M7","accessType":"GUESTREG","accountId":"246711","userType":"NORMAL","privilegeType":"BASIC","email":"joshoacr13@gmail.com"},"access":{"openAccess":true,"openArchive":true,"license":"http://www.elsevier.com/open-access/userlicense/1.0/"},"aipType":"none","articleEntitlement":{"entitled":true,"isCasaUser":false,"usageInfo":"(17847196,U|291352,D|228598,A|3,P|2,PL)(SDFE,CON|dff920209ded104b702a862821bec5b80b2egxrqb,SSO|REG_GUEST,ACCESS_TYPE)","entitledByAccount":false,"entitlementOrigin":"SD"},"crawlerInformation":{"canCrawlPDFContent":false,"isCrawler":false},"dates":{"Available online":"4 August 2022","Revised":[],"Accepted":"22 April 2022","Publication date":"1 August 2022","Version of Record":"4 August 2022"},"downloadFullIssue":true,"entitlementReason":"openarchive","features":["aamAttachments","references","preview"],"hasBody":true,"has-large-authors":false,"hasScholarlyAbstract":true,"headerConfig":{"helpUrl":"https://service.elsevier.com/ci/pta/login/redirect/home/supporthub/sciencedirect/p_li/OUjuK9K7N5XLymooN4~iYJzc6mDq2m3HVgiU6CpJ5pv2PqbC1G8StBEM3ffoF4FCTI_QPShve3l07_bfdZ8xQkTaYsO9rD8Mj_6a_FCSvBokwW4uPVPJuHvJguXB9Tkn~HULFTMIoz4niUj_FDpgJOdD4wNFdF_xSaFD7_USdB_qaRLHbKYq5AbkZEoEXI6lWgE41_t4G_ltTu81dEOqxTEo89ZLlsi5h4UkN2erlX5fOIjXJs7b6VE7TKkgtbOzIAOM4KPcOAUpvEdFIkngXdLJnimhbLOTjlOm~FuwLT_D73CQZF43sLcJcYBqyEs3AQ5Og6MgVopqL~ejA0_XTBACQYzNAxutk9SFVmogOobxG4opoTNzn2i1BjI6j1M7","contactUrl":"https://service.elsevier.com/ci/pta/login/redirect/contact/supporthub/sciencedirect/p_li/OUjuK9K7N5XLymooN4~iYJzc6mDq2m3HVgiU6CpJ5pv2PqbC1G8StBEM3ffoF4FCTI_QPShve3l07_bfdZ8xQkTaYsO9rD8Mj_6a_FCSvBokwW4uPVPJuHvJguXB9Tkn~HULFTMIoz4niUj_FDpgJOdD4wNFdF_xSaFD7_USdB_qaRLHbKYq5AbkZEoEXI6lWgE41_t4G_ltTu81dEOqxTEo89ZLlsi5h4UkN2erlX5fOIjXJs7b6VE7TKkgtbOzIAOM4KPcOAUpvEdFIkngXdLJnimhbLOTjlOm~FuwLT_D73CQZF43sLcJcYBqyEs3AQ5Og6MgVopqL~ejA0_XTBACQYzNAxutk9SFVmogOobxG4opoTNzn2i1BjI6j1M7","userName":"José Andrés Castillo Rivas","userEmail":"joshoacr13@gmail.com","orgName":"ClinicalKey Global Guest Users","webUserId":"17847196","libraryBanner":null,"shib_regUrl":"","tick_regUrl":"","recentInstitutions":[],"canActivatePersonalization":false,"hasInstitutionalAssociation":false,"hasMultiOrg":false,"userType":"GUESTREG","userAnonymity":"INDIVIDUAL","allowCart":true,"environment":"prod","cdnAssetsHost":"https://sdfestaticassets-eu-west-1.sciencedirectassets.com","supportUserData":"OUjuK9K7N5XLymooN4~iYJzc6mDq2m3HVgiU6CpJ5pv2PqbC1G8StBEM3ffoF4FCTI_QPShve3l07_bfdZ8xQkTaYsO9rD8Mj_6a_FCSvBokwW4uPVPJuHvJguXB9Tkn~HULFTMIoz4niUj_FDpgJOdD4wNFdF_xSaFD7_USdB_qaRLHbKYq5AbkZEoEXI6lWgE41_t4G_ltTu81dEOqxTEo89ZLlsi5h4UkN2erlX5fOIjXJs7b6VE7TKkgtbOzIAOM4KPcOAUpvEdFIkngXdLJnimhbLOTjlOm~FuwLT_D73CQZF43sLcJcYBqyEs3AQ5Og6MgVopqL~ejA0_XTBACQYzNAxutk9SFVmogOobxG4opoTNzn2i1BjI6j1M7"},"isCorpReq":false,"isPdfFullText":false,"issn":"15251578","issn-primary-formatted":"1525-1578","issRange":"8","isThirdParty":false,"pageCount":11,"pdfDownload":{"isPdfFullText":false,"urlMetadata":{"queryParams":{"md5":"e8814061aa6a6d230d9dc16eda9f1ace","pid":"1-s2.0-S1525157822001337-main.pdf"},"pii":"S1525157822001337","pdfExtension":"/pdfft","path":"science/article/pii"}},"publication-content":{"noElsevierLogo":false,"imprintPublisher":{"displayName":"Elsevier","id":"47"},"isSpecialIssue":false,"isSampleIssue":false,"transactionsBlocked":false,"publicationOpenAccess":{"oaStatus":"Partial","oaArticleCount":1838,"openArchiveStatus":true,"openArchiveArticleCount":1442,"openAccessStartDate":"","oaAllowsAuthorPaid":true},"issue-cover":{"attachment":[{"attachment-eid":"1-s2.0-S1525157821X00190-cov200h.gif","file-basename":"cov200h","extension":"gif","filename":"cov200h.gif","ucs-locator":["https://s3-eu-west-1.amazonaws.com/prod-ucs-content-store-eu-west/content/pii:S1525157821X00190/cover/DOWNSAMPLED200/image/gif/208d977cd32361128bb2d1a884134f6b/cov200h.gif"],"attachment-type":"IMAGE-COVER-H200","filesize":"8709","pixel-height":"200","pixel-width":"149"},{"attachment-eid":"1-s2.0-S1525157821X00190-cov150h.gif","file-basename":"cov150h","extension":"gif","filename":"cov150h.gif","ucs-locator":["https://s3-eu-west-1.amazonaws.com/prod-ucs-content-store-eu-west/content/pii:S1525157821X00190/cover/DOWNSAMPLED/image/gif/45aa96ae4e3a36af6cf09cc6c0358ae2/cov150h.gif"],"attachment-type":"IMAGE-COVER-H150","filesize":"5966","pixel-height":"150","pixel-width":"112"}]},"smallCoverUrl":"https://ars.els-cdn.com/content/image/S15251578.gif","title":"the-journal-of-molecular-diagnostics","contentTypeCode":"JL","images":{"coverImage":"https://ars.els-cdn.com/content/image/1-s2.0-S1525157821X00190-cov150h.gif","logo":"https://sdfestaticassets-eu-west-1.sciencedirectassets.com/prod/5f37568fb8c8e6bc810d6b0e6e66d8f143fc0e6e/image/elsevier-non-solus.png","logoAltText":"Elsevier"},"publicationCoverImageUrl":"https://ars.els-cdn.com/content/image/1-s2.0-S1525157821X00190-cov150h.gif"},"volRange":"24","titleString":"Analytical Validation and Performance Characteristics of Molecular Serum Biomarkers, miR-371a-3p and miR-372-3p, for Male Germ Cell Tumors, in a Clinical Laboratory Setting","openManuscriptUrl":"/science/article/am/pii/S1525157822001337","ssrn":{},"renderingMode":"Article","isAbstract":false,"isContentVisible":false,"ajaxLinks":{"citingArticles":true,"referenceLinks":true,"references":true,"referredToBy":true,"toc":true,"body":true,"recommendations":true,"authorMetadata":true,"substances":true},"pdfEmbed":false,"displayViewFullText":false},"authors":{"content":[{"#name":"author-group","$":{"id":"augrp0010"},"$$":[{"#name":"author","$":{"id":"au1","orcid":"0000-0002-6167-0142","author-id":"S1525157822001337-92012d4a18af8d60fd610c447323cadc"},"$$":[{"#name":"given-name","_":"Fei"},{"#name":"surname","_":"Ye"},{"#name":"cross-ref","$":{"refid":"aff1","id":"crosref0010"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"∗"}]},{"#name":"cross-ref","$":{"refid":"cor1","id":"crosref0015"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"∗"}]},{"#name":"encoded-e-address","__encoded":"JTdCJTIyJTIzbmFtZSUyMiUzQSUyMmUtYWRkcmVzcyUyMiUyQyUyMiUyNCUyMiUzQSU3QiUyMnhtbG5zJTNBeGxpbmslMjIlM0F0cnVlJTJDJTIydHlwZSUyMiUzQSUyMmVtYWlsJTIyJTJDJTIyaHJlZiUyMiUzQSUyMm1haWx0byUzQXllZiU0MG1za2NjLm9yZyUyMiUyQyUyMmlkJTIyJTNBJTIyZWFkZDAwMTAlMjIlN0QlMkMlMjJfJTIyJTNBJTIyeWVmJTQwbXNrY2Mub3JnJTIyJTdE"}]},{"#name":"author","$":{"id":"au2","author-id":"S1525157822001337-5ac0c128a1f015dcc6e9e01b4dadd788"},"$$":[{"#name":"given-name","_":"Darren R."},{"#name":"surname","_":"Feldman"},{"#name":"cross-ref","$":{"refid":"aff2","id":"crosref0020"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"†"}]},{"#name":"cross-ref","$":{"refid":"aff3","id":"crosref0025"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"‡"}]}]},{"#name":"author","$":{"id":"au3","author-id":"S1525157822001337-916b7dbc66a624629ddd1bfe8354ee6e"},"$$":[{"#name":"given-name","_":"Alisa"},{"#name":"surname","_":"Valentino"},{"#name":"cross-ref","$":{"refid":"aff1","id":"crosref0030"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"∗"}]}]},{"#name":"author","$":{"id":"au4","orcid":"0000-0001-7022-1595","author-id":"S1525157822001337-bf2c79ea44eb931f1b165273db375515"},"$$":[{"#name":"given-name","_":"Rachel"},{"#name":"surname","_":"So"},{"#name":"cross-ref","$":{"refid":"aff1","id":"crosref0035"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"∗"}]}]},{"#name":"author","$":{"id":"au5","author-id":"S1525157822001337-21203d3c728f7c731e61dca5b4170327"},"$$":[{"#name":"given-name","_":"Maria"},{"#name":"surname","_":"Bromberg"},{"#name":"cross-ref","$":{"refid":"aff2","id":"crosref0040"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"†"}]}]},{"#name":"author","$":{"id":"au6","author-id":"S1525157822001337-0321cbaab91269126ee3122a8ac062b7"},"$$":[{"#name":"given-name","_":"Shah"},{"#name":"surname","_":"Khan"},{"#name":"cross-ref","$":{"refid":"aff2","id":"crosref0045"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"†"}]}]},{"#name":"author","$":{"id":"au7","orcid":"0000-0002-7235-6319","author-id":"S1525157822001337-87db96111d15a094f19796ef0809cc8a"},"$$":[{"#name":"given-name","_":"Samuel A."},{"#name":"surname","_":"Funt"},{"#name":"cross-ref","$":{"refid":"aff2","id":"crosref0050"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"†"}]},{"#name":"cross-ref","$":{"refid":"aff3","id":"crosref0055"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"‡"}]}]},{"#name":"author","$":{"id":"au8","author-id":"S1525157822001337-8fcbea0b241a67bc95b87a1cea7f5257"},"$$":[{"#name":"given-name","_":"Joel"},{"#name":"surname","_":"Sheinfeld"},{"#name":"cross-ref","$":{"refid":"aff4","id":"crosref0060"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"§"}]}]},{"#name":"author","$":{"id":"au9","author-id":"S1525157822001337-cd93c266b3865ab9e0542c12408c6510"},"$$":[{"#name":"given-name","_":"David B."},{"#name":"surname","_":"Solit"},{"#name":"cross-ref","$":{"refid":"aff2","id":"crosref0065"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"†"}]},{"#name":"cross-ref","$":{"refid":"aff3","id":"crosref0070"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"‡"}]},{"#name":"cross-ref","$":{"refid":"aff5","id":"crosref0075"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"¶"}]}]},{"#name":"author","$":{"id":"au10","orcid":"0000-0003-3990-6494","author-id":"S1525157822001337-0aaa3724e0159a6c32a616603c5feeb7"},"$$":[{"#name":"given-name","_":"Melissa S."},{"#name":"surname","_":"Pessin"},{"#name":"cross-ref","$":{"refid":"aff1","id":"crosref0080"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"∗"}]}]},{"#name":"author","$":{"id":"au11","author-id":"S1525157822001337-8e3052c5108e70480481ad606c608f6a"},"$$":[{"#name":"given-name","_":"Ellinor I."},{"#name":"surname","_":"Peerschke"},{"#name":"cross-ref","$":{"refid":"aff1","id":"crosref0085"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"∗"}]},{"#name":"cross-ref","$":{"refid":"cor1","id":"crosref0090"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"∗"}]},{"#name":"encoded-e-address","__encoded":"JTdCJTIyJTIzbmFtZSUyMiUzQSUyMmUtYWRkcmVzcyUyMiUyQyUyMiUyNCUyMiUzQSU3QiUyMnhtbG5zJTNBeGxpbmslMjIlM0F0cnVlJTJDJTIydHlwZSUyMiUzQSUyMmVtYWlsJTIyJTJDJTIyaHJlZiUyMiUzQSUyMm1haWx0byUzQXBlZXJzY2hlJTQwbXNrY2Mub3JnJTIyJTJDJTIyaWQlMjIlM0ElMjJlYWRkMDAxNSUyMiU3RCUyQyUyMl8lMjIlM0ElMjJwZWVyc2NoZSU0MG1za2NjLm9yZyUyMiU3RA=="}]},{"#name":"affiliation","$":{"id":"aff1","affiliation-id":"S1525157822001337-38764f1544d403815f2c5294e67924a2"},"$$":[{"#name":"label","_":"∗"},{"#name":"textfn","_":"Department of Pathology and Laboratory Medicine, Memorial Sloan Kettering Cancer Center, New York, New York"},{"#name":"affiliation","$":{"xmlns:sa":true},"$$":[{"#name":"organization","_":"Department of Pathology and Laboratory Medicine"},{"#name":"organization","_":"Memorial Sloan Kettering Cancer Center"},{"#name":"city","_":"New York"},{"#name":"state","_":"New York"}]},{"#name":"source-text","$":{"id":"srct0010"},"_":"Department of Pathology and Laboratory Medicine, Memorial Sloan Kettering Cancer Center, New York, New York"}]},{"#name":"affiliation","$":{"id":"aff2","affiliation-id":"S1525157822001337-e5b38ab2abba6ec0216fb1cd4466b543"},"$$":[{"#name":"label","_":"†"},{"#name":"textfn","_":"Department of Medicine, Memorial Sloan Kettering Cancer Center, New York, New York"},{"#name":"affiliation","$":{"xmlns:sa":true},"$$":[{"#name":"organization","_":"Department of Medicine"},{"#name":"organization","_":"Memorial Sloan Kettering Cancer Center"},{"#name":"city","_":"New York"},{"#name":"state","_":"New York"}]},{"#name":"source-text","$":{"id":"srct0015"},"_":"Department of Medicine, Memorial Sloan Kettering Cancer Center, New York, New York"}]},{"#name":"affiliation","$":{"id":"aff3","affiliation-id":"S1525157822001337-9db7a16f906443141e5e2c6fa5ed20a1"},"$$":[{"#name":"label","_":"‡"},{"#name":"textfn","_":"Department of Medicine, Weil Cornell Medical College, New York, New York"},{"#name":"affiliation","$":{"xmlns:sa":true},"$$":[{"#name":"organization","_":"Department of Medicine"},{"#name":"organization","_":"Weil Cornell Medical College"},{"#name":"city","_":"New York"},{"#name":"state","_":"New York"}]},{"#name":"source-text","$":{"id":"srct0020"},"_":"Department of Medicine, Weil Cornell Medical College, New York, New York"}]},{"#name":"affiliation","$":{"id":"aff4","affiliation-id":"S1525157822001337-27fac9fed16b6937fe41777b3aa21694"},"$$":[{"#name":"label","_":"§"},{"#name":"textfn","_":"Department of Surgery, Memorial Sloan Kettering Cancer Center, New York, New York"},{"#name":"affiliation","$":{"xmlns:sa":true},"$$":[{"#name":"organization","_":"Department of Surgery"},{"#name":"organization","_":"Memorial Sloan Kettering Cancer Center"},{"#name":"city","_":"New York"},{"#name":"state","_":"New York"}]},{"#name":"source-text","$":{"id":"srct0025"},"_":"Department of Surgery, Memorial Sloan Kettering Cancer Center, New York, New York"}]},{"#name":"affiliation","$":{"id":"aff5","affiliation-id":"S1525157822001337-6e2fb1413e7ba3b7f2267dc5c24c7122"},"$$":[{"#name":"label","_":"¶"},{"#name":"textfn","_":"Kravis Center for Molecular Oncology, Memorial Sloan Kettering Cancer Center, New York, New York"},{"#name":"affiliation","$":{"xmlns:sa":true},"$$":[{"#name":"organization","_":"Kravis Center for Molecular Oncology"},{"#name":"organization","_":"Memorial Sloan Kettering Cancer Center"},{"#name":"city","_":"New York"},{"#name":"state","_":"New York"}]},{"#name":"source-text","$":{"id":"srct0030"},"_":"Kravis Center for Molecular Oncology, Memorial Sloan Kettering Cancer Center, New York, New York"}]},{"#name":"correspondence","$":{"id":"cor1"},"$$":[{"#name":"label","_":"∗"},{"#name":"text","_":"Address correspondence to Fei Ye, Ph.D., or Ellinor I. Peerschke, Ph.D., Department of Laboratory Medicine, Memorial Sloan Kettering Cancer Center, 327 E. 64th St., New York, NY 10065."},{"#name":"affiliation","$":{"xmlns:sa":true},"$$":[{"#name":"organization","_":"Department of Laboratory Medicine"},{"#name":"organization","_":"Memorial Sloan Kettering Cancer Center"},{"#name":"address-line","_":"327 E. 64th St."},{"#name":"city","_":"New York"},{"#name":"state","_":"NY"},{"#name":"postal-code","_":"10065"}]}]}]}],"floats":[],"footnotes":[],"affiliations":{"aff1":{"#name":"affiliation","$":{"id":"aff1","affiliation-id":"S1525157822001337-38764f1544d403815f2c5294e67924a2"},"$$":[{"#name":"label","_":"∗"},{"#name":"textfn","_":"Department of Pathology and Laboratory Medicine, Memorial Sloan Kettering Cancer Center, New York, New York"},{"#name":"affiliation","$":{"xmlns:sa":true},"$$":[{"#name":"organization","_":"Department of Pathology and Laboratory Medicine"},{"#name":"organization","_":"Memorial Sloan Kettering Cancer Center"},{"#name":"city","_":"New York"},{"#name":"state","_":"New York"}]},{"#name":"source-text","$":{"id":"srct0010"},"_":"Department of Pathology and Laboratory Medicine, Memorial Sloan Kettering Cancer Center, New York, New York"}]},"aff2":{"#name":"affiliation","$":{"id":"aff2","affiliation-id":"S1525157822001337-e5b38ab2abba6ec0216fb1cd4466b543"},"$$":[{"#name":"label","_":"†"},{"#name":"textfn","_":"Department of Medicine, Memorial Sloan Kettering Cancer Center, New York, New York"},{"#name":"affiliation","$":{"xmlns:sa":true},"$$":[{"#name":"organization","_":"Department of Medicine"},{"#name":"organization","_":"Memorial Sloan Kettering Cancer Center"},{"#name":"city","_":"New York"},{"#name":"state","_":"New York"}]},{"#name":"source-text","$":{"id":"srct0015"},"_":"Department of Medicine, Memorial Sloan Kettering Cancer Center, New York, New York"}]},"aff3":{"#name":"affiliation","$":{"id":"aff3","affiliation-id":"S1525157822001337-9db7a16f906443141e5e2c6fa5ed20a1"},"$$":[{"#name":"label","_":"‡"},{"#name":"textfn","_":"Department of Medicine, Weil Cornell Medical College, New York, New York"},{"#name":"affiliation","$":{"xmlns:sa":true},"$$":[{"#name":"organization","_":"Department of Medicine"},{"#name":"organization","_":"Weil Cornell Medical College"},{"#name":"city","_":"New York"},{"#name":"state","_":"New York"}]},{"#name":"source-text","$":{"id":"srct0020"},"_":"Department of Medicine, Weil Cornell Medical College, New York, New York"}]},"aff4":{"#name":"affiliation","$":{"id":"aff4","affiliation-id":"S1525157822001337-27fac9fed16b6937fe41777b3aa21694"},"$$":[{"#name":"label","_":"§"},{"#name":"textfn","_":"Department of Surgery, Memorial Sloan Kettering Cancer Center, New York, New York"},{"#name":"affiliation","$":{"xmlns:sa":true},"$$":[{"#name":"organization","_":"Department of Surgery"},{"#name":"organization","_":"Memorial Sloan Kettering Cancer Center"},{"#name":"city","_":"New York"},{"#name":"state","_":"New York"}]},{"#name":"source-text","$":{"id":"srct0025"},"_":"Department of Surgery, Memorial Sloan Kettering Cancer Center, New York, New York"}]},"aff5":{"#name":"affiliation","$":{"id":"aff5","affiliation-id":"S1525157822001337-6e2fb1413e7ba3b7f2267dc5c24c7122"},"$$":[{"#name":"label","_":"¶"},{"#name":"textfn","_":"Kravis Center for Molecular Oncology, Memorial Sloan Kettering Cancer Center, New York, New York"},{"#name":"affiliation","$":{"xmlns:sa":true},"$$":[{"#name":"organization","_":"Kravis Center for Molecular Oncology"},{"#name":"organization","_":"Memorial Sloan Kettering Cancer Center"},{"#name":"city","_":"New York"},{"#name":"state","_":"New York"}]},{"#name":"source-text","$":{"id":"srct0030"},"_":"Kravis Center for Molecular Oncology, Memorial Sloan Kettering Cancer Center, New York, New York"}]}},"correspondences":{"cor1":{"#name":"correspondence","$":{"id":"cor1"},"$$":[{"#name":"label","_":"∗"},{"#name":"text","_":"Address correspondence to Fei Ye, Ph.D., or Ellinor I. Peerschke, Ph.D., Department of Laboratory Medicine, Memorial Sloan Kettering Cancer Center, 327 E. 64th St., New York, NY 10065."},{"#name":"affiliation","$":{"xmlns:sa":true},"$$":[{"#name":"organization","_":"Department of Laboratory Medicine"},{"#name":"organization","_":"Memorial Sloan Kettering Cancer Center"},{"#name":"address-line","_":"327 E. 64th St."},{"#name":"city","_":"New York"},{"#name":"state","_":"NY"},{"#name":"postal-code","_":"10065"}]}]}},"attachments":[],"scopusAuthorIds":{},"articles":{}},"authorMetadata":[],"banner":{"expanded":false},"biographies":{},"body":{},"chapters":{"toc":[],"isLoading":false},"changeViewLinks":{"showFullTextLink":false,"showAbstractLink":false},"citingArticles":{},"combinedContentItems":{},"crossMark":{"isOpen":false},"domainConfig":{"cdnAssetsHost":"https://sdfestaticassets-eu-west-1.sciencedirectassets.com","assetRoute":"https://sdfestaticassets-eu-west-1.sciencedirectassets.com/prod/5f37568fb8c8e6bc810d6b0e6e66d8f143fc0e6e"},"downloadIssue":{"openOnPageLoad":false,"isOpen":false,"downloadCapOpen":false,"articles":[],"selected":[]},"enrichedContent":{"tableOfContents":false,"researchData":{"hasResearchData":false,"dataProfile":{},"openData":{},"mendeleyData":{},"databaseLinking":{}},"geospatialData":{"attachments":[]},"interactiveCaseInsights":{},"virtualMicroscope":{}},"entitledRecommendations":{"openOnPageLoad":false,"isOpen":false,"articles":[],"selected":[],"currentPage":1,"totalPages":1},"exam":{},"helpText":{"keyDates":{"html":"<div class=\"key-dates-help\"><h3 class=\"u-margin-s-bottom u-h4\">Publication milestones</h3><p class=\"u-margin-m-bottom\">The dates displayed for an article provide information on when various publication milestones were reached at the journal that has published the article. Where applicable, activities on preceding journals at which the article was previously under consideration are not shown (for instance submission, revisions, rejection).</p><p class=\"u-margin-xs-bottom\">The publication milestones include:</p><ul class=\"key-dates-help-list u-margin-m-bottom u-padding-s-left\"><li><span class=\"u-text-italic\">Received</span>: The date the article was originally submitted to the journal.</li><li><span class=\"u-text-italic\">Revised</span>: The date the most recent revision of the article was submitted to the journal. Dates corresponding to intermediate revisions are not shown.</li><li><span class=\"u-text-italic\">Accepted</span>: The date the article was accepted for publication in the journal.</li><li><span class=\"u-text-italic\">Available online</span>: The date a version of the article was made available online in the journal.</li><li><span class=\"u-text-italic\">Version of Record</span>: The date the finalized version of the article was made available in the journal.</li></ul><p>More information on publishing policies can be found on the <a class=\"anchor anchor-secondary u-display-inline anchor-underline\" href=\"https://www.elsevier.com/about/policies-and-standards/publishing-ethics\" target=\"_blank\"><span class=\"anchor-text-container\"><span class=\"anchor-text\">Publishing Ethics Policies</span></span></a> page. View our <a class=\"anchor anchor-secondary u-display-inline anchor-underline\" href=\"https://www.elsevier.com/researcher/author/submit-your-paper\" target=\"_blank\"><span class=\"anchor-text-container\"><span class=\"anchor-text\">Publishing with Elsevier: step-by-step</span></span></a> page to learn more about the publishing process. For any questions on your own submission or other questions related to publishing an article, <a class=\"anchor anchor-secondary u-display-inline anchor-underline\" href=\"https://service.elsevier.com/app/phone/supporthub/publishing\" target=\"_blank\"><span class=\"anchor-text-container\"><span class=\"anchor-text\">contact our Researcher support team.</span></span></a></p></div>","title":"What do these dates mean?"}},"glossary":{},"issueNavigation":{"previous":{},"next":{}},"linkingHubLinks":{},"metrics":{"metricGroup":{"citations":[],"captures":[],"mentions":[],"socialMedia":[]},"isLoading":false,"error":false},"preview":{},"rawtext":"","recommendations":{},"references":{},"referenceLinks":{"internal":[],"internalLoaded":false,"external":[]},"refersTo":{},"referredToBy":{},"relatedContent":{"isModal":false,"isOpenSpecialIssueArticles":false,"isOpenVirtualSpecialIssueLink":false,"isOpenRecommendations":true,"isOpenSubstances":true,"citingArticles":[false,false,false,false,false,false],"recommendations":[false,false,false,false,false,false]},"seamlessAccess":{},"specialIssueArticles":{},"substances":{},"supplementaryFilesData":[],"tableOfContents":{"showEntitledTocLinks":true},"tail":{},"transientError":{"isOpen":false},"sidePanel":{"openState":1},"viewConfig":{"articleFeature":{"rightsAndContentLink":true,"sdAnswersButton":false},"pathPrefix":""},"virtualSpecialIssue":{"showVirtualSpecialIssueLink":false},"usageProps":{"itemStage":"S300","isAip":false,"tombAip":"0","sample":false},"userCookiePreferences":{"STRICTLY_NECESSARY":true,"PERFORMANCE":false,"FUNCTIONAL":false,"TARGETING":false}};
      </script>
      <noscript>
      JavaScript is disabled on your browser.
      Please enable JavaScript to use all the features on this page.
      <img src=https://smetrics.elsevier.com/b/ss/elsevier-sd-prod/1/G.4--NS/1737318715139?pageName=sd%3Aproduct%3Ajournal%3Aarticle&c16=els%3Arp%3Ast&c2=sd&v185=img&v33=ae%3AREG_GUEST&c1=ae%3A246711&c12=ae%3A17847196 />
    </noscript>
      <div id="elementForFocusReset" tabindex="-1"></div><a class="anchor sr-only sr-only-focusable u-display-inline anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#screen-reader-main-content"><span class="anchor-text-container"><span class="anchor-text">Skip to main content</span></span></a><a class="anchor sr-only sr-only-focusable u-display-inline anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#screen-reader-main-title"><span class="anchor-text-container"><span class="anchor-text">Skip to article</span></span></a>
      <div id="root"><div class="App" id="app" data-aa-name="root"><div class="page"><div class="sd-flex-container"><div class="sd-flex-content"><header id="gh-cnt"><div id="gh-main-cnt" class="u-flex-center-ver u-position-relative u-padding-s-hor u-padding-l-hor-from-xl"><a id="gh-branding" class="u-flex-center-ver" href="https://www.sciencedirect.com/" aria-label="ScienceDirect home page" data-aa-region="header" data-aa-name="ScienceDirect"><img class="gh-logo" src="./ye2022analytical_files/elsevier-non-solus-new-grey.svg" alt="Elsevier logo" height="48" width="54"><svg xmlns="http://www.w3.org/2000/svg" version="1.1" height="15" viewBox="0 0 190 23" role="img" class="gh-wordmark u-margin-s-left" aria-labelledby="gh-wm-science-direct" focusable="false" aria-hidden="true" alt="ScienceDirect Wordmark"><title id="gh-wm-science-direct">ScienceDirect</title><g><path fill="#EB6500" d="M3.81 6.9c0-1.48 0.86-3.04 3.7-3.04 1.42 0 3.1 0.43 4.65 1.32l0.13-2.64c-1.42-0.63-2.97-0.96-4.78-0.96 -4.62 0-6.6 2.44-6.6 5.45 0 5.61 8.78 6.14 8.78 9.93 0 1.48-1.15 3.04-3.86 3.04 -1.72 0-3.4-0.56-4.72-1.39l-0.36 2.64c1.55 0.76 3.57 1.06 5.15 1.06 4.26 0 6.7-2.48 6.7-5.51C12.59 11.49 3.81 10.76 3.81 6.9M20.27 9.01c0.23-0.13 0.69-0.26 1.72-0.26 1.72 0 2.41 0.3 2.41 1.58h2.38c0-0.36 0-0.79-0.03-1.09 -0.23-1.98-2.15-2.67-4.88-2.67 -3 0-6.7 2.31-6.7 7.76 0 5.22 2.77 7.99 6.63 7.99 1.68 0 3.47-0.36 4.95-1.39l-0.2-2.31c-0.99 0.82-2.84 1.52-4.06 1.52 -2.14 0-4.55-1.71-4.55-5.91C17.93 10.2 20.01 9.18 20.27 9.01"></path><rect x="29.42" y="6.97" fill="#EB6500" width="2.54" height="14.95"></rect><path fill="#EB6500" d="M30.67 0.7c-0.92 0-1.65 0.92-1.65 1.81 0 0.93 0.76 1.85 1.65 1.85 0.89 0 1.68-0.96 1.68-1.88C32.35 1.55 31.56 0.7 30.67 0.7M48.06 14.13c0-5.18-1.42-7.56-6.01-7.56 -3.86 0-6.67 2.77-6.67 7.92 0 4.92 2.97 7.82 6.73 7.82 2.81 0 4.36-0.63 5.68-1.42l-0.2-2.31c-0.89 0.79-2.94 1.55-4.69 1.55 -3.14 0-4.88-1.95-4.88-5.51v-0.49H48.06M39.91 9.18c0.17-0.17 1.29-0.46 1.98-0.46 2.48 0 3.76 0.53 3.86 3.43h-7.46C38.56 10.27 39.71 9.37 39.91 9.18zM58.82 6.57c-2.24 0-3.63 1.12-4.85 2.61l-0.4-2.21h-2.34l0.13 1.19c0.1 0.76 0.13 1.78 0.13 2.97v10.79h2.54V11.88c0.69-0.96 2.15-2.48 2.48-2.64 0.23-0.13 1.29-0.4 2.08-0.4 2.28 0 2.48 1.15 2.54 3.43 0.03 1.19 0.03 3.17 0.03 3.17 0.03 3-0.1 6.47-0.1 6.47h2.54c0 0 0.07-4.49 0.07-6.96 0-1.48 0.03-2.97-0.1-4.46C63.31 7.43 61.49 6.57 58.82 6.57M72.12 9.01c0.23-0.13 0.69-0.26 1.72-0.26 1.72 0 2.41 0.3 2.41 1.58h2.38c0-0.36 0-0.79-0.03-1.09 -0.23-1.98-2.15-2.67-4.88-2.67 -3 0-6.7 2.31-6.7 7.76 0 5.22 2.77 7.99 6.63 7.99 1.68 0 3.47-0.36 4.95-1.39l-0.2-2.31c-0.99 0.82-2.84 1.52-4.06 1.52 -2.15 0-4.55-1.71-4.55-5.91C69.77 10.2 71.85 9.18 72.12 9.01M92.74 14.13c0-5.18-1.42-7.56-6.01-7.56 -3.86 0-6.67 2.77-6.67 7.92 0 4.92 2.97 7.82 6.73 7.82 2.81 0 4.36-0.63 5.68-1.42l-0.2-2.31c-0.89 0.79-2.94 1.55-4.69 1.55 -3.14 0-4.88-1.95-4.88-5.51v-0.49H92.74M84.59 9.18c0.17-0.17 1.29-0.46 1.98-0.46 2.48 0 3.76 0.53 3.86 3.43h-7.46C83.24 10.27 84.39 9.37 84.59 9.18zM103.9 1.98h-7.13v19.93h6.83c7.26 0 9.77-5.68 9.77-10.03C113.37 7.33 110.93 1.98 103.9 1.98M103.14 19.8h-3.76V4.1h4.09c5.38 0 6.96 4.39 6.96 7.79C110.43 16.87 108.19 19.8 103.14 19.8zM118.38 0.7c-0.92 0-1.65 0.92-1.65 1.81 0 0.93 0.76 1.85 1.65 1.85 0.89 0 1.69-0.96 1.69-1.88C120.07 1.55 119.28 0.7 118.38 0.7"></path><rect x="117.13" y="6.97" fill="#EB6500" width="2.54" height="14.95"></rect><path fill="#EB6500" d="M130.2 6.6c-1.62 0-2.87 1.45-3.4 2.74l-0.43-2.37h-2.34l0.13 1.19c0.1 0.76 0.13 1.75 0.13 2.9v10.86h2.54v-9.51c0.53-1.29 1.72-3.7 3.17-3.7 0.96 0 1.06 0.99 1.06 1.22l2.08-0.6V9.18c0-0.03-0.03-0.17-0.06-0.4C132.8 7.36 131.91 6.6 130.2 6.6M145.87 14.13c0-5.18-1.42-7.56-6.01-7.56 -3.86 0-6.67 2.77-6.67 7.92 0 4.92 2.97 7.82 6.73 7.82 2.81 0 4.36-0.63 5.68-1.42l-0.2-2.31c-0.89 0.79-2.94 1.55-4.69 1.55 -3.14 0-4.89-1.95-4.89-5.51v-0.49H145.87M137.72 9.18c0.17-0.17 1.29-0.46 1.98-0.46 2.48 0 3.76 0.53 3.86 3.43h-7.46C136.37 10.27 137.52 9.37 137.72 9.18zM153.23 9.01c0.23-0.13 0.69-0.26 1.72-0.26 1.72 0 2.41 0.3 2.41 1.58h2.38c0-0.36 0-0.79-0.03-1.09 -0.23-1.98-2.14-2.67-4.88-2.67 -3 0-6.7 2.31-6.7 7.76 0 5.22 2.77 7.99 6.63 7.99 1.69 0 3.47-0.36 4.95-1.39l-0.2-2.31c-0.99 0.82-2.84 1.52-4.06 1.52 -2.15 0-4.55-1.71-4.55-5.91C150.89 10.2 152.97 9.18 153.23 9.01M170 19.44c-0.92 0.36-1.72 0.69-2.51 0.69 -1.16 0-1.58-0.66-1.58-2.34V8.95h3.93V6.97h-3.93V2.97h-2.48v3.99h-2.71v1.98h2.71v9.67c0 2.64 1.39 3.73 3.33 3.73 1.15 0 2.54-0.39 3.43-0.79L170 19.44M173.68 5.96c-1.09 0-2-0.87-2-1.97 0-1.1 0.91-1.97 2-1.97s1.98 0.88 1.98 1.98C175.66 5.09 174.77 5.96 173.68 5.96zM173.67 2.46c-0.85 0-1.54 0.67-1.54 1.52 0 0.85 0.69 1.54 1.54 1.54 0.85 0 1.54-0.69 1.54-1.54C175.21 3.13 174.52 2.46 173.67 2.46zM174.17 5.05c-0.09-0.09-0.17-0.19-0.25-0.3l-0.41-0.56h-0.16v0.87h-0.39V2.92c0.22-0.01 0.47-0.03 0.66-0.03 0.41 0 0.82 0.16 0.82 0.64 0 0.29-0.21 0.55-0.49 0.63 0.23 0.32 0.45 0.62 0.73 0.91H174.17zM173.56 3.22l-0.22 0.01v0.63h0.22c0.26 0 0.43-0.05 0.43-0.34C174 3.28 173.83 3.21 173.56 3.22z"></path></g></svg></a><div class="gh-nav-cnt u-hide-from-print"><div class="gh-nav-links-container gh-nav-links-container-h u-hide-from-print gh-nav-content-container"><nav aria-label="links" class="gh-nav gh-nav-links gh-nav-h"><ul class="gh-nav-list u-list-reset"><li class="gh-nav-item gh-move-to-spine"><a class="anchor gh-nav-action text-s anchor-secondary anchor-medium" href="https://www.sciencedirect.com/browse/journals-and-books" id="gh-journals-books-link" data-aa-region="header" data-aa-name="Journals &amp; Books"><span class="anchor-text-container"><span class="anchor-text">Journals &amp; Books</span></span></a></li></ul></nav><nav aria-label="utilities" class="gh-nav gh-nav-utilities gh-nav-h"><ul class="gh-nav-list u-list-reset"><li class="gh-nav-help text-s u-flex-center-ver u-gap-6 gh-nav-action"><div class="gh-move-to-spine gh-help-button gh-help-icon gh-nav-item"><div class="popover" id="gh-help-icon-popover"><div id="popover-trigger-gh-help-icon-popover"><input type="hidden"><button class="button-link button-link-secondary gh-icon-btn button-link-medium button-link-icon-left" title="Help" aria-expanded="false" type="button"><svg focusable="false" viewBox="0 0 114 128" height="20" width="20" class="icon icon-help gh-icon"><path d="M57 8C35.69 7.69 15.11 21.17 6.68 40.71c-8.81 19.38-4.91 43.67 9.63 59.25 13.81 15.59 36.85 21.93 56.71 15.68 21.49-6.26 37.84-26.81 38.88-49.21 1.59-21.15-10.47-42.41-29.29-52.1C74.76 10.17 65.88 7.99 57 8zm0 10c20.38-.37 39.57 14.94 43.85 34.85 4.59 18.53-4.25 39.23-20.76 48.79-17.05 10.59-40.96 7.62-54.9-6.83-14.45-13.94-17.42-37.85-6.83-54.9C26.28 26.5 41.39 17.83 57 18zm-.14 14C45.31 32.26 40 40.43 40 50v2h10v-2c0-4.22 2.22-9.66 8-9.24 5.5.4 6.32 5.14 5.78 8.14C62.68 55.06 52 58.4 52 69.4V76h10v-5.56c0-8.16 11.22-11.52 12-21.7.74-9.86-5.56-16.52-16-16.74-.39-.01-.76-.01-1.14 0zM52 82v10h10V82H52z"></path></svg><span class="button-link-text-container"><span class="button-link-text">Help</span></span></button></div></div></div></li><li class="gh-nav-search text-s u-flex-center-ver u-gap-6 gh-nav-action"><div class="gh-search-toggle gh-nav-item search-button-link"><a class="anchor button-link-secondary anchor-secondary u-margin-l-left gh-nav-action gh-icon-btn anchor-medium anchor-icon-left anchor-with-icon" href="https://www.sciencedirect.com/search" id="gh-search-link" title="Search" data-aa-button="search-in-header-opened-from-article" role="button"><svg focusable="false" viewBox="0 0 100 128" height="20" class="icon icon-search gh-icon"><path d="M19.22 76.91c-5.84-5.84-9.05-13.6-9.05-21.85s3.21-16.01 9.05-21.85c5.84-5.83 13.59-9.05 21.85-9.05 8.25 0 16.01 3.22 21.84 9.05 5.84 5.84 9.05 13.6 9.05 21.85s-3.21 16.01-9.05 21.85c-5.83 5.83-13.59 9.05-21.84 9.05-8.26 0-16.01-3.22-21.85-9.05zm80.33 29.6L73.23 80.19c5.61-7.15 8.68-15.9 8.68-25.13 0-10.91-4.25-21.17-11.96-28.88-7.72-7.71-17.97-11.96-28.88-11.96S19.9 18.47 12.19 26.18C4.47 33.89.22 44.15.22 55.06s4.25 21.17 11.97 28.88C19.9 91.65 30.16 95.9 41.07 95.9c9.23 0 17.98-3.07 25.13-8.68l26.32 26.32 7.03-7.03"></path></svg><span class="anchor-text-container"><span class="anchor-text">Search</span></span></a></div></li></ul></nav></div></div><div class="gh-profile-container u-hide-from-print"><div id="gh-profile-cnt" class="u-flex-center-ver gh-move-to-spine"><div class="popover" id="gh-profile-dropdown"><div id="popover-trigger-gh-profile-dropdown"><input type="hidden"><button class="button-link gh-icon-btn gh-user-icon u-margin-l-left gh-truncate text-s button-link-secondary button-link-medium button-link-icon-left" title="José Andrés Castillo Rivas" aria-expanded="false" type="button"><svg focusable="false" viewBox="0 0 106 128" height="20" class="icon icon-person"><path d="M11.07 120l.84-9.29C13.88 91.92 35.25 87.78 53 87.78c17.74 0 39.11 4.13 41.08 22.84l.84 9.38h10.04l-.93-10.34C101.88 89.23 83.89 78 53 78S4.11 89.22 1.95 109.73L1.04 120h10.03M53 17.71c-9.72 0-18.24 8.69-18.24 18.59 0 13.67 7.84 23.98 18.24 23.98S71.24 49.97 71.24 36.3c0-9.9-8.52-18.59-18.24-18.59zM53 70c-15.96 0-28-14.48-28-33.67C25 20.97 37.82 8 53 8s28 12.97 28 28.33C81 55.52 68.96 70 53 70"></path></svg><span class="button-link-text-container"><span class="button-link-text">José Andrés Castillo Rivas</span></span></button></div></div></div></div><a class="anchor text-s u-clr-grey8 gh-move-to-spine u-hide-from-print u-margin-l-left anchor-secondary gh-icon-btn anchor-medium anchor-icon-left anchor-with-icon" href="https://www.sciencedirect.com/user/institution/login?targetURL=%2Fscience%2Farticle%2Fpii%2FS1525157822001337" id="gh-institutionalsignin-btn" data-aa-region="header" data-aa-name="institutionalsignin"><svg focusable="false" viewBox="0 0 106 128" height="20" aria-hidden="true" class="icon icon-institution gh-cta-btn-icon"><path d="M84 98h10v10H12V98h10V52h14v46h10V52h14v46h10V52h14v46zM12 36.86l41-20.84 41 20.84V42H12v-5.14zM104 52V30.74L53 4.8 2 30.74V52h10v36H2v30h102V88H94V52h10z"></path></svg><span class="anchor-text-container"><span class="anchor-text">Sign in</span></span></a><div id="gh-mobile-menu" class="mobile-menu u-hide-from-print"><div class="gh-hamburger u-fill-grey7"><button class="button-link u-flex-center-ver button-link-primary button-link-icon-left" aria-label="Toggle mobile menu" aria-expanded="false" type="button"><svg class="gh-hamburger-svg-el gh-hamburger-closed" role="img" aria-hidden="true" height="20" width="20"><path d="M0 14h40v2H0zm0-7h40v2H0zm0-7h40v2H0z"></path></svg></button></div><div id="gh-overlay" class="mobile-menu-overlay u-overlay u-display-none" role="button" tabindex="-1"></div><div id="gh-drawer" aria-label="Mobile menu" class="" role="navigation"></div></div></div></header><div class="Article" id="mathjax-container" role="main"><div class="accessbar-sticky"><div id="screen-reader-main-content"></div><div role="region" aria-label="Download options and search"><div class="accessbar"><div class="accessbar-label"></div><ul aria-label="PDF Options"><li class="accessbar-item-show-from-initial accessbar-item-show-from-xs accessbar-item-show-from-md ViewPDF"><a class="link-button accessbar-utility-component accessbar-utility-link link-button-primary link-button-icon-left" target="_blank" aria-label="View PDF. Opens in a new window." href="https://www.sciencedirect.com/science/article/pii/S1525157822001337/pdfft?md5=e8814061aa6a6d230d9dc16eda9f1ace&amp;pid=1-s2.0-S1525157822001337-main.pdf" rel="nofollow"><svg focusable="false" viewBox="0 0 35 32" height="20" class="icon icon-pdf-multicolor"><path d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path d="M.167 2.592H22.39V9.72H.166z" fill="#da0000"></path><path fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="link-button-text-container"><span class="link-button-text"><span>View&nbsp;<strong>PDF</strong></span></span></span></a></li><li class="accessbar-item-hide-from-initial accessbar-item-hide-from-xs accessbar-item-show-from-md DownloadFullIssue"><button class="button-link accessbar-utility-component button-link-primary" aria-label="Download full issue" type="button"><span class="button-link-text-container"><span class="button-link-text"><span>Download full issue</span></span></span></button></li><li class="accessbar-item-hide-from-initial accessbar-item-hide-from-xs accessbar-item-show-from-md Divider"><span class="accessbar-divider"></span></li><li class="accessbar-item-hide-from-initial accessbar-item-hide-from-xs accessbar-item-show-from-md OpenManuscript"><a class="anchor accessbar-utility-component accessbar-utility-link anchor-primary" href="https://www.sciencedirect.com/science/article/am/pii/S1525157822001337" target="_blank" aria-label="View Open Manuscript"><span class="anchor-text-container"><span class="anchor-text"><span>View Open Manuscript</span></span></span></a></li><li class="accessbar-item-show-from-initial accessbar-item-show-from-xs accessbar-item-hide-from-md OverflowPopover"><div class="popover accessbar-overflow-popover" id="OverflowPopoverAnchor"><div id="popover-trigger-OverflowPopoverAnchor"><button class="button-link accessbar-utility-component button-link-primary button-link-icon-right" aria-label="Other access options" type="button"><span class="button-link-text-container"><span class="button-link-text"><span>Other access options</span></span></span><svg focusable="false" viewBox="0 0 92 128" height="20" class="icon icon-navigate-down"><path d="M1 51l7-7 38 38 38-38 7 7-45 45z"></path></svg></button></div></div></li></ul><form class="QuickSearch" action="https://www.sciencedirect.com/search#submit" method="get" aria-label="form"><div class="search-input"><div class="search-input-container search-input-container-no-label"><label class="search-input-label u-hide-visually" for="article-quick-search">Search ScienceDirect</label><input type="text" id="article-quick-search" name="qs" class="search-input-field" aria-describedby="article-quick-search-description-message" aria-invalid="false" aria-label="Search ScienceDirect" placeholder="Search ScienceDirect" value=""></div><div class="search-input-message-container"><div class="search-input-validation-error" aria-live="polite"></div><div id="article-quick-search-description-message"></div></div></div><button type="submit" class="button u-margin-xs-left button-primary small button-icon-only" aria-disabled="false" aria-label="Submit search"><svg focusable="false" viewBox="0 0 100 128" height="20" class="icon icon-search"><path d="M19.22 76.91c-5.84-5.84-9.05-13.6-9.05-21.85s3.21-16.01 9.05-21.85c5.84-5.83 13.59-9.05 21.85-9.05 8.25 0 16.01 3.22 21.84 9.05 5.84 5.84 9.05 13.6 9.05 21.85s-3.21 16.01-9.05 21.85c-5.83 5.83-13.59 9.05-21.84 9.05-8.26 0-16.01-3.22-21.85-9.05zm80.33 29.6L73.23 80.19c5.61-7.15 8.68-15.9 8.68-25.13 0-10.91-4.25-21.17-11.96-28.88-7.72-7.71-17.97-11.96-28.88-11.96S19.9 18.47 12.19 26.18C4.47 33.89.22 44.15.22 55.06s4.25 21.17 11.97 28.88C19.9 91.65 30.16 95.9 41.07 95.9c9.23 0 17.98-3.07 25.13-8.68l26.32 26.32 7.03-7.03"></path></svg></button><input type="hidden" name="origin" value="article"><input type="hidden" name="zone" value="qSearch"></form></div></div></div><div class="article-wrapper grid row"><div role="navigation" class="u-display-block-from-lg col-lg-6 u-padding-s-top sticky-table-of-contents" aria-label="Table of contents"><div class="TableOfContents" lang="en"><div class="Outline" id="toc-outline"><h2 class="u-h4">Outline</h2><ol class="u-padding-xs-bottom"><li class="ai-components-toc-entry" id="ai-components-toc-entry"></li><li class="toc-list-entry-outline-padding"><a class="anchor u-truncate-anchor-text anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#sec1" data-aa-button="sd:product:journal:article:type=anchor:name=outlinelink" title="Materials and Methods"><span class="anchor-text-container"><span class="anchor-text">Materials and Methods</span></span></a></li><li class="toc-list-entry-outline-padding"><a class="anchor u-truncate-anchor-text anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#sec2" data-aa-button="sd:product:journal:article:type=anchor:name=outlinelink" title="Results"><span class="anchor-text-container"><span class="anchor-text">Results</span></span></a></li><li class="toc-list-entry-outline-padding"><a class="anchor u-truncate-anchor-text anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#sec3" data-aa-button="sd:product:journal:article:type=anchor:name=outlinelink" title="Discussion"><span class="anchor-text-container"><span class="anchor-text">Discussion</span></span></a></li><li class="toc-list-entry-outline-padding"><a class="anchor u-truncate-anchor-text anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#appsec1" data-aa-button="sd:product:journal:article:type=anchor:name=outlinelink" title="Supplemental Data"><span class="anchor-text-container"><span class="anchor-text">Supplemental Data</span></span></a></li><li class="toc-list-entry-outline-padding"><a class="anchor u-truncate-anchor-text anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#cebib0010" data-aa-button="sd:product:journal:article:type=anchor:name=outlinelink" title="References"><span class="anchor-text-container"><span class="anchor-text">References</span></span></a></li></ol><button class="button-link u-margin-xs-top u-margin-s-bottom button-link-primary button-link-icon-right" aria-expanded="false" data-aa-button="sd:product:journal:article:type=menu:name=show-full-outline" type="button"><span class="button-link-text-container"><span class="button-link-text">Show full outline</span></span><svg focusable="false" viewBox="0 0 92 128" height="20" class="icon icon-navigate-down"><path d="M1 51l7-7 38 38 38-38 7 7-45 45z"></path></svg></button><div class="PageDivider"></div></div><div class="CitedBy" id="toc-cited-by"><h2 class="u-h4"><a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#section-cited-by"><span class="anchor-text-container"><span class="anchor-text">Cited by (12)</span></span></a></h2><div class="PageDivider"></div></div><div class="Figures" id="toc-figures"><h2 class="u-h4">Figures (6)</h2><ol class="u-margin-s-bottom"><li><a class="anchor u-display-block anchor-primary anchor-icon-only anchor-with-icon" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#fig1" data-aa-button="sd:product:journal:article:type=anchor:name=figure"><img alt="Figure1. Expression (CT values) of exogenous nonhuman spike-in cel-miR-39-3p and…" class="u-display-block" height="44px" src="./ye2022analytical_files/1-s2.0-S1525157822001337-gr1.sml" width="219px"></a></li><li><a class="anchor u-display-block anchor-primary anchor-icon-only anchor-with-icon" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#fig2" data-aa-button="sd:product:journal:article:type=anchor:name=figure"><img alt="Figure2. Results of linearity studies for detection of miR-371a-3p and miR-372-3p using…" class="u-display-block" height="144px" src="./ye2022analytical_files/1-s2.0-S1525157822001337-gr2.sml" width="219px"></a></li><li><a class="anchor u-display-block anchor-primary anchor-icon-only anchor-with-icon" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#fig3" data-aa-button="sd:product:journal:article:type=anchor:name=figure"><img alt="Figure3. Reproducibility of serum miR-371a-3p and miR-372-3p quantitation within and…" class="u-display-block" height="60px" src="./ye2022analytical_files/1-s2.0-S1525157822001337-gr3.sml" width="219px"></a></li><li><a class="anchor u-display-block anchor-primary anchor-icon-only anchor-with-icon" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#fig4" data-aa-button="sd:product:journal:article:type=anchor:name=figure"><img alt="Figure4. Receiver operating characteristic analysis, demonstrating performance of serum…" class="u-display-block" height="164px" src="./ye2022analytical_files/1-s2.0-S1525157822001337-gr4.sml" width="174px"></a></li><li><a class="anchor u-display-block anchor-primary anchor-icon-only anchor-with-icon" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#fig5" data-aa-button="sd:product:journal:article:type=anchor:name=figure"><img alt="Figure5. Stability of miR-371a-3p (A) and miR-372-3p (B) targets in whole blood stored…" class="u-display-block" height="78px" src="./ye2022analytical_files/1-s2.0-S1525157822001337-gr5.sml" width="219px"></a></li><li><a class="anchor u-display-block anchor-primary anchor-icon-only anchor-with-icon" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#fig6" data-aa-button="sd:product:journal:article:type=anchor:name=figure"><img alt="Figure6. Effect of hemolysis on endogenous housekeeping miRNA miR-30b-5p and red blood…" class="u-display-block" height="89px" src="./ye2022analytical_files/1-s2.0-S1525157822001337-gr6.sml" width="219px"></a></li></ol><div class="PageDivider"></div></div><div class="Tables" id="toc-tables"><h2 class="u-h4">Tables (5)</h2><ol class="u-padding-s-bottom"><li class="toc-list-entry-outline-padding"><a class="anchor u-truncate-anchor-text anchor-primary anchor-icon-left anchor-with-icon" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#tbl1" data-aa-button="sd:product:journal:article:type=anchor:name=table" title="Patient Demographics (N=48; 51 Samples from 48 Individual Patients)"><svg focusable="false" viewBox="0 0 98 128" height="20" class="icon icon-table"><path d="M54 68h32v32H54V68zm-42 0h32v32H12V68zm0-42h32v32H12V26zm42 0h32v32H54V26zM2 110h94V16H2v94z"></path></svg><span class="anchor-text-container"><span class="anchor-text">Table&nbsp;1</span></span></a></li><li class="toc-list-entry-outline-padding"><a class="anchor u-truncate-anchor-text anchor-primary anchor-icon-left anchor-with-icon" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#tbl2" data-aa-button="sd:product:journal:article:type=anchor:name=table" title="TaqMan Assays and miRNA Sequences Used in Reverse Transcription, Pre-Amplification, and qPCR"><svg focusable="false" viewBox="0 0 98 128" height="20" class="icon icon-table"><path d="M54 68h32v32H54V68zm-42 0h32v32H12V68zm0-42h32v32H12V26zm42 0h32v32H54V26zM2 110h94V16H2v94z"></path></svg><span class="anchor-text-container"><span class="anchor-text">Table&nbsp;2</span></span></a></li><li class="toc-list-entry-outline-padding"><a class="anchor u-truncate-anchor-text anchor-primary anchor-icon-left anchor-with-icon" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#tbl3" data-aa-button="sd:product:journal:article:type=anchor:name=table" title="Analytical Sensitivity"><svg focusable="false" viewBox="0 0 98 128" height="20" class="icon icon-table"><path d="M54 68h32v32H54V68zm-42 0h32v32H12V68zm0-42h32v32H12V26zm42 0h32v32H54V26zM2 110h94V16H2v94z"></path></svg><span class="anchor-text-container"><span class="anchor-text">Table&nbsp;3</span></span></a></li><li class="toc-list-entry-outline-padding"><a class="anchor u-truncate-anchor-text anchor-primary anchor-icon-left anchor-with-icon" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#tbl4" data-aa-button="sd:product:journal:article:type=anchor:name=table" title="Analytical Accuracy: Truth Tables"><svg focusable="false" viewBox="0 0 98 128" height="20" class="icon icon-table"><path d="M54 68h32v32H54V68zm-42 0h32v32H12V68zm0-42h32v32H12V26zm42 0h32v32H54V26zM2 110h94V16H2v94z"></path></svg><span class="anchor-text-container"><span class="anchor-text">Table&nbsp;4</span></span></a></li><li class="toc-list-entry-outline-padding"><a class="anchor u-truncate-anchor-text anchor-primary anchor-icon-left anchor-with-icon" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#tbl5" data-aa-button="sd:product:journal:article:type=anchor:name=table" title="Analytical Accuracy: Calculations of Sensitivity and Specificity"><svg focusable="false" viewBox="0 0 98 128" height="20" class="icon icon-table"><path d="M54 68h32v32H54V68zm-42 0h32v32H12V68zm0-42h32v32H12V26zm42 0h32v32H54V26zM2 110h94V16H2v94z"></path></svg><span class="anchor-text-container"><span class="anchor-text">Table&nbsp;5</span></span></a></li></ol><div class="PageDivider"></div></div><div class="Extras" id="toc-extras"><h2 class="u-h4">Extras (5)</h2><ol class="u-padding-s-bottom"><li><span class="download-all-supplemental-data"><span class="in-toc"><iframe title="download_all" name="download_all" class="u-hide" src="./ye2022analytical_files/saved_resource.html"></iframe><form method="post" action="https://www.sciencedirect.com/sdfe/pdf/download/S1525157822001337/attachments" target="download_all"><input name="eids" type="hidden" value="1-s2.0-S1525157822001337-mmc1.docx"><input name="eids" type="hidden" value="1-s2.0-S1525157822001337-mmc2.docx"><input name="eids" type="hidden" value="1-s2.0-S1525157822001337-mmc3.docx"><input name="eids" type="hidden" value="1-s2.0-S1525157822001337-mmc4.docx"><input name="eids" type="hidden" value="1-s2.0-S1525157822001337-mmc5.docx"><button class="button-link download-all-button u-font-sans u-margin-xs-bottom button-link-primary button-link-icon-left" type="submit"><svg focusable="false" viewBox="0 0 98 128" height="20" class="icon icon-download"><path d="M77.38 56.18l-6.6-7.06L54 66.36V26H44v40.34L26.28 49.1l-7.3 7.08 29.2 29.32 29.2-29.32M88 74v20H10V74H0v30h98V74H88"></path></svg><span class="button-link-text-container"><span class="button-link-text"><span class="download-all-title">Download all </span></span></span></button></form></span></span></li><li class="toc-list-entry-outline-padding"><a class="anchor u-truncate-anchor-text anchor-primary anchor-icon-left anchor-with-icon" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#mmc1" data-aa-button="sd:product:journal:article:type=anchor:region=outline:name=sdf" title="Supplemental Table S1"><svg focusable="false" viewBox="0 0 92 128" height="20" class="icon icon-text-document"><path d="M1 10v108h57.22L91 85.09V10H1m80 70.96L54.07 108H11V20h70v60.96zM43 100h10V80h20V70H43v30M25 32h42v10H25V32zm0 18h42v10H25V50z"></path></svg><span class="anchor-text-container"><span class="anchor-text">Supplemental Table S1</span></span></a></li><li class="toc-list-entry-outline-padding"><a class="anchor u-truncate-anchor-text anchor-primary anchor-icon-left anchor-with-icon" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#mmc2" data-aa-button="sd:product:journal:article:type=anchor:region=outline:name=sdf" title="Supplemental Table S2"><svg focusable="false" viewBox="0 0 92 128" height="20" class="icon icon-text-document"><path d="M1 10v108h57.22L91 85.09V10H1m80 70.96L54.07 108H11V20h70v60.96zM43 100h10V80h20V70H43v30M25 32h42v10H25V32zm0 18h42v10H25V50z"></path></svg><span class="anchor-text-container"><span class="anchor-text">Supplemental Table S2</span></span></a></li><li class="toc-list-entry-outline-padding"><a class="anchor u-truncate-anchor-text anchor-primary anchor-icon-left anchor-with-icon" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#mmc3" data-aa-button="sd:product:journal:article:type=anchor:region=outline:name=sdf" title="Supplemental Table S3"><svg focusable="false" viewBox="0 0 92 128" height="20" class="icon icon-text-document"><path d="M1 10v108h57.22L91 85.09V10H1m80 70.96L54.07 108H11V20h70v60.96zM43 100h10V80h20V70H43v30M25 32h42v10H25V32zm0 18h42v10H25V50z"></path></svg><span class="anchor-text-container"><span class="anchor-text">Supplemental Table S3</span></span></a></li><li class="toc-list-entry-outline-padding"><a class="anchor u-truncate-anchor-text anchor-primary anchor-icon-left anchor-with-icon" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#mmc4" data-aa-button="sd:product:journal:article:type=anchor:region=outline:name=sdf" title="Supplemental Table S4"><svg focusable="false" viewBox="0 0 92 128" height="20" class="icon icon-text-document"><path d="M1 10v108h57.22L91 85.09V10H1m80 70.96L54.07 108H11V20h70v60.96zM43 100h10V80h20V70H43v30M25 32h42v10H25V32zm0 18h42v10H25V50z"></path></svg><span class="anchor-text-container"><span class="anchor-text">Supplemental Table S4</span></span></a></li><li class="toc-list-entry-outline-padding"><a class="anchor u-truncate-anchor-text anchor-primary anchor-icon-left anchor-with-icon" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#mmc5" data-aa-button="sd:product:journal:article:type=anchor:region=outline:name=sdf" title="Supplemental Table S5"><svg focusable="false" viewBox="0 0 92 128" height="20" class="icon icon-text-document"><path d="M1 10v108h57.22L91 85.09V10H1m80 70.96L54.07 108H11V20h70v60.96zM43 100h10V80h20V70H43v30M25 32h42v10H25V32zm0 18h42v10H25V50z"></path></svg><span class="anchor-text-container"><span class="anchor-text">Supplemental Table S5</span></span></a></li></ol><div class="PageDivider"></div></div></div></div><article class="col-lg-12 col-md-16 pad-left pad-right u-padding-s-top" lang="en"><div class="Publication" id="publication"><div class="publication-brand u-display-block-from-sm"><a class="anchor u-display-flex anchor-primary" href="https://www.sciencedirect.com/journal/the-journal-of-molecular-diagnostics" title="Go to The Journal of Molecular Diagnostics on ScienceDirect"><span class="anchor-text-container"><span class="anchor-text"><img class="publication-brand-image" src="./ye2022analytical_files/elsevier-non-solus.png" alt="Elsevier"></span></span></a></div><div class="publication-volume u-text-center"><h2 class="publication-title u-h3" id="publication-title"><a class="anchor anchor-secondary publication-title-link" href="https://www.sciencedirect.com/journal/the-journal-of-molecular-diagnostics" title="Go to The Journal of Molecular Diagnostics on ScienceDirect"><span class="anchor-text-container"><span class="anchor-text">The Journal of Molecular Diagnostics</span></span></a></h2><div class="text-xs"><a class="anchor anchor-primary" href="https://www.sciencedirect.com/journal/the-journal-of-molecular-diagnostics/vol/24/issue/8" title="Go to table of contents for this volume/issue"><span class="anchor-text-container"><span class="anchor-text">Volume 24, Issue 8</span></span></a>, <!-- -->August 2022<!-- -->, Pages 867-877</div></div><div class="publication-cover u-display-block-from-sm"><a class="anchor u-display-flex anchor-primary" href="https://www.sciencedirect.com/journal/the-journal-of-molecular-diagnostics/vol/24/issue/8"><span class="anchor-text-container"><span class="anchor-text"><img class="publication-cover-image" src="./ye2022analytical_files/1-s2.0-S1525157821X00190-cov150h.gif" alt="The Journal of Molecular Diagnostics"></span></span></a></div></div><h1 id="screen-reader-main-title" class="Head u-font-serif u-h2 u-margin-s-ver"><div class="article-dochead"><span>Regular article</span></div><span class="title-text">Analytical Validation and Performance Characteristics of Molecular Serum Biomarkers, miR-371a-3p and miR-372-3p, for Male Germ Cell Tumors, in a Clinical Laboratory Setting</span></h1><div class="Banner" id="banner"><div class="wrapper truncated"><div aria-live="polite"></div><div class="AuthorGroups"><div class="author-group" id="author-group"><span class="sr-only">Author links open overlay panel</span><button class="button-link button-link-secondary button-link-underline" data-sd-ui-side-panel-opener="true" data-xocs-content-type="author" data-xocs-content-id="au1" type="button"><span class="button-link-text-container"><span class="button-link-text"><span class="react-xocs-alternative-link"><span class="given-name">Fei</span> <span class="text surname">Ye</span> </span><span class="author-ref" id="baff1"><sup>∗</sup></span><svg focusable="false" viewBox="0 0 106 128" height="20" title="Correspondence author icon" class="icon icon-person react-xocs-author-icon u-fill-grey8"><path d="M11.07 120l.84-9.29C13.88 91.92 35.25 87.78 53 87.78c17.74 0 39.11 4.13 41.08 22.84l.84 9.38h10.04l-.93-10.34C101.88 89.23 83.89 78 53 78S4.11 89.22 1.95 109.73L1.04 120h10.03M53 17.71c-9.72 0-18.24 8.69-18.24 18.59 0 13.67 7.84 23.98 18.24 23.98S71.24 49.97 71.24 36.3c0-9.9-8.52-18.59-18.24-18.59zM53 70c-15.96 0-28-14.48-28-33.67C25 20.97 37.82 8 53 8s28 12.97 28 28.33C81 55.52 68.96 70 53 70"></path></svg><svg focusable="false" viewBox="0 0 102 128" height="20" title="Author email or social media contact details icon" class="icon icon-envelope react-xocs-author-icon u-fill-grey8"><path d="M55.8 57.2c-1.78 1.31-5.14 1.31-6.9 0L17.58 34h69.54L55.8 57.19zM0 32.42l42.94 32.62c2.64 1.95 6.02 2.93 9.4 2.93s6.78-.98 9.42-2.93L102 34.34V24H0zM92 88.9L73.94 66.16l-8.04 5.95L83.28 94H18.74l18.38-23.12-8.04-5.96L10 88.94V51.36L0 42.9V104h102V44.82l-10 8.46V88.9"></path></svg></span></span></button>, <button class="button-link button-link-secondary button-link-underline" data-sd-ui-side-panel-opener="true" data-xocs-content-type="author" data-xocs-content-id="au2" type="button"><span class="button-link-text-container"><span class="button-link-text"><span class="react-xocs-alternative-link"><span class="given-name">Darren R.</span> <span class="text surname">Feldman</span> </span><span class="author-ref" id="baff2"><sup>†</sup></span> <span class="author-ref" id="baff3"><sup>‡</sup></span></span></span></button>, <button class="button-link button-link-secondary button-link-underline" data-sd-ui-side-panel-opener="true" data-xocs-content-type="author" data-xocs-content-id="au3" type="button"><span class="button-link-text-container"><span class="button-link-text"><span class="react-xocs-alternative-link"><span class="given-name">Alisa</span> <span class="text surname">Valentino</span> </span><span class="author-ref" id="baff1"><sup>∗</sup></span></span></span></button>, <button class="button-link button-link-secondary button-link-underline" data-sd-ui-side-panel-opener="true" data-xocs-content-type="author" data-xocs-content-id="au4" type="button"><span class="button-link-text-container"><span class="button-link-text"><span class="react-xocs-alternative-link"><span class="given-name">Rachel</span> <span class="text surname">So</span> </span><span class="author-ref" id="baff1"><sup>∗</sup></span></span></span></button>, <button class="button-link button-link-secondary button-link-underline" data-sd-ui-side-panel-opener="true" data-xocs-content-type="author" data-xocs-content-id="au5" type="button"><span class="button-link-text-container"><span class="button-link-text"><span class="react-xocs-alternative-link"><span class="given-name">Maria</span> <span class="text surname">Bromberg</span> </span><span class="author-ref" id="baff2"><sup>†</sup></span></span></span></button>, <button class="button-link button-link-secondary button-link-underline" data-sd-ui-side-panel-opener="true" data-xocs-content-type="author" data-xocs-content-id="au6" type="button"><span class="button-link-text-container"><span class="button-link-text"><span class="react-xocs-alternative-link"><span class="given-name">Shah</span> <span class="text surname">Khan</span> </span><span class="author-ref" id="baff2"><sup>†</sup></span></span></span></button>, <button class="button-link button-link-secondary button-link-underline" data-sd-ui-side-panel-opener="true" data-xocs-content-type="author" data-xocs-content-id="au7" type="button"><span class="button-link-text-container"><span class="button-link-text"><span class="react-xocs-alternative-link"><span class="given-name">Samuel A.</span> <span class="text surname">Funt</span> </span><span class="author-ref" id="baff2"><sup>†</sup></span> <span class="author-ref" id="baff3"><sup>‡</sup></span></span></span></button>, <button class="button-link button-link-secondary button-link-underline" data-sd-ui-side-panel-opener="true" data-xocs-content-type="author" data-xocs-content-id="au8" type="button"><span class="button-link-text-container"><span class="button-link-text"><span class="react-xocs-alternative-link"><span class="given-name">Joel</span> <span class="text surname">Sheinfeld</span> </span><span class="author-ref" id="baff4"><sup>§</sup></span></span></span></button>, <button class="button-link button-link-secondary button-link-underline" data-sd-ui-side-panel-opener="true" data-xocs-content-type="author" data-xocs-content-id="au9" type="button"><span class="button-link-text-container"><span class="button-link-text"><span class="react-xocs-alternative-link"><span class="given-name">David B.</span> <span class="text surname">Solit</span> </span><span class="author-ref" id="baff2"><sup>†</sup></span> <span class="author-ref" id="baff3"><sup>‡</sup></span> <span class="author-ref" id="baff5"><sup>¶</sup></span></span></span></button>, <button class="button-link button-link-secondary button-link-underline" data-sd-ui-side-panel-opener="true" data-xocs-content-type="author" data-xocs-content-id="au10" type="button"><span class="button-link-text-container"><span class="button-link-text"><span class="react-xocs-alternative-link"><span class="given-name">Melissa S.</span> <span class="text surname">Pessin</span> </span><span class="author-ref" id="baff1"><sup>∗</sup></span></span></span></button>, <button class="button-link button-link-secondary button-link-underline" data-sd-ui-side-panel-opener="true" data-xocs-content-type="author" data-xocs-content-id="au11" type="button"><span class="button-link-text-container"><span class="button-link-text"><span class="react-xocs-alternative-link"><span class="given-name">Ellinor I.</span> <span class="text surname">Peerschke</span> </span><span class="author-ref" id="baff1"><sup>∗</sup></span><svg focusable="false" viewBox="0 0 106 128" height="20" title="Correspondence author icon" class="icon icon-person react-xocs-author-icon u-fill-grey8"><path d="M11.07 120l.84-9.29C13.88 91.92 35.25 87.78 53 87.78c17.74 0 39.11 4.13 41.08 22.84l.84 9.38h10.04l-.93-10.34C101.88 89.23 83.89 78 53 78S4.11 89.22 1.95 109.73L1.04 120h10.03M53 17.71c-9.72 0-18.24 8.69-18.24 18.59 0 13.67 7.84 23.98 18.24 23.98S71.24 49.97 71.24 36.3c0-9.9-8.52-18.59-18.24-18.59zM53 70c-15.96 0-28-14.48-28-33.67C25 20.97 37.82 8 53 8s28 12.97 28 28.33C81 55.52 68.96 70 53 70"></path></svg><svg focusable="false" viewBox="0 0 102 128" height="20" title="Author email or social media contact details icon" class="icon icon-envelope react-xocs-author-icon u-fill-grey8"><path d="M55.8 57.2c-1.78 1.31-5.14 1.31-6.9 0L17.58 34h69.54L55.8 57.19zM0 32.42l42.94 32.62c2.64 1.95 6.02 2.93 9.4 2.93s6.78-.98 9.42-2.93L102 34.34V24H0zM92 88.9L73.94 66.16l-8.04 5.95L83.28 94H18.74l18.38-23.12-8.04-5.96L10 88.94V51.36L0 42.9V104h102V44.82l-10 8.46V88.9"></path></svg></span></span></button></div></div></div><button class="button-link u-margin-s-ver button-link-primary button-link-icon-right" id="show-more-btn" type="button" data-aa-button="icon-expand"><span class="button-link-text-container"><span class="button-link-text">Show more</span></span><svg focusable="false" viewBox="0 0 92 128" height="20" class="icon icon-navigate-down"><path d="M1 51l7-7 38 38 38-38 7 7-45 45z"></path></svg></button><div class="banner-options u-padding-xs-bottom"><div class="toc-button-wrap u-display-inline-block u-display-none-from-lg u-margin-s-right"><button class="button-link button-link-secondary button-link-icon-left button-link-has-colored-icon" type="button"><svg focusable="false" viewBox="0 0 128 128" height="20" class="icon icon-list"><path d="M23 26a9 9 0 0 0-9 9 9 9 0 0 0 9 9 9 9 0 0 0 9-9 9 9 0 0 0-9-9zm23 4v10h68V30zM23 56a9 9 0 0 0-9 9 9 9 0 0 0 9 9 9 9 0 0 0 9-9 9 9 0 0 0-9-9zm23 4v10h68V60zM23 86a9 9 0 0 0-9 9 9 9 0 0 0 9 9 9 9 0 0 0 9-9 9 9 0 0 0-9-9zm23 4v10h68V90z"></path></svg><span class="button-link-text-container"><span class="button-link-text">Outline</span></span></button></div><button class="button-link AddToMendeley button-link-secondary u-margin-s-right u-display-inline-flex-from-md button-link-icon-left button-link-has-colored-icon" type="button"><svg focusable="false" viewBox="0 0 86 128" height="20" class="icon icon-plus"><path d="M48 58V20H38v38H0v10h38v38h10V68h38V58z"></path></svg><span class="button-link-text-container"><span class="button-link-text">Add to Mendeley</span></span></button><div class="Social u-display-inline-block" id="social"><div class="popover social-popover" id="social-popover"><div id="popover-trigger-social-popover"><button class="button-link button-link-secondary u-margin-s-right button-link-icon-left button-link-has-colored-icon" aria-expanded="false" aria-haspopup="true" type="button"><svg focusable="false" viewBox="0 0 114 128" height="20" class="icon icon-share"><path d="M90 112c-6.62 0-12-5.38-12-12s5.38-12 12-12 12 5.38 12 12-5.38 12-12 12zM24 76c-6.62 0-12-5.38-12-12s5.38-12 12-12 12 5.38 12 12-5.38 12-12 12zm66-60c6.62 0 12 5.38 12 12s-5.38 12-12 12-12-5.38-12-12 5.38-12 12-12zm0 62c-6.56 0-12.44 2.9-16.48 7.48L45.1 70.2c.58-1.98.9-4.04.9-6.2s-.32-4.22-.9-6.2l28.42-15.28C77.56 47.1 83.44 50 90 50c12.14 0 22-9.86 22-22S102.14 6 90 6s-22 9.86-22 22c0 1.98.28 3.9.78 5.72L40.14 49.1C36.12 44.76 30.38 42 24 42 11.86 42 2 51.86 2 64s9.86 22 22 22c6.38 0 12.12-2.76 16.14-7.12l28.64 15.38c-.5 1.84-.78 3.76-.78 5.74 0 12.14 9.86 22 22 22s22-9.86 22-22-9.86-22-22-22z"></path></svg><span class="button-link-text-container"><span class="button-link-text">Share</span></span></button></div></div></div><div class="ExportCitation u-display-inline-block" id="export-citation"><div class="popover export-citation-popover" id="export-citation-popover"><div id="popover-trigger-export-citation-popover"><button class="button-link button-link-secondary button-link-icon-left button-link-has-colored-icon" aria-expanded="false" aria-haspopup="true" type="button"><svg focusable="false" viewBox="0 0 104 128" height="20" class="icon icon-cited-by-66"><path d="M2 58.78V106h44V64H12v-5.22C12 40.28 29.08 32 46 32V22C20.1 22 2 37.12 2 58.78zM102 32V22c-25.9 0-44 15.12-44 36.78V106h44V64H68v-5.22C68 40.28 85.08 32 102 32z"></path></svg><span class="button-link-text-container"><span class="button-link-text">Cite</span></span></button></div></div></div></div></div><div class="ArticleIdentifierLinks u-margin-xs-bottom text-xs" id="article-identifier-links"><a class="anchor doi anchor-primary" href="https://doi.org/10.1016/j.jmoldx.2022.04.007" target="_blank" rel="noreferrer noopener" aria-label="Persistent link using digital object identifier" title="Persistent link using digital object identifier"><span class="anchor-text-container"><span class="anchor-text">https://doi.org/10.1016/j.jmoldx.2022.04.007</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor rights-and-content anchor-primary" href="https://s100.copyright.com/AppDispatchServlet?publisherName=ELS&amp;contentID=S1525157822001337&amp;orderBeanReset=true" target="_blank" rel="noreferrer noopener"><span class="anchor-text-container"><span class="anchor-text">Get rights and content</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div><div class="LicenseInfo text-xs u-margin-xs-bottom"><div class="License"><span>Under an Elsevier </span><a class="anchor anchor-primary" href="http://www.elsevier.com/open-access/userlicense/1.0/" target="_blank" rel="noreferrer noopener"><span class="anchor-text-container"><span class="anchor-text">user license</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div><div class="OpenAccessLabel"><span class="access-indicator"></span>open archive</div></div><section class="ReferencedArticles"></section><section class="ReferencedArticles"></section><div class="PageDivider border-black-mobile"></div><div class="Abstracts u-font-serif" id="abstracts"><div class="abstract author" id="abs0010" lang="en"><div id="abssec0010"><div class="u-margin-s-bottom" id="abspara0010"><span><span>Detection of serum embryonic <a href="https://www.sciencedirect.com/topics/medicine-and-dentistry/microrna" title="Learn more about miRNAs from ScienceDirect&#39;s AI-generated Topic Pages" class="topic-link">miRNAs</a><span> miR-371a-3p and miR-372-3p has been proposed to aid in diagnosis, prognosis, and management of patients with testicular <a href="https://www.sciencedirect.com/topics/pharmacology-toxicology-and-pharmaceutical-science/germ-cell-tumor" title="Learn more about germ cell tumors from ScienceDirect&#39;s AI-generated Topic Pages" class="topic-link">germ cell tumors</a> (GCTs). This study describes the analytical validation and performance of a laboratory-developed test to detect these </span></span><a href="https://www.sciencedirect.com/topics/agricultural-and-biological-sciences/microrna" title="Learn more about miRNA from ScienceDirect&#39;s AI-generated Topic Pages" class="topic-link">miRNA</a><span><span> targets by <a href="https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/stem-loop" title="Learn more about stem loop from ScienceDirect&#39;s AI-generated Topic Pages" class="topic-link">stem loop</a> real-time quantitative RT-PCR (RT-qPCR) in serum from patients with GCTs. The assay was standardized using an exogenous spike-in control of nonhuman </span><a href="https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/microrna" title="Learn more about miRNA from ScienceDirect&#39;s AI-generated Topic Pages" class="topic-link">miRNA</a> from </span></span><span><span><a href="https://www.sciencedirect.com/topics/medicine-and-dentistry/caenorhabditis-elegans" title="Learn more about Caenorhabditis elegans from ScienceDirect&#39;s AI-generated Topic Pages" class="topic-link">Caenorhabditis elegans</a></span></span><span> (cel-miR-39-3p) to assess extraction efficiency, and an endogenous housekeeping miRNA, miR-30b-5p, to control for miRNA normalization. miRNA results were expressed as relative expression level, using the comparative <a href="https://www.sciencedirect.com/topics/immunology-and-microbiology/cycle-threshold" title="Learn more about threshold cycle from ScienceDirect&#39;s AI-generated Topic Pages" class="topic-link">threshold cycle</a> method (2</span><sup>ΔΔC</sup><sup>T</sup>). Analytical sensitivity of miR-371a-3p and miR-372-3p was 12.5 and 1.25 copies/μL, respectively. Clinical accuracy was evaluated using GCT patients with (<em>n</em>&nbsp;=&nbsp;34) and without (<em>n</em><span>&nbsp;=&nbsp;17) active disease. Positive/negative cutoffs and indeterminate findings were established on the basis of results from <a href="https://www.sciencedirect.com/topics/immunology-and-microbiology/normal-human" title="Learn more about healthy volunteers from ScienceDirect&#39;s AI-generated Topic Pages" class="topic-link">healthy volunteers</a> (</span><em>n</em>&nbsp;=&nbsp;25) and assay precision. miR-371a-3p and miR-372-3p exhibited a sensitivity of 81.8% and 87.5%, respectively, and a specificity of 100% and 94%, respectively, and an area under the receiver operating characteristic curve of 0.93 and 0.95, respectively. Taken together, RT-qPCR testing for serum miR-371a-3p and miR-372-3p represents a robust, sensitive, and specific clinical assay to aid in the clinical management of patients with GCT.</div></div></div></div><div id="reading-assistant-main-body-section"></div><ul id="issue-navigation" class="issue-navigation u-margin-s-bottom u-bg-grey1"><li class="previous move-left u-padding-s-ver u-padding-s-left"><a class="button-alternative button-alternative-tertiary u-display-flex button-alternative-icon-left" href="https://www.sciencedirect.com/science/article/pii/S1525157822001453"><svg focusable="false" viewBox="0 0 54 128" height="20" class="icon icon-navigate-left"><path d="M1 61l45-45 7 7-38 38 38 38-7 7z"></path></svg><span class="button-alternative-text-container"><span class="button-alternative-text">Previous <span class="extra-detail-1">article</span><span class="extra-detail-2"> in issue</span></span></span></a></li><li class="next move-right u-padding-s-ver u-padding-s-right"><a class="button-alternative button-alternative-tertiary u-display-flex button-alternative-icon-right" href="https://www.sciencedirect.com/science/article/pii/S1525157822001581"><span class="button-alternative-text-container"><span class="button-alternative-text">Next <span class="extra-detail-1">article</span><span class="extra-detail-2"> in issue</span></span></span><svg focusable="false" viewBox="0 0 54 128" height="20" class="icon icon-navigate-right"><path d="M1 99l38-38L1 23l7-7 45 45-45 45z"></path></svg></a></li></ul><div class="Body u-font-serif" id="body"><div><div class="u-margin-s-bottom" id="p0010"><span>miRNAs are small <a href="https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/non-coding-rna" title="Learn more about noncoding RNAs from ScienceDirect&#39;s AI-generated Topic Pages" class="topic-link">noncoding RNAs</a> that function in the regulation of gene expression</span><a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#bib1" name="bbib1" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib1"><span class="anchor-text-container"><span class="anchor-text"><sup>1</sup></span></span></a> and may be dysregulated in cancer.<a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#bib2" name="bbib2" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib2"><span class="anchor-text-container"><span class="anchor-text"><sup>2</sup></span></span></a><sup>,</sup><a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#bib3" name="bbib3" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib3"><span class="anchor-text-container"><span class="anchor-text"><sup>3</sup></span></span></a><span><span> Indeed, miRNAs have been shown to participate in carcinogenesis by modulating <a href="https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/cell-proliferation" title="Learn more about cell proliferation from ScienceDirect&#39;s AI-generated Topic Pages" class="topic-link">cell proliferation</a>, resisting cell death, activating invasion and metastasis, and inducing </span><a href="https://www.sciencedirect.com/topics/immunology-and-microbiology/angiogenesis" title="Learn more about angiogenesis from ScienceDirect&#39;s AI-generated Topic Pages" class="topic-link">angiogenesis</a>.</span><a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#bib2" name="bbib2" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib2"><span class="anchor-text-container"><span class="anchor-text"><sup>2</sup></span></span></a><span> Evidence is accumulating to support the concept that miRNA signatures may serve as diagnostic and <a href="https://www.sciencedirect.com/topics/immunology-and-microbiology/prognostic-biomarker" title="Learn more about prognostic biomarkers from ScienceDirect&#39;s AI-generated Topic Pages" class="topic-link">prognostic biomarkers</a> and may also predict therapy response.</span><a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#bib4" name="bbib4" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib4"><span class="anchor-text-container"><span class="anchor-text"><sup>4</sup></span></span></a></div><div class="u-margin-s-bottom" id="p0015">Germ cell tumors (GCTs) are a heterogeneous group of neoplasms originating at various points in germ cell development.<a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#bib5" name="bbib5" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib5"><span class="anchor-text-container"><span class="anchor-text"><sup>5</sup></span></span></a><span> Depending on disease stage and <a href="https://www.sciencedirect.com/topics/medicine-and-dentistry/tumor-histology" title="Learn more about tumor histology from ScienceDirect&#39;s AI-generated Topic Pages" class="topic-link">tumor histology</a>, GCTs are treated with surgery, followed by radiation and/or chemotherapy.</span><a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#bib6" name="bbib6" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib6"><span class="anchor-text-container"><span class="anchor-text"><sup>6</sup></span></span></a><sup>,</sup><a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#bib7" name="bbib7" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib7"><span class="anchor-text-container"><span class="anchor-text"><sup>7</sup></span></span></a> Significant long-term effects of systemic therapy have been observed and, therefore, early and accurate diagnosis is important to achieve optimal clinical outcomes.<a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#bib8" name="bbib8" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib8"><span class="anchor-text-container"><span class="anchor-text"><sup>8</sup></span></span></a></div><div class="u-margin-s-bottom" id="p0020"><span><span>Guidelines for diagnosis, <a href="https://www.sciencedirect.com/topics/medicine-and-dentistry/prognostic-assessment" title="Learn more about prognostic assessment from ScienceDirect&#39;s AI-generated Topic Pages" class="topic-link">prognostic assessment</a>, and management of patients with GCTs include evaluation of serum biomarkers, α-fetoprotein (AFP), </span><a href="https://www.sciencedirect.com/topics/pharmacology-toxicology-and-pharmaceutical-science/chorionic-gonadotropin" title="Learn more about human chorionic gonadotropin from ScienceDirect&#39;s AI-generated Topic Pages" class="topic-link">human chorionic gonadotropin</a> (HCG), and lactate dehydrogenase (LDH).</span><a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#bib9" name="bbib9" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib9"><span class="anchor-text-container"><span class="anchor-text"><sup>9</sup></span></span></a> These serum markers, however, lack sensitivity and specificity for GCTs<a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#bib10" name="bbib10" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib10"><span class="anchor-text-container"><span class="anchor-text"><sup>10</sup></span></span></a><sup>,</sup><a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#bib11" name="bbib11" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib11"><span class="anchor-text-container"><span class="anchor-text"><sup>11</sup></span></span></a><span> and are less informative in patients with <a href="https://www.sciencedirect.com/topics/medicine-and-dentistry/seminoma" title="Learn more about seminoma from ScienceDirect&#39;s AI-generated Topic Pages" class="topic-link">seminoma</a><span><span>, <a href="https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/teratoma" title="Learn more about teratoma from ScienceDirect&#39;s AI-generated Topic Pages" class="topic-link">teratoma</a>, and </span><a href="https://www.sciencedirect.com/topics/agricultural-and-biological-sciences/embryonal-carcinoma" title="Learn more about embryonal carcinoma from ScienceDirect&#39;s AI-generated Topic Pages" class="topic-link">embryonal carcinoma</a>, which represent a large proportion of GCT patients.</span></span><a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#bib6" name="bbib6" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib6"><span class="anchor-text-container"><span class="anchor-text"><sup>6</sup></span></span></a> Recently, the presence of serum embryonic miRNA clusters, miR-371 to miR-373 and miR-302/367, in patients with GCT has been described.<a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#bib12" name="bbib12" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib12"><span class="anchor-text-container"><span class="anchor-text">12</span></span></a>, <a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#bib13" name="bbib13" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib13"><span class="anchor-text-container"><span class="anchor-text">13</span></span></a>, <a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#bib14" name="bbib14" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib14"><span class="anchor-text-container"><span class="anchor-text">14</span></span></a>, <a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#bib15" name="bbib15" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib15"><span class="anchor-text-container"><span class="anchor-text">15</span></span></a>, <a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#bib16" name="bbib16" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib16"><span class="anchor-text-container"><span class="anchor-text">16</span></span></a><span> Of the two miRNA clusters, miR-371a-3p appears the most sensitive and specific, with an area under the receiver operating characteristic curve of &gt;0.90, a sensitivity of 89% to 96%, and a specificity of &gt;90% for both <a href="https://www.sciencedirect.com/topics/pharmacology-toxicology-and-pharmaceutical-science/seminoma" title="Learn more about seminoma from ScienceDirect&#39;s AI-generated Topic Pages" class="topic-link">seminoma</a> and nonseminoma.</span><a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#bib17" name="bbib17" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib17"><span class="anchor-text-container"><span class="anchor-text"><sup>17</sup></span></span></a><span> Results of a multicenter study indicate that detection of serum miR-371a-3p outperforms the sensitivity and specificity of standard-of-care serum <a href="https://www.sciencedirect.com/topics/medicine-and-dentistry/biochemical-marker" title="Learn more about biochemical markers from ScienceDirect&#39;s AI-generated Topic Pages" class="topic-link">biochemical markers</a> in this setting.</span><a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#bib14" name="bbib14" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib14"><span class="anchor-text-container"><span class="anchor-text"><sup>14</sup></span></span></a> However, the sensitivity of this marker for detection of relapse is decreased to 82.6%.<a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#bib14" name="bbib14" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib14"><span class="anchor-text-container"><span class="anchor-text"><sup>14</sup></span></span></a> Furthermore, miR-371a-3p alone was not sufficient to identify some malignant GCTs.<a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#bib18" name="bbib18" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib18"><span class="anchor-text-container"><span class="anchor-text"><sup>18</sup></span></span></a> Moreover, the particularly rapid reduction in miR-371a-3p levels following tumor removal in stage 1 disease<a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#bib19" name="bbib19" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib19"><span class="anchor-text-container"><span class="anchor-text"><sup>19</sup></span></span></a><sup>,</sup><a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#bib20" name="bbib20" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib20"><span class="anchor-text-container"><span class="anchor-text"><sup>20</sup></span></span></a> will lead to negative results in samples taken &gt;1 day after surgery, unlike the other miRNAs, such as miR-372-3p, which has a longer half-life.<a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#bib18" name="bbib18" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib18"><span class="anchor-text-container"><span class="anchor-text"><sup>18</sup></span></span></a><span><span><span> Furthermore, of the miR-371 to miR-373 cluster (miR-371, miR-372, and miR-373), <a href="https://www.sciencedirect.com/topics/immunology-and-microbiology/blood-level" title="Learn more about serum level from ScienceDirect&#39;s AI-generated Topic Pages" class="topic-link">serum level</a> of miR-372-3p was shown to be significantly elevated at the time of diagnosis of a </span><a href="https://www.sciencedirect.com/topics/medicine-and-dentistry/pediatrics" title="Learn more about pediatric from ScienceDirect&#39;s AI-generated Topic Pages" class="topic-link">pediatric</a> </span><a href="https://www.sciencedirect.com/topics/pharmacology-toxicology-and-pharmaceutical-science/yolk-sac-tumor" title="Learn more about yolk sac tumor from ScienceDirect&#39;s AI-generated Topic Pages" class="topic-link">yolk sac tumor</a>.</span><a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#bib21" name="bbib21" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib21"><span class="anchor-text-container"><span class="anchor-text"><sup>21</sup></span></span></a><span> Functionally, both miR-372-3p and miR-373-3p contribute to cancer development in the <a href="https://www.sciencedirect.com/topics/immunology-and-microbiology/testis" title="Learn more about testis from ScienceDirect&#39;s AI-generated Topic Pages" class="topic-link">testis</a> by disabling the p53 pathway via targeting of the </span><em>LATS2</em><span> <a href="https://www.sciencedirect.com/topics/medicine-and-dentistry/tumor-suppressor-gene" title="Learn more about tumor suppressor gene from ScienceDirect&#39;s AI-generated Topic Pages" class="topic-link">tumor suppressor gene</a>, although the full pathway has yet to be elucidated.</span><a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#bib17" name="bbib17" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib17"><span class="anchor-text-container"><span class="anchor-text"><sup>17</sup></span></span></a> Potentially, more than one miRNA biomarker can be subject to considerable variation in levels for technical as well as biological reasons. In addition, combination of miRNA markers resulted in higher sensitivity and higher specificity compared with an individual marker.<a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#bib16" name="bbib16" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib16"><span class="anchor-text-container"><span class="anchor-text"><sup>16</sup></span></span></a> Therefore, the two miRNAs (miR-371a-3p and miR-372-3p) were included in this study.</div><div class="u-margin-s-bottom" id="p0025">Clinical application of miRNA testing in patients with GCTs is being considered, requiring the evaluation of analytical performance outside of research settings.<a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#bib22" name="bbib22" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib22"><span class="anchor-text-container"><span class="anchor-text"><sup>22</sup></span></span></a><span> Analytical performance of miRNA testing requires a high degree of technical expertise and robust protocols and procedures that address specimen collection, RNA extraction, <a href="https://www.sciencedirect.com/topics/pharmacology-toxicology-and-pharmaceutical-science/rna-directed-dna-polymerase" title="Learn more about reverse transcriptase from ScienceDirect&#39;s AI-generated Topic Pages" class="topic-link">reverse transcriptase</a> PCR measurement, result interpretation, and reporting.</span><a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#bib23" name="bbib23" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib23"><span class="anchor-text-container"><span class="anchor-text"><sup>23</sup></span></span></a><span> Major sources of analytical variability are associated with <a href="https://www.sciencedirect.com/topics/immunology-and-microbiology/rna-isolation" title="Learn more about RNA isolation from ScienceDirect&#39;s AI-generated Topic Pages" class="topic-link">RNA isolation</a> methods,</span><a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#bib24" name="bbib24" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib24"><span class="anchor-text-container"><span class="anchor-text"><sup>24</sup></span></span></a> preanalytical sample processing,<a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#bib25" name="bbib25" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib25"><span class="anchor-text-container"><span class="anchor-text"><sup>25</sup></span></span></a> including sample storage and hemolysis,<a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#bib25" name="bbib25" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib25"><span class="anchor-text-container"><span class="anchor-text"><sup>25</sup></span></span></a><sup>,</sup><a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#bib26" name="bbib26" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib26"><span class="anchor-text-container"><span class="anchor-text"><sup>26</sup></span></span></a> as well as quantitative PCR (qPCR) measurement systems,<a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#bib27" name="bbib27" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib27"><span class="anchor-text-container"><span class="anchor-text"><sup>27</sup></span></span></a> and data processing and normalization.<a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#bib10" name="bbib10" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib10"><span class="anchor-text-container"><span class="anchor-text"><sup>10</sup></span></span></a> The present study describes the validation of a clinical laboratory-developed test for the detection of miR-371a-3p and miR-372-3p in serum from patients with GCTs, using stem loop real-time quantitative&nbsp;RT-PCR (RT-qPCR), with a focus on analytical performance, including sensitivity, specificity, precision, accuracy, and result interpretation.</div><section id="sec1"><h2 id="sectitle0010" class="u-h4 u-margin-l-top u-margin-xs-bottom">Materials and Methods</h2><section id="sec1.1"><h3 id="sectitle0015" class="u-h4 u-margin-m-top u-margin-xs-bottom">Blood Specimen Collection and Processing</h3><div class="u-margin-s-bottom"><div id="p0030"><span><span>Blood specimens were collected from patients with a history of testicular <a href="https://www.sciencedirect.com/topics/pharmacology-toxicology-and-pharmaceutical-science/germ-cell-tumor" title="Learn more about GCTs from ScienceDirect&#39;s AI-generated Topic Pages" class="topic-link">GCTs</a>, including patients with and without active disease. Disease status was determined on the basis of clinical assessment, </span><a href="https://www.sciencedirect.com/topics/medicine-and-dentistry/biochemical-marker" title="Learn more about biochemical markers from ScienceDirect&#39;s AI-generated Topic Pages" class="topic-link">biochemical markers</a><span>, and radiological studies. The patient population consisted mainly of individuals with nonseminomatous tumors and relapsed and/or <a href="https://www.sciencedirect.com/topics/medicine-and-dentistry/metastatic-carcinoma" title="Learn more about metastatic disease from ScienceDirect&#39;s AI-generated Topic Pages" class="topic-link">metastatic disease</a>. Patient demographics are summarized in </span></span><a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#tbl1" name="btbl1" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="tbl1"><span class="anchor-text-container"><span class="anchor-text">Table&nbsp;1</span></span></a><span><span>. <a href="https://www.sciencedirect.com/topics/medicine-and-dentistry/informed-consent" title="Learn more about Informed consent from ScienceDirect&#39;s AI-generated Topic Pages" class="topic-link">Informed consent</a> for blood collection was obtained under a Memorial Sloan Kettering (FWA: 00004998) approved institutional review board protocol (institutional review board number 15-220). Deidentified blood specimens from </span><a href="https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/normal-human" title="Learn more about healthy adult from ScienceDirect&#39;s AI-generated Topic Pages" class="topic-link">healthy adult</a> volunteers (</span><em>n</em><span><span>&nbsp;=&nbsp;25) were obtained to establish reference range cutoffs. The <a href="https://www.sciencedirect.com/topics/immunology-and-microbiology/normal-human" title="Learn more about healthy adult from ScienceDirect&#39;s AI-generated Topic Pages" class="topic-link">healthy adult</a> volunteers were of the same ages and consisted of 30% male and 70% female. There was no difference in </span><a href="https://www.sciencedirect.com/topics/agricultural-and-biological-sciences/microrna" title="Learn more about miRNA from ScienceDirect&#39;s AI-generated Topic Pages" class="topic-link">miRNA</a> expression level whether the sample was from a male or a female donor (data not shown).</span></div><div class="tables frame-topbot colsep-0 rowsep-0" id="tbl1"><span class="captions text-s"><span id="cap0040"><p id="tspara0010"><span class="label">Table&nbsp;1</span>. Patient Demographics (<em>N</em>&nbsp;=&nbsp;48; 51 Samples from 48 Individual Patients)</p></span></span><div class="groups"><table><thead><tr class="rowsep-1 valign-top"><th scope="col">Patient clinical characteristics</th><th scope="col">Active disease, <em>n</em> (%)</th><th scope="col">No evidence of disease, <em>n</em> (%)</th></tr></thead><tbody><tr class="valign-top"><td>Samples, <em>n</em></td><td>34</td><td>17</td></tr><tr class="valign-top"><td>Male sex</td><td>34</td><td>17</td></tr><tr class="valign-top"><td>Age, median (range), years</td><td class="align-char">35 (18–66)</td><td class="align-char">35 (21–49)</td></tr><tr class="valign-top"><td>Primary tumor site</td><td></td><td></td></tr><tr class="valign-top"><td>&nbsp;Testis</td><td class="align-char">25 (74)</td><td class="align-char">16 (94)</td></tr><tr class="valign-top"><td>&nbsp;Mediastinum</td><td class="align-char">9 (26)</td><td class="align-char">1 (6)</td></tr><tr class="valign-top"><td>Overall histology</td><td></td><td></td></tr><tr class="valign-top"><td>&nbsp;NSGCT (including SEM with elevated AFP)</td><td class="align-char">27 (79)</td><td class="align-char">11 (65)</td></tr><tr class="valign-top"><td>&nbsp;SEM</td><td class="align-char">7 (21)</td><td class="align-char">6 (35)</td></tr><tr class="valign-top"><td>Disease status at time of sample collection</td><td></td><td></td></tr><tr class="valign-top"><td>&nbsp;Metastatic disease</td><td class="align-char">33 (97)</td><td>Not applicable</td></tr><tr class="valign-top"><td>&nbsp;Local disease only</td><td class="align-char">1 (3)</td><td>Not applicable</td></tr><tr class="valign-top"><td>Relapse status at time of sample collection</td><td></td><td></td></tr><tr class="valign-top"><td>&nbsp;<em>De novo</em> disease at time of sample collection</td><td class="align-char">4 (12)</td><td>Not applicable</td></tr><tr class="valign-top"><td>&nbsp;Relapsed disease at time of sample collection</td><td class="align-char">30 (88)</td><td>Not applicable</td></tr></tbody></table></div><div class="legend"><div class="u-margin-s-bottom" id="tspara0015">Three patients had repeated samples collected at different time points in the context of changing disease status.</div><div class="u-margin-s-bottom" id="tspara0020"><span>AFP, α-fetoprotein; NSGCT, <a href="https://www.sciencedirect.com/topics/medicine-and-dentistry/non-seminomatous-germinoma" title="Learn more about nonseminomatous germ cell tumor from ScienceDirect&#39;s AI-generated Topic Pages" class="topic-link">nonseminomatous germ cell tumor</a>; SEM, </span><a href="https://www.sciencedirect.com/topics/pharmacology-toxicology-and-pharmaceutical-science/seminoma" title="Learn more about seminoma from ScienceDirect&#39;s AI-generated Topic Pages" class="topic-link">seminoma</a>.</div></div></div></div><div class="u-margin-s-bottom" id="p0035">Whole blood was collected into gold-top blood collection tubes (SST; Becton Dickinson, Franklin Lakes, NJ) between January 2020 and April 2021. Samples were allowed to clot for at least 30 minutes at room temperature. Subsequently and within 4 hours of blood collection, samples were centrifuged at room temperature (1900&nbsp;×&nbsp;<em>g</em><span> for 15 minutes). The supernatant serum was removed, aliquoted, and stored at −80°C until use. Frozen serum aliquots were thawed on ice before <a href="https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/rna-extraction" title="Learn more about RNA extraction from ScienceDirect&#39;s AI-generated Topic Pages" class="topic-link">RNA extraction</a>.</span></div></section><section id="sec1.2"><h3 id="sectitle0020" class="u-h4 u-margin-m-top u-margin-xs-bottom">RNA Extraction</h3><div class="u-margin-s-bottom" id="p0040"><span>Total <a href="https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/rna" title="Learn more about RNA from ScienceDirect&#39;s AI-generated Topic Pages" class="topic-link">RNA</a><span>, including small RNAs, was extracted from 200 μL of thawed serum using the miRNeasy Serum/Plasma Advanced kit (Qiagen, Germantown, MD; catalog number 217204), according to manufacturer's instructions. To measure <a href="https://www.sciencedirect.com/topics/medicine-and-dentistry/microrna" title="Learn more about miRNA from ScienceDirect&#39;s AI-generated Topic Pages" class="topic-link">miRNA</a> extraction efficiency, a nonhuman spike-in </span></span><span><span><a href="https://www.sciencedirect.com/topics/medicine-and-dentistry/caenorhabditis-elegans" title="Learn more about Caenorhabditis elegans from ScienceDirect&#39;s AI-generated Topic Pages" class="topic-link">Caenorhabditis elegans</a></span></span><span> (cel)-miR-39-3p <a href="https://www.sciencedirect.com/topics/pharmacology-toxicology-and-pharmaceutical-science/oligonucleotide" title="Learn more about oligonucleotide from ScienceDirect&#39;s AI-generated Topic Pages" class="topic-link">oligonucleotide</a> control was added to each serum sample at a fixed quantity (5.6&nbsp;×&nbsp;10</span><sup>8</sup><span> copies) before <a href="https://www.sciencedirect.com/topics/immunology-and-microbiology/rna-extraction" title="Learn more about RNA extraction from ScienceDirect&#39;s AI-generated Topic Pages" class="topic-link">RNA extraction</a>, as per the manufacturer's instruction. Total RNA was eluted in 20 μL of nuclease-free water and stored at −80°C until further use.</span></div></section><section id="sec1.3"><h3 id="sectitle0025" class="u-h4 u-margin-m-top u-margin-xs-bottom">Stem-Loop RT-qPCR (TaqMan Assay)</h3><div class="u-margin-s-bottom"><div id="p0045"><span>Specific <a href="https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/microrna" title="Learn more about miRNA from ScienceDirect&#39;s AI-generated Topic Pages" class="topic-link">miRNA</a> levels were determined using stem-loop RT-qPCR, as described previously.</span><a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#bib14" name="bbib14" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib14"><span class="anchor-text-container"><span class="anchor-text"><sup>14</sup></span></span></a><sup>,</sup><a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#bib18" name="bbib18" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib18"><span class="anchor-text-container"><span class="anchor-text"><sup>18</sup></span></span></a><span> Briefly, a fixed amount of extracted total RNA (5 μL per reaction) was reverse transcribed into cDNA using the <a href="https://www.sciencedirect.com/topics/agricultural-and-biological-sciences/taqman" title="Learn more about TaqMan from ScienceDirect&#39;s AI-generated Topic Pages" class="topic-link">TaqMan</a><span><span> MicroRNA <a href="https://www.sciencedirect.com/topics/medicine-and-dentistry/reverse-transcription" title="Learn more about Reverse Transcription from ScienceDirect&#39;s AI-generated Topic Pages" class="topic-link">Reverse Transcription</a><span> Kit (Thermo Fisher Scientific, Waltham, MA; catalog number 4366596) in singleplex. <a href="https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/taqman" title="Learn more about TaqMan from ScienceDirect&#39;s AI-generated Topic Pages" class="topic-link">TaqMan</a><span> assays were used for cDNA synthesis and quantification of miRNAs, including miR-371a-3p, miR-372-3p, and cel-miR-39-3p. In addition, endogenous cellular markers, miR-30b-5p and hsa-miR-451a, were evaluated as internal monitors of RT-qPCR efficiency, and to assess the effect of hemolysis, respectively. TaqMan assays and <a href="https://www.sciencedirect.com/topics/pharmacology-toxicology-and-pharmaceutical-science/microrna" title="Learn more about miRNA from ScienceDirect&#39;s AI-generated Topic Pages" class="topic-link">miRNA</a> sequences used in reverse transcription, pre-amplification, and </span></span></span><a href="https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/real-time-polymerase-chain-reaction" title="Learn more about qPCR from ScienceDirect&#39;s AI-generated Topic Pages" class="topic-link">qPCR</a> are detailed in </span></span><a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#tbl2" name="btbl2" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="tbl2"><span class="anchor-text-container"><span class="anchor-text">Table&nbsp;2</span></span></a>.</div><div class="tables frame-topbot colsep-0 rowsep-0" id="tbl2"><span class="captions text-s"><span id="cap0045"><p id="tspara0025"><span class="label">Table&nbsp;2</span>. <span><a href="https://www.sciencedirect.com/topics/agricultural-and-biological-sciences/taqman" title="Learn more about TaqMan from ScienceDirect&#39;s AI-generated Topic Pages" class="topic-link">TaqMan</a> Assays and miRNA Sequences Used in </span><a href="https://www.sciencedirect.com/topics/agricultural-and-biological-sciences/reverse-transcription" title="Learn more about Reverse Transcription from ScienceDirect&#39;s AI-generated Topic Pages" class="topic-link">Reverse Transcription</a>, Pre-Amplification, and qPCR</p></span></span><div class="groups"><table><thead><tr class="rowsep-1 valign-top"><th scope="col">Assay name</th><th scope="col">Accession no.</th><th scope="col">miRNA sequence</th><th scope="col">Assay ID, Thermo Fisher Scientific</th><th scope="col">Function</th></tr></thead><tbody><tr class="valign-top"><td>miR-371a-3p</td><td>MIMAT0000723</td><td><span class="monospace">5′-AAGUGCCGCCAUCUUUUGAGUGU-3′</span></td><td class="align-char">002124</td><td>GCT miRNA marker</td></tr><tr class="valign-top"><td>miR-372-3p</td><td>MIMAT0000724</td><td><span class="monospace">5′-AAAGUGCUGCGACAUUUGAGCGU-3′</span></td><td class="align-char">000560</td><td>GCT miRNA marker</td></tr><tr class="valign-top"><td>miR-30b-5p</td><td>MIMAT0000420</td><td><span class="monospace">5′-UGUAAACAUCCUACACUCAGCU-3′</span></td><td class="align-char">000602</td><td>Endogenous housekeeping control miRNA</td></tr><tr class="valign-top"><td>cel-miR-39-3p</td><td>MIMAT0000010</td><td><span class="monospace">5′-UCACCGGGUGUAAAUCAGCUUG-3′</span></td><td class="align-char">000200</td><td>Exogenous miRNA extraction efficiency control miRNA</td></tr><tr class="valign-top"><td>miR-451a</td><td>MIMAT0001631</td><td><span class="monospace">5′-AAACCGUUACCAUUACUGAGUU-3′</span></td><td class="align-char">001141</td><td>Hemolysis assessment</td></tr></tbody></table></div><div class="legend"><div class="u-margin-s-bottom" id="tspara0030">Accession numbers from <a class="anchor anchor-primary" href="https://www.mirbase.org/" target="_blank"><span class="anchor-text-container"><span class="anchor-text"><em>https://www.mirbase.org</em></span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a> (last accessed May 9, 2022).</div><div class="u-margin-s-bottom" id="tspara0035">GCT, germ cell tumor; ID, identifier; qPCR, quantitative PCR.</div></div></div></div><div class="u-margin-s-bottom" id="p0050"><span>The reverse transcription reaction mixture was incubated in a thermocycler at 16°C for 30 minutes and 42°C for 30 minutes, followed by 85°C for 5 minutes. The resulting cDNA was pre-amplified using 2 × TaqMan Preamp master mix and selected miRNA primers. Pre-amplification occurred in a thermocycler with 1 cycle each of 95°C for 10 minutes, 55°C for 2 minutes, and 72°C for 2 minutes, followed by 14 cycles each of 95°C for 15 seconds, and 60°C for 4 minutes. The pre-amplification product was diluted 1:8 in nuclease-free water and taken into <a href="https://www.sciencedirect.com/topics/agricultural-and-biological-sciences/real-time-polymerase-chain-reaction" title="Learn more about qPCR from ScienceDirect&#39;s AI-generated Topic Pages" class="topic-link">qPCR</a> using 2 × TaqMan Fast Advanced master mix and miRNA primers and probes shown in </span><a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#tbl2" name="btbl2" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="tbl2"><span class="anchor-text-container"><span class="anchor-text">Table&nbsp;2</span></span></a><span>, using a QuantStudio 6 Flex Real-Time PCR instrument (Thermo Fisher Scientific). The qPCR cycling conditions were 95°C for 10 minutes, followed by 40 cycles of 95°C for 15 seconds and 60°C for 60 seconds. Relative quantification of miR-371a-3p and miR-372-3p was calculated using the comparative <a href="https://www.sciencedirect.com/topics/immunology-and-microbiology/cycle-threshold" title="Learn more about threshold cycle from ScienceDirect&#39;s AI-generated Topic Pages" class="topic-link">threshold cycle</a> (C</span><sub>T</sub>) method (2<sup>ΔΔCT</sup>)<a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#bib18" name="bbib18" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib18"><span class="anchor-text-container"><span class="anchor-text"><sup>18</sup></span></span></a><sup>,</sup><a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#bib28" name="bbib28" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib28"><span class="anchor-text-container"><span class="anchor-text"><sup>28</sup></span></span></a> as follows:<span class="display"><span id="ufd1" class="formula"><span class="label">(1)</span><span class="math"><span class="MathJax_Preview" style=""></span><span class="MathJax_SVG" id="MathJax-Element-2-Frame" tabindex="0" style="font-size: 90%; display: inline-block; position: relative;" data-mathml="&lt;math xmlns=&quot;http://www.w3.org/1998/Math/MathML&quot;&gt;&lt;mrow is=&quot;true&quot;&gt;&lt;mtext is=&quot;true&quot;&gt;Relative&amp;#xA0;Expression&amp;#xA0;Level&lt;/mtext&gt;&lt;mo linebreak=&quot;badbreak&quot; is=&quot;true&quot;&gt;=&lt;/mo&gt;&lt;msup is=&quot;true&quot;&gt;&lt;mn is=&quot;true&quot;&gt;2&lt;/mn&gt;&lt;mrow is=&quot;true&quot;&gt;&lt;mi is=&quot;true&quot;&gt;&amp;#x394;&lt;/mi&gt;&lt;mi is=&quot;true&quot;&gt;&amp;#x394;&lt;/mi&gt;&lt;mtext is=&quot;true&quot;&gt;Ct&lt;/mtext&gt;&lt;/mrow&gt;&lt;/msup&gt;&lt;/mrow&gt;&lt;/math&gt;" role="presentation"><svg xmlns:xlink="http://www.w3.org/1999/xlink" width="34.906ex" height="2.894ex" viewBox="0 -995.6 15028.9 1246" role="img" focusable="false" style="vertical-align: -0.582ex;" aria-hidden="true"><g stroke="currentColor" fill="currentColor" stroke-width="0" transform="matrix(1 0 0 -1 0 0)"><g is="true"><g is="true"><use href="#MJMAIN-52"></use><use href="#MJMAIN-65" x="736" y="0"></use><use href="#MJMAIN-6C" x="1181" y="0"></use><use href="#MJMAIN-61" x="1459" y="0"></use><use href="#MJMAIN-74" x="1960" y="0"></use><use href="#MJMAIN-69" x="2349" y="0"></use><use href="#MJMAIN-76" x="2628" y="0"></use><use href="#MJMAIN-65" x="3156" y="0"></use><use href="#MJMAIN-45" x="3851" y="0"></use><use href="#MJMAIN-78" x="4532" y="0"></use><use href="#MJMAIN-70" x="5061" y="0"></use><use href="#MJMAIN-72" x="5617" y="0"></use><use href="#MJMAIN-65" x="6010" y="0"></use><use href="#MJMAIN-73" x="6454" y="0"></use><use href="#MJMAIN-73" x="6849" y="0"></use><use href="#MJMAIN-69" x="7243" y="0"></use><use href="#MJMAIN-6F" x="7522" y="0"></use><use href="#MJMAIN-6E" x="8022" y="0"></use><use href="#MJMAIN-4C" x="8829" y="0"></use><use href="#MJMAIN-65" x="9454" y="0"></use><use href="#MJMAIN-76" x="9899" y="0"></use><use href="#MJMAIN-65" x="10427" y="0"></use><use href="#MJMAIN-6C" x="10872" y="0"></use></g><g is="true" transform="translate(11428,0)"><use href="#MJMAIN-3D"></use></g><g is="true" transform="translate(12484,0)"><g is="true"><use href="#MJMAIN-32"></use></g><g is="true" transform="translate(500,393)"><g is="true"><use transform="scale(0.707)" href="#MJMAINI-394"></use></g><g is="true" transform="translate(578,0)"><use transform="scale(0.707)" href="#MJMAINI-394"></use></g><g is="true" transform="translate(1157,0)"><use transform="scale(0.707)" href="#MJMAIN-43"></use><use transform="scale(0.707)" href="#MJMAIN-74" x="722" y="0"></use></g></g></g></g></g></svg><span class="MJX_Assistive_MathML" role="presentation"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow is="true"><mtext is="true">Relative&nbsp;Expression&nbsp;Level</mtext><mo linebreak="badbreak" is="true">=</mo><msup is="true"><mn is="true">2</mn><mrow is="true"><mi is="true">Δ</mi><mi is="true">Δ</mi><mtext is="true">Ct</mtext></mrow></msup></mrow></math></span></span><script type="math/mml" id="MathJax-Element-2"><math><mrow is="true"><mtext is="true">Relative Expression Level</mtext><mo linebreak="badbreak" is="true">=</mo><msup is="true"><mn is="true">2</mn><mrow is="true"><mi is="true">Δ</mi><mi is="true">Δ</mi><mtext is="true">Ct</mtext></mrow></msup></mrow></math></script></span></span></span>Where: ΔΔC<sub>T</sub>&nbsp;=&nbsp;ΔC<sub>T(</sub><sub>healthy donor</sub><sub>)</sub> − ΔC<sub>T(</sub><sub>sample</sub><sub>)</sub></div><div class="u-margin-s-bottom" id="p0055">ΔC<sub>T(</sub><sub>sample</sub><sub>)</sub>&nbsp;=&nbsp;C<sub>T(</sub><sub>target miRNA</sub><sub>)</sub> − C<sub>T(</sub><sub>Corrected miR-30b-5p</sub><sub>)</sub></div><div class="u-margin-s-bottom" id="p0060">C<sub>T(</sub><sub>Corrected miR-30b-5p</sub><sub>)</sub>&nbsp;=&nbsp;C<sub>T(</sub><sub>miR-30b-5p</sub><sub>)</sub> – (C<sub>T(</sub><sub>cel-miR-39-3p sample</sub><sub>)</sub> − C<sub>T(</sub><sub>cel-miR-39-3p lowest in same run</sub><sub>)</sub>)</div><div class="u-margin-s-bottom" id="p0065">This formula corrects for RNA extraction and amplification variations by subtracting the lowest within-run C<sub>T</sub> value of the exogenous control, cel-miR-39-3p, from the other cel-miR-39-3p C<sub>T</sub> levels of the samples in the same&nbsp;run. The formula additionally normalizes the sample to its endogenously expressed housekeeping miRNA, miR-30b-5p, as well as to a nontumor control group (ΔC<sub>T(</sub><sub>healthy donor</sub><sub>)</sub>). The ΔC<sub>T</sub> of the nontumor control group was calculated using the average C<sub>T</sub> value of each target obtained from the 25 healthy donors.</div></section><section id="sec1.4"><h3 id="sectitle0030" class="u-h4 u-margin-m-top u-margin-xs-bottom">Statistical Analysis</h3><div class="u-margin-s-bottom" id="p0070">Assay performance was evaluated using GraphPad Prism 9.2.0 (GraphPad Inc., San Diego, CA) for receiver operating characteristic analysis and by determining clinical sensitivity and specificity, as well as positive and negative predictive values.<a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#bib29" name="bbib29" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib29"><span class="anchor-text-container"><span class="anchor-text"><sup>29</sup></span></span></a> Calculations were based on miR-371a-3p and miR-372-3p results (relative expression levels and cutoffs) compared with the clinical outcome (active disease versus nonactive disease). Clinical outcome was defined by the clinical oncologist based on the results of serum tumor markers and radiographic evidence.</div></section></section><section id="sec2"><h2 id="sectitle0035" class="u-h4 u-margin-l-top u-margin-xs-bottom">Results</h2><section id="sec2.1"><h3 id="sectitle0040" class="u-h4 u-margin-m-top u-margin-xs-bottom">Assay Standardization</h3><div class="u-margin-s-bottom" id="p0075"><span><span>Several <a href="https://www.sciencedirect.com/topics/agricultural-and-biological-sciences/quality-control" title="Learn more about quality control from ScienceDirect&#39;s AI-generated Topic Pages" class="topic-link">quality control</a> steps were incorporated into the assay to ensure robust serum RT-qPCR detection and relative quantification. The exogenous nonhuman spike-in control, cel-miR-39-3p, was used to monitor </span><a href="https://www.sciencedirect.com/topics/agricultural-and-biological-sciences/microrna" title="Learn more about miRNA from ScienceDirect&#39;s AI-generated Topic Pages" class="topic-link">miRNA</a><span> extraction efficiency. An endogenous housekeeping <a href="https://www.sciencedirect.com/topics/medicine-and-dentistry/microrna" title="Learn more about miRNA from ScienceDirect&#39;s AI-generated Topic Pages" class="topic-link">miRNA</a>, miR-30b-5p, reported to be highly stable in serum,</span></span><a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#bib16" name="bbib16" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib16"><span class="anchor-text-container"><span class="anchor-text"><sup>16</sup></span></span></a><sup>,</sup><a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#bib18" name="bbib18" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib18"><span class="anchor-text-container"><span class="anchor-text"><sup>18</sup></span></span></a><sup>,</sup><a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#bib30" name="bbib30" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib30"><span class="anchor-text-container"><span class="anchor-text"><sup>30</sup></span></span></a><span> was used to control for <a href="https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/microrna" title="Learn more about miRNA from ScienceDirect&#39;s AI-generated Topic Pages" class="topic-link">miRNA</a> load and normalization purposes. For further quality control and to monitor contamination, each RT-qPCR run included a negative control (healthy donor serum) and a nontemplate control.</span></div><div class="u-margin-s-bottom"><div id="p0080"><a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#fig1" name="bfig1" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="fig1"><span class="anchor-text-container"><span class="anchor-text">Figure&nbsp;1</span></span></a><span> shows results for amplification of cel-miR-39-3p and the housekeeping miR-30b-5p in 76 serum samples (25 from healthy donors, and 51 from <a href="https://www.sciencedirect.com/topics/pharmacology-toxicology-and-pharmaceutical-science/germ-cell-tumor" title="Learn more about GCT from ScienceDirect&#39;s AI-generated Topic Pages" class="topic-link">GCT</a> patients). C</span><sub>T</sub><span> values, 15.5&nbsp;±&nbsp;3.6 (mean&nbsp;±&nbsp;3 SDs), for the exogenous <a href="https://www.sciencedirect.com/topics/pharmacology-toxicology-and-pharmaceutical-science/microrna" title="Learn more about miRNA from ScienceDirect&#39;s AI-generated Topic Pages" class="topic-link">miRNA</a> cel-miR-39-3p were consistent across samples, with a CV of 7.5%, indicating acceptable miRNA extraction efficiency and reproducibility.</span><a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#bib31" name="bbib31" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib31"><span class="anchor-text-container"><span class="anchor-text"><sup>31</sup></span></span></a> Similarly, C<sub>T</sub> values, 19.4&nbsp;±&nbsp;4.8 (mean&nbsp;±&nbsp;3 SDs), were highly reproducible for the endogenous miRNA control, miR-30b-5p, used for normalization, with a CV of 8.4%. A single sample (<a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#fig1" name="bfig1" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="fig1"><span class="anchor-text-container"><span class="anchor-text">Figure&nbsp;1</span></span></a>) demonstrated out-of-range C<sub>T</sub> values for both cel-miR-39-3p and miR-30b-5p, indicating that miRNA extraction efficiency was poor. When miRNA re-extraction was performed, C<sub>T</sub> values of 16.8 and 21.1 for cel-miR-39-3p and miR-30b-5p, respectively, were within acceptable limits. These findings suggested that miR-30b-5p is abundant and stably expressed in serum from healthy donors and patients with GCTs. On the basis of these results, acceptable control ranges were set at C<sub>T</sub> values of 15 to 24 and 12 to 19 for cel-miR-39-3p and miR-30b-5p, respectively, representing a 3-SD range.</div><figure class="figure text-xs" id="fig1"><span><img src="./ye2022analytical_files/1-s2.0-S1525157822001337-gr1.jpg" height="113" alt="" aria-describedby="cap0010"><ol class="u-margin-s-bottom"><li><a class="anchor download-link u-font-sans anchor-primary" href="https://ars.els-cdn.com/content/image/1-s2.0-S1525157822001337-gr1_lrg.jpg" target="_blank" download="" title="Download high-res image (132KB)"><span class="anchor-text-container"><span class="anchor-text">Download: <span class="download-link-title">Download high-res image (132KB)</span></span></span></a></li><li><a class="anchor download-link u-font-sans anchor-primary" href="./ye2022analytical_files/1-s2.0-S1525157822001337-gr1.jpg" target="_blank" download="" title="Download full-size image"><span class="anchor-text-container"><span class="anchor-text">Download: <span class="download-link-title">Download full-size image</span></span></span></a></li></ol></span><span class="captions text-s"><span id="cap0010"><p id="fspara0010"><span class="label">Figure&nbsp;1</span>. Expression (C<sub>T</sub> values) of exogenous nonhuman spike-in cel-miR-39-3p and endogenous housekeeping miRNA miR-30b-5p in study samples. Results were used to determine acceptability ranges encompassing mean&nbsp;±&nbsp;3 SDs for quality control purposes. A single sample, identified using an <strong>arrow</strong>, demonstrated out-of-range C<sub>T</sub> values for both cel-miR-39-3p and miR-30b-5p. <em>n</em>&nbsp;=&nbsp;76.</p></span></span></figure></div></section><section id="sec2.2"><h3 id="sectitle0045" class="u-h4 u-margin-m-top u-margin-xs-bottom">Assay Performance</h3><section id="sec2.2.1"><h4 id="sectitle0050" class="u-margin-m-top u-margin-xs-bottom">Linearity</h4><div class="u-margin-s-bottom"><div id="p0085"><span>The linear range for miR-371a-3p and miR-372-3p detection was determined using two synthetic <a href="https://www.sciencedirect.com/topics/pharmacology-toxicology-and-pharmaceutical-science/oligonucleotide" title="Learn more about oligonucleotides from ScienceDirect&#39;s AI-generated Topic Pages" class="topic-link">oligonucleotides</a> specific to miR-371a-3p and miR-372-3p, which were serially diluted from 1.25&nbsp;×&nbsp;10</span><sup>6</sup><span> copies/μL using nuclease-free water. In addition, two serum samples from patients with active disease were serially diluted using serum from a healthy donor in ratios of 1:10, 1:100, 1:1000, and 1:10,000. Each <a href="https://www.sciencedirect.com/topics/immunology-and-microbiology/dilution" title="Learn more about dilution from ScienceDirect&#39;s AI-generated Topic Pages" class="topic-link">dilution</a><span> was assayed using the <a href="https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/stem-loop" title="Learn more about stem loop from ScienceDirect&#39;s AI-generated Topic Pages" class="topic-link">stem loop</a> real-time RT-qPCR for the two miRNAs (miR-371a-3p and miR-372-3p) in three independent runs. The results are summarized in </span></span><a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#fig2" name="bfig2" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="fig2"><span class="anchor-text-container"><span class="anchor-text">Figure&nbsp;2</span></span></a>. The data demonstrate excellent linearity for both miRNAs, miR-371a-3p and miR-372-3p, over a wide range of copy numbers.</div><figure class="figure text-xs" id="fig2"><span><img src="./ye2022analytical_files/1-s2.0-S1525157822001337-gr2.jpg" height="371" alt="" aria-describedby="cap0015"><ol class="u-margin-s-bottom"><li><a class="anchor download-link u-font-sans anchor-primary" href="https://ars.els-cdn.com/content/image/1-s2.0-S1525157822001337-gr2_lrg.jpg" target="_blank" download="" title="Download high-res image (228KB)"><span class="anchor-text-container"><span class="anchor-text">Download: <span class="download-link-title">Download high-res image (228KB)</span></span></span></a></li><li><a class="anchor download-link u-font-sans anchor-primary" href="./ye2022analytical_files/1-s2.0-S1525157822001337-gr2.jpg" target="_blank" download="" title="Download full-size image"><span class="anchor-text-container"><span class="anchor-text">Download: <span class="download-link-title">Download full-size image</span></span></span></a></li></ol></span><span class="captions text-s"><span id="cap0015"><p id="fspara0015"><span class="label">Figure&nbsp;2</span>. Results of linearity studies for detection of miR-371a-3p and miR-372-3p using synthetic oligonucleotides (<strong>A</strong>) or serum samples from two patients (<strong>B</strong>) designated by different symbols. Analyses were performed in singlicate over three independent runs. Data represent means&nbsp;±&nbsp;SD for C<sub>T</sub> values.</p></span></span></figure></div></section><section id="sec2.2.2"><h4 id="sectitle0055" class="u-margin-m-top u-margin-xs-bottom">PCR Efficiency</h4><div class="u-margin-s-bottom" id="p0090"><span>The lowest PCR efficiency observed across three runs was 86%. PCR efficiency depends on many factors, such as: assay performance (which depends on primer and <a href="https://www.sciencedirect.com/topics/medicine-and-dentistry/dna-template" title="Learn more about template from ScienceDirect&#39;s AI-generated Topic Pages" class="topic-link">template</a> sequences and structures); sample matrix (which may contain inhibitors and other interfering substances); reagents used; and short miRNA targets. Therefore, on the basis of the resultant data (representative data shown in </span><a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#fig2" name="bfig2" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="fig2"><span class="anchor-text-container"><span class="anchor-text">Figure&nbsp;2</span></span></a><span><span>) and generally accepted principles of <a href="https://www.sciencedirect.com/topics/agricultural-and-biological-sciences/good-laboratory-practice" title="Learn more about good laboratory practice from ScienceDirect&#39;s AI-generated Topic Pages" class="topic-link">good laboratory practice</a>, a PCR efficiency range of 85% to 115% was considered acceptable for stem-loop RT-PCR for miRNA </span><a href="https://www.sciencedirect.com/topics/agricultural-and-biological-sciences/germinoma" title="Learn more about GCT from ScienceDirect&#39;s AI-generated Topic Pages" class="topic-link">GCT</a> testing.</span></div></section><section id="sec2.2.3"><h4 id="sectitle0060" class="u-margin-m-top u-margin-xs-bottom">Analytical Sensitivity</h4><div class="u-margin-s-bottom"><div id="p0095"><span>To determine analytical sensitivity, two synthetic <a href="https://www.sciencedirect.com/topics/agricultural-and-biological-sciences/oligonucleotide" title="Learn more about oligonucleotides from ScienceDirect&#39;s AI-generated Topic Pages" class="topic-link">oligonucleotides</a> specific for miR-371a-3p and miR-372-3p, with known copy numbers, were serially diluted 10-fold from 12,500 to 0.0125 copies/μL using nucleic acid-free water. Each dilution was analyzed in triplicate in three independent runs. The results are shown in </span><a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#tbl3" name="btbl3" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="tbl3"><span class="anchor-text-container"><span class="anchor-text">Table&nbsp;3</span></span></a>. The analytical sensitivity for miR-371a-3p was 12.5 copies/μL, and it was 1.25 copies/μL for miR-372-3p, with acceptable CVs of &lt;10%.</div><div class="tables frame-topbot colsep-0 rowsep-0" id="tbl3"><span class="captions text-s"><span id="cap0050"><p id="tspara0040"><span class="label">Table&nbsp;3</span>. Analytical Sensitivity</p></span></span><div class="groups"><table><thead><tr class="rowsep-1 valign-top"><th scope="col" rowspan="2">Expected concentration, copies/μL</th><th scope="col" colspan="3">miR-371a-3p</th><th scope="col" colspan="3">miR-372-3p</th></tr><tr class="rowsep-1 valign-top"><th scope="col">Mean C<sub>T</sub> value</th><th scope="col">SD</th><th scope="col">% CV</th><th scope="col">Mean C<sub>T</sub> value</th><th scope="col">SD</th><th scope="col">% CV</th></tr></thead><tbody><tr class="valign-top"><td>12,500</td><td class="align-char">21.1</td><td class="align-char">0.6</td><td class="align-char">2.8</td><td class="align-char">20.5</td><td class="align-char">0.9</td><td class="align-char">4.6</td></tr><tr class="valign-top"><td>1250</td><td class="align-char">24.4</td><td class="align-char">0.5</td><td class="align-char">2.1</td><td class="align-char">23.9</td><td class="align-char">1.2</td><td class="align-char">5.1</td></tr><tr class="valign-top"><td>125</td><td class="align-char">27.6</td><td class="align-char">0.5</td><td class="align-char">1.8</td><td class="align-char">27.3</td><td class="align-char">1.2</td><td class="align-char">4.3</td></tr><tr class="valign-top"><td>12.5</td><td class="align-char">31.3</td><td class="align-char">1.0</td><td class="align-char">3.3</td><td class="align-char">30.6</td><td class="align-char">1.8</td><td class="align-char">6.0</td></tr><tr class="valign-top"><td>1.25</td><td>Not detected</td><td>—</td><td>—</td><td class="align-char">34.1</td><td class="align-char">2.6</td><td class="align-char">7.7</td></tr><tr class="valign-top"><td>0.125</td><td>Not detected</td><td>—</td><td>—</td><td>Not detected</td><td>—</td><td>—</td></tr><tr class="valign-top"><td>0.0125</td><td>Not detected</td><td>—</td><td>—</td><td>Not detected</td><td>—</td><td>—</td></tr></tbody></table></div></div></div></section><section id="sec2.2.4"><h4 id="sectitle0065" class="u-margin-m-top u-margin-xs-bottom">Reproducibility</h4><div class="u-margin-s-bottom" id="p0100"><span>The within-run and between-run reproducibility was assessed by analyzing four clinical samples from <a href="https://www.sciencedirect.com/topics/medicine-and-dentistry/germinoma" title="Learn more about GCT from ScienceDirect&#39;s AI-generated Topic Pages" class="topic-link">GCT</a> patients with active disease. The samples were analyzed in triplicate in three independent runs with a week interval between the runs. C</span><sub>T</sub> values of these four clinical samples ranged from 21 to 31 and 20 to 30 for miR-371a-3p and miR-372-3p, respectively. The within-assay CVs using C<sub>T</sub> values ranged from 1.2% to 2.8% and from 0.3% to 1.0% for miR-371a-3p and miR-372-3p, respectively. The interassay precision ranged from 0.3% to 3.7% and from 2.2% to 3.5% for miR-371a-3p and miR-372-3p, respectively (data not shown).</div><div class="u-margin-s-bottom"><div id="p0105">When C<sub>T</sub> values were normalized on the basis of exogenous nonhuman spike-in cel-miR-39-3p and endogenous housekeeping miRNA miR-30b-5p and results expressed as ΔC<sub>T</sub> values, the reproducibility between replicates was more variable. CVs ranged from 3.9% to 32% for miR-371a-3p, and from 6.5% to 20.8% for miR-372-3p, in the intra-assay precision studies (<a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#fig3" name="bfig3" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="fig3"><span class="anchor-text-container"><span class="anchor-text">Figure&nbsp;3</span></span></a>A). The interassay precision ranged from 2.7% to 15% and from 2.2% to 31.5% for miR-371a-3p and miR-372-3p, respectively (<a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#fig3" name="bfig3" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="fig3"><span class="anchor-text-container"><span class="anchor-text">Figure&nbsp;3</span></span></a>B). The increased variability of normalized expression levels relative to C<sub>T</sub><span> values is explained by the <a href="https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/real-time-polymerase-chain-reaction" title="Learn more about qPCR from ScienceDirect&#39;s AI-generated Topic Pages" class="topic-link">qPCR</a> method principle,</span><a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#bib32" name="bbib32" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib32"><span class="anchor-text-container"><span class="anchor-text"><sup>32</sup></span></span></a> with a C<sub>T</sub> variability of up to one cycle, which can result in an error of up to 0.5 or 2 times the true value. The expected variability in relative expression for miR-371a-3p and miR-372-3p was taken into consideration during result interpretation, with the generation of an indeterminate category, which reflects variability around the target detection threshold.</div><figure class="figure text-xs" id="fig3"><span><img src="./ye2022analytical_files/1-s2.0-S1525157822001337-gr3.jpg" height="207" alt="" aria-describedby="cap0020"><ol class="u-margin-s-bottom"><li><a class="anchor download-link u-font-sans anchor-primary" href="https://ars.els-cdn.com/content/image/1-s2.0-S1525157822001337-gr3_lrg.jpg" target="_blank" download="" title="Download high-res image (141KB)"><span class="anchor-text-container"><span class="anchor-text">Download: <span class="download-link-title">Download high-res image (141KB)</span></span></span></a></li><li><a class="anchor download-link u-font-sans anchor-primary" href="./ye2022analytical_files/1-s2.0-S1525157822001337-gr3.jpg" target="_blank" download="" title="Download full-size image"><span class="anchor-text-container"><span class="anchor-text">Download: <span class="download-link-title">Download full-size image</span></span></span></a></li></ol></span><span class="captions text-s"><span id="cap0020"><p id="fspara0020"><span class="label">Figure&nbsp;3</span>. Reproducibility of serum miR-371a-3p and miR-372-3p quantitation within and between analytical runs. Patient samples, designated by different symbols, were analyzed in triplicate within a single run (<strong>A</strong>) or in singlicate over three independent runs (<strong>B</strong>). Results are expressed as ΔC<sub>T</sub> values after normalization to exogenous nonhuman spike-in cel-miR-39-3p and endogenous housekeeping miRNA miR-30b-5p. <em>n</em>&nbsp;=&nbsp;4.</p></span></span></figure></div></section><section id="sec2.2.5"><h4 id="sectitle0070" class="u-margin-m-top u-margin-xs-bottom">Result Interpretation</h4><div class="u-margin-s-bottom" id="p0110">To establish negative cutoff values for both C<sub>T</sub> and relative expression levels for miR-371a-3p and miR-372-3p using the stem loop real-time RT-qPCR assay, a total of 25 negative serum samples from healthy donors were analyzed. The cutoff C<sub>T</sub> values and relative expression levels were defined as mean&nbsp;±&nbsp;SD (90% CI). C<sub>T</sub> value cutoffs of 34 and 30 and relative expression levels of 0.5 and 1.0 were determined for miR-371a-3p and miR-372-3p, respectively (data not shown).</div><div class="u-margin-s-bottom" id="p0115">An indeterminate category was established for miR-371a-3p and miR-372-3p based on the assay reproducibility. The indeterminate relative expression range for each target was calculated by applying the mean CV% for intra-assay variability to the relative expression cutoff. Thus, three result categories were established: negative, indeterminate, and positive. Relative expression levels of ≤0.49 and&nbsp;≤0.99 were considered negative for miR-371a-3p and miR-372-3p, respectively. Relative expression levels of ≥0.75 and&nbsp;≥1.5 were considered positive, for the respective targets. Values falling between these cutoffs were considered indeterminate.</div></section><section id="sec2.2.6"><h4 id="sectitle0075" class="u-margin-m-top u-margin-xs-bottom">Accuracy: Assay Sensitivity and Specificity</h4><div class="u-margin-s-bottom" id="p0120">To determine the clinical accuracy of miR-371a-3p and miR-372-3p detection by the stem loop real-time RT-qPCR assay, a total of 51 clinical serum samples obtained during diagnosis, treatment, or follow-up from GCT patients were examined. These included 34 blood samples from patients with active disease and 17 blood samples from patients without active disease. Three patients contributed two samples each, obtained at different time points. Disease status was defined by the clinical oncologist (D.R.F.) based on results of standard serum tumor markers (HCG, AFP, and LDH) and radiographic evidence.</div><div class="u-margin-s-bottom"><div id="p0125">Using the relative expression cutoffs of ≥0.75 for miR-371a-3p and ≥1.5 for miR-372-3p, miR-371a-3p and miR-372-3p were detected in 27 and 28 samples from patients with active disease, respectively. None of the 17 samples from patients without active disease were positive for miR-371a-3p, but one sample was positive for miR-372-3p (<a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#tbl4" name="btbl4" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="tbl4"><span class="anchor-text-container"><span class="anchor-text">Table&nbsp;4</span></span></a> and <a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#appsec1" name="bappsec1" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="appsec1"><span class="anchor-text-container"><span class="anchor-text">Supplemental Table S1</span></span></a>). These results suggest a clinical sensitivity of 81.8% or 87.5%, respectively, for miR-371a-3p and miR-372-3p, and clinical specificity of 100% and 94%, respectively (<a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#tbl5" name="btbl5" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="tbl5"><span class="anchor-text-container"><span class="anchor-text">Table&nbsp;5</span></span></a>). Receiver operating characteristic analysis of miR-371a-3p and miR-372a-3p discriminated between patients with and without active disease with an area under the curve of ≥0.93 and&nbsp;≥0.95, respectively (<a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#fig4" name="bfig4" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="fig4"><span class="anchor-text-container"><span class="anchor-text">Figure&nbsp;4</span></span></a>). These results indicate that both miR-371a-3p and miR-372-3p are sensitive and specific biomarkers for testicular germ cell tumors.</div><div class="tables frame-topbot colsep-0 rowsep-0" id="tbl4"><span class="captions text-s"><span id="cap0055"><p id="tspara0045"><span class="label">Table&nbsp;4</span>. Analytical Accuracy: Truth Tables</p></span></span><div class="groups"><table><thead><tr class="rowsep-1 valign-top"><th scope="col" rowspan="2">Variable</th><th scope="col" colspan="2">miR-371a-3p</th></tr><tr class="rowsep-1 valign-top"><th scope="col">Disease present</th><th scope="col">Disease absent</th></tr></thead><tbody><tr class="valign-top"><td>Detected</td><td class="align-char">27</td><td class="align-char">0</td></tr><tr class="valign-top"><td>Not detected</td><td class="align-char">6</td><td class="align-char">17</td></tr></tbody></table><table><thead><tr class="rowsep-1 valign-top"><td scope="col" rowspan="2"><span class="screen-reader-only">Empty Cell</span></td><th scope="col" colspan="2">miR-372-3p</th></tr><tr class="rowsep-1 valign-top"><th scope="col">Disease present</th><th scope="col">Disease absent</th></tr></thead><tbody><tr class="valign-top"><td>Detected</td><td class="align-char">28</td><td class="align-char">1</td></tr><tr class="valign-top"><td>Not detected</td><td class="align-char">4</td><td class="align-char">15</td></tr></tbody></table><table><thead><tr class="rowsep-1 valign-top"><td scope="col" rowspan="2"><span class="screen-reader-only">Empty Cell</span></td><th scope="col" colspan="2">miR-371a-3p&nbsp;+&nbsp;miR-372-3p</th></tr><tr class="rowsep-1 valign-top"><th scope="col">Disease present</th><th scope="col">Disease absent</th></tr></thead><tbody><tr class="valign-top"><td>Detected</td><td class="align-char">29</td><td class="align-char">1</td></tr><tr class="valign-top rowsep-1"><td>Not detected</td><td class="align-char">4</td><td class="align-char">15</td></tr></tbody></table></div><div class="legend"><div class="u-margin-s-bottom" id="tspara0050">Truth tables of germ cell tumor targets, miR-371a-3p and miR-372-3p. A total of 51 patient samples were tested, consisting of 34 samples from patients having active disease and samples from 17 patients without evidence of disease. Samples (<em>n</em>&nbsp;=&nbsp;3) with indeterminate relative expression levels were excluded. These include one sample with indeterminate results for both miR-371-3p and miR-372-3p from a patient with active disease, and two samples with indeterminate results for miR-372-3p, one from a patient with active disease and one from a patient with no evidence of disease.</div></div></div><div class="tables frame-topbot colsep-0 rowsep-0" id="tbl5"><span class="captions text-s"><span id="cap0060"><p id="tspara0055"><span class="label">Table&nbsp;5</span>. Analytical Accuracy: Calculations of Sensitivity and Specificity</p></span></span><div class="groups"><table><thead><tr class="rowsep-1 valign-top"><th scope="col">Relative expression</th><th scope="col">miR-371a-3p</th><th scope="col">miR-372-3p</th></tr></thead><tbody><tr class="valign-top"><td>Not&nbsp;detected</td><td>&lt;0.5</td><td>&lt;1.0</td></tr><tr class="valign-top"><td>Indeterminate</td><td>≥0.5 and&nbsp;&lt;0.75</td><td>≥1.0 and&nbsp;&lt;1.5</td></tr><tr class="valign-top"><td>Detected</td><td>≥0.75</td><td>≥1.5</td></tr><tr class="valign-top"><td>Sensitivity, %</td><td>81.8</td><td>87.5</td></tr><tr class="valign-top"><td>Specificity, %</td><td>100</td><td>94</td></tr><tr class="valign-top"><td>PPV, %</td><td>100</td><td>97</td></tr><tr class="rowsep-1 valign-top"><td>NPV, %</td><td>73.9</td><td>78.9</td></tr></tbody></table><table><thead><tr class="rowsep-1 valign-top"><td scope="col"><span class="screen-reader-only">Empty Cell</span></td><th scope="col" colspan="2">miR-371a-3p&nbsp;+&nbsp;miR-372-3p</th></tr></thead><tbody><tr class="valign-top"><td>Sensitivity, %</td><td colspan="2">87.9</td></tr><tr class="valign-top"><td>Specificity, %</td><td colspan="2">94</td></tr><tr class="valign-top"><td>PPV, %</td><td colspan="2">97</td></tr><tr class="valign-top rowsep-1"><td>NPV, %</td><td colspan="2">78.9</td></tr></tbody></table></div><div class="legend"><div class="u-margin-s-bottom" id="tspara0060">Analysis of assay sensitivity, specificity, PPV, and NPV of germ cell tumor (GCT) targets, miR-371a-3p and miR-372-3p. Relative expression levels of &gt;0.75 and &gt;1.5 for miR-371a-3p and miR-372-3p, respectively, were used to determine GCT target detection. Results were calculated on the basis of detection of miR-371a-3p and miR-372-3p separately (top table) and collectively (bottom table), where detection of one target deemed the sample as detected.</div><div class="u-margin-s-bottom" id="tspara0065">NPV, negative predictive value; PPV, positive predictive value.</div></div></div><figure class="figure text-xs" id="fig4"><span><img src="./ye2022analytical_files/1-s2.0-S1525157822001337-gr4.jpg" height="532" alt="" aria-describedby="cap0025"><ol class="u-margin-s-bottom"><li><a class="anchor download-link u-font-sans anchor-primary" href="https://ars.els-cdn.com/content/image/1-s2.0-S1525157822001337-gr4_lrg.jpg" target="_blank" download="" title="Download high-res image (326KB)"><span class="anchor-text-container"><span class="anchor-text">Download: <span class="download-link-title">Download high-res image (326KB)</span></span></span></a></li><li><a class="anchor download-link u-font-sans anchor-primary" href="./ye2022analytical_files/1-s2.0-S1525157822001337-gr4.jpg" target="_blank" download="" title="Download full-size image"><span class="anchor-text-container"><span class="anchor-text">Download: <span class="download-link-title">Download full-size image</span></span></span></a></li></ol></span><span class="captions text-s"><span id="cap0025"><p id="fspara0025"><span class="label">Figure&nbsp;4</span>. Receiver operating characteristic analysis, demonstrating performance of serum miR-371a-3p and miR-372-3p targets reported as relative expression levels (<strong>A</strong> and <strong>B</strong>) or raw C<sub>T</sub> values (<strong>C</strong> and <strong>D</strong>) based on samples from 51 patients with a history of germ cell tumors, including 34 patients with active disease and 17 patients without evidence of disease. AUC, area under the curve.</p></span></span></figure></div><div class="u-margin-s-bottom" id="p0130">Three patient results were interpreted as indeterminate for miR-371a-3p and/or miR-372-3p. One sample showed indeterminate results for both miRNA targets. One sample was positive for miR-371a-3p and indeterminate for miR-372-3p, and another was negative for miR-371a-3p and indeterminate for miR-372-3p. As detection of either miRNA target is considered potential evidence for the presence of active disease, the overall clinical interpretation would have changed for two samples.</div><div class="u-margin-s-bottom" id="p0135"><span><span><span>The GCT patients were further stratified into two <a href="https://www.sciencedirect.com/topics/pharmacology-toxicology-and-pharmaceutical-science/primary-tumor" title="Learn more about primary tumor from ScienceDirect&#39;s AI-generated Topic Pages" class="topic-link">primary tumor</a> site subgroups (41 from </span><a href="https://www.sciencedirect.com/topics/immunology-and-microbiology/testis" title="Learn more about testis from ScienceDirect&#39;s AI-generated Topic Pages" class="topic-link">testis</a> and 10 from mediastinum). Although the number of samples in primary mediastinal site is relatively small, miR-371a-3p and miR-372-3p were detected in seven of nine and eight of nine samples from </span><a href="https://www.sciencedirect.com/topics/medicine-and-dentistry/mediastinal-germ-cell-tumor" title="Learn more about mediastinal GCTs from ScienceDirect&#39;s AI-generated Topic Pages" class="topic-link">mediastinal GCTs</a> with active disease, respectively, and not detected in the one primary mediastinal patient without active disease, resulting in a clinical sensitivity of 77.8% or 88.9%, respectively, for miR-371a-3p and miR-372-3p, with a clinical specificity of 100% for both (</span><a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#appsec1" name="bappsec1" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="appsec1"><span class="anchor-text-container"><span class="anchor-text">Supplemental Tables S1–S3</span></span></a>). The sensitivity for GCT with a testicular primary tumor site was 83.3% for miR-371a-3p, which is slightly higher than that for patients with a primary mediastinal GCT. The clinical specificity of miR-371a-3p for testicular GCTs was 100% (<a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#appsec1" name="bappsec1" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="appsec1"><span class="anchor-text-container"><span class="anchor-text">Supplemental Tables S1</span></span></a>, <a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#appsec1" name="bappsec1" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="appsec1"><span class="anchor-text-container"><span class="anchor-text">S4</span></span></a>, and <a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#appsec1" name="bappsec1" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="appsec1"><span class="anchor-text-container"><span class="anchor-text">S5</span></span></a>). For miR-372-3p, the clinical sensitivity was 87% and specificity was 93.3% for testicular GCTs (<a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#appsec1" name="bappsec1" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="appsec1"><span class="anchor-text-container"><span class="anchor-text">Supplemental Tables S1</span></span></a>, <a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#appsec1" name="bappsec1" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="appsec1"><span class="anchor-text-container"><span class="anchor-text">S4</span></span></a>, and <a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#appsec1" name="bappsec1" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="appsec1"><span class="anchor-text-container"><span class="anchor-text">S5</span></span></a>), which is similar to the sensitivity and specificity of miR-372-3p among patients with a mediastinal primary tumor site.</div><div class="u-margin-s-bottom" id="p0140"><span><span>In addition, the sensitivity of the two miRNA markers was compared with traditional tumor markers currently used as the gold standard [AFP, beta <a href="https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/human-chorionic-gonadotropin" title="Learn more about HCG from ScienceDirect&#39;s AI-generated Topic Pages" class="topic-link">HCG</a> (bHCG), and LDH]. As expected, sensitivity is lower at 47.1%, 64.7%, and 52.9% when using AFP, bHCG, or </span><a href="https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/lactate-dehydrogenase" title="Learn more about LDH from ScienceDirect&#39;s AI-generated Topic Pages" class="topic-link">LDH</a> to correlate with patient disease status, respectively, and is consistent with other studies.</span><a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#bib10" name="bbib10" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib10"><span class="anchor-text-container"><span class="anchor-text"><sup>10</sup></span></span></a><sup>,</sup><a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#bib12" name="bbib12" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib12"><span class="anchor-text-container"><span class="anchor-text"><sup>12</sup></span></span></a><sup>,</sup><a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#bib13" name="bbib13" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib13"><span class="anchor-text-container"><span class="anchor-text"><sup>13</sup></span></span></a> The assay sensitivity established when using GCT miRNA targets, miR-371a-3p and miR-372-3p, demonstrated a drastic improvement in distinguishing patients with active disease from no active disease, compared with when using traditional biomarkers (AFP, bHCG, and LDH).</div></section></section><section id="sec2.3"><h3 id="sectitle0080" class="u-h4 u-margin-m-top u-margin-xs-bottom">Preanalytical Variables</h3><section id="sec2.3.1"><h4 id="sectitle0085" class="u-margin-m-top u-margin-xs-bottom">Specimen Stability</h4><div class="u-margin-s-bottom"><div id="p0145"><span><span>The stability of miR-371a-3p and miR-372-3p in whole blood was evaluated for up to 96 hours after <a href="https://www.sciencedirect.com/topics/immunology-and-microbiology/blood-sampling" title="Learn more about blood collection from ScienceDirect&#39;s AI-generated Topic Pages" class="topic-link">blood collection</a> to mimic clinical blood sample handling and transport to a central </span><a href="https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/laboratory-test" title="Learn more about testing laboratory from ScienceDirect&#39;s AI-generated Topic Pages" class="topic-link">testing laboratory</a><span>. Blood was stored at 4°C and processed to obtain serum at 4, 24, 48, 72, and 96 hours after <a href="https://www.sciencedirect.com/topics/agricultural-and-biological-sciences/blood-sampling" title="Learn more about blood collection from ScienceDirect&#39;s AI-generated Topic Pages" class="topic-link">blood collection</a>. </span></span><span>miRNA levels were stable for 72 hours after <a href="https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/blood-sampling" title="Learn more about blood collection from ScienceDirect&#39;s AI-generated Topic Pages" class="topic-link">blood collection</a> at 4°C (</span><a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#fig5" name="bfig5" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="fig5"><span class="anchor-text-container"><span class="anchor-text">Figure 5</span></span></a>). Stability was defined as a ΔC<sub>T</sub> value&nbsp;&lt;0.5 when comparing the designated time point with the initial time point at 4 hours after blood collection.</div><figure class="figure text-xs" id="fig5"><span><img src="./ye2022analytical_files/1-s2.0-S1525157822001337-gr5.jpg" height="197" alt="" aria-describedby="cap0030"><ol class="u-margin-s-bottom"><li><a class="anchor download-link u-font-sans anchor-primary" href="https://ars.els-cdn.com/content/image/1-s2.0-S1525157822001337-gr5_lrg.jpg" target="_blank" download="" title="Download high-res image (144KB)"><span class="anchor-text-container"><span class="anchor-text">Download: <span class="download-link-title">Download high-res image (144KB)</span></span></span></a></li><li><a class="anchor download-link u-font-sans anchor-primary" href="./ye2022analytical_files/1-s2.0-S1525157822001337-gr5.jpg" target="_blank" download="" title="Download full-size image"><span class="anchor-text-container"><span class="anchor-text">Download: <span class="download-link-title">Download full-size image</span></span></span></a></li></ol></span><span class="captions text-s"><span id="cap0030"><p id="fspara0030"><span class="label">Figure&nbsp;5</span>. Stability of miR-371a-3p (<strong>A</strong>) and miR-372-3p (<strong>B</strong>) targets in whole blood stored at 4°C from 4 to 96 hours after collection. Data represent ΔC<sub>T</sub> values of singlicate analysis at designated time points.</p></span></span></figure></div></section><section id="sec2.3.2"><h4 id="sectitle0090" class="u-margin-m-top u-margin-xs-bottom">Effect of Hemolysis</h4><div class="u-margin-s-bottom"><div id="p0150"><span>Because miR-30b-5p is highly expressed in red <a href="https://www.sciencedirect.com/topics/immunology-and-microbiology/hemocyte" title="Learn more about blood cells from ScienceDirect&#39;s AI-generated Topic Pages" class="topic-link">blood cells</a><span>, the impact of sample hemolysis was examined. The data show that serum miR-30b-5p levels are higher in hemolyzed samples with a <a href="https://www.sciencedirect.com/topics/medicine-and-dentistry/hemoglobin-determination" title="Learn more about hemoglobin concentration from ScienceDirect&#39;s AI-generated Topic Pages" class="topic-link">hemoglobin concentration</a> of ≥120 mg/dL (</span></span><a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#fig6" name="bfig6" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="fig6"><span class="anchor-text-container"><span class="anchor-text">Figure&nbsp;6</span></span></a>). This degree of hemolysis imparts a subtle red tinge of color to serum and correlates with increased hemolysis-dependent miR-451a levels.<a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#bib33" name="bbib33" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib33"><span class="anchor-text-container"><span class="anchor-text"><sup>33</sup></span></span></a><span> Expression of miR-371a-3p and miR-372-3p cannot be normalized accurately in the setting of hemolysis and will result in an underestimation of the relative expression level for these <a href="https://www.sciencedirect.com/topics/immunology-and-microbiology/genetic-marker" title="Learn more about molecular markers from ScienceDirect&#39;s AI-generated Topic Pages" class="topic-link">molecular markers</a>. This is particularly relevant for samples with negative or indeterminate results. A result comment indicating this limitation will thus be included in future patient reports, as well as a specific notation for samples that are visibly hemolyzed.</span></div><figure class="figure text-xs" id="fig6"><span><img src="./ye2022analytical_files/1-s2.0-S1525157822001337-gr6.jpg" height="207" alt="" aria-describedby="cap0035"><ol class="u-margin-s-bottom"><li><a class="anchor download-link u-font-sans anchor-primary" href="https://ars.els-cdn.com/content/image/1-s2.0-S1525157822001337-gr6_lrg.jpg" target="_blank" download="" title="Download high-res image (137KB)"><span class="anchor-text-container"><span class="anchor-text">Download: <span class="download-link-title">Download high-res image (137KB)</span></span></span></a></li><li><a class="anchor download-link u-font-sans anchor-primary" href="./ye2022analytical_files/1-s2.0-S1525157822001337-gr6.jpg" target="_blank" download="" title="Download full-size image"><span class="anchor-text-container"><span class="anchor-text">Download: <span class="download-link-title">Download full-size image</span></span></span></a></li></ol></span><span class="captions text-s"><span id="cap0035"><p id="fspara0035"><span class="label">Figure&nbsp;6</span>. Effect of hemolysis on endogenous housekeeping miRNA miR-30b-5p and red blood cell marker miRNA miR-451a. <strong>A:</strong> Increasing concentrations of lysed red blood cells were spiked into normal serum before miRNA extraction and analysis. The data show expression (C<sub>T</sub><span> value) of miR-30b-5p and hemolysis marker miR-451a relative to <a href="https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/hemoglobin-blood-level" title="Learn more about serum hemoglobin from ScienceDirect&#39;s AI-generated Topic Pages" class="topic-link">serum hemoglobin</a> concentration. Samples were analyzed in singlicate. </span><strong>B:</strong><span> Slight visual hemolysis could be detected at <a href="https://www.sciencedirect.com/topics/immunology-and-microbiology/hemoglobin-blood-level" title="Learn more about serum hemoglobin from ScienceDirect&#39;s AI-generated Topic Pages" class="topic-link">serum hemoglobin</a> concentration of 120 mg/dL.</span></p></span></span></figure></div></section></section></section><section id="sec3"><h2 id="sectitle0095" class="u-h4 u-margin-l-top u-margin-xs-bottom">Discussion</h2><div class="u-margin-s-bottom" id="p0155"><span><span>The data show that stem-loop RT-qPCR testing for miR-371a-3p and miR-372-3p in the serum of patients with history of GCTs can be performed accurately and reliably in a clinical laboratory setting, with appropriate <a href="https://www.sciencedirect.com/topics/agricultural-and-biological-sciences/quality-control" title="Learn more about quality control from ScienceDirect&#39;s AI-generated Topic Pages" class="topic-link">quality control</a> procedures. Following stringent preanalytical, analytical, and postanalytical procedures, the sensitivity and specificity of miRNA miR-371a-3p and miR-372-3p testing showed good correlation with the presence or </span><a href="https://www.sciencedirect.com/topics/pharmacology-toxicology-and-pharmaceutical-science/epileptic-absence" title="Learn more about absence from ScienceDirect&#39;s AI-generated Topic Pages" class="topic-link">absence</a> of disease among patients with GCTs. These findings support previous reports</span><a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#bib13" name="bbib13" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib13"><span class="anchor-text-container"><span class="anchor-text"><sup>13</sup></span></span></a><sup>,</sup><a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#bib14" name="bbib14" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib14"><span class="anchor-text-container"><span class="anchor-text"><sup>14</sup></span></span></a> and extend findings to patients with mediastinal primary GCTs and more extensive treatment.</div><div class="u-margin-s-bottom" id="p0160">Interestingly, in the current study, miR-372-3p showed a higher sensitivity than miR-371a-3p (87.5% versus 81.8%; areas under the curve of 0.95 and 0.93, respectively), which contradicts the results of Dieckmann et&nbsp;al,<a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#bib13" name="bbib13" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib13"><span class="anchor-text-container"><span class="anchor-text"><sup>13</sup></span></span></a><span> which indicated that miR-371a-3p was the most sensitive GCT serum biomarker among the miR-371 to miR-373 and miR-302/367 clusters. However, those studies included few patients with <a href="https://www.sciencedirect.com/topics/medicine-and-dentistry/metastatic-carcinoma" title="Learn more about metastases from ScienceDirect&#39;s AI-generated Topic Pages" class="topic-link">metastases</a><span> or relapsed disease, whereas in the current study, most patients had relapsed disease and metastatic disease at the time of sample collection. Moreover, this study differs from others because the comparison control group consisted of GCT patients without active disease as opposed to prior publications that used <a href="https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/normal-human" title="Learn more about healthy patients from ScienceDirect&#39;s AI-generated Topic Pages" class="topic-link">healthy patients</a><span> or those with non-GCT <a href="https://www.sciencedirect.com/topics/medicine-and-dentistry/cancer" title="Learn more about malignancies from ScienceDirect&#39;s AI-generated Topic Pages" class="topic-link">malignancies</a> as controls. Therefore, these findings must be assessed with caution because miRNA level may not be the same among healthy control, non-GCT control, and GCT patients without active disease.</span></span></span><a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#bib14" name="bbib14" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib14"><span class="anchor-text-container"><span class="anchor-text"><sup>14</sup></span></span></a><sup>,</sup><a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#bib15" name="bbib15" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib15"><span class="anchor-text-container"><span class="anchor-text"><sup>15</sup></span></span></a> In addition, the current study showed a sensitivity of 81.8% for miR-371a-3p, which is lower than the 90.1% in the primary GCT subgroup but consistent with a sensitivity of only 82.6% in the subgroup of relapsing cases reported in the largest study so far.<a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#bib14" name="bbib14" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib14"><span class="anchor-text-container"><span class="anchor-text"><sup>14</sup></span></span></a><span> It is possible that the lower sensitivity of miR-371a-3p in this assay is a result of including mostly patients with relapsed disease as well as the inclusion of some with primary <a href="https://www.sciencedirect.com/topics/medicine-and-dentistry/mediastinal-germ-cell-tumor" title="Learn more about mediastinal GCTs from ScienceDirect&#39;s AI-generated Topic Pages" class="topic-link">mediastinal GCTs</a> (</span><em>n</em>&nbsp;=&nbsp;10), and the sensitivity of which may be somewhat lower in these populations.</div><div class="u-margin-s-bottom" id="p0165">Previous studies of circulating miRNAs for patients with the primary mediastinal GCT malignant-GCT subtype has been exceptionally limited.<a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#bib34" name="bbib34" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib34"><span class="anchor-text-container"><span class="anchor-text"><sup>34</sup></span></span></a><span> A separate analysis was performed to compare mediastinal GCT test results of miRNAs miR-371a-3p and miR-372-3p with those of the testicular GCT test results, which indicated that the sensitivity for GCT with primary mediastinal GCT is slightly lower than those in testicular <a href="https://www.sciencedirect.com/topics/pharmacology-toxicology-and-pharmaceutical-science/primary-tumor" title="Learn more about primary tumor from ScienceDirect&#39;s AI-generated Topic Pages" class="topic-link">primary tumor</a> site. One study</span><a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#bib34" name="bbib34" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib34"><span class="anchor-text-container"><span class="anchor-text"><sup>34</sup></span></span></a><span> identified a panel of C19MC miRNAs (chromosome 19 miRNA cluster, or miR-515 to miR-526) that were highly elevated in one patient with a widely metastatic primary mediastinal <a href="https://www.sciencedirect.com/topics/medicine-and-dentistry/non-seminomatous-germinoma" title="Learn more about nonseminomatous GCT from ScienceDirect&#39;s AI-generated Topic Pages" class="topic-link">nonseminomatous GCT</a> at diagnosis, and confirmed elevation of the established miRNA marker miR-371a-3p34, which is consistent with the current findings.</span></div><div class="u-margin-s-bottom" id="p0170"><span>Both miRNA biomarkers showed an excellent positive predictive value. However, the negative predictive value of &lt;80% of either miRNA alone or both together indicates the possibility of false-negative results. Depending on the intended clinical use, the reference range cutoffs could be adjusted to eliminate the chance of false-negative results, at the cost of increasing the false positivity rate. To improve result reliability, assay reproducibility was considered in the interpretation of miRNA results, such that result interpretation included three categories: target detected, target not detected, and indeterminate. In addition, the data suggest that <a href="https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/rna" title="Learn more about RNA from ScienceDirect&#39;s AI-generated Topic Pages" class="topic-link">RNA</a> should be re-extracted if the C</span><sub>T</sub> value for cel-miR-39-3p is out of acceptable range, and a new specimen requested if the C<sub>T</sub> value for miR-30b-5p is out of acceptable range. To further improve assay performance for samples with low levels of biomarkers, both C<sub>T</sub> value and relative expression levels for each miRNA are used together during result interpretation. Samples with mismatched positive/negative C<sub>T</sub> value and relative expression level should be re-extracted and reanalyzed.</div><div class="u-margin-s-bottom" id="p0175"><span>To achieve robust and reproducible quantification of miRNAs, normalization against at least two reference miRNAs, which are present at stable levels across the sample, should be used to correct for variations in sample composition, <a href="https://www.sciencedirect.com/topics/immunology-and-microbiology/rna-extraction" title="Learn more about RNA extraction from ScienceDirect&#39;s AI-generated Topic Pages" class="topic-link">RNA extraction</a>, RNA handling and storage, RNA load, and the RT-qPCR protocol. In the current protocol, a spiked-in </span><em>C. elegans</em><span> miRNA control and an endogenous <a href="https://www.sciencedirect.com/topics/medicine-and-dentistry/housekeeping-gene" title="Learn more about housekeeping gene from ScienceDirect&#39;s AI-generated Topic Pages" class="topic-link">housekeeping gene</a> were used to fulfill this requirement, and results were expressed as relative expression levels, rather than raw C</span><sub>T</sub> values. In addition, each analytical run included a negative control, a serum sample from a healthy volunteer, and a positive control, representing a previously analyzed positive patient sample. Challenges with incorporating a positive assay control into each run included the lack of commercially available cell lines expressing the desired miRNAs and the risk of contamination with use of specific oligonucleotides. In addition, a no target control was included to detect contamination, and any sample without target amplification was repeated to control for manual testing errors.</div><div class="u-margin-s-bottom" id="p0180"><span><span>Most of the highly expressed miRNAs in serum are derived from <a href="https://www.sciencedirect.com/topics/immunology-and-microbiology/hemocyte" title="Learn more about blood cells from ScienceDirect&#39;s AI-generated Topic Pages" class="topic-link">blood cells</a> and are released into serum following </span><a href="https://www.sciencedirect.com/topics/medicine-and-dentistry/cytolysis" title="Learn more about cell lysis from ScienceDirect&#39;s AI-generated Topic Pages" class="topic-link">cell lysis</a>.</span><a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#bib35" name="bbib35" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib35"><span class="anchor-text-container"><span class="anchor-text"><sup>35</sup></span></span></a> Therefore, the effect of hemolysis on target miRNAs was evaluated. Not only did hemoglobin interfere with PCR, but the results indicate that hemolysis-susceptible reference miRNA (miR-30b-5p) will impact the determination of relative target expression levels. As reported previously, samples that were visually hemolyzed showed decreased target relative expression levels.<a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#bib26" name="bbib26" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib26"><span class="anchor-text-container"><span class="anchor-text"><sup>26</sup></span></span></a> Therefore, the inclusion of an interpretive comment with results when a serum sample is visually hemolyzed to indicate the potential reduction in relative expression is recommended. This would be most relevant for samples testing negative for miR-371a-3p and miR-372-3p or samples with indeterminate results. The use of an alternative reference miRNA could also be considered. For example, Myklebust et&nbsp;al<a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#bib26" name="bbib26" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib26"><span class="anchor-text-container"><span class="anchor-text"><sup>26</sup></span></span></a> reported that miRNA miR-191-5p represents a relatively stable alternative reference miRNA in serum samples with mild hemolysis.</div><div class="u-margin-s-bottom" id="p0185"><span><span><span>Patients with GCTs require intensive follow-up after diagnosis and treatment. Unlike the <a href="https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/plasma-protein" title="Learn more about serum protein from ScienceDirect&#39;s AI-generated Topic Pages" class="topic-link">serum protein</a> markers AFP, β-HCG, and </span><a href="https://www.sciencedirect.com/topics/pharmacology-toxicology-and-pharmaceutical-science/lactate-dehydrogenase" title="Learn more about LDH from ScienceDirect&#39;s AI-generated Topic Pages" class="topic-link">LDH</a>, which are not uniformly elevated in patients with GCT, are significantly associated with </span><a href="https://www.sciencedirect.com/topics/medicine-and-dentistry/tumor-histology" title="Learn more about tumor histology from ScienceDirect&#39;s AI-generated Topic Pages" class="topic-link">tumor histology</a>, clinical and pathologic stages, tumor size, and younger age, and may change on relapse,</span><a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#bib7" name="bbib7" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib7"><span class="anchor-text-container"><span class="anchor-text"><sup>7</sup></span></span></a><sup>,</sup><a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#bib9" name="bbib9" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib9"><span class="anchor-text-container"><span class="anchor-text">9</span></span></a>, <a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#bib10" name="bbib10" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib10"><span class="anchor-text-container"><span class="anchor-text">10</span></span></a>, <a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#bib11" name="bbib11" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib11"><span class="anchor-text-container"><span class="anchor-text">11</span></span></a> evaluation of serum miR-371a-3p and miR-372-3p represents a noninvasive method with potentially greater sensitivity and specificity.<a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#bib12" name="bbib12" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib12"><span class="anchor-text-container"><span class="anchor-text"><sup>12</sup></span></span></a><sup>,</sup><a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#bib22" name="bbib22" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib22"><span class="anchor-text-container"><span class="anchor-text"><sup>22</sup></span></span></a><span> The integration of serum miRNA analysis into the evaluation of patients with GCTs, using validated clinical laboratory processes and procedures, could thus have significant positive impact on <a href="https://www.sciencedirect.com/topics/medicine-and-dentistry/clinical-trial" title="Learn more about clinical trials from ScienceDirect&#39;s AI-generated Topic Pages" class="topic-link">clinical trials</a> and the diagnosis and management of patients with GCTs.</span><a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#bib17" name="bbib17" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib17"><span class="anchor-text-container"><span class="anchor-text"><sup>17</sup></span></span></a><sup>,</sup><a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#bib36" name="bbib36" data-sd-ui-side-panel-opener="true" data-xocs-content-type="reference" data-xocs-content-id="bib36"><span class="anchor-text-container"><span class="anchor-text"><sup>36</sup></span></span></a><span> This study demonstrates that serum miR-371a-3p and miR-372-3p analysis can be validated for clinical applications and that prospective studies should be initiated to determine whether miRNA monitoring during <a href="https://www.sciencedirect.com/topics/medicine-and-dentistry/cancer-treatment" title="Learn more about GCT treatment from ScienceDirect&#39;s AI-generated Topic Pages" class="topic-link">GCT treatment</a> and follow-up care can improve patient outcomes.</span></div></section></div><div class="Appendices"><section id="appsec1"><h2 id="sectitle0100" class="u-h4 u-margin-l-top u-margin-xs-bottom">Supplemental Data</h2><span class="download-all-supplemental-data"><span class="article-attachment"><iframe title="download_all" name="download_all" class="u-hide" src="./ye2022analytical_files/saved_resource(1).html"></iframe><form method="post" action="https://www.sciencedirect.com/sdfe/pdf/download/S1525157822001337/attachments" target="download_all"><input name="eids" type="hidden" value="1-s2.0-S1525157822001337-mmc1.docx"><input name="eids" type="hidden" value="1-s2.0-S1525157822001337-mmc2.docx"><input name="eids" type="hidden" value="1-s2.0-S1525157822001337-mmc3.docx"><input name="eids" type="hidden" value="1-s2.0-S1525157822001337-mmc4.docx"><input name="eids" type="hidden" value="1-s2.0-S1525157822001337-mmc5.docx"><button class="button-link download-all-button u-font-sans button-link-primary button-link-icon-left" type="submit"><svg focusable="false" viewBox="0 0 98 128" height="20" class="icon icon-download"><path d="M77.38 56.18l-6.6-7.06L54 66.36V26H44v40.34L26.28 49.1l-7.3 7.08 29.2 29.32 29.2-29.32M88 74v20H10V74H0v30h98V74H88"></path></svg><span class="button-link-text-container"><span class="button-link-text"><span class="download-all-title">Download all </span>supplementary files<span class="u-show-inline-from-sm"> included with this article</span></span></span></button></form><a class="anchor help-link u-font-sans u-margin-s-left anchor-primary anchor-has-colored-icon anchor-icon-left anchor-with-icon" href="https://service.elsevier.com/app/answers/detail/a_id/19286/supporthub/sciencedirect/" target="_blank" title="What’s this? (Opens in new window)"><svg focusable="false" viewBox="0 0 114 128" height="20" class="icon icon-help"><path d="M57 8C35.69 7.69 15.11 21.17 6.68 40.71c-8.81 19.38-4.91 43.67 9.63 59.25 13.81 15.59 36.85 21.93 56.71 15.68 21.49-6.26 37.84-26.81 38.88-49.21 1.59-21.15-10.47-42.41-29.29-52.1C74.76 10.17 65.88 7.99 57 8zm0 10c20.38-.37 39.57 14.94 43.85 34.85 4.59 18.53-4.25 39.23-20.76 48.79-17.05 10.59-40.96 7.62-54.9-6.83-14.45-13.94-17.42-37.85-6.83-54.9C26.28 26.5 41.39 17.83 57 18zm-.14 14C45.31 32.26 40 40.43 40 50v2h10v-2c0-4.22 2.22-9.66 8-9.24 5.5.4 6.32 5.14 5.78 8.14C62.68 55.06 52 58.4 52 69.4V76h10v-5.56c0-8.16 11.22-11.52 12-21.7.74-9.86-5.56-16.52-16-16.74-.39-.01-.76-.01-1.14 0zM52 82v10h10V82H52z"></path></svg><span class="anchor-text-container"><span class="anchor-text">What’s this?</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></span></span><div class="u-margin-s-bottom" id="p0190"><span class="display"><span class="e-component e-component-mmc1" id="mmc1"><span class="article-attachment u-margin-xs-bottom"><a class="anchor download-link u-font-sans anchor-primary anchor-icon-left anchor-with-icon" href="https://ars.els-cdn.com/content/image/1-s2.0-S1525157822001337-mmc1.docx" target="_blank" download="" title="Download Word document (25KB)"><svg focusable="false" viewBox="0 0 92 128" height="20" width="20" class="icon icon-text-document"><path d="M1 10v108h57.22L91 85.09V10H1m80 70.96L54.07 108H11V20h70v60.96zM43 100h10V80h20V70H43v30M25 32h42v10H25V32zm0 18h42v10H25V50z"></path></svg><span class="anchor-text-container"><span class="anchor-text">Download: <span class="download-link-title">Download Word document (25KB)</span></span></span></a></span><span class="captions text-s"><span><p><span class="label">Supplemental Table S1</span>. </p></span></span></span></span><span class="display"><span class="e-component e-component-mmc2" id="mmc2"><span class="article-attachment u-margin-xs-bottom"><a class="anchor download-link u-font-sans anchor-primary anchor-icon-left anchor-with-icon" href="https://ars.els-cdn.com/content/image/1-s2.0-S1525157822001337-mmc2.docx" target="_blank" download="" title="Download Word document (14KB)"><svg focusable="false" viewBox="0 0 92 128" height="20" width="20" class="icon icon-text-document"><path d="M1 10v108h57.22L91 85.09V10H1m80 70.96L54.07 108H11V20h70v60.96zM43 100h10V80h20V70H43v30M25 32h42v10H25V32zm0 18h42v10H25V50z"></path></svg><span class="anchor-text-container"><span class="anchor-text">Download: <span class="download-link-title">Download Word document (14KB)</span></span></span></a></span><span class="captions text-s"><span><p><span class="label">Supplemental Table S2</span>. </p></span></span></span></span><span class="display"><span class="e-component e-component-mmc3" id="mmc3"><span class="article-attachment u-margin-xs-bottom"><a class="anchor download-link u-font-sans anchor-primary anchor-icon-left anchor-with-icon" href="https://ars.els-cdn.com/content/image/1-s2.0-S1525157822001337-mmc3.docx" target="_blank" download="" title="Download Word document (14KB)"><svg focusable="false" viewBox="0 0 92 128" height="20" width="20" class="icon icon-text-document"><path d="M1 10v108h57.22L91 85.09V10H1m80 70.96L54.07 108H11V20h70v60.96zM43 100h10V80h20V70H43v30M25 32h42v10H25V32zm0 18h42v10H25V50z"></path></svg><span class="anchor-text-container"><span class="anchor-text">Download: <span class="download-link-title">Download Word document (14KB)</span></span></span></a></span><span class="captions text-s"><span><p><span class="label">Supplemental Table S3</span>. </p></span></span></span></span><span class="display"><span class="e-component e-component-mmc4" id="mmc4"><span class="article-attachment u-margin-xs-bottom"><a class="anchor download-link u-font-sans anchor-primary anchor-icon-left anchor-with-icon" href="https://ars.els-cdn.com/content/image/1-s2.0-S1525157822001337-mmc4.docx" target="_blank" download="" title="Download Word document (13KB)"><svg focusable="false" viewBox="0 0 92 128" height="20" width="20" class="icon icon-text-document"><path d="M1 10v108h57.22L91 85.09V10H1m80 70.96L54.07 108H11V20h70v60.96zM43 100h10V80h20V70H43v30M25 32h42v10H25V32zm0 18h42v10H25V50z"></path></svg><span class="anchor-text-container"><span class="anchor-text">Download: <span class="download-link-title">Download Word document (13KB)</span></span></span></a></span><span class="captions text-s"><span><p><span class="label">Supplemental Table S4</span>. </p></span></span></span></span><span class="display"><span class="e-component e-component-mmc5" id="mmc5"><span class="article-attachment u-margin-xs-bottom"><a class="anchor download-link u-font-sans anchor-primary anchor-icon-left anchor-with-icon" href="https://ars.els-cdn.com/content/image/1-s2.0-S1525157822001337-mmc5.docx" target="_blank" download="" title="Download Word document (13KB)"><svg focusable="false" viewBox="0 0 92 128" height="20" width="20" class="icon icon-text-document"><path d="M1 10v108h57.22L91 85.09V10H1m80 70.96L54.07 108H11V20h70v60.96zM43 100h10V80h20V70H43v30M25 32h42v10H25V32zm0 18h42v10H25V50z"></path></svg><span class="anchor-text-container"><span class="anchor-text">Download: <span class="download-link-title">Download Word document (13KB)</span></span></span></a></span><span class="captions text-s"><span><p><span class="label">Supplemental Table S5</span>. </p></span></span></span></span></div></section></div></div><div class="related-content-links u-display-none-from-md"><button class="button-link button-link-primary button-link-small" type="button"><span class="button-link-text-container"><span class="button-link-text">Recommended articles</span></span></button></div><div class="Tail"></div><div><section class="bibliography u-font-serif text-s" id="cebib0010"><h2 class="section-title u-h4 u-margin-l-top u-margin-xs-bottom">References</h2><section class="bibliography-sec" id="cebibsec0010"><ol class="references" id="reference-links-cebibsec0010"><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#bbib1" id="ref-id-bib1" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">1</span></span></a></span><span class="reference" id="sref1"><div class="contribution"><div class="authors u-font-sans">M. Weiland, X.H. Gao, L. Zhou, Q.S. Mi</div><div id="ref-id-sref1" class="title text-m">Small RNAs have a large impact: circulating microRNAs as biomarkers for human diseases</div></div><div class="host u-font-sans">RNA Biol, 9 (2012), pp. 850-859</div><div class="ReferenceLinks u-font-sans"><span class="link lazy-third-party-pdf-link"><span></span></span><a class="anchor link anchor-primary" href="https://doi.org/10.4161/rna.20378" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref1"><span class="anchor-text-container"><span class="anchor-text">Crossref</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-84863425849&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref1"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Small%20RNAs%20have%20a%20large%20impact%3A%20circulating%20microRNAs%20as%20biomarkers%20for%20human%20diseases&amp;publication_year=2012&amp;author=M.%20Weiland&amp;author=X.H.%20Gao&amp;author=L.%20Zhou&amp;author=Q.S.%20Mi" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref1"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#bbib2" id="ref-id-bib2" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">2</span></span></a></span><span class="reference" id="sref2"><div class="contribution"><div class="authors u-font-sans">Y. Peng, C.M. Croce</div><div id="ref-id-sref2" class="title text-m">The role of MicroRNAs in human cancer</div></div><div class="host u-font-sans">Signal Transduct Target Ther, 1 (2016), p. 15004</div><div class="ReferenceLinks u-font-sans"><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-85059644548&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref2"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=The%20role%20of%20MicroRNAs%20in%20human%20cancer&amp;publication_year=2016&amp;author=Y.%20Peng&amp;author=C.M.%20Croce" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref2"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#bbib3" id="ref-id-bib3" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">3</span></span></a></span><span class="reference" id="sref3"><div class="contribution"><div class="authors u-font-sans">P.S. Mitchell, R.K. Parkin, E.M. Kroh, B.R. Fritz, S.K. Wyman, E.L. Pogosova-Agadjanyan, A. Peterson, J. Noteboom, K.C. O'Briant, A. Allen, D.W. Lin, N. Urban, C.W. Drescher, B.S. Knudsen, D.L. Stirewalt, R. Gentleman, R.L. Vessella, P.S. Nelson, D.B. Martin, M. Tewari</div><div id="ref-id-sref3" class="title text-m">Circulating microRNAs as stable blood-based markers for cancer detection</div></div><div class="host u-font-sans">Proc Natl Acad Sci U S A, 105 (2008), pp. 10513-10518</div><div class="ReferenceLinks u-font-sans"><span class="link lazy-third-party-pdf-link"><span></span></span><a class="anchor link anchor-primary" href="https://doi.org/10.1073/pnas.0804549105" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref3"><span class="anchor-text-container"><span class="anchor-text">Crossref</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-48749122914&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref3"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Circulating%20microRNAs%20as%20stable%20blood-based%20markers%20for%20cancer%20detection&amp;publication_year=2008&amp;author=P.S.%20Mitchell&amp;author=R.K.%20Parkin&amp;author=E.M.%20Kroh&amp;author=B.R.%20Fritz&amp;author=S.K.%20Wyman&amp;author=E.L.%20Pogosova-Agadjanyan&amp;author=A.%20Peterson&amp;author=J.%20Noteboom&amp;author=K.C.%20O%27Briant&amp;author=A.%20Allen&amp;author=D.W.%20Lin&amp;author=N.%20Urban&amp;author=C.W.%20Drescher&amp;author=B.S.%20Knudsen&amp;author=D.L.%20Stirewalt&amp;author=R.%20Gentleman&amp;author=R.L.%20Vessella&amp;author=P.S.%20Nelson&amp;author=D.B.%20Martin&amp;author=M.%20Tewari" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref3"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#bbib4" id="ref-id-bib4" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">4</span></span></a></span><span class="reference" id="sref4"><div class="contribution"><div class="authors u-font-sans">K.E.A. Max, K. Bertram, K.M. Akat, K.A. Bogardus, J. Li, P. Morozov, I.Z. Ben-Dov, X. Li, Z.R. Weiss, A. Azizian, A. Sopeyin, T.G. Diacovo, C. Adamidi, Z. Williams, T. Tuschl</div><div id="ref-id-sref4" class="title text-m">Human plasma and serum extracellular small RNA reference profiles and their clinical utility</div></div><div class="host u-font-sans">Proc Natl Acad Sci U S A, 115 (2018), pp. e5334-e5343</div><div class="ReferenceLinks u-font-sans"><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Human%20plasma%20and%20serum%20extracellular%20small%20RNA%20reference%20profiles%20and%20their%20clinical%20utility&amp;publication_year=2018&amp;author=K.E.A.%20Max&amp;author=K.%20Bertram&amp;author=K.M.%20Akat&amp;author=K.A.%20Bogardus&amp;author=J.%20Li&amp;author=P.%20Morozov&amp;author=I.Z.%20Ben-Dov&amp;author=X.%20Li&amp;author=Z.R.%20Weiss&amp;author=A.%20Azizian&amp;author=A.%20Sopeyin&amp;author=T.G.%20Diacovo&amp;author=C.%20Adamidi&amp;author=Z.%20Williams&amp;author=T.%20Tuschl" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref4"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#bbib5" id="ref-id-bib5" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">5</span></span></a></span><span class="reference" id="sref5"><div class="contribution"><div class="authors u-font-sans">J.W. Oosterhuis, L.H. Looijenga</div><div id="ref-id-sref5" class="title text-m">Testicular germ-cell tumours in a broader perspective</div></div><div class="host u-font-sans">Nat Rev Cancer, 5 (2005), pp. 210-222</div><div class="ReferenceLinks u-font-sans"><span class="link lazy-third-party-pdf-link"><span></span></span><a class="anchor link anchor-primary" href="https://doi.org/10.1038/nrc1568" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref5"><span class="anchor-text-container"><span class="anchor-text">Crossref</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-14644396631&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref5"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Testicular%20germ-cell%20tumours%20in%20a%20broader%20perspective&amp;publication_year=2005&amp;author=J.W.%20Oosterhuis&amp;author=L.H.%20Looijenga" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref5"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#bbib6" id="ref-id-bib6" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">6</span></span></a></span><span class="reference" id="sref6"><div class="contribution"><div class="authors u-font-sans">A. Horwich, J. Shipley, R. Huddart</div><div id="ref-id-sref6" class="title text-m">Testicular germ-cell cancer</div></div><div class="host u-font-sans">Lancet, 367 (2006), pp. 754-765</div><div class="ReferenceLinks u-font-sans"><a class="anchor pdf link anchor-primary anchor-icon-left anchor-with-icon" href="https://www.sciencedirect.com/science/article/pii/S0140673606683050/pdfft?md5=abbf20492079ad20b75cba7f15f4ea92&amp;pid=1-s2.0-S0140673606683050-main.pdf" target="_blank" rel="nofollow" aria-describedby="ref-id-sref6"><svg focusable="false" viewBox="0 0 35 32" height="20" class="icon icon-pdf-multicolor"><path d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path d="M.167 2.592H22.39V9.72H.166z" fill="#da0000"></path><path fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="anchor-text-container"><span class="anchor-text">View PDF</span></span></a><a class="anchor link anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S0140673606683050" aria-describedby="ref-id-sref6"><span class="anchor-text-container"><span class="anchor-text">View article</span></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-33644648880&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref6"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Testicular%20germ-cell%20cancer&amp;publication_year=2006&amp;author=A.%20Horwich&amp;author=J.%20Shipley&amp;author=R.%20Huddart" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref6"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#bbib7" id="ref-id-bib7" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">7</span></span></a></span><span class="reference" id="sref7"><div class="contribution"><div class="authors u-font-sans">D.R. Feldman</div><div id="ref-id-sref7" class="title text-m">State-of-the-art management of germ cell tumors</div></div><div class="host u-font-sans">Am Soc Clin Oncol Educ Book, 38 (2018), pp. 319-323</div><div class="ReferenceLinks u-font-sans"><span class="link lazy-third-party-pdf-link"><span></span></span><a class="anchor link anchor-primary" href="https://doi.org/10.1200/edbk_201139" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref7"><span class="anchor-text-container"><span class="anchor-text">Crossref</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-85057242574&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref7"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=State-of-the-art%20management%20of%20germ%20cell%20tumors&amp;publication_year=2018&amp;author=D.R.%20Feldman" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref7"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#bbib8" id="ref-id-bib8" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">8</span></span></a></span><span class="reference" id="sref8"><div class="contribution"><div class="authors u-font-sans">H.S. Haugnes, T. Wethal, N. Aass, O. Dahl, O. Klepp, C.W. Langberg, T. Wilsgaard, R.M. Bremnes, S.D. Fosså</div><div id="ref-id-sref8" class="title text-m">Cardiovascular risk factors and morbidity in long-term survivors of testicular cancer: a 20-year follow-up study</div></div><div class="host u-font-sans">J Clin Oncol, 28 (2010), pp. 4649-4657</div><div class="ReferenceLinks u-font-sans"><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-78049525650&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref8"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Cardiovascular%20risk%20factors%20and%20morbidity%20in%20long-term%20survivors%20of%20testicular%20cancer%3A%20a%2020-year%20follow-up%20study&amp;publication_year=2010&amp;author=H.S.%20Haugnes&amp;author=T.%20Wethal&amp;author=N.%20Aass&amp;author=O.%20Dahl&amp;author=O.%20Klepp&amp;author=C.W.%20Langberg&amp;author=T.%20Wilsgaard&amp;author=R.M.%20Bremnes&amp;author=S.D.%20Foss%C3%A5" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref8"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#bbib9" id="ref-id-bib9" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">9</span></span></a></span><span class="reference" id="sref9"><div class="contribution"><div class="authors u-font-sans">K.P. Dieckmann, H. Simonsen-Richter, M. Kulejewski, P. Anheuser, H. Zecha, H. Isbarn, U. Pichlmeier</div><div id="ref-id-sref9" class="title text-m">Serum tumour markers in testicular germ cell tumours: frequencies of elevated levels and extents of marker elevation are significantly associated with clinical parameters and with response to treatment</div></div><div class="host u-font-sans">Biomed Res Int, 2019 (2019), p. 5030349</div><div class="ReferenceLinks u-font-sans"><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-85067119938&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref9"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Serum%20tumour%20markers%20in%20testicular%20germ%20cell%20tumours%3A%20frequencies%20of%20elevated%20levels%20and%20extents%20of%20marker%20elevation%20are%20significantly%20associated%20with%20clinical%20parameters%20and%20with%20response%20to%20treatment&amp;publication_year=2019&amp;author=K.P.%20Dieckmann&amp;author=H.%20Simonsen-Richter&amp;author=M.%20Kulejewski&amp;author=P.%20Anheuser&amp;author=H.%20Zecha&amp;author=H.%20Isbarn&amp;author=U.%20Pichlmeier" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref9"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#bbib10" id="ref-id-bib10" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">10</span></span></a></span><span class="reference" id="sref10"><div class="contribution"><div class="authors u-font-sans">M.J. Murray, R.A. Huddart, N. Coleman</div><div id="ref-id-sref10" class="title text-m">The present and future of serum diagnostic tests for testicular germ cell tumours</div></div><div class="host u-font-sans">Nat Rev Urol, 13 (2016), pp. 715-725</div><div class="ReferenceLinks u-font-sans"><span class="link lazy-third-party-pdf-link"><span></span></span><a class="anchor link anchor-primary" href="https://doi.org/10.1038/nrurol.2016.170" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref10"><span class="anchor-text-container"><span class="anchor-text">Crossref</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-84991738345&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref10"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=The%20present%20and%20future%20of%20serum%20diagnostic%20tests%20for%20testicular%20germ%20cell%20tumours&amp;publication_year=2016&amp;author=M.J.%20Murray&amp;author=R.A.%20Huddart&amp;author=N.%20Coleman" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref10"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#bbib11" id="ref-id-bib11" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">11</span></span></a></span><span class="reference" id="sref11"><div class="contribution"><div class="authors u-font-sans">F.E. von Eyben</div><div id="ref-id-sref11" class="title text-m">Laboratory markers and germ cell tumors</div></div><div class="host u-font-sans">Crit Rev Clin Lab Sci, 40 (2003), pp. 377-427</div><div class="ReferenceLinks u-font-sans"><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-0141453090&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref11"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Laboratory%20markers%20and%20germ%20cell%20tumors&amp;publication_year=2003&amp;author=F.E.%20von%20Eyben" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref11"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#bbib12" id="ref-id-bib12" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">12</span></span></a></span><span class="reference" id="sref12"><div class="contribution"><div class="authors u-font-sans">A.J. Gillis, M.A. Rijlaarsdam, R. Eini, L.C. Dorssers, K. Biermann, M.J. Murray, J.C. Nicholson, N. Coleman, K.P. Dieckmann, G. Belge, J. Bullerdiek, T. Xu, N. Bernard, L.H. Looijenga</div><div id="ref-id-sref12" class="title text-m">Targeted serum miRNA (TSmiR) test for diagnosis and follow-up of (testicular) germ cell cancer patients: a proof of principle</div></div><div class="host u-font-sans">Mol Oncol, 7 (2013), pp. 1083-1092</div><div class="ReferenceLinks u-font-sans"><a class="anchor pdf link anchor-primary anchor-icon-left anchor-with-icon" href="https://www.sciencedirect.com/science/article/pii/S1574789113001130/pdfft?md5=075d633dd79a3f2b4efc76427c2c0d36&amp;pid=1-s2.0-S1574789113001130-main.pdf" target="_blank" rel="nofollow" aria-describedby="ref-id-sref12"><svg focusable="false" viewBox="0 0 35 32" height="20" class="icon icon-pdf-multicolor"><path d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path d="M.167 2.592H22.39V9.72H.166z" fill="#da0000"></path><path fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="anchor-text-container"><span class="anchor-text">View PDF</span></span></a><a class="anchor link anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1574789113001130" aria-describedby="ref-id-sref12"><span class="anchor-text-container"><span class="anchor-text">View article</span></span></a><a class="anchor link anchor-primary" href="https://doi.org/10.1016/j.molonc.2013.08.002" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref12"><span class="anchor-text-container"><span class="anchor-text">Crossref</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-84887997858&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref12"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Targeted%20serum%20miRNA%20%20test%20for%20diagnosis%20and%20follow-up%20of%20%20germ%20cell%20cancer%20patients%3A%20a%20proof%20of%20principle&amp;publication_year=2013&amp;author=A.J.%20Gillis&amp;author=M.A.%20Rijlaarsdam&amp;author=R.%20Eini&amp;author=L.C.%20Dorssers&amp;author=K.%20Biermann&amp;author=M.J.%20Murray&amp;author=J.C.%20Nicholson&amp;author=N.%20Coleman&amp;author=K.P.%20Dieckmann&amp;author=G.%20Belge&amp;author=J.%20Bullerdiek&amp;author=T.%20Xu&amp;author=N.%20Bernard&amp;author=L.H.%20Looijenga" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref12"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#bbib13" id="ref-id-bib13" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">13</span></span></a></span><span class="reference" id="sref13"><div class="contribution"><div class="authors u-font-sans">K.P. Dieckmann, A. Radtke, M. Spiekermann, T. Balks, C. Matthies, P. Becker, C. Ruf, C. Oing, K. Oechsle, C. Bokemeyer, J. Hammel, S. Melchior, W. Wosniok, G. Belge</div><div id="ref-id-sref13" class="title text-m">Serum levels of MicroRNA miR-371a-3p: a sensitive and specific new biomarker for germ cell tumours</div></div><div class="host u-font-sans">Eur Urol, 71 (2017), pp. 213-220</div><div class="ReferenceLinks u-font-sans"><a class="anchor pdf link anchor-primary anchor-icon-left anchor-with-icon" href="https://www.sciencedirect.com/science/article/pii/S0302283816304341/pdfft?md5=447f266a8af73ed8182ea32879917b19&amp;pid=1-s2.0-S0302283816304341-mainext.pdf" target="_blank" rel="nofollow" aria-describedby="ref-id-sref13"><svg focusable="false" viewBox="0 0 35 32" height="20" class="icon icon-pdf-multicolor"><path d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path d="M.167 2.592H22.39V9.72H.166z" fill="#da0000"></path><path fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="anchor-text-container"><span class="anchor-text">View PDF</span></span></a><a class="anchor link anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S0302283816304341" aria-describedby="ref-id-sref13"><span class="anchor-text-container"><span class="anchor-text">View article</span></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-84997765050&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref13"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Serum%20levels%20of%20MicroRNA%20miR-371a-3p%3A%20a%20sensitive%20and%20specific%20new%20biomarker%20for%20germ%20cell%20tumours&amp;publication_year=2017&amp;author=K.P.%20Dieckmann&amp;author=A.%20Radtke&amp;author=M.%20Spiekermann&amp;author=T.%20Balks&amp;author=C.%20Matthies&amp;author=P.%20Becker&amp;author=C.%20Ruf&amp;author=C.%20Oing&amp;author=K.%20Oechsle&amp;author=C.%20Bokemeyer&amp;author=J.%20Hammel&amp;author=S.%20Melchior&amp;author=W.%20Wosniok&amp;author=G.%20Belge" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref13"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#bbib14" id="ref-id-bib14" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">14</span></span></a></span><span class="reference" id="sref14"><div class="contribution"><div class="authors u-font-sans">K.P. Dieckmann, A. Radtke, L. Geczi, C. Matthies, P. Anheuser, U. Eckardt, <em> et al.</em></div><div id="ref-id-sref14" class="title text-m">Serum levels of MicroRNA-371a-3p (M371 test) as a new biomarker of testicular germ cell tumors: results of a prospective multicentric study</div></div><div class="host u-font-sans">J Clin Oncol, 37 (2019), pp. 1412-1423</div><div class="ReferenceLinks u-font-sans"><span class="link lazy-third-party-pdf-link"><span></span></span><a class="anchor link anchor-primary" href="https://doi.org/10.1200/jco.18.01480" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref14"><span class="anchor-text-container"><span class="anchor-text">Crossref</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-85066237611&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref14"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Serum%20levels%20of%20MicroRNA-371a-3p%20%20as%20a%20new%20biomarker%20of%20testicular%20germ%20cell%20tumors%3A%20results%20of%20a%20prospective%20multicentric%20study&amp;publication_year=2019&amp;author=K.P.%20Dieckmann&amp;author=A.%20Radtke&amp;author=L.%20Geczi&amp;author=C.%20Matthies&amp;author=P.%20Anheuser&amp;author=U.%20Eckardt" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref14"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#bbib15" id="ref-id-bib15" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">15</span></span></a></span><span class="reference" id="sref15"><div class="contribution"><div class="authors u-font-sans">I. Syring, J. Bartels, S. Holdenrieder, G. Kristiansen, S.C. Müller, J. Ellinger</div><div id="ref-id-sref15" class="title text-m">Circulating serum miRNA (miR-367-3p, miR-371a-3p, miR-372-3p and miR-373-3p) as biomarkers in patients with testicular germ cell cancer</div></div><div class="host u-font-sans">J Urol, 193 (2015), pp. 331-337</div><div class="ReferenceLinks u-font-sans"><a class="anchor pdf link anchor-primary anchor-icon-left anchor-with-icon" href="https://www.sciencedirect.com/science/article/pii/S0022534714039573/pdfft?md5=682e250cf16387219692db7e9a21a2e9&amp;pid=1-s2.0-S0022534714039573-main.pdf" target="_blank" rel="nofollow" aria-describedby="ref-id-sref15"><svg focusable="false" viewBox="0 0 35 32" height="20" class="icon icon-pdf-multicolor"><path d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path d="M.167 2.592H22.39V9.72H.166z" fill="#da0000"></path><path fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="anchor-text-container"><span class="anchor-text">View PDF</span></span></a><a class="anchor link anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S0022534714039573" aria-describedby="ref-id-sref15"><span class="anchor-text-container"><span class="anchor-text">View article</span></span></a><a class="anchor link anchor-primary" href="https://doi.org/10.1016/j.juro.2014.07.010" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref15"><span class="anchor-text-container"><span class="anchor-text">Crossref</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-84925600621&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref15"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Circulating%20serum%20miRNA%20%20as%20biomarkers%20in%20patients%20with%20testicular%20germ%20cell%20cancer&amp;publication_year=2015&amp;author=I.%20Syring&amp;author=J.%20Bartels&amp;author=S.%20Holdenrieder&amp;author=G.%20Kristiansen&amp;author=S.C.%20M%C3%BCller&amp;author=J.%20Ellinger" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref15"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#bbib16" id="ref-id-bib16" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">16</span></span></a></span><span class="reference" id="sref16"><div class="contribution"><div class="authors u-font-sans">T. van Agthoven, L.H.J. Looijenga</div><div id="ref-id-sref16" class="title text-m">Accurate primary germ cell cancer diagnosis using serum based microRNA detection (ampTSmiR test)</div></div><div class="host u-font-sans">Oncotarget, 8 (2017), pp. 58037-58049</div><div class="ReferenceLinks u-font-sans"><span class="link lazy-third-party-pdf-link"><span></span></span><a class="anchor link anchor-primary" href="https://doi.org/10.18632/oncotarget.10867" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref16"><span class="anchor-text-container"><span class="anchor-text">Crossref</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-85029087995&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref16"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Accurate%20primary%20germ%20cell%20cancer%20diagnosis%20using%20serum%20based%20microRNA%20detection%20&amp;publication_year=2017&amp;author=T.%20van%20Agthoven&amp;author=L.H.J.%20Looijenga" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref16"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#bbib17" id="ref-id-bib17" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">17</span></span></a></span><span class="reference" id="sref17"><div class="contribution"><div class="authors u-font-sans">R. Leão, M. Albersen, L.H.J. Looijenga, T. Tandstad, C. Kollmannsberger, M.J. Murray, S. Culine, N. Coleman, G. Belge, R.J. Hamilton, K.P. Dieckmann</div><div id="ref-id-sref17" class="title text-m">Circulating MicroRNAs, the next-generation serum biomarkers in testicular germ cell tumours: a systematic review</div></div><div class="host u-font-sans">Eur Urol, 80 (2021), pp. 456-466</div><div class="ReferenceLinks u-font-sans"><a class="anchor pdf link anchor-primary anchor-icon-left anchor-with-icon" href="https://www.sciencedirect.com/science/article/pii/S0302283821018121/pdfft?md5=10185778f171177e930d65909af1ac16&amp;pid=1-s2.0-S0302283821018121-main.pdf" target="_blank" rel="nofollow" aria-describedby="ref-id-sref17"><svg focusable="false" viewBox="0 0 35 32" height="20" class="icon icon-pdf-multicolor"><path d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path d="M.167 2.592H22.39V9.72H.166z" fill="#da0000"></path><path fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="anchor-text-container"><span class="anchor-text">View PDF</span></span></a><a class="anchor link anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S0302283821018121" aria-describedby="ref-id-sref17"><span class="anchor-text-container"><span class="anchor-text">View article</span></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-85108573908&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref17"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Circulating%20MicroRNAs%2C%20the%20next-generation%20serum%20biomarkers%20in%20testicular%20germ%20cell%20tumours%3A%20a%20systematic%20review&amp;publication_year=2021&amp;author=R.%20Le%C3%A3o&amp;author=M.%20Albersen&amp;author=L.H.J.%20Looijenga&amp;author=T.%20Tandstad&amp;author=C.%20Kollmannsberger&amp;author=M.J.%20Murray&amp;author=S.%20Culine&amp;author=N.%20Coleman&amp;author=G.%20Belge&amp;author=R.J.%20Hamilton&amp;author=K.P.%20Dieckmann" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref17"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#bbib18" id="ref-id-bib18" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">18</span></span></a></span><span class="reference" id="sref18"><div class="contribution"><div class="authors u-font-sans">M.J. Murray, E. Bell, K.L. Raby, M.A. Rijlaarsdam, A.J. Gillis, L.H. Looijenga, H. Brown, B. Destenaves, J.C. Nicholson, N. Coleman</div><div id="ref-id-sref18" class="title text-m">A pipeline to quantify serum and cerebrospinal fluid microRNAs for diagnosis and detection of relapse in paediatric malignant germ-cell tumours</div></div><div class="host u-font-sans">Br J Cancer, 114 (2016), pp. 151-162</div><div class="ReferenceLinks u-font-sans"><span class="link lazy-third-party-pdf-link"><span></span></span><a class="anchor link anchor-primary" href="https://doi.org/10.1038/bjc.2015.429" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref18"><span class="anchor-text-container"><span class="anchor-text">Crossref</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-84955203155&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref18"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=A%20pipeline%20to%20quantify%20serum%20and%20cerebrospinal%20fluid%20microRNAs%20for%20diagnosis%20and%20detection%20of%20relapse%20in%20paediatric%20malignant%20germ-cell%20tumours&amp;publication_year=2016&amp;author=M.J.%20Murray&amp;author=E.%20Bell&amp;author=K.L.%20Raby&amp;author=M.A.%20Rijlaarsdam&amp;author=A.J.%20Gillis&amp;author=L.H.%20Looijenga&amp;author=H.%20Brown&amp;author=B.%20Destenaves&amp;author=J.C.%20Nicholson&amp;author=N.%20Coleman" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref18"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#bbib19" id="ref-id-bib19" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">19</span></span></a></span><span class="reference" id="sref19"><div class="contribution"><div class="authors u-font-sans">A. Radtke, F. Hennig, R. Ikogho, J. Hammel, P. Anheuser, C. Wülfing, G. Belge, K.P. Dieckmann</div><div id="ref-id-sref19" class="title text-m">The novel biomarker of germ cell tumours, Micro-RNA-371a-3p, has a very rapid decay in patients with clinical stage 1</div></div><div class="host u-font-sans">Urol Int, 100 (2018), pp. 470-475</div><div class="ReferenceLinks u-font-sans"><span class="link lazy-third-party-pdf-link"><span></span></span><a class="anchor link anchor-primary" href="https://doi.org/10.1159/000488771" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref19"><span class="anchor-text-container"><span class="anchor-text">Crossref</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-85046008488&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref19"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=The%20novel%20biomarker%20of%20germ%20cell%20tumours%2C%20Micro-RNA-371a-3p%2C%20has%20a%20very%20rapid%20decay%20in%20patients%20with%20clinical%20stage%201&amp;publication_year=2018&amp;author=A.%20Radtke&amp;author=F.%20Hennig&amp;author=R.%20Ikogho&amp;author=J.%20Hammel&amp;author=P.%20Anheuser&amp;author=C.%20W%C3%BClfing&amp;author=G.%20Belge&amp;author=K.P.%20Dieckmann" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref19"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#bbib20" id="ref-id-bib20" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">20</span></span></a></span><span class="reference" id="sref20"><div class="contribution"><div class="authors u-font-sans">M. Spiekermann, G. Belge, N. Winter, R. Ikogho, T. Balks, J. Bullerdiek, K.P. Dieckmann</div><div id="ref-id-sref20" class="title text-m">MicroRNA miR-371a-3p in serum of patients with germ cell tumours: evaluations for establishing a serum biomarker</div></div><div class="host u-font-sans">Andrology, 3 (2015), pp. 78-84</div><div class="ReferenceLinks u-font-sans"><span class="link lazy-third-party-pdf-link"><span></span></span><a class="anchor link anchor-primary" href="https://doi.org/10.1111/j.2047-2927.2014.00269.x" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref20"><span class="anchor-text-container"><span class="anchor-text">Crossref</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-84923259069&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref20"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=MicroRNA%20miR-371a-3p%20in%20serum%20of%20patients%20with%20germ%20cell%20tumours%3A%20evaluations%20for%20establishing%20a%20serum%20biomarker&amp;publication_year=2015&amp;author=M.%20Spiekermann&amp;author=G.%20Belge&amp;author=N.%20Winter&amp;author=R.%20Ikogho&amp;author=T.%20Balks&amp;author=J.%20Bullerdiek&amp;author=K.P.%20Dieckmann" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref20"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#bbib21" id="ref-id-bib21" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">21</span></span></a></span><span class="reference" id="sref21"><div class="contribution"><div class="authors u-font-sans">M.J. Murray, N. Coleman</div><div id="ref-id-sref21" class="title text-m">Testicular cancer: a new generation of biomarkers for malignant germ cell tumours</div></div><div class="host u-font-sans">Nat Rev Urol, 9 (2012), pp. 298-300</div><div class="ReferenceLinks u-font-sans"><span class="link lazy-third-party-pdf-link"><span></span></span><a class="anchor link anchor-primary" href="https://doi.org/10.1038/nrurol.2012.86" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref21"><span class="anchor-text-container"><span class="anchor-text">Crossref</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-84862209160&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref21"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Testicular%20cancer%3A%20a%20new%20generation%20of%20biomarkers%20for%20malignant%20germ%20cell%20tumours&amp;publication_year=2012&amp;author=M.J.%20Murray&amp;author=N.%20Coleman" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref21"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#bbib22" id="ref-id-bib22" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">22</span></span></a></span><span class="reference" id="sref22"><div class="contribution"><div class="authors u-font-sans">M. Alkassis, H.R. Kourie, J. Sarkis, E. Nemr</div><div id="ref-id-sref22" class="title text-m">Predictive biomarkers in bladder cancer</div></div><div class="host u-font-sans">Biomark Med, 15 (2021), pp. 241-246</div><div class="ReferenceLinks u-font-sans"><span class="link lazy-third-party-pdf-link"><span></span></span><a class="anchor link anchor-primary" href="https://doi.org/10.2217/bmm-2020-0575" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref22"><span class="anchor-text-container"><span class="anchor-text">Crossref</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-85101686309&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref22"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Predictive%20biomarkers%20in%20bladder%20cancer&amp;publication_year=2021&amp;author=M.%20Alkassis&amp;author=H.R.%20Kourie&amp;author=J.%20Sarkis&amp;author=E.%20Nemr" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref22"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#bbib23" id="ref-id-bib23" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">23</span></span></a></span><span class="reference" id="sref23"><div class="contribution"><div class="authors u-font-sans">I. Lee, D. Baxter, M.Y. Lee, K. Scherler, K. Wang</div><div id="ref-id-sref23" class="title text-m">The importance of standardization on analyzing circulating RNA</div></div><div class="host u-font-sans">Mol Diagn Ther, 21 (2017), pp. 259-268</div><div class="ReferenceLinks u-font-sans"><span class="link lazy-third-party-pdf-link"><span></span></span><a class="anchor link anchor-primary" href="https://doi.org/10.1007/s40291-016-0251-y" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref23"><span class="anchor-text-container"><span class="anchor-text">Crossref</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-85007483164&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref23"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=The%20importance%20of%20standardization%20on%20analyzing%20circulating%20RNA&amp;publication_year=2017&amp;author=I.%20Lee&amp;author=D.%20Baxter&amp;author=M.Y.%20Lee&amp;author=K.%20Scherler&amp;author=K.%20Wang" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref23"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#bbib24" id="ref-id-bib24" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">24</span></span></a></span><span class="reference" id="sref24"><div class="contribution"><div class="authors u-font-sans">M.J. Marzi, F. Montani, R.M. Carletti, F. Dezi, E. Dama, G. Bonizzi, M.T. Sandri, C. Rampinelli, M. Bellomi, P. Maisonneuve, L. Spaggiari, G. Veronesi, F. Bianchi, P.P. Di Fiore, F. Nicassio</div><div id="ref-id-sref24" class="title text-m">Optimization and standardization of circulating MicroRNA detection for clinical application: the miR-test case</div></div><div class="host u-font-sans">Clin Chem, 62 (2016), pp. 743-754</div><div class="ReferenceLinks u-font-sans"><span class="link lazy-third-party-pdf-link"><span></span></span><a class="anchor link anchor-primary" href="https://doi.org/10.1373/clinchem.2015.251942" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref24"><span class="anchor-text-container"><span class="anchor-text">Crossref</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-84964799354&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref24"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Optimization%20and%20standardization%20of%20circulating%20MicroRNA%20detection%20for%20clinical%20application%3A%20the%20miR-test%20case&amp;publication_year=2016&amp;author=M.J.%20Marzi&amp;author=F.%20Montani&amp;author=R.M.%20Carletti&amp;author=F.%20Dezi&amp;author=E.%20Dama&amp;author=G.%20Bonizzi&amp;author=M.T.%20Sandri&amp;author=C.%20Rampinelli&amp;author=M.%20Bellomi&amp;author=P.%20Maisonneuve&amp;author=L.%20Spaggiari&amp;author=G.%20Veronesi&amp;author=F.%20Bianchi&amp;author=P.P.%20Di%20Fiore&amp;author=F.%20Nicassio" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref24"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#bbib25" id="ref-id-bib25" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">25</span></span></a></span><span class="reference" id="sref25"><div class="contribution"><div class="authors u-font-sans">X. Feng, Y. Liu, N. Wan</div><div id="ref-id-sref25" class="title text-m">Plasma microRNA detection standardization test</div></div><div class="host u-font-sans">J Clin Lab Anal, 34 (2020), p. e23058</div><div class="ReferenceLinks u-font-sans"><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-85074110536&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref25"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Plasma%20microRNA%20detection%20standardization%20test&amp;publication_year=2020&amp;author=X.%20Feng&amp;author=Y.%20Liu&amp;author=N.%20Wan" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref25"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#bbib26" id="ref-id-bib26" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">26</span></span></a></span><span class="reference" id="sref26"><div class="contribution"><div class="authors u-font-sans">M.P. Myklebust, B. Rosenlund, P. Gjengsto, B.S. Bercea, A. Karlsdottir, M. Brydoy, O. Dahl</div><div id="ref-id-sref26" class="title text-m">Quantitative PCR measurement of miR-371a-3p and miR-372-p is influenced by hemolysis</div></div><div class="host u-font-sans">Front Genet, 10 (2019), p. 463</div><div class="ReferenceLinks u-font-sans"><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-85067853844&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref26"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Quantitative%20PCR%20measurement%20of%20miR-371a-3p%20and%20miR-372-p%20is%20influenced%20by%20hemolysis&amp;publication_year=2019&amp;author=M.P.%20Myklebust&amp;author=B.%20Rosenlund&amp;author=P.%20Gjengsto&amp;author=B.S.%20Bercea&amp;author=A.%20Karlsdottir&amp;author=M.%20Brydoy&amp;author=O.%20Dahl" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref26"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#bbib27" id="ref-id-bib27" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">27</span></span></a></span><span class="reference" id="sref27"><div class="contribution"><div class="authors u-font-sans">G.W. Tan, A.S. Khoo, L.P. Tan</div><div id="ref-id-sref27" class="title text-m">Evaluation of extraction kits and RT-qPCR systems adapted to high-throughput platform for circulating miRNAs</div></div><div class="host u-font-sans">Sci Rep, 5 (2015), p. 9430</div><div class="ReferenceLinks u-font-sans"><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-84925373069&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref27"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Evaluation%20of%20extraction%20kits%20and%20RT-qPCR%20systems%20adapted%20to%20high-throughput%20platform%20for%20circulating%20miRNAs&amp;publication_year=2015&amp;author=G.W.%20Tan&amp;author=A.S.%20Khoo&amp;author=L.P.%20Tan" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref27"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#bbib28" id="ref-id-bib28" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">28</span></span></a></span><span class="reference" id="sref28"><div class="contribution"><div class="authors u-font-sans">K.J. Livak, T.D. Schmittgen</div><div id="ref-id-sref28" class="title text-m">Analysis of relative gene expression data using real-time quantitative PCR and the 2(-Delta Delta C(T)) method</div></div><div class="host u-font-sans">Methods, 25 (2001), pp. 402-408</div><div class="ReferenceLinks u-font-sans"><a class="anchor pdf link anchor-primary anchor-icon-left anchor-with-icon" href="https://www.sciencedirect.com/science/article/pii/S1046202301912629/pdf?md5=168f8e22f2c7d76f6021c5cbe37a3ba6&amp;pid=1-s2.0-S1046202301912629-main.pdf" target="_blank" rel="nofollow" aria-describedby="ref-id-sref28"><svg focusable="false" viewBox="0 0 35 32" height="20" class="icon icon-pdf-multicolor"><path d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path d="M.167 2.592H22.39V9.72H.166z" fill="#da0000"></path><path fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="anchor-text-container"><span class="anchor-text">View PDF</span></span></a><a class="anchor link anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1046202301912629" aria-describedby="ref-id-sref28"><span class="anchor-text-container"><span class="anchor-text">View article</span></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Analysis%20of%20relative%20gene%20expression%20data%20using%20real-time%20quantitative%20PCR%20and%20the%202%20method&amp;publication_year=2001&amp;author=K.J.%20Livak&amp;author=T.D.%20Schmittgen" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref28"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#bbib29" id="ref-id-bib29" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">29</span></span></a></span><span class="reference" id="sref29"><div class="contribution"><div class="authors u-font-sans">A. Baratloo, M. Hosseini, A. Negida, G. El Ashal</div><div id="ref-id-sref29" class="title text-m">Part 1: simple definition and calculation of accuracy, sensitivity and specificity</div></div><div class="host u-font-sans">Emerg (Tehran), 3 (2015), pp. 48-49</div><div class="ReferenceLinks u-font-sans"><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-84956933214&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref29"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Part%201%3A%20simple%20definition%20and%20calculation%20of%20accuracy%2C%20sensitivity%20and%20specificity&amp;publication_year=2015&amp;author=A.%20Baratloo&amp;author=M.%20Hosseini&amp;author=A.%20Negida&amp;author=G.%20El%20Ashal" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref29"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#bbib30" id="ref-id-bib30" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">30</span></span></a></span><span class="reference" id="sref30"><div class="contribution"><div class="authors u-font-sans">R. Leão, T. van Agthoven, A. Figueiredo, M.A.S. Jewett, K. Fadaak, J. Sweet, A.E. Ahmad, L. Anson-Cartwright, P. Chung, A. Hansen, P. Warde, P. Castelo-Branco, M. O'Malley, P.L. Bedard, L.H.J. Looijenga, R.J. Hamilton</div><div id="ref-id-sref30" class="title text-m">Serum miRNA predicts viable disease after chemotherapy in patients with testicular nonseminoma germ cell tumor</div></div><div class="host u-font-sans">J Urol, 200 (2018), pp. 126-135</div><div class="ReferenceLinks u-font-sans"><a class="anchor pdf link anchor-primary anchor-icon-left anchor-with-icon" href="https://www.sciencedirect.com/science/article/pii/S0022534718393741/pdfft?md5=5ffae618445033d52f7c8df226668fe8&amp;pid=1-s2.0-S0022534718393741-main.pdf" target="_blank" rel="nofollow" aria-describedby="ref-id-sref30"><svg focusable="false" viewBox="0 0 35 32" height="20" class="icon icon-pdf-multicolor"><path d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path d="M.167 2.592H22.39V9.72H.166z" fill="#da0000"></path><path fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="anchor-text-container"><span class="anchor-text">View PDF</span></span></a><a class="anchor link anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S0022534718393741" aria-describedby="ref-id-sref30"><span class="anchor-text-container"><span class="anchor-text">View article</span></span></a><a class="anchor link anchor-primary" href="https://doi.org/10.1016/j.juro.2018.02.068" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref30"><span class="anchor-text-container"><span class="anchor-text">Crossref</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-85044591680&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref30"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Serum%20miRNA%20predicts%20viable%20disease%20after%20chemotherapy%20in%20patients%20with%20testicular%20nonseminoma%20germ%20cell%20tumor&amp;publication_year=2018&amp;author=R.%20Le%C3%A3o&amp;author=T.%20van%20Agthoven&amp;author=A.%20Figueiredo&amp;author=M.A.S.%20Jewett&amp;author=K.%20Fadaak&amp;author=J.%20Sweet&amp;author=A.E.%20Ahmad&amp;author=L.%20Anson-Cartwright&amp;author=P.%20Chung&amp;author=A.%20Hansen&amp;author=P.%20Warde&amp;author=P.%20Castelo-Branco&amp;author=M.%20O%27Malley&amp;author=P.L.%20Bedard&amp;author=L.H.J.%20Looijenga&amp;author=R.J.%20Hamilton" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref30"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#bbib31" id="ref-id-bib31" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">31</span></span></a></span><span class="reference" id="sref31"><div class="contribution"><div class="authors u-font-sans">M.J. Murray, H.L. Watson, D. Ward, S. Bailey, M. Ferraresso, J.C. Nicholson, V.J. Gnanapragasam, B. Thomas, C.G. Scarpini, N. Coleman</div><div id="ref-id-sref31" class="title text-m">“Future-proofing” blood processing for measurement of circulating miRNAs in samples from biobanks and prospective clinical trials</div></div><div class="host u-font-sans">Cancer Epidemiol Biomarkers Prev, 27 (2018), pp. 208-218</div><div class="ReferenceLinks u-font-sans"><span class="link lazy-third-party-pdf-link"><span></span></span><a class="anchor link anchor-primary" href="https://doi.org/10.1158/1055-9965.epi-17-0657" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref31"><span class="anchor-text-container"><span class="anchor-text">Crossref</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-85041389948&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref31"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Future-proofing%20blood%20processing%20for%20measurement%20of%20circulating%20miRNAs%20in%20samples%20from%20biobanks%20and%20prospective%20clinical%20trials&amp;publication_year=2018&amp;author=M.J.%20Murray&amp;author=H.L.%20Watson&amp;author=D.%20Ward&amp;author=S.%20Bailey&amp;author=M.%20Ferraresso&amp;author=J.C.%20Nicholson&amp;author=V.J.%20Gnanapragasam&amp;author=B.%20Thomas&amp;author=C.G.%20Scarpini&amp;author=N.%20Coleman" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref31"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#bbib32" id="ref-id-bib32" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">32</span></span></a></span><span class="reference" id="sref32"><div class="contribution"><div class="authors u-font-sans">M. Alizadeh, M. Bernard, B. Danic, C. Dauriac, B. Birebent, C. Lapart, T. Lamy, P.Y. Le Prisé, A. Beauplet, D. Bories, G. Semana, E. Quelvennec</div><div id="ref-id-sref32" class="title text-m">Quantitative assessment of hematopoietic chimerism after bone marrow transplantation by real-time quantitative polymerase chain reaction</div></div><div class="host u-font-sans">Blood, 99 (2002), pp. 4618-4625</div><div class="ReferenceLinks u-font-sans"><a class="anchor pdf link anchor-primary anchor-icon-left anchor-with-icon" href="https://www.sciencedirect.com/science/article/pii/S0006497120600696/pdfft?md5=48be5ec91280f9b314f749a668c9e14d&amp;pid=1-s2.0-S0006497120600696-main.pdf" target="_blank" rel="nofollow" aria-describedby="ref-id-sref32"><svg focusable="false" viewBox="0 0 35 32" height="20" class="icon icon-pdf-multicolor"><path d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path d="M.167 2.592H22.39V9.72H.166z" fill="#da0000"></path><path fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="anchor-text-container"><span class="anchor-text">View PDF</span></span></a><a class="anchor link anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S0006497120600696" aria-describedby="ref-id-sref32"><span class="anchor-text-container"><span class="anchor-text">View article</span></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-0037097590&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref32"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Quantitative%20assessment%20of%20hematopoietic%20chimerism%20after%20bone%20marrow%20transplantation%20by%20real-time%20quantitative%20polymerase%20chain%20reaction&amp;publication_year=2002&amp;author=M.%20Alizadeh&amp;author=M.%20Bernard&amp;author=B.%20Danic&amp;author=C.%20Dauriac&amp;author=B.%20Birebent&amp;author=C.%20Lapart&amp;author=T.%20Lamy&amp;author=P.Y.%20Le%20Pris%C3%A9&amp;author=A.%20Beauplet&amp;author=D.%20Bories&amp;author=G.%20Semana&amp;author=E.%20Quelvennec" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref32"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#bbib33" id="ref-id-bib33" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">33</span></span></a></span><span class="reference" id="sref33"><div class="contribution"><div class="authors u-font-sans">K.D. Rasmussen, S. Simmini, C. Abreu-Goodger, N. Bartonicek, M. Di Giacomo, D. Bilbao-Cortes, R. Horos, M. Von Lindern, A.J. Enright, D. O'Carroll</div><div id="ref-id-sref33" class="title text-m">The miR-144/451 locus is required for erythroid homeostasis</div></div><div class="host u-font-sans">J Exp Med, 207 (2010), pp. 1351-1358</div><div class="ReferenceLinks u-font-sans"><span class="link lazy-third-party-pdf-link"><span></span></span><a class="anchor link anchor-primary" href="https://doi.org/10.1084/jem.20100458" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref33"><span class="anchor-text-container"><span class="anchor-text">Crossref</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-77954408884&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref33"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=The%20miR-144451%20locus%20is%20required%20for%20erythroid%20homeostasis&amp;publication_year=2010&amp;author=K.D.%20Rasmussen&amp;author=S.%20Simmini&amp;author=C.%20Abreu-Goodger&amp;author=N.%20Bartonicek&amp;author=M.%20Di%20Giacomo&amp;author=D.%20Bilbao-Cortes&amp;author=R.%20Horos&amp;author=M.%20Von%20Lindern&amp;author=A.J.%20Enright&amp;author=D.%20O%27Carroll" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref33"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#bbib34" id="ref-id-bib34" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">34</span></span></a></span><span class="reference" id="sref34"><div class="contribution"><div class="authors u-font-sans">M.J. Murray, S. Smith, D. Ward, L. Verduci, J.C. Nicholson, C.G. Scarpini, N. Coleman</div><div id="ref-id-sref34" class="title text-m">Circulating microRNAs as biomarkers to assist the management of the malignant germ-cell-tumour subtype choriocarcinoma</div></div><div class="host u-font-sans">Transl Oncol, 14 (2021), p. 100904</div><div class="ReferenceLinks u-font-sans"><a class="anchor pdf link anchor-primary anchor-icon-left anchor-with-icon" href="https://www.sciencedirect.com/science/article/pii/S193652332030396X/pdfft?md5=856f422a7c371cb37d5308a4b8afb49b&amp;pid=1-s2.0-S193652332030396X-main.pdf" target="_blank" rel="nofollow" aria-describedby="ref-id-sref34"><svg focusable="false" viewBox="0 0 35 32" height="20" class="icon icon-pdf-multicolor"><path d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path d="M.167 2.592H22.39V9.72H.166z" fill="#da0000"></path><path fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="anchor-text-container"><span class="anchor-text">View PDF</span></span></a><a class="anchor link anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S193652332030396X" aria-describedby="ref-id-sref34"><span class="anchor-text-container"><span class="anchor-text">View article</span></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-85092274774&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref34"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Circulating%20microRNAs%20as%20biomarkers%20to%20assist%20the%20management%20of%20the%20malignant%20germ-cell-tumour%20subtype%20choriocarcinoma&amp;publication_year=2021&amp;author=M.J.%20Murray&amp;author=S.%20Smith&amp;author=D.%20Ward&amp;author=L.%20Verduci&amp;author=J.C.%20Nicholson&amp;author=C.G.%20Scarpini&amp;author=N.%20Coleman" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref34"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#bbib35" id="ref-id-bib35" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">35</span></span></a></span><span class="reference" id="sref35"><div class="contribution"><div class="authors u-font-sans">M.B. Kirschner, J.J. Edelman, S.C. Kao, M.P. Vallely, N. van Zandwijk, G. Reid</div><div id="ref-id-sref35" class="title text-m">The impact of hemolysis on cell-free microRNA biomarkers</div></div><div class="host u-font-sans">Front Genet, 4 (2013), p. 94</div><div class="ReferenceLinks u-font-sans"><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=The%20impact%20of%20hemolysis%20on%20cell-free%20microRNA%20biomarkers&amp;publication_year=2013&amp;author=M.B.%20Kirschner&amp;author=J.J.%20Edelman&amp;author=S.C.%20Kao&amp;author=M.P.%20Vallely&amp;author=N.%20van%20Zandwijk&amp;author=G.%20Reid" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref35"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li><li><span class="label u-font-sans"><a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#bbib36" id="ref-id-bib36" data-aa-button="sd:product:journal:article:location=references:type=anchor:name=citation-name"><span class="anchor-text-container"><span class="anchor-text">36</span></span></a></span><span class="reference" id="sref36"><div class="contribution"><div class="authors u-font-sans">K. Almstrup, J. Lobo, N. Mørup, G. Belge, E. Rajpert-De Meyts, L.H.J. Looijenga, K.P. Dieckmann</div><div id="ref-id-sref36" class="title text-m">Application of miRNAs in the diagnosis and monitoring of testicular germ cell tumours</div></div><div class="host u-font-sans">Nat Rev Urol, 17 (2020), pp. 201-213</div><div class="ReferenceLinks u-font-sans"><span class="link lazy-third-party-pdf-link"><span></span></span><a class="anchor link anchor-primary" href="https://doi.org/10.1038/s41585-020-0296-x" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref36"><span class="anchor-text-container"><span class="anchor-text">Crossref</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-85081642889&amp;partnerID=10&amp;rel=R3.0.0" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref36"><span class="anchor-text-container"><span class="anchor-text">View in Scopus</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a><a class="anchor link anchor-primary" href="https://scholar.google.com/scholar_lookup?title=Application%20of%20miRNAs%20in%20the%20diagnosis%20and%20monitoring%20of%20testicular%20germ%20cell%20tumours&amp;publication_year=2020&amp;author=K.%20Almstrup&amp;author=J.%20Lobo&amp;author=N.%20M%C3%B8rup&amp;author=G.%20Belge&amp;author=E.%20Rajpert-De%20Meyts&amp;author=L.H.J.%20Looijenga&amp;author=K.P.%20Dieckmann" target="_blank" rel="noopener noreferrer" aria-describedby="ref-id-sref36"><span class="anchor-text-container"><span class="anchor-text">Google Scholar</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></span></li></ol></section></section></div><div id="section-cited-by"><section aria-label="Cited by" class="ListArticles preview"><div class="PageDivider"></div><header id="citing-articles-header"><h2 class="u-h4 u-margin-l-ver u-font-serif">Cited by (12)</h2></header><div aria-describedby="citing-articles-header"><div class="citing-articles u-margin-l-bottom"><ul><li class="ListArticleItem u-margin-l-bottom"><div class="sub-heading u-margin-xs-bottom"><h3 class="u-font-serif" id="citing-articles-article-0-title"><a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S2666168324006190"><span class="anchor-text-container"><span class="anchor-text">Perioperative Serum MicroRNA 371a-3p and 372-3p Levels in Patients with Clinically Localized Testicular Masses</span></span></a></h3><div>2024, European Urology Open Science</div><div class="CitedSection u-margin-s-top"><div class="u-margin-s-left"><div class="cite-header u-text-italic u-font-sans">Citation Excerpt :</div><p class="u-font-serif text-xs">Our previously described combination assay (miR-371a-3p and miR-372-3p, collectively Memorial Sloan Kettering Cancer Center [MSK] miRNA assay [MMA]) was created based on the prevailing performance characteristics of miR-371a-3p and the potential benefit of miR-372-3p. The extent literature suggests that miR-371a-3p alone demonstrates lower sensitivity in the detection of relapse; miR-372-3p has been shown to be markedly elevated in pediatric yolk sac tumors and has a longer half-life, and therefore was selected for inclusion in our assay [15]. We sought to study diagnostic test characteristics of our combined assay in patients with localized testicular masses to determine possible marginal benefit of this approach, to determine what patient and tumor factors may influence levels of these markers, and to explore postorchiectomy kinetics.</p></div></div></div><div class="buttons"><button class="button-link button-link-primary button-link-icon-right" data-aa-button="sd:product:journal:article:location=citing-articles:type=view-details" aria-describedby="citing-articles-article-0-title" aria-controls="citing-articles-article-0" aria-expanded="false" type="button"><span class="button-link-text-container"><span class="button-link-text">Show abstract</span></span><svg focusable="false" viewBox="0 0 92 128" height="20" class="icon icon-navigate-down"><path d="M1 51l7-7 38 38 38-38 7 7-45 45z"></path></svg></button></div><div class="u-display-none" aria-hidden="true"><div class="abstract u-margin-xs-top u-margin-m-bottom u-font-serif" id="reference-abstract"><div class="u-margin-ver-m"><div class="u-margin-s-bottom" id="sp0010">MicroRNAs (miRNAs) show promise as blood-based tumor markers for germ cell tumors (GCTs), with miRNA-371-3p being the most studied. The marginal benefit of including other candidate miRNAs to aid with the management of testicular GCTs remains unclear.</div><div class="u-margin-s-bottom" id="sp0015">To assess the performance of our combined miRNA assay (371a-3p and 372-3p) in patients with clinically localized testicular masses.</div><div class="u-margin-s-bottom" id="sp0020">This was a retrospective review of patients prospectively enrolled in an ongoing protocol collecting serum miR-371a-3p and miR-372-3p levels (together, Memorial Sloan Kettering Cancer Center [MSK] miRNA assay [MMA]) in patients with a suspected or diagnosed testicular GCT.</div><div class="u-margin-s-bottom" id="sp0025">The coprimary outcomes of interest were sensitivity and specificity of miR-371a-3p and 372-3p, individually and together, to detect nonteratomatous GCTs in the orchiectomy specimen. Secondary outcomes included additional assay diagnostic parameters, the relationship of patient and disease factors with variations in miRNA levels, and temporal patterns of miRNA normalization after orchiectomy.</div><div class="u-margin-s-bottom" id="sp0030">Sixty-two patients were included, 52 had a viable GCT at orchiectomy, and ten had no cancer or a non-GCT. Forty-six patients with a GCT had positive preorchiectomy MMA (sensitivity 88.5% [95% confidence interval {CI}: 79.8, 97.2]), and one patient had positive preorchiectomy MMA but no GCT (specificity 90.0% [95% CI: 71.4, 100]). The diagnostic performance of miR-371a-3-p and miR-372-3p was similar. The time for miRNA to decrease to undetectable levels varied, with some patients having positive levels up to 3 wk after orchiectomy.</div><div class="u-margin-s-bottom" id="sp0035">The biomarkers miR-371a-3p and miR-372-3p demonstrated high sensitivity and specificity for localized testicular GCTs, but causes of variation in relative miRNA levels and time to normalization for individual patients remain unclear.</div><div class="u-margin-s-bottom" id="sp0040">We studied the ability of the blood-based biomarkers miR-371a-3p and miR-372-3p to detect testicular cancer (germ cell tumors) in patients with small testicular masses. We found that together and individually these were sensitive and specific for testicular cancer.</div></div></div></div></li><li class="ListArticleItem u-margin-l-bottom"><div class="sub-heading u-margin-xs-bottom"><h3 class="u-font-serif" id="citing-articles-article-1-title"><a class="anchor anchor-primary" href="https://doi.org/10.1158/1078-0432.CCR-23-0730" target="_blank"><span class="anchor-text-container"><span class="anchor-text">Detection of Recurrence through microRNA-371a-3p Serum Levels in a Follow-up of Stage I Testicular Germ Cell Tumors in the DRKS-00019223 Study</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></h3><div>2024, Clinical Cancer Research</div></div><div class="buttons"></div><div class="u-display-none" aria-hidden="true"><div class="abstract u-margin-xs-top u-margin-m-bottom u-font-serif" id="reference-abstract"></div></div></li><li class="ListArticleItem u-margin-l-bottom"><div class="sub-heading u-margin-xs-bottom"><h3 class="u-font-serif" id="citing-articles-article-2-title"><a class="anchor anchor-primary" href="https://doi.org/10.1038/s41598-023-37271-1" target="_blank"><span class="anchor-text-container"><span class="anchor-text">Refining the serum miR-371a-3p test for viable germ cell tumor detection</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></h3><div>2023, Scientific Reports</div></div><div class="buttons"></div><div class="u-display-none" aria-hidden="true"><div class="abstract u-margin-xs-top u-margin-m-bottom u-font-serif" id="reference-abstract"></div></div></li><li class="ListArticleItem u-margin-l-bottom"><div class="sub-heading u-margin-xs-bottom"><h3 class="u-font-serif" id="citing-articles-article-3-title"><a class="anchor anchor-primary" href="https://doi.org/10.3390/cancers15153944" target="_blank"><span class="anchor-text-container"><span class="anchor-text">MicroRNA-371a-3p—The Novel Serum Biomarker in Testicular Germ Cell Tumors</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></h3><div>2023, Cancers</div></div><div class="buttons"></div><div class="u-display-none" aria-hidden="true"><div class="abstract u-margin-xs-top u-margin-m-bottom u-font-serif" id="reference-abstract"></div></div></li><li class="ListArticleItem u-margin-l-bottom"><div class="sub-heading u-margin-xs-bottom"><h3 class="u-font-serif" id="citing-articles-article-4-title"><a class="anchor anchor-primary" href="https://doi.org/10.1007/s00432-023-04664-8" target="_blank"><span class="anchor-text-container"><span class="anchor-text">Testicular neoplasms: the interrelationships of serum levels of microRNA-371a-3p (M371) and classical tumor markers with histology, clinical staging, and age—a statistical analysis</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></h3><div>2023, Journal of Cancer Research and Clinical Oncology</div></div><div class="buttons"></div><div class="u-display-none" aria-hidden="true"><div class="abstract u-margin-xs-top u-margin-m-bottom u-font-serif" id="reference-abstract"></div></div></li><li class="ListArticleItem u-margin-l-bottom"><div class="sub-heading u-margin-xs-bottom"><h3 class="u-font-serif" id="citing-articles-article-5-title"><a class="anchor anchor-primary" href="https://doi.org/10.3389/fonc.2022.893877" target="_blank"><span class="anchor-text-container"><span class="anchor-text">The values of systemic immune-inflammation index and neutrophil-lymphocyte ratio in predicting testicular germ cell tumors: A retrospective clinical study</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></h3><div>2022, Frontiers in Oncology</div></div><div class="buttons"></div><div class="u-display-none" aria-hidden="true"><div class="abstract u-margin-xs-top u-margin-m-bottom u-font-serif" id="reference-abstract"></div></div></li></ul><a class="button-alternative button-alternative-secondary large-alternative button-alternative-icon-left" href="http://www.scopus.com/scopus/inward/citedby.url?partnerID=10&amp;rel=3.0.0&amp;eid=2-s2.0-85135549912&amp;md5=f091eb2fd39f5c5436523024dd3c98a" target="_blank" id="citing-articles-view-all-btn"><svg focusable="false" viewBox="0 0 54 128" height="20" class="icon icon-navigate-right"><path d="M1 99l38-38L1 23l7-7 45 45-45 45z"></path></svg><span class="button-alternative-text-container"><span class="button-alternative-text">View all citing articles on Scopus</span></span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></a></div></div></section></div><div class="Footnotes text-xs"><dl class="footnote"><dt class="footnote-label"></dt><dd class="footnote-detail u-padding-xs-top"><div class="u-margin-s-bottom" id="ntpara0010">Supported in part by the <span id="gs6">Department of Laboratory Medicine</span> (E.I.P.), the Kravis Center for Molecular Oncology at <span id="gs2">Memorial Sloan Kettering Cancer Center</span> (D.B.S.), <span id="gs7">Cycle for Survival</span> (D.R.F.), and the <span id="gs4">NIH</span>/<span id="gs9">National Cancer Institute Cancer Center</span> Support grant <a class="anchor anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001337#gs9"><span class="anchor-text-container"><span class="anchor-text">P30 CA008748</span></span></a>.</div></dd></dl><dl class="footnote"><dt class="footnote-label"></dt><dd class="footnote-detail u-padding-xs-top"><div class="u-margin-s-bottom" id="ntpara0015">F.Y. and D.R.F. are co–first authors and contributed equally to this work.</div></dd></dl><dl class="footnote"><dt class="footnote-label"></dt><dd class="footnote-detail u-padding-xs-top"><div class="u-margin-s-bottom" id="ntpara0020">Disclosures: D.B.S. has consulted for/received honoraria from Pfizer, Loxo/Lilly Oncology, Vividion Therapeutics, FORE Therapeutics, Scorpion Therapeutics, and BridgeBio. S.A.F. receives research funding from <span id="gs10">AstraZeneca</span> and <span id="gs8">Genetech</span>/<span id="gs11">Roche</span>, has consulted for/received honoraria from Merck, and has stock/equity ownership in Allogene Therapeutics, Urogen, Kronos Bio, Neogene Therapeutics, IconOVir, and Vida Ventures. The other authors declare no conflicts of interest.</div></dd></dl></div><div class="Copyright"><span class="copyright-line">© 2022 Association for Molecular Pathology and American Society for Investigative Pathology. Published by Elsevier Inc.</span></div></article><div class="u-display-block-from-md col-lg-6 col-md-8 pad-right u-padding-s-top"><aside class="RelatedContent u-clr-grey8" aria-label="Related content"><section class="RelatedContentPanel u-margin-s-bottom"><header id="recommended-articles-header" class="related-content-panel-header u-margin-s-bottom"><button class="button-link button-link-secondary related-content-panel-toggle is-up button-link-icon-right button-link-has-colored-icon" aria-expanded="true" data-aa-button="sd:product:journal:article:location=recommended-articles:type=close" type="button"><span class="button-link-text-container"><span class="button-link-text"><h2 class="section-title u-h4"><span class="related-content-panel-title-text">Recommended articles</span></h2></span></span><svg focusable="false" viewBox="0 0 92 128" height="20" class="icon icon-navigate-down"><path d="M1 51l7-7 38 38 38-38 7 7-45 45z"></path></svg></button></header><div class="" aria-hidden="false" aria-describedby="recommended-articles-header"><div id="recommended-articles" class="text-xs"><ul><li class="RelatedContentPanelItem u-display-block"><div class="sub-heading u-padding-xs-bottom"><h3 class="related-content-panel-list-entry-outline-padding text-s u-font-serif" id="recommended-articles-article0-title"><a class="anchor u-clamp-2-lines anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S0014482718303525" title="Hepatitis B virus promotes proliferation and metastasis in male Chinese hepatocellular carcinoma patients through the LEF-1/miR-371a-5p/SRCIN1/pleiotrophin/Slug pathway"><span class="anchor-text-container"><span class="anchor-text"><span>Hepatitis B virus promotes proliferation and metastasis in male Chinese hepatocellular carcinoma patients through the LEF-1/miR-371a-5p/SRCIN1/pleiotrophin/Slug pathway</span></span></span></a></h3><div class="article-source u-clr-grey6"><div class="source">Experimental Cell Research, Volume 370, Issue 1, 2018, pp. 174-188</div></div><div class="authors"><span>Pei-song</span> <span>Bai</span>, …, <span>Ying</span> <span>Kong</span></div></div><div class="buttons"></div></li><li class="RelatedContentPanelItem u-display-block"><div class="sub-heading u-padding-xs-bottom"><h3 class="related-content-panel-list-entry-outline-padding text-s u-font-serif" id="recommended-articles-article1-title"><a class="anchor u-clamp-2-lines anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S0021925820311662" title="MicroRNA-451a in extracellular, blood-resident vesicles attenuates macrophage and dendritic cell responses to influenza whole-virus vaccine"><span class="anchor-text-container"><span class="anchor-text"><span>MicroRNA-451a in extracellular, blood-resident vesicles attenuates macrophage and dendritic cell responses to influenza whole-virus vaccine</span></span></span></a></h3><div class="article-source u-clr-grey6"><div class="source">Journal of Biological Chemistry, Volume 293, Issue 48, 2018, pp. 18585-18600</div></div><div class="authors"><span>Masaaki</span> <span>Okamoto</span>, …, <span>Hiroyuki</span> <span>Oshiumi</span></div></div><div class="buttons"><a class="anchor anchor-primary anchor-icon-left anchor-with-icon" href="https://www.sciencedirect.com/science/article/pii/S0021925820311662/pdfft?md5=20790d797c5af1602f142ec869661223&amp;pid=1-s2.0-S0021925820311662-main.pdf" target="_blank" rel="nofollow" aria-describedby="recommended-articles-article1-title"><svg focusable="false" viewBox="0 0 35 32" height="20" class="icon icon-pdf-multicolor"><path d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path d="M.167 2.592H22.39V9.72H.166z" fill="#da0000"></path><path fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="anchor-text-container"><span class="anchor-text">View PDF</span></span></a></div></li><li class="RelatedContentPanelItem u-display-block"><div class="sub-heading u-padding-xs-bottom"><h3 class="related-content-panel-list-entry-outline-padding text-s u-font-serif" id="recommended-articles-article2-title"><a class="anchor u-clamp-2-lines anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S2214442022001887" title="Malignant juxtaglomerular cell tumor"><span class="anchor-text-container"><span class="anchor-text"><span>Malignant juxtaglomerular cell tumor</span></span></span></a></h3><div class="article-source u-clr-grey6"><div class="source">Urology Case Reports, Volume 45, 2022, Article 102176</div></div><div class="authors"><span>Daniel</span> <span>Geisler</span>, …, <span>Sheldon</span> <span>Bastacky</span></div></div><div class="buttons"><a class="anchor anchor-primary anchor-icon-left anchor-with-icon" href="https://www.sciencedirect.com/science/article/pii/S2214442022001887/pdfft?md5=38e0e6a94c3c52f2e97b50882c8ec16e&amp;pid=1-s2.0-S2214442022001887-main.pdf" target="_blank" rel="nofollow" aria-describedby="recommended-articles-article2-title"><svg focusable="false" viewBox="0 0 35 32" height="20" class="icon icon-pdf-multicolor"><path d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path d="M.167 2.592H22.39V9.72H.166z" fill="#da0000"></path><path fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="anchor-text-container"><span class="anchor-text">View PDF</span></span></a></div></li><li class="RelatedContentPanelItem u-display-none"><div class="sub-heading u-padding-xs-bottom"><h3 class="related-content-panel-list-entry-outline-padding text-s u-font-serif" id="recommended-articles-article3-title"><a class="anchor u-clamp-2-lines anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822000654" title="Validation of a New High-Throughput BD COR System Using the BD CTGCTV2 Assay"><span class="anchor-text-container"><span class="anchor-text"><span>Validation of a New High-Throughput BD COR System Using the BD CTGCTV2 Assay</span></span></span></a></h3><div class="article-source u-clr-grey6"><div class="source">The Journal of Molecular Diagnostics, Volume 24, Issue 5, 2022, pp. 485-493</div></div><div class="authors"><span>Stephanie</span> <span>Taylor</span>, …, <span>Wallace</span> <span>Greene</span></div></div><div class="buttons"><a class="anchor anchor-primary anchor-icon-left anchor-with-icon" href="https://www.sciencedirect.com/science/article/pii/S1525157822000654/pdfft?md5=341d933ae257a6ce871fe7b34d3f7094&amp;pid=1-s2.0-S1525157822000654-main.pdf" target="_blank" rel="nofollow" aria-describedby="recommended-articles-article3-title"><svg focusable="false" viewBox="0 0 35 32" height="20" class="icon icon-pdf-multicolor"><path d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path d="M.167 2.592H22.39V9.72H.166z" fill="#da0000"></path><path fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="anchor-text-container"><span class="anchor-text">View PDF</span></span></a></div></li><li class="RelatedContentPanelItem u-display-none"><div class="sub-heading u-padding-xs-bottom"><h3 class="related-content-panel-list-entry-outline-padding text-s u-font-serif" id="recommended-articles-article4-title"><a class="anchor u-clamp-2-lines anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822001180" title="MET Exon 14 Skipping Mutations"><span class="anchor-text-container"><span class="anchor-text"><span><em>MET</em> Exon 14 Skipping Mutations: Essential Considerations for Current Management of Non–Small-Cell Lung Cancer</span></span></span></a></h3><div class="article-source u-clr-grey6"><div class="source">The Journal of Molecular Diagnostics, Volume 24, Issue 8, 2022, pp. 841-843</div></div><div class="authors"><span>Kurtis D.</span> <span>Davies</span>, …, <span>Nikoletta</span> <span>Sidiropoulos</span></div></div><div class="buttons"><a class="anchor anchor-primary anchor-icon-left anchor-with-icon" href="https://www.sciencedirect.com/science/article/pii/S1525157822001180/pdfft?md5=604fe8f8e9435457ee1fc527a7a9300f&amp;pid=1-s2.0-S1525157822001180-main.pdf" target="_blank" rel="nofollow" aria-describedby="recommended-articles-article4-title"><svg focusable="false" viewBox="0 0 35 32" height="20" class="icon icon-pdf-multicolor"><path d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path d="M.167 2.592H22.39V9.72H.166z" fill="#da0000"></path><path fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="anchor-text-container"><span class="anchor-text">View PDF</span></span></a></div></li><li class="RelatedContentPanelItem u-display-none"><div class="sub-heading u-padding-xs-bottom"><h3 class="related-content-panel-list-entry-outline-padding text-s u-font-serif" id="recommended-articles-article5-title"><a class="anchor u-clamp-2-lines anchor-primary" href="https://www.sciencedirect.com/science/article/pii/S1525157822000204" title="PirePred"><span class="anchor-text-container"><span class="anchor-text"><span>PirePred: An Accurate Online Consensus Tool to Interpret Newborn Screening–Related Genetic Variants in Structural Context</span></span></span></a></h3><div class="article-source u-clr-grey6"><div class="source">The Journal of Molecular Diagnostics, Volume 24, Issue 4, 2022, pp. 406-425</div></div><div class="authors"><span>Juan José</span> <span>Galano-Frutos</span>, …, <span>Javier</span> <span>Sancho</span></div></div><div class="buttons"><a class="anchor anchor-primary anchor-icon-left anchor-with-icon" href="https://www.sciencedirect.com/science/article/pii/S1525157822000204/pdfft?md5=e1dbf60b58b5b81b0b1fbcb4402d7f41&amp;pid=1-s2.0-S1525157822000204-main.pdf" target="_blank" rel="nofollow" aria-describedby="recommended-articles-article5-title"><svg focusable="false" viewBox="0 0 35 32" height="20" class="icon icon-pdf-multicolor"><path d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path d="M.167 2.592H22.39V9.72H.166z" fill="#da0000"></path><path fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="anchor-text-container"><span class="anchor-text">View PDF</span></span></a></div></li></ul></div><button class="button-link more-recommendations-button u-margin-s-bottom button-link-primary button-link-icon-right" type="button"><span class="button-link-text-container"><span class="button-link-text">Show 3 more articles</span></span><svg focusable="false" viewBox="0 0 92 128" height="20" class="icon icon-navigate-down"><path d="M1 51l7-7 38 38 38-38 7 7-45 45z"></path></svg></button></div></section><section class="RelatedContentPanel u-margin-s-bottom"><header id="metrics-header" class="related-content-panel-header u-margin-s-bottom"><button class="button-link button-link-secondary related-content-panel-toggle is-up button-link-icon-right button-link-has-colored-icon" aria-expanded="true" type="button"><span class="button-link-text-container"><span class="button-link-text"><h2 class="section-title u-h4"><span class="related-content-panel-title-text">Article Metrics</span></h2></span></span><svg focusable="false" viewBox="0 0 92 128" height="20" class="icon icon-navigate-down"><path d="M1 51l7-7 38 38 38-38 7 7-45 45z"></path></svg></button></header><div class="" aria-hidden="false" aria-describedby="metrics-header"><div class="plumX-metrics"><h3 class="text-s metric-title metric-title-citations">Citations</h3><ul><li class="text-xs metrics"><span>Citation Indexes</span><span>11</span></li></ul><h3 class="text-s metric-title metric-title-captures">Captures</h3><ul><li class="text-xs metrics"><span>Readers</span><span>2</span></li></ul><div class="metrics u-margin-m-top u-margin-s-bottom"><img src="./ye2022analytical_files/plumx-logo.png" class="plumX-logo" alt="PlumX Metrics Logo"><a class="anchor text-xs anchor-primary" href="https://plu.mx/plum/a/?doi=10.1016%2Fj.jmoldx.2022.04.007&amp;theme=plum-sciencedirect-theme&amp;hideUsage=true" target="_blank"><span class="anchor-text-container"><span class="anchor-text">View details</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></div></div></div></section></aside></div></div></div></div><footer role="contentinfo" class="els-footer u-bg-white text-xs u-padding-s-hor u-padding-m-hor-from-sm u-padding-l-hor-from-md u-padding-l-ver u-margin-l-top u-margin-xl-top-from-sm u-margin-l-top-from-md"><div class="els-footer-elsevier u-margin-m-bottom u-margin-0-bottom-from-md u-margin-s-right u-margin-m-right-from-md u-margin-l-right-from-lg"><a class="anchor anchor-primary anchor-icon-left anchor-with-icon" href="https://www.elsevier.com/" target="_blank" aria-label="Elsevier home page (opens in a new tab)" rel="nofollow"><img class="footer-logo" src="./ye2022analytical_files/elsevier-non-solus-new-with-wordmark.svg" alt="Elsevier logo with wordmark" height="64" width="58" loading="lazy"></a></div><div class="els-footer-content"><div class="u-remove-if-print"><ul class="els-footer-links u-margin-xs-bottom" style="list-style:none"><li><a class="anchor u-display-flex u-clr-grey8 u-margin-s-bottom u-margin-0-bottom-from-sm u-margin-m-right-from-sm u-margin-l-right-from-md anchor-primary anchor-small" href="https://www.elsevier.com/solutions/sciencedirect" target="_blank" id="els-footer-about-science-direct" rel="nofollow"><span class="anchor-text-container"><span class="anchor-text">About ScienceDirect</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></li><li><a class="anchor u-display-flex u-clr-grey8 u-margin-s-bottom u-margin-0-bottom-from-sm u-margin-m-right-from-sm u-margin-l-right-from-md anchor-primary anchor-small" href="https://www.sciencedirect.com/user/institution/login?targetURL=%2Fscience%2Farticle%2Fpii%2FS1525157822001337" id="els-footer-remote-access" rel="nofollow"><span class="anchor-text-container"><span class="anchor-text">Remote access</span></span></a></li><li><a class="anchor u-display-flex u-clr-grey8 u-margin-s-bottom u-margin-0-bottom-from-sm u-margin-m-right-from-sm u-margin-l-right-from-md anchor-primary anchor-small" href="https://sd-cart.elsevier.com/?" target="_blank" id="els-footer-shopping-cart" rel="nofollow"><span class="anchor-text-container"><span class="anchor-text">Shopping cart</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></li><li><a class="anchor u-display-flex u-clr-grey8 u-margin-s-bottom u-margin-0-bottom-from-sm u-margin-m-right-from-sm u-margin-l-right-from-md anchor-primary anchor-small" href="https://www.elsmediakits.com/" target="_blank" id="els-footer-advertise" rel="nofollow"><span class="anchor-text-container"><span class="anchor-text">Advertise</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></li><li><a class="anchor u-display-flex u-clr-grey8 u-margin-s-bottom u-margin-0-bottom-from-sm u-margin-m-right-from-sm u-margin-l-right-from-md anchor-primary anchor-small" href="https://service.elsevier.com/ci/pta/login/redirect/contact/supporthub/sciencedirect/p_li/OUjuK9K7N5XLymooN4~iYJzc6mDq2m3HVgiU6CpJ5pv2PqbC1G8StBEM3ffoF4FCTI_QPShve3l07_bfdZ8xQkTaYsO9rD8Mj_6a_FCSvBokwW4uPVPJuHvJguXB9Tkn~HULFTMIoz4niUj_FDpgJOdD4wNFdF_xSaFD7_USdB_qaRLHbKYq5AbkZEoEXI6lWgE41_t4G_ltTu81dEOqxTEo89ZLlsi5h4UkN2erlX5fOIjXJs7b6VE7TKkgtbOzIAOM4KPcOAUpvEdFIkngXdLJnimhbLOTjlOm~FuwLT_D73CQZF43sLcJcYBqyEs3AQ5Og6MgVopqL~ejA0_XTBACQYzNAxutk9SFVmogOobxG4opoTNzn2i1BjI6j1M7" target="_blank" id="els-footer-contact-support" rel="nofollow"><span class="anchor-text-container"><span class="anchor-text">Contact and support</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></li><li><a class="anchor u-display-flex u-clr-grey8 u-margin-s-bottom u-margin-0-bottom-from-sm u-margin-m-right-from-sm u-margin-l-right-from-md anchor-primary anchor-small" href="https://www.elsevier.com/legal/elsevier-website-terms-and-conditions" target="_blank" id="els-footer-terms-condition" rel="nofollow"><span class="anchor-text-container"><span class="anchor-text">Terms and conditions</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></li><li><a class="anchor u-display-flex u-clr-grey8 u-margin-s-bottom u-margin-0-bottom-from-sm u-margin-m-right-from-sm u-margin-l-right-from-md anchor-primary anchor-small" href="https://www.elsevier.com/legal/privacy-policy" target="_blank" id="els-footer-privacy-policy" rel="nofollow"><span class="anchor-text-container"><span class="anchor-text">Privacy policy</span><svg focusable="false" viewBox="0 0 8 8" height="20" aria-label="Opens in new window" class="icon icon-arrow-up-right-tiny arrow-external-link"><path d="M1.12949 2.1072V1H7V6.85795H5.89111V2.90281L0.784057 8L0 7.21635L5.11902 2.1072H1.12949Z"></path></svg></span></a></li></ul></div><p id="els-footer-cookie-message" class="u-remove-if-print">Cookies are used by this site. <!-- --> <button class="button-link ot-sdk-show-settings cookie-btn button-link-primary button-link-small" id="ot-sdk-btn" type="button"><span class="button-link-text-container"><span class="button-link-text"><strong>Cookie Settings</strong></span></span></button></p><p id="els-footer-copyright">All content on this site: Copyright © <!-- -->2025<!-- --> Elsevier B.V., its licensors, and contributors. All rights are reserved, including those for text and data mining, AI training, and similar technologies. For all open access content, the Creative Commons licensing terms apply.</p></div><div class="els-footer-relx u-margin-0-top u-margin-m-top-from-xs u-margin-0-top-from-md"><a class="anchor anchor-primary anchor-icon-left anchor-with-icon" href="https://www.relx.com/" target="_blank" aria-label="RELX home page (opens in a new tab)" id="els-footer-relx" rel="nofollow"><img loading="lazy" src="./ye2022analytical_files/logo-relx-tm.svg" width="93" height="20" alt="RELX group home page"></a></div></footer></div></div></div></div>
      <div id="floating-ui-node" class="floating-ui-node" data-sd-ui-floating-ui="true"></div>
      
      <script src="./ye2022analytical_files/launch-a6263b31083f.min.js" type="image/ot-performance" async=""></script>
      
<script type="text/javascript">
    window.pageData = {"content":[{"contentType":"JL","format":"MIME-XHTML","id":"sd:article:pii:S1525157822001337","type":"sd:article:JL:scope-full","detail":"sd:article:subtype:fla","publicationType":"journal","issn":"1525-1578","volumeNumber":"24","issueNumber":"8","provider":"elsevier","entitlementType":"openarchive"}],"page":{"businessUnit":"ELS:RP:ST","language":"en","name":"product:journal:article","noTracking":"false","productAppVersion":"full-direct","productName":"SD","type":"CP-CA","environment":"prod","loadTimestamp":1737318715125,"loadTime":""},"visitor":{"accessType":"ae:REG_GUEST","accountId":"ae:246711","accountName":"ae:ClinicalKey Global Guest Users","loginStatus":"logged in","userId":"ae:17847196","ipAddress":"188.240.139.246","appSessionId":"40c299f9-7f3b-44b6-9d04-e4d0ac73172b"}};
    window.pageData.page.loadTime = performance ? Math.round(performance.now()).toString() : '';

    try {
      appData.push({
      event: 'pageLoad',
      page: pageData.page,
      visitor: pageData.visitor,
      content: pageData.content
      })
    } catch(e) {
        console.warn("There was an error loading or running Adobe DTM: ", e);
    }
</script>
      <script nomodule="" src="./ye2022analytical_files/core-js.es.minified.js" type="text/javascript"></script>
      <script src="./ye2022analytical_files/react.production.min.js" type="text/javascript"></script>
      <script src="./ye2022analytical_files/react-dom.production.min.js" type="text/javascript"></script>
      <script async="" src="./ye2022analytical_files/arp.js" type="text/javascript"></script>
      <script type="text/javascript">
    const pendoData = {"visitor":{"pageName":"SD:product:journal:article","pageType":"CP-CA","pageProduct":"SD","pageLanguage":"en","pageEnvironment":"prod","accessType":"ae:REG_GUEST","countryCode":"ES"},"account":{"id":"ae:246711","name":"ae:ClinicalKey Global Guest Users"},"events":{}};;
    pendoData.events = {
      ready: function () {
        pendo.addAltText();
      },
    };
    function runPendo(data, options) {
  const {
    firstDelay,
    maxRetries,
    urlPrefix,
    urlSuffix,
    apiKey
  } = options;
  (function (apiKey) {
    (function (p, e, n, d, o) {
      var v, w, x, y, z;
      o = p[d] = p[d] || {};
      o._q = [];
      v = ['initialize', 'identify', 'updateOptions', 'pageLoad'];
      for (w = 0, x = v.length; w < x; ++w) (function (m) {
        o[m] = o[m] || function () {
          o._q[m === v[0] ? 'unshift' : 'push']([m].concat([].slice.call(arguments, 0)));
        };
      })(v[w]);
      y = e.createElement(n);
      y.async = !0;
      y.src = urlPrefix + apiKey + urlSuffix;
      z = e.getElementsByTagName(n)[0];
      z.parentNode.insertBefore(y, z);
    })(window, document, 'script', 'pendo');
    pendo.addAltText = function () {
      var target = document.querySelector('body');
      var observer = new MutationObserver(function (mutations) {
        mutations.forEach(function (mutation) {
          if (mutation?.addedNodes?.length) {
            if (mutation.addedNodes[0]?.className?.includes("_pendo-badge")) {
              const badge = mutation.addedNodes[0];
              const altText = badge?.attributes['aria-label'].value ? badge?.attributes['aria-label'].value : 'Feedback';
              const pendoBadgeImage = pendo.dom(`#${badge?.attributes?.id.value} img`);
              if (pendoBadgeImage.length) {
                pendoBadgeImage[0]?.setAttribute('alt', altText);
              }
            }
          }
        });
      });
      var config = {
        attributeFilter: ['data-layout'],
        attributes: true,
        childList: true,
        characterData: true,
        subtree: false
      };
      observer.observe(target, config);
    };
  })(apiKey);
  (function watchAndSetPendo(nextDelay, retryAttempt) {
    if (typeof pageDataTracker === 'object' && typeof pageDataTracker.getVisitorId === 'function' && pageDataTracker.getVisitorId()) {
      data.visitor.id = pageDataTracker.getVisitorId();
      console.debug(`initializing pendo`);
      pendo.initialize(data);
    } else {
      if (retryAttempt > 0) {
        return setTimeout(function () {
          watchAndSetPendo(nextDelay * 2, retryAttempt - 1);
        }, nextDelay);
      }
      pendo.initialize(data);
      console.debug(`gave up ... pendo initialized`);
    }
  })(firstDelay, maxRetries);
}
    runPendo(pendoData, {
      firstDelay: 100,
      maxRetries: 5,
      urlPrefix: 'https://cdn.pendo.io/agent/static/',
      urlSuffix: '/pendo.js',
      apiKey: 'd6c1d995-bc7e-4e53-77f1-2ea4ecbb9565',
    });
  </script>
      <span id="pendo-answer-rating"></span>
      <script type="text/x-mathjax-config;executed=true">
        MathJax.Hub.Config({
          displayAlign: 'left',
          "fast-preview": {
            disabled: true
          },
          CommonHTML: { linebreaks: { automatic: true } },
          PreviewHTML: { linebreaks: { automatic: true } },
          'HTML-CSS': { linebreaks: { automatic: true } },
          SVG: {
            scale: 90,
            linebreaks: { automatic: true }
          }
        });
      </script>
      <script async="" src="./ye2022analytical_files/MathJax.js" type="text/javascript"></script>
      <script async="" src="./ye2022analytical_files/f.txt" type="text/javascript"></script>
      <script async="" src="./ye2022analytical_files/f(1).txt" type="text/javascript"></script>
      <script data-cfasync="false">
      (function initOneTrust()  {
        const monitor = {
  init: () => {},
  loaded: () => {},
};
        function enableGroup(group) {
  document.querySelectorAll(`script[type*="ot-${group}"]`).forEach(script => {
    script.type = 'text/javascript';
    document.head.appendChild(script);
  });
}
        function runOneTrustCookies(doClear, monitor) {
  const oneTrustConsentSdkId = 'onetrust-consent-sdk';
  const emptyNodeSelectors = 'h3.ot-host-name, h4.ot-host-desc, button.ot-host-box';
  const ariaLabelledByButtonNodes = 'div.ot-accordion-layout > button';
  const ariaAttribute = 'aria-labelledby';
  function adjustOneTrustDOM() {
    const oneTrustRoot = document.getElementById('onetrust-consent-sdk');

    /* remove empty nodes */
    [...(oneTrustRoot?.querySelectorAll(emptyNodeSelectors) ?? [])].filter(e => e.textContent === '').forEach(e => e.remove());

    /* remove invalid aria-labelledby values */
    oneTrustRoot?.querySelectorAll(ariaLabelledByButtonNodes).forEach(e => {
      const presentIdValue = e.getAttribute(ariaAttribute)?.split(' ').filter(label => document.getElementById(label)).join(' ');
      if (presentIdValue) {
        e.setAttribute(ariaAttribute, presentIdValue);
      }
    });
  }
  function observeOneTrustLoaded(shouldSetOTDefaults, isConsentPresent) {
    const cb = (mutationList, observer) => {
      const oneTrustRoot = mutationList.filter(mutationRecord => mutationRecord.type === 'childList' && mutationRecord.addedNodes.length).map(mutationRecord => [...mutationRecord.addedNodes]).flat().find(e => e.id === oneTrustConsentSdkId);
      if (oneTrustRoot && typeof OneTrust !== 'undefined') {
        monitor.loaded(true);
        OneTrust.OnConsentChanged(() => {
          const perfAllowed = decodeURIComponent(document.cookie.match('(^| )OptanonConsent=([^;]+)')?.[2])?.match('groups=([0-9:0|1,?]+)&?')?.[1]?.match('2:([0|1])')[1] === '1';
          if (perfAllowed) {
            enableGroup('performance');
          }
        });
        if (!isConsentPresent && (shouldSetOTDefaults || OneTrust.GetDomainData().ConsentModel.Name === 'implied consent')) {
          OneTrust.AllowAll();
        }
        document.dispatchEvent(new CustomEvent('@sdtech/onetrust/loaded', {}));
        observer.disconnect();
        adjustOneTrustDOM();
      }
    };
    const observer = new MutationObserver(cb);
    observer.observe(document.querySelector('body'), {
      childList: true
    });
  }
  if (doClear) {
    document.cookie = 'OptanonAlertBoxClosed=; expires=Thu, 01 Jan 1970 00:00:00 UTC; samesite=lax; path=/';
  }
  const isConsentPresent = !!decodeURIComponent(document.cookie.match('(^| )OptanonConsent=([^;]+)')?.[2])?.match('groups=([0-9:0|1,?]+)&?')?.[1];
  const shouldSetOTDefaults = 'true' === 'false' && !document.cookie?.match('OptanonAlertBoxClosed=');
  if (shouldSetOTDefaults) {
    const date = new Date();
    date.setFullYear(date.getFullYear() + 1);
    document.cookie = `OptanonAlertBoxClosed=${new Date().toISOString()}; expires=${date.toUTCString()}; samesite=lax; path=/; domain=sciencedirect.com`;
  }
  observeOneTrustLoaded(shouldSetOTDefaults, isConsentPresent, monitor);
  window.addOTScript = () => {
    const otSDK = document.createElement('script');
    otSDK.setAttribute('data-cfasync', 'false');
    otSDK.setAttribute('src', 'https://cdn.cookielaw.org/scripttemplates/otSDKStub.js');
    otSDK.setAttribute('data-document-language', 'true');
    otSDK.setAttribute('data-domain-script', '865ea198-88cc-4e41-8952-1df75d554d02');
    window.addOTScript = () => {};
    document.head.appendChild(otSDK);
    monitor.init();
  };
  window.addEventListener('load', () => window.addOTScript());
}
        if (document.location.host.match(/.sciencedirect.com$/)) {
          runOneTrustCookies(true, monitor);
        }
        else {
          window.addEventListener('load', (event) => {
            enableGroup('performance');
          });
        }
      }());
    </script>
    
  <iframe id="dJnZ6CSL" frameborder="0" src="./ye2022analytical_files/translateSandbox(1).html" style="width: 0px; height: 0px; display: none;"></iframe><iframe id="ps_mxoirt" src="./ye2022analytical_files/saved_resource(2).html" style="display: none; position: absolute; top: -999px; left: -999px;"></iframe><div class="js-react-modal"></div><div class="js-react-modal"></div><div class="js-react-modal"></div><div class="js-react-modal"></div><div class="js-react-modal"></div><div class="js-react-modal"></div><div class="js-react-modal"></div></body></html>