# Methodological Framework
## Study Population
The analyzed data originates from the FATe cohort [@torres2015fat], a longitudinal study of patients with obesity undergoing bariatric surgery at Miguel Servet University Hospital (HUMS, Zaragoza, Spain). For this research, 78 patients with varying degrees of adiposity were selected based on subcutaneous adipose tissue (scWAT) samples, which are registered at the regional Biobank (Biobanco Aragón) and approved by the CEICA ethics committee. Patients with alcohol or drug abuse, autoimmune diseases, chronic inflammatory conditions, or infectious diseases (HIV, HBV, HCV) were excluded during the screening process. The FATe cohort is characterized by a range of clinical and demographic variables, including sex, age, body mass index (BMI), steatosis, metabolic dysfunction-associated steatohepatitis (MASH), hepatocytic ballooning, lobular inflammation, diabetes, hyperlipidemia, and metabolic dysfunction-associated steatotic liver disease (MASLD).

## Collection and characterization of subcutaneous white adipose tissue
White adipose tissue biopsies (\~3 cm³) from the subcutaneous depot were obtained during laparoscopic bariatric surgery using a bipolar/ultrasonic device (Thunderbeat). The samples were extracted through a 12 mm trocar (Applied Medical) inserted into the left hypochondrium. An experienced pathologist evaluated pathological features such as steatosis, lobular inflammation, hepatocellular ballooning, and fibrosis according to the criteria established by the Nonalcoholic Steatohepatitis Clinical Research Network (@kleiner2005design).

## RNA isolation
Total RNA was extracted from frozen biopsies of subcutaneous white adipose tissue (scWAT) and cell cultures using TRIzol (#T9424, Sigma Aldrich) following the manufacturer's instructions. For lysing adipose tissue samples, 1 ml of TRIzol was added *per* sample in a homogenizer, while for cell cultures, 1 ml of TRIzol was used *per* 10 cm² of the culture plate, along with a scraper. The resulting cell lysates or tissue disaggregates were transferred to a vial and incubated for 5 minutes at room temperature to dissociate nuclear components. Subsequently, 0.2 ml of 100% chloroform *per* ml of TRIzol was added. The mixture was shaken vigorously, incubated for 15 minutes at room temperature, and then centrifuged for 15 minutes at 12,000 g and 4 ºC. 

Following centrifugation, three distinct phases were formed. The aqueous phase, which contained the ribonucleic acids, was collected, and 0.5 ml of 100% isopropanol was added to precipitate the RNA. This mixture was unified, incubated on ice for 10 minutes, and then centrifuged at 12,000 g for 15 minutes at 4 ºC. Afterward, the supernatant was carefully decanted, and the RNA pellet was resuspended in 1 ml of 75% ethanol for washing. The pellet was homogenized and centrifuged at 7,500 g for 5 minutes at 4 ºC. The supernatant was discarded, and the pellet was allowed to dry for 10 minutes at room temperature. Finally, the RNA was resuspended in DEPC-treated water. To eliminate any genomic DNA contamination, all RNA samples were treated with RNase-Free DNase (Life Technologies). The concentration and purity of the RNA were assessed by measuring absorbance at 260/280 nm and 260/230 nm using a Nanodrop 2000 (Thermo Fisher Scientific Inc.) and Qubit 4 Fluorometer (Thermo Fisher Scientific Inc.). The quality of the extracted RNA was visualized using an agarose gel.

## RNA Sequencing
RNA integrity was evaluated using the RNA Integrity Number (RIN) on the Agilent 2200 TapeStation with the RNA ScreenTape assay. Stranded mRNA libraries were prepared with the Novogene NGS RNA Library Prep Set (PT042), which included mRNA isolation using poly-T oligo-attached magnetic beads, cDNA synthesis, adapter ligation, and PCR amplification. Libraries that passed quality control checks were sequenced (2x150 bp) on the Illumina Novaseq X Plus platform.

## Analysis of small RNA-seq Data with *nf-core/smrnaseq*
For the analysis of small RNA sequencing (sRNA-seq) data, version 2.4.0 of the *nf-core/smrnaseq* pipeline [@alexanderpeltzer2024] was used, which is specifically designed for the automated processing of miRNAs data.

### Execution of the *nf-core/smrnaseq* Pipeline
The installation of *nf-core/smrnaseq* was carried out following the instructions provided by the authors in *nf-core* [@ewels2020nf], available at <https://nf-co.re/smrnaseq/2.4.0> .

To ensure the proper installation and execution of the pipeline, the following key components were installed beforehand:

1.  *Nextflow*: Version 24.04.4 of Nextflow was used, following the detailed instructions at <https://nf-co.re/usage/installation>.

2.  *Java Runtime Environment (JRE)*: Version 11.0.25 of the Java Runtime Environment was installed, as it is required for compatibility with Nextflow and the *nf-core/smrnaseq* pipeline.

To ensure reproducibility and streamline the pipeline execution, one of the available Docker containers was utilized. These containers provide the necessary instructions and configurations required to run the pipeline. The configuration is specified at runtime using the `profile` argument. For this analysis, the Docker image *nf-core/smrnaseq*, available at [Docker Hub](https://hub.docker.com/r/nfcore/smrnaseq), was employed.

The pipeline was executed on a server with 8 CPUs, 16 GB of RAM, and a Linux operating system. The following command was used in the terminal, which configures the main options, including the reference genome, input data, and output file location:

```         
nextflow run nf-core/smrnaseq -r 2.4.0 
-profile docker,ci  
--genome GRCh38  
--input '/home/joshoacr13/Documentos/TFM/nfcore-smrnaseq/input/samples.csv' 
--fasta 'https://github.com/nf-core/test-datasets/raw/smrnaseq/reference/genome.fa' 
--mirtrace_species 'hsa' 
--outdir /home/joshoacr13/Documentos/TFM/nfcore-smrnaseq/workdir  
-resume -c /home/joshoacr13/Documentos/TFM/nfcore-smrnaseq/nextflow_memory.config 
--filter_contamination 
```

The pipeline was executed three times to accommodate the large number of samples, processing 26 samples *per* run.

### Description of the Parameters Used

-   `-profile docker,ci`: Runs the pipeline inside a Docker container to ensure reproducibility and sets up a continuous integration (CI) profile.

-   `--genome GRCh38`: Specifies the human genome (version GRCh38) as the reference for sequence mapping.

-   `--input`: Provides the path to the CSV file containing metadata and the paths to the FASTQ files.

-   `--fasta`: URL to the FASTA file of the reference genome.

-   `--mirtrace_species hsa`: Defines the species as Homo sapiens (hsa) for miRNAs analysis with miRTrace.

-   `--outdir`: Sets the working directory for the processed results.

-   `-resume`: Allows continuation of a previous analysis without restarting from the beginning.

-   `-c`: Specifies a custom configuration file (nextflow_memory.config) to adjust resource usage.

-   `--filter_contamination`: Enables the contamination filtering.

### Analysis Workflow and Tools Used
The *nf-core/smrnaseq* pipeline is performed following the next steps:

1.  **Quality Control** An initial quality assessment of the raw reads was conducted using *FastQC* (version 0.12.1) [@andrews2010fastqc]. Additionally, 3' adapter trimming was performed using *fastp* (version 0.23.4) [@chen2018fastp], followed by quality and length filtering. A second quality assessment of the trimmed reads was conducted with *FastQC*.

2.  **miRNA Quality Control**: A more specific quality control process for miRNA sequencing was implemented, whereby samples that failed to meet the minimum quality thresholds established by *miRTrace* (version 1.0.1) [@kang2018mirtrace] were excluded from further analysis. This tool incorporates the following steps to ensure the integrity and reliability of the data:
-   **Verify Read Length Distribution**: The majority of reads fell within the expected range of 18–24 nucleotides, indicative of high-quality small RNA data.
-   **Identify Contaminants**: Potential contaminants such as tRNA, rRNA, and other non-target molecules were flagged.
-   **Taxonomic Classification**: Reads were classified taxonomically to ensure that most sequences originated from the organism of interest (*Homo sapiens*).

3.  **miRNAs Quantification**:
-   **Alignment**: The filtered reads were aligned against mature miRNA sequences in the miRBase database using *Bowtie1* (version 1.3.1)[@langmead2009ultrafast]. Unmapped reads were aligned against "hairpin" sequences to identify miRNA precursors.
-   **Post-Alignment Processing**: *SAMtools* (version 1.16.1) [@10.1093/gigascience/giab008] was used to process the mapping results.

4.  **IsomiR Annotation**: The collapsed reads were processed with *mirtop* (version 0.4.28) [@desvignes2020unification] to identify miRNA variants (isomiRs).

5.  **Analysis and Visualization of Results**: The overall pipeline metrics, encompassing quality assessments, mapping statistics, and expression analysis results, were consolidated and summarized using *MultiQC* (version 1.25.1) [@MultiQC]. Visualization of the results was performed using the *ggplot2* package in R [@ggplot2], which facilitated the creation of clear and informative graphical representations.

## Differential expression analysis according to steatosis using *DEseq2*
For the differential expression analysis, the R statistical software [@rproject], version 4.4.1 (2024-06-14) (<https://cran.r-project.org/>), was used. This analysis was performed using the RStudio integrated development environment (IDE) [@RStudio], version 2023.12.0+369, designed for Ubuntu Jammy (<https://www.rstudio.com/>).

The script used to perform the differential expression analysis is available in the file "`miRNA_steatosis.qmd`" which can be accessed at the following link: <https://github.com/joshoandres13/miRNAs>.

The analysis began by loading essential R packages: *tidyverse* (version 2.0.0)[@tidyverse] for data manipulation, *isomiRs* [@isomiRs] (version 1.32.1) for analyze isomirs and miRNAs from sRNA-seq, *DESeq2* (version 1.44.0) [@deseq] for differential expresion analysis, *org.Hs.eg.db* (version 3.19.1) [@org] for gene annotation.

### Data Preparation
Metadata were imported, and sample identifiers were stablished as row names. The variable *Steatosis* is categorized in four distinct groups. Isomirs count data were subsequently obtained where rows represent the identified isomiRs and columns correspond to the experimental samples.

Using isomirs count data and metadata, an object of class `IsomirDataSeq` was created. This object enables efficient management of information derived from small RNA sequencing studies, streamlining differential expression analyses and facilitating the interpretation of biological findings.

### Filtering and Processing of isomiRs
The filtering process enables the grouping of isomiRs into distinct categories, associating them with a single variant of a miRNA. This grouping is essential for ensuring consistency and accuracy in differential expression analyses. To reduce technical noise and highlight biologically meaningful signals, a stringent filtering criterion was applied: only isomiRs with a minimum of 20 counts in at least 40 samples were retained.

### Differential expression in scWAT
In this study, the *DESeq2* package was used to perform differential expression analysis on scWAT samples with varying degrees of liver steatosis. The *DESeq2* object was configured using the Likelihood Ratio Test (LRT), enabling the analysis to account for all four steatosis categories: \<5% (no steatosis), 5–33% (mild steatosis), 33–66% (moderate steatosis), and \>66% (severe steatosis). This approach allowed for a comprehensive assessment of gene expression changes across the entire spectrum of steatosis progression.

#### Testing for differential expression
The criteria for identifying significant differentially expressed miRNAs across the steatosis categories involved applying a false discovery rate (FDR) cutoff of less than 0.05. Following the identification of significant miRNAs, the expression patterns of the selected miRNAs were further analyzed by representing their normalized counts in boxplots across the four steatosis groups. This approach enabled the observation of expression variations of these miRNAs across different steatosis degrees, offering insights into their potential roles in steatosis progression. Differences between steatosis groups were evaluated using the Kruskal-Wallis test

## Target mRNA Selection and Validation Using *multiMiR*
The selected miRNAs were used for the identification of mRNA targets through the *multiMiR* bioinformatics package [@multiMiR; @microRNAtarget], version 2.4.0 in R. *multiMiR* facilitates a systematic search and annotation of miRNA targets, providing functional analysis to elucidate biological mechanisms. For this analysis, only validated interaction data were utilized.

### Filtering Parameters
The validated target table provided by *multiMiR* was used during the selection process. Key columns included:

1.  **database**: Source database of validated interactions, such as *miRTarBase*, *TarBase*, or *miRecords*.
2.  **mature_mirna_id**: Standard format identifier for the miRNA.
3.  **target_symbol**: Target gene symbol.
4.  **experiment**: Experimental methods used for validation, including luciferase assays, Western blot, or qRT-PCR.
5.  **support_type**: Level of experimental support, such as "Functional MTI" (miRNA-mRNA functional interaction).
6.  **pubmed_id**: References to PubMed articles reporting the interaction.
7.  **type**: Specifies whether the interaction is "validated" or "predicted."

### Selection Criteria
To ensure reliable results, databases were filtered according to update criteria and the following selection parameters:

-   Databases up-to-date at the time of analysis were prioritized (*miRTarBase* and *TarBase*).

-   Only interactions classified as "validated" were included.

-   Interactions backed by robust experimental methods, such as luciferase assays or Western blot, were prioritized.

-   Interactions with functional support ("Functional MTI") and verifiable references in PubMed were selected.

This approach ensured the identification of mRNA targets with high reliability and experimental backing, facilitating the analysis of potential regulatory functions of the studied miRNAs.

### Functional Analysis
To explore the biological functions associated with the validated target genes, an enrichment analysis was performed using the *KEGGREST* package [@KEGG] in R. Gene symbols for validated target genes associated with selected miRNAs were extracted using *multiMiR* previously, with duplicates removed to ensure a unique gene list. The Entrez IDs of these genes were mapped to specific metabolic pathways in KEGG, using parameters that controlled the false discovery rate (FDR) with the Benjamini-Hochberg method and significance cutoff values for `qvalue` and `pvalue` set at 0.05. The results included a bar plot displaying the top 5 significantly enriched KEGG pathways, highlighting their statistical significance and the number of genes associated with each pathway. This analysis provided insights into key metabolic pathways and biological processes involving miRNA-regulated target genes.

## Functional validation of miRNAs
### Cell Culture and Transfections with miRNA Mimics
The human hepatoma HepG2 cell line (American Type Culture Collection, ATCC® HB-8065™; Manassas, VA, USA) was cultured in an incubator at 37 ºC and 5% CO$_2$ using high-glucose Dulbecco’s Modified Eagle Medium L-GlutaMAX (DMEM) (Gibco, Thermo Fisher Scientific Inc., Waltham, MA, USA) supplemented with 10% fetal bovine serum (FBS; Gibco, Thermo Fisher Scientific Inc., Brazil). Experiments were carried out when the cells reached 70-80% confluence.

For the experiments, HepG2 cells were plated at a density of 100,000 cells per well in 12 well-plates for gene expression assays, in DMEM high-glucose (1 g/L) L-GlutaMAX supplemented with 10% FBS.

HepG2 cells were reverse-transfected with Lipofectamine RNAiMAX Reagent (Thermo Fisher Scientific Inc.) and 50 nM of the following mirVana™ synthetic miRNAs (Thermo Fisher Scientific Inc.): a scramble sequence as a negative control (mirVana™ miRNA Mimic, Negative Control #1), negative control (mirVana™ miRNA Inhibitor, Negative Control #1), *miR-144-3p* ($5'-UACAGUAUAGAUGAUGUACU-3'$; assay ID H11051), and *miR-372-3p* ($5'-AAAGUGCUGCGACAUUUGAGCGU-3'$; assay ID MC10165). The miRNA mimics were diluted in Opti-MEM I Reduced Serum Medium (Gibco, Thermo Fisher Scientific Inc.) and added to the wells according to the manufacturer's instructions. Following this, Lipofectamine was added to the wells containing the diluted miRNA mimics and incubated for 15 minutes at room temperature to form miRNA mimic–lipofectamine complexes. HepG2 cells diluted in DMEM with 10% FBS were then plated into the wells. The cells were incubated for 24 hours to assess transfection efficiency. To induce lipid accumulation and simulate steatosis [@gomez2007human], HepG2 cells were treated with oleic acid 24 hours after miRNA mimic transfections, mixed in DMEM GlutaMAX-I supplemented with 10% FBS, at a final concentration of 0.5 mM used to mimic hepatic steatosis, which reflects the physiological range of free fatty acids [@belfort2006placebo].

HepG2 cells were frozen on dry ice and stored at -80 °C until RNA extraction. Cells used for miRNA expression analyses were washed previously with phosphate-buffered saline (PBS) to completely remove potential unabsorbed miRNA mimics.

### Gene Expression Analyses
mRNA expression was evaluated in HepG2 cells transfected with miRNA mimics for 48 h. RNA was reverse transcribed using PrimeScript Reverse Transcriptase (Takara Bio), with 100 ng of RNA utilized for each reaction in a total volume of 10 $\mu$l. The process was carried out using an Applied Biosystems 2720 Thermal Cycler, following this protocol: 10 minutes at 25 °C, 2 hours at 37 °C, and finally, 5 minutes at 85 °C. The cDNA product were amplified using Quantitative-real time PCR (qPCR) in a total reaction volume of 15 $\mu$l with SYBR Select Master Mix (Applied Biosystems), to which 0.5 $\mu$l of gene-specific primers at a concentration of 10 $\mu$M was added. The primers utilized are detailed in [@tbl-primers] qPCR was performed on a StepOnePlus system (Applied Biosystems) with the following protocol: an initial step at 95 °C for 10 minutes, followed by 40 cycles of 15 seconds at 95 °C and 1 minute at 60 °C, concluding with 15 seconds at 95 °C, 1 minute at 60 °C, and a final 15 seconds at 95 °C. The gene $\beta$-actin was used as a housekeeping control to normalize gene expression levels ($\triangle Cq$). Comparisons between gene expression levels after miRNA mimic transfections vs. scramble-sequence-transfected cells (negative control) were established using the the $2 -\triangle\triangle CT$ method, determining relative gene expression [@rao2013improvement].

```{r my-table, echo=FALSE, results='asis', message=FALSE, warning=FALSE}
#| tbl-cap: "Primers designed for qPCR mRNA gene expression analysis"
#| label: tbl-primers
library(kableExtra)
library(knitr)
library(flextable)

primers <- data.frame(
  Gene_Name = c("ACCA", "ATGL (PNPLA2)","DGAT2","FAS","PPARG",   "ACTB-2"),
  Forward_Primer = c("AGGTGCCTAGAGGGTTGAAGA",
                     "TGGAGACTGAGGAGAACAAG",
                     "AGTGGCAATGCTATCATCATCGT",
                     "AAGGACCTGTCTAGGTTTGATGC",
                     "AGATGACAGCGACTTGGCAAT",
                     
                     "ACCGAGCGCGGCTACAG"), 

    Reverse_Primer = c("TCGGCCCTGCTTTACTAGGT",
                     "ATCCCTGCTTGCACATCTCT",
                     "TCTTCTGGACCCATCGGCCCCAGGA",
                     "TGGCTTCATAGGTGACTTCCA",
                     "ACTCAGGGTGGTTCAGCTTC",

                     "CTTAATGTCACGCACGATTTCC"))

knitr::kable(primers, 
             caption =  "Primers designed for qPCR mRNA gene expression analysis",
             col.names = c("Gene Name", "Forward Primer (5'→ 3')", "Reverse Primer (5'→ 3')"),
             format = "latex", 
             booktabs = TRUE,
             longtable = TRUE) %>%
  kable_styling(
    latex_options = c("hold_position", "scale_down"), # Escala automáticamente
    font_size = 10.5) %>%
  row_spec(0, bold = TRUE) %>%
  column_spec(1, italic = TRUE )

cat("
\\vspace{-1em}
\\begin{flushleft}
\\footnotesize{\\textbf{Abbreviations:} 
\\textit{ACCA} (Acetyl-CoA Carboxylase Alpha), 
\\textit{DGAT2} (Diacylglycerol O-Acyltransferase 2), 
\\textit{FAS} (Fatty Acid Synthase), 
\\textit{PNPLA2} (Patatin-like phospholipase domain-containing protein 2), 
\\textit{PPARG} (Peroxisome Proliferator Activated Receptor Gamma).}
\\end{flushleft}
")
```

## Statistical Analysis

The results are expressed as median \[interquartile range\] and number of cases (%). Pairwise group comparisons for continuous variables were calculated using Student’s t-test for variables with a Gaussian distribution and the Mann–Whitney U test for data that do not follow this distribution. Categorical variables were analyzed using the chi-square test.  

The data obtained from the gene expression experiments were analyzed to determine the statistical significance of the differences observed between the experimental groups treated with synthetic miRNAs and the negative controls. The results are presented as the percentage of relative regulation (fold change) with respect to the negative control, expressed as percentages (mean ± standard error of the mean, SEM). The fold change was calculated by dividing the gene expression of each sample by the mean expression in the negative control. A Student's t-test for independent samples was used to evaluate the differences in the expression of the target genes. Statistical significance was set at p < 0.05.
