<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.6.39">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>results</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
/* CSS for syntax highlighting */
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
div.sourceCode { margin: 1em 0; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { display: inline-block; text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
  }
pre.numberSource { margin-left: 3em;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
</style>


<script src="Results_files/libs/clipboard/clipboard.min.js"></script>
<script src="Results_files/libs/quarto-html/quarto.js"></script>
<script src="Results_files/libs/quarto-html/popper.min.js"></script>
<script src="Results_files/libs/quarto-html/tippy.umd.min.js"></script>
<script src="Results_files/libs/quarto-html/anchor.min.js"></script>
<link href="Results_files/libs/quarto-html/tippy.css" rel="stylesheet">
<link href="Results_files/libs/quarto-html/quarto-syntax-highlighting-e26003cea8cd680ca0c55a263523d882.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="Results_files/libs/bootstrap/bootstrap.min.js"></script>
<link href="Results_files/libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="Results_files/libs/bootstrap/bootstrap-afb7ef7cec2bb82ae451d0cdffc16375.min.css" rel="stylesheet" append-hash="true" id="quarto-bootstrap" data-mode="light">
<script src="Results_files/libs/kePrint-0.0.1/kePrint.js"></script>
<link href="Results_files/libs/lightable-0.0.1/lightable.css" rel="stylesheet">


</head>

<body class="fullcontent">

<div id="quarto-content" class="page-columns page-rows-contents page-layout-article">

<main class="content" id="quarto-document-content"><header id="title-block-header" class="quarto-title-block"></header>




<section id="results" class="level1">
<h1>Results</h1>
<section id="phenotypic-characterization" class="level2">
<h2 class="anchored" data-anchor-id="phenotypic-characterization">Phenotypic Characterization</h2>
<p><a href="#tbl-description" class="quarto-xref">Table&nbsp;1</a> describes the phenotypical features of FATe patients selected for this study. This cohort encomprisses 78 subjects (24% males; 76% females), aged between 22 to 61 with obesity as regards of BMI levels of 46.11 ± 6.13 kg/m². These patients had an overall presence of general variables/descriptors of metabolic syndrome features of particular interest those related to liver/hepatic status (steatosis, lobular inflammation and the prevalence of associated metabolic diseases). According to the hepatic fat scale, 35.9% of patients had less than 5% liver fat, indicating a normal or minimal steatosis state. A total of 32.1% exhibited mild steatosis (5–33%), while 25.6% showed moderate fat accumulation (33–66%). Only 6.4% of patients presented severe steatosis (&gt;66%), with no significant differences between men and women (p=0.818).</p>
<div id="tbl-description" class="cell quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-tbl figure">
<figcaption class="quarto-float-caption-top quarto-float-caption quarto-float-tbl" id="tbl-description-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Table&nbsp;1: Clinical characteristics of the FATe cohort and their statistical differences. Data are presented as number (%) or median [interquartile range]. Differences between groups were tested with the Mann–Whitney U test and chi-square test; <em>BMI</em>: Body Mass Index (kg/m²); <em>NASH</em>: Non-alcoholic steatohepatitis.
</figcaption>
<div aria-describedby="tbl-description-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">

</div>
</figure>
</div>
<p>Regarding lobular inflammation, most patients (66.7%) had no inflammatory foci, while 24.4% exhibited fewer than two foci per microscopic field, and 3.8% showed severe inflammation (&gt;4 foci). Hepatocyte ballooning analysis revealed that 74.4% of patients did not display significant damage, although 9.0% showed severe ballooning.</p>
<p>Among most prevalent comorbidities, 26.9% of patients were diagnosed with diabetes, and 34.6% presented hyperlipidemia, with the latter being significantly more prevalent in men (63.2%) compared to women (25.4%, p=0.006). Additionally, 15.4% of patients were classified with non-alcoholic steatohepatitis (NASH), although this proportion showed no statistically significant differences between genders (p=0.249). Collectively, these data highlight the heterogeneity in the clinical characteristics of the cohort, emphasizing the complexity of the relationship between obesity and liver disease in this group of patients.</p>
</section>
<section id="quality-control-qc-and-reads-preprocessing" class="level2">
<h2 class="anchored" data-anchor-id="quality-control-qc-and-reads-preprocessing">Quality Control (QC) and reads preprocessing</h2>
<p>All QC results from the various steps in the <em>nf-core/smrnaseq</em> pipeline are summarized in the following figures and tables. The evaluation of samples, both before and after processing, was conducted using <em>fastp</em>, a rapid tool designed for preprocessing RNA sequencing data. This tool includes features for adapter trimming, quality filtering, and report generation, making it essential for assessing sample quality. Overall, <a href="#tbl-metrics" class="quarto-xref">Table&nbsp;2</a> presents a descriptive analysis that includes key metrics related to the sequencing data.</p>
<div class="cell">
<div class="sourceCode cell-code" id="cb1"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb1-1"><a href="#cb1-1" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(readr)</span>
<span id="cb1-2"><a href="#cb1-2" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(dplyr)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>
Adjuntando el paquete: 'dplyr'</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>The following object is masked from 'package:kableExtra':

    group_rows</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>The following objects are masked from 'package:stats':

    filter, lag</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>The following objects are masked from 'package:base':

    intersect, setdiff, setequal, union</code></pre>
</div>
<div class="sourceCode cell-code" id="cb6"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb6-1"><a href="#cb6-1" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(knitr)</span>
<span id="cb6-2"><a href="#cb6-2" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(tidyr)</span>
<span id="cb6-3"><a href="#cb6-3" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(psych)</span>
<span id="cb6-4"><a href="#cb6-4" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(kableExtra)</span>
<span id="cb6-5"><a href="#cb6-5" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(ggplot2)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>
Adjuntando el paquete: 'ggplot2'</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>The following objects are masked from 'package:psych':

    %+%, alpha</code></pre>
</div>
<div class="sourceCode cell-code" id="cb9"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb9-1"><a href="#cb9-1" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(ggrepel)</span>
<span id="cb9-2"><a href="#cb9-2" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(patchwork)</span>
<span id="cb9-3"><a href="#cb9-3" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(scales)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>
Adjuntando el paquete: 'scales'</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>The following objects are masked from 'package:psych':

    alpha, rescale</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>The following object is masked from 'package:readr':

    col_factor</code></pre>
</div>
<div class="sourceCode cell-code" id="cb13"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb13-1"><a href="#cb13-1" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(gridExtra)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>
Adjuntando el paquete: 'gridExtra'</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>The following object is masked from 'package:dplyr':

    combine</code></pre>
</div>
<div class="sourceCode cell-code" id="cb16"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb16-1"><a href="#cb16-1" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(tidyverse)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>── Attaching core tidyverse packages ──────────────────────── tidyverse 2.0.0 ──
✔ forcats   1.0.0     ✔ stringr   1.5.1
✔ lubridate 1.9.4     ✔ tibble    3.2.1
✔ purrr     1.0.2     </code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>── Conflicts ────────────────────────────────────────── tidyverse_conflicts() ──
✖ ggplot2::%+%()       masks psych::%+%()
✖ scales::alpha()      masks ggplot2::alpha(), psych::alpha()
✖ scales::col_factor() masks readr::col_factor()
✖ gridExtra::combine() masks dplyr::combine()
✖ purrr::discard()     masks scales::discard()
✖ dplyr::filter()      masks stats::filter()
✖ dplyr::group_rows()  masks kableExtra::group_rows()
✖ dplyr::lag()         masks stats::lag()
ℹ Use the conflicted package (&lt;http://conflicted.r-lib.org/&gt;) to force all conflicts to become errors</code></pre>
</div>
<div class="sourceCode cell-code" id="cb19"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb19-1"><a href="#cb19-1" aria-hidden="true" tabindex="-1"></a>file_path_data <span class="ot">&lt;-</span> <span class="st">"data/general_stats_table.tsv"</span>  <span class="co"># Cambia esto por la ruta real</span></span>
<span id="cb19-2"><a href="#cb19-2" aria-hidden="true" tabindex="-1"></a>general_stats_data <span class="ot">&lt;-</span> <span class="fu">read.delim</span>(file_path_data, <span class="at">header =</span> <span class="cn">TRUE</span>, <span class="at">sep =</span> <span class="st">"</span><span class="sc">\t</span><span class="st">"</span>)</span>
<span id="cb19-3"><a href="#cb19-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb19-4"><a href="#cb19-4" aria-hidden="true" tabindex="-1"></a>general_stats_data <span class="ot">&lt;-</span> <span class="fu">read_delim</span>(file_path_data, <span class="at">delim =</span> <span class="st">"</span><span class="sc">\t</span><span class="st">"</span>, <span class="at">col_names =</span> <span class="cn">TRUE</span>, <span class="at">na =</span> <span class="fu">c</span>(<span class="st">"."</span>, <span class="st">"NA"</span>), <span class="at">guess_max =</span> <span class="dv">10000</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>New names:
• `Seqs` -&gt; `Seqs...16`
• `Seqs` -&gt; `Seqs...21`</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>Warning: One or more parsing issues, call `problems()` on your data frame for details,
e.g.:
  dat &lt;- vroom(...)
  problems(dat)</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>Rows: 390 Columns: 21
── Column specification ────────────────────────────────────────────────────────
Delimiter: "\t"
chr  (2): Sample, Seqs...21
dbl (19): % Duplication, Reads After Filtering, GC content, % PF, % Adapter,...

ℹ Use `spec()` to retrieve the full column specification for this data.
ℹ Specify the column types or set `show_col_types = FALSE` to quiet this message.</code></pre>
</div>
<div class="sourceCode cell-code" id="cb23"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb23-1"><a href="#cb23-1" aria-hidden="true" tabindex="-1"></a>filtrados <span class="ot">&lt;-</span> general_stats_data[<span class="fu">grep</span>(<span class="st">"^SC_</span><span class="sc">\\</span><span class="st">d+$"</span>, general_stats_data<span class="sc">$</span>Sample), ]</span>
<span id="cb23-2"><a href="#cb23-2" aria-hidden="true" tabindex="-1"></a><span class="co"># Eliminar columnas sin informacion</span></span>
<span id="cb23-3"><a href="#cb23-3" aria-hidden="true" tabindex="-1"></a>filtrados <span class="ot">&lt;-</span> filtrados <span class="sc">%&gt;%</span> <span class="fu">select</span>(<span class="sc">-</span><span class="st">`</span><span class="at">IsomiR %</span><span class="st">`</span>, <span class="sc">-</span><span class="st">`</span><span class="at">M IsomiR reads</span><span class="st">`</span>,<span class="sc">-</span><span class="st">`</span><span class="at">Error rate</span><span class="st">`</span>,<span class="sc">-</span><span class="st">`</span><span class="at">Non-primary</span><span class="st">`</span>,<span class="sc">-</span><span class="st">`</span><span class="at">Reads mapped</span><span class="st">`</span>,<span class="sc">-</span><span class="st">`</span><span class="at">Total seqs</span><span class="st">`</span>,<span class="sc">-</span><span class="st">`</span><span class="at">% Mapped</span><span class="st">`</span>,<span class="sc">-</span>Seqs...<span class="dv">16</span>,<span class="sc">-</span>Seqs...<span class="dv">21</span>,<span class="sc">-</span><span class="st">`</span><span class="at">M Ref miRNA reads</span><span class="st">`</span>,<span class="sc">-</span><span class="st">`</span><span class="at">M Reads</span><span class="st">`</span>,<span class="sc">-</span><span class="st">`</span><span class="at">Dups</span><span class="st">`</span>,<span class="sc">-</span><span class="st">`</span><span class="at">GC</span><span class="st">`</span>,<span class="sc">-</span><span class="st">`</span><span class="at">Avg len</span><span class="st">`</span>,<span class="sc">-</span><span class="st">`</span><span class="at">Median len</span><span class="st">`</span>)</span>
<span id="cb23-4"><a href="#cb23-4" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb23-5"><a href="#cb23-5" aria-hidden="true" tabindex="-1"></a><span class="co"># Calcular estadísticas descriptivas</span></span>
<span id="cb23-6"><a href="#cb23-6" aria-hidden="true" tabindex="-1"></a>summary_metrics <span class="ot">&lt;-</span> filtrados <span class="sc">%&gt;%</span></span>
<span id="cb23-7"><a href="#cb23-7" aria-hidden="true" tabindex="-1"></a>  <span class="fu">summarise</span>(</span>
<span id="cb23-8"><a href="#cb23-8" aria-hidden="true" tabindex="-1"></a>    <span class="at">Mean_Percent_Duplication =</span> <span class="fu">mean</span>(<span class="st">`</span><span class="at">% Duplication</span><span class="st">`</span>),</span>
<span id="cb23-9"><a href="#cb23-9" aria-hidden="true" tabindex="-1"></a>    <span class="at">Median_Percent_Duplication =</span> <span class="fu">median</span>(<span class="st">`</span><span class="at">% Duplication</span><span class="st">`</span>),</span>
<span id="cb23-10"><a href="#cb23-10" aria-hidden="true" tabindex="-1"></a>    <span class="at">IQR_Percent_Duplication =</span> <span class="fu">IQR</span>(<span class="st">`</span><span class="at">% Duplication</span><span class="st">`</span>),</span>
<span id="cb23-11"><a href="#cb23-11" aria-hidden="true" tabindex="-1"></a>    </span>
<span id="cb23-12"><a href="#cb23-12" aria-hidden="true" tabindex="-1"></a>    <span class="at">Mean_Reads_After_Filtering =</span> <span class="fu">mean</span>(<span class="st">`</span><span class="at">Reads After Filtering</span><span class="st">`</span>),</span>
<span id="cb23-13"><a href="#cb23-13" aria-hidden="true" tabindex="-1"></a>    <span class="at">Median_Reads_After_Filtering =</span> <span class="fu">median</span>(<span class="st">`</span><span class="at">Reads After Filtering</span><span class="st">`</span>),</span>
<span id="cb23-14"><a href="#cb23-14" aria-hidden="true" tabindex="-1"></a>    <span class="at">IQR_Reads_After_Filtering =</span> <span class="fu">IQR</span>(<span class="st">`</span><span class="at">Reads After Filtering</span><span class="st">`</span>),</span>
<span id="cb23-15"><a href="#cb23-15" aria-hidden="true" tabindex="-1"></a>    </span>
<span id="cb23-16"><a href="#cb23-16" aria-hidden="true" tabindex="-1"></a>    <span class="at">Mean_Percent_PF =</span> <span class="fu">mean</span>(<span class="st">`</span><span class="at">% PF</span><span class="st">`</span>),</span>
<span id="cb23-17"><a href="#cb23-17" aria-hidden="true" tabindex="-1"></a>    <span class="at">Median_Percent_PF =</span> <span class="fu">median</span>(<span class="st">`</span><span class="at">% PF</span><span class="st">`</span>),</span>
<span id="cb23-18"><a href="#cb23-18" aria-hidden="true" tabindex="-1"></a>    <span class="at">IQR_Percent_PF =</span> <span class="fu">IQR</span>(<span class="st">`</span><span class="at">% PF</span><span class="st">`</span>),</span>
<span id="cb23-19"><a href="#cb23-19" aria-hidden="true" tabindex="-1"></a>    </span>
<span id="cb23-20"><a href="#cb23-20" aria-hidden="true" tabindex="-1"></a>    <span class="at">Mean_Percent_Adapter =</span> <span class="fu">mean</span>(<span class="st">`</span><span class="at">% Adapter</span><span class="st">`</span>),</span>
<span id="cb23-21"><a href="#cb23-21" aria-hidden="true" tabindex="-1"></a>    <span class="at">Median_Percent_Adapter =</span> <span class="fu">median</span>(<span class="st">`</span><span class="at">% Adapter</span><span class="st">`</span>),</span>
<span id="cb23-22"><a href="#cb23-22" aria-hidden="true" tabindex="-1"></a>    <span class="at">IQR_Percent_Adapter =</span> <span class="fu">IQR</span>(<span class="st">`</span><span class="at">% Adapter</span><span class="st">`</span>),</span>
<span id="cb23-23"><a href="#cb23-23" aria-hidden="true" tabindex="-1"></a>    </span>
<span id="cb23-24"><a href="#cb23-24" aria-hidden="true" tabindex="-1"></a>    <span class="at">Mean_GC_content =</span> <span class="fu">mean</span>(<span class="st">`</span><span class="at">GC content</span><span class="st">`</span>),</span>
<span id="cb23-25"><a href="#cb23-25" aria-hidden="true" tabindex="-1"></a>    <span class="at">Median_GC_content =</span> <span class="fu">median</span>(<span class="st">`</span><span class="at">GC content</span><span class="st">`</span>),</span>
<span id="cb23-26"><a href="#cb23-26" aria-hidden="true" tabindex="-1"></a>    <span class="at">IQR_GC_content =</span> <span class="fu">IQR</span>(<span class="st">`</span><span class="at">GC content</span><span class="st">`</span>)</span>
<span id="cb23-27"><a href="#cb23-27" aria-hidden="true" tabindex="-1"></a>  )</span>
<span id="cb23-28"><a href="#cb23-28" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb23-29"><a href="#cb23-29" aria-hidden="true" tabindex="-1"></a>summary_stats <span class="ot">&lt;-</span> <span class="fu">describe</span>(filtrados[ , <span class="sc">-</span><span class="dv">1</span>])  <span class="co"># Excluir la columna 'Sample' si no es numérica</span></span>
<span id="cb23-30"><a href="#cb23-30" aria-hidden="true" tabindex="-1"></a>summary_stats <span class="ot">&lt;-</span> summary_stats <span class="sc">%&gt;%</span> <span class="fu">select</span>(<span class="sc">-</span>vars, <span class="sc">-</span>se, <span class="sc">-</span>trimmed, <span class="sc">-</span>mad,<span class="sc">-</span>n, <span class="sc">-</span>skew,<span class="sc">-</span>kurtosis)</span>
<span id="cb23-31"><a href="#cb23-31" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb23-32"><a href="#cb23-32" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb23-33"><a href="#cb23-33" aria-hidden="true" tabindex="-1"></a><span class="fu">rownames</span>(summary_stats) <span class="ot">&lt;-</span> <span class="fu">c</span>(<span class="st">"% Duplication"</span>, <span class="st">"Reads After Filtering (M)"</span>, <span class="st">"% GC content"</span>,<span class="st">"% PF"</span>, <span class="st">"% Adapter"</span>)</span>
<span id="cb23-34"><a href="#cb23-34" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb23-35"><a href="#cb23-35" aria-hidden="true" tabindex="-1"></a>summary_stats[, <span class="dv">2</span><span class="sc">:</span><span class="fu">ncol</span>(summary_stats)] <span class="ot">&lt;-</span> <span class="fu">round</span>(summary_stats[, <span class="dv">2</span><span class="sc">:</span><span class="fu">ncol</span>(summary_stats)], <span class="dv">2</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<div class="cell">
<div class="sourceCode cell-code" id="cb24"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb24-1"><a href="#cb24-1" aria-hidden="true" tabindex="-1"></a>knitr<span class="sc">::</span><span class="fu">kable</span>(summary_stats,</span>
<span id="cb24-2"><a href="#cb24-2" aria-hidden="true" tabindex="-1"></a>             <span class="at">col.names =</span> <span class="fu">c</span>(<span class="st">""</span>, <span class="st">"Mean"</span>, </span>
<span id="cb24-3"><a href="#cb24-3" aria-hidden="true" tabindex="-1"></a>                              <span class="st">"sd"</span>, <span class="st">"Median"</span>, </span>
<span id="cb24-4"><a href="#cb24-4" aria-hidden="true" tabindex="-1"></a>                              <span class="st">"Minimum"</span>, <span class="st">"Maximum"</span>, <span class="st">"Range"</span>),)<span class="sc">%&gt;%</span></span>
<span id="cb24-5"><a href="#cb24-5" aria-hidden="true" tabindex="-1"></a>  <span class="fu">kable_styling</span>(</span>
<span id="cb24-6"><a href="#cb24-6" aria-hidden="true" tabindex="-1"></a>    <span class="at">latex_options =</span> <span class="fu">c</span>(<span class="st">"hold_position"</span>, <span class="st">"scale_down"</span>), <span class="co"># Escala automáticamente</span></span>
<span id="cb24-7"><a href="#cb24-7" aria-hidden="true" tabindex="-1"></a>    <span class="at">font_size =</span> <span class="dv">11</span>   <span class="co"># Ajustar tamaño de fuente</span></span>
<span id="cb24-8"><a href="#cb24-8" aria-hidden="true" tabindex="-1"></a>  ) <span class="sc">%&gt;%</span></span>
<span id="cb24-9"><a href="#cb24-9" aria-hidden="true" tabindex="-1"></a>  <span class="fu">row_spec</span>(<span class="dv">0</span>, <span class="at">bold =</span> <span class="cn">TRUE</span>) <span class="co"># Encabezado en negrita</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div id="tbl-metrics" class="cell quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-tbl figure">
<figcaption class="quarto-float-caption-top quarto-float-caption quarto-float-tbl" id="tbl-metrics-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Table&nbsp;2: Descriptive statistics of the analyzed metrics with fastp. <em>% Duplication</em>: Duplication rate before filtering; <em>Reads After Filtering</em>: Total reads after filtering in millions; <em>% GC</em> content: GC content after filtering; <em>% PF</em>: Percent reads passing filter; <em>% Adapter</em>: Percentage adapter-trimmed reads
</figcaption>
<div aria-describedby="tbl-metrics-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<div class="cell-output-display">
<table class="table do-not-create-environment cell caption-top table-sm table-striped small" data-quarto-postprocess="true">
<thead>
<tr class="header">
<th data-quarto-table-cell-role="th" style="text-align: left; font-weight: bold;"></th>
<th data-quarto-table-cell-role="th" style="text-align: right; font-weight: bold;">Mean</th>
<th data-quarto-table-cell-role="th" style="text-align: right; font-weight: bold;">sd</th>
<th data-quarto-table-cell-role="th" style="text-align: right; font-weight: bold;">Median</th>
<th data-quarto-table-cell-role="th" style="text-align: right; font-weight: bold;">Minimum</th>
<th data-quarto-table-cell-role="th" style="text-align: right; font-weight: bold;">Maximum</th>
<th data-quarto-table-cell-role="th" style="text-align: right; font-weight: bold;">Range</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">% Duplication</td>
<td style="text-align: right;">98.30513</td>
<td style="text-align: right;">0.68</td>
<td style="text-align: right;">98.45</td>
<td style="text-align: right;">94.31</td>
<td style="text-align: right;">99.09</td>
<td style="text-align: right;">4.78</td>
</tr>
<tr class="even">
<td style="text-align: left;">Reads After Filtering (M)</td>
<td style="text-align: right;">25.23152</td>
<td style="text-align: right;">5.67</td>
<td style="text-align: right;">25.82</td>
<td style="text-align: right;">1.51</td>
<td style="text-align: right;">35.47</td>
<td style="text-align: right;">33.96</td>
</tr>
<tr class="odd">
<td style="text-align: left;">% GC content</td>
<td style="text-align: right;">46.71982</td>
<td style="text-align: right;">1.68</td>
<td style="text-align: right;">46.43</td>
<td style="text-align: right;">43.26</td>
<td style="text-align: right;">50.89</td>
<td style="text-align: right;">7.63</td>
</tr>
<tr class="even">
<td style="text-align: left;">% PF</td>
<td style="text-align: right;">99.02358</td>
<td style="text-align: right;">1.21</td>
<td style="text-align: right;">99.36</td>
<td style="text-align: right;">90.13</td>
<td style="text-align: right;">99.76</td>
<td style="text-align: right;">9.63</td>
</tr>
<tr class="odd">
<td style="text-align: left;">% Adapter</td>
<td style="text-align: right;">99.38240</td>
<td style="text-align: right;">0.35</td>
<td style="text-align: right;">99.45</td>
<td style="text-align: right;">96.96</td>
<td style="text-align: right;">99.62</td>
<td style="text-align: right;">2.66</td>
</tr>
</tbody>
</table>


</div>
</div>
</figure>
</div>
</div>
<p>The metrics obtained from the fastp analysis provide a detailed overview of the preprocessing performance across the samples. The percentage of duplicated reads before filtering was notably high, with a mean of 98.31% (±0.68), a median of 98.45%, and a range between 94.31% and 99.09%. After filtering, the number of reads retained per sample averaged 25.23 million (±5.67M) suggesting a reasonable amount of reads obtained, with a median of 25.82M and a range from 1.51M to 35.47M.</p>
<p>The GC content showed consistency across the samples, with an average of 46.72% (±1.68), a median of 46.43%, and a range between 43.26% and 50.89%. The percentage of pass-filtered (PF%) reads was high, with an average of 99.02% (±1.21), reaching a maximum of 99.76%, as shown in <a href="#fig-filter_reads" class="quarto-xref">Figure&nbsp;1</a>.</p>
<div class="cell">
<div class="sourceCode cell-code" id="cb25"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb25-1"><a href="#cb25-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Cargar datos</span></span>
<span id="cb25-2"><a href="#cb25-2" aria-hidden="true" tabindex="-1"></a>filter_fastp <span class="ot">&lt;-</span> <span class="fu">read.table</span>(<span class="st">"data/fastp_filtered_reads_plot.tsv"</span>, <span class="at">header =</span> <span class="cn">TRUE</span>, <span class="at">sep =</span> <span class="st">"</span><span class="sc">\t</span><span class="st">"</span>)</span>
<span id="cb25-3"><a href="#cb25-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-4"><a href="#cb25-4" aria-hidden="true" tabindex="-1"></a><span class="co"># Transformación de los datos</span></span>
<span id="cb25-5"><a href="#cb25-5" aria-hidden="true" tabindex="-1"></a>data_fastp <span class="ot">&lt;-</span> filter_fastp <span class="sc">%&gt;%</span></span>
<span id="cb25-6"><a href="#cb25-6" aria-hidden="true" tabindex="-1"></a>  <span class="fu">pivot_longer</span>(<span class="at">cols =</span> <span class="sc">-</span>Sample, <span class="at">names_to =</span> <span class="st">"Category"</span>, <span class="at">values_to =</span> <span class="st">"Reads"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb25-7"><a href="#cb25-7" aria-hidden="true" tabindex="-1"></a>  <span class="fu">group_by</span>(Sample) <span class="sc">%&gt;%</span></span>
<span id="cb25-8"><a href="#cb25-8" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">Proportion =</span> Reads <span class="sc">/</span> <span class="fu">sum</span>(Reads)) <span class="sc">%&gt;%</span>  <span class="co"># Calcular proporciones por muestra</span></span>
<span id="cb25-9"><a href="#cb25-9" aria-hidden="true" tabindex="-1"></a>  <span class="fu">ungroup</span>()</span>
<span id="cb25-10"><a href="#cb25-10" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-11"><a href="#cb25-11" aria-hidden="true" tabindex="-1"></a><span class="co"># Gráfico de barras apiladas al 100%</span></span>
<span id="cb25-12"><a href="#cb25-12" aria-hidden="true" tabindex="-1"></a><span class="fu">ggplot</span>(data_fastp, <span class="fu">aes</span>(<span class="at">x =</span> Sample, <span class="at">y =</span> Proportion, <span class="at">fill =</span> Category)) <span class="sc">+</span></span>
<span id="cb25-13"><a href="#cb25-13" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_bar</span>(<span class="at">stat =</span> <span class="st">"identity"</span>, <span class="at">position =</span> <span class="st">"stack"</span>, <span class="at">alpha =</span> <span class="fl">0.9</span>) <span class="sc">+</span></span>
<span id="cb25-14"><a href="#cb25-14" aria-hidden="true" tabindex="-1"></a>  <span class="fu">labs</span>(</span>
<span id="cb25-15"><a href="#cb25-15" aria-hidden="true" tabindex="-1"></a>    <span class="at">title =</span> <span class="st">""</span>,</span>
<span id="cb25-16"><a href="#cb25-16" aria-hidden="true" tabindex="-1"></a>    <span class="at">x =</span> <span class="st">""</span>,</span>
<span id="cb25-17"><a href="#cb25-17" aria-hidden="true" tabindex="-1"></a>    <span class="at">y =</span> <span class="st">"Reads (%)"</span></span>
<span id="cb25-18"><a href="#cb25-18" aria-hidden="true" tabindex="-1"></a>  ) <span class="sc">+</span></span>
<span id="cb25-19"><a href="#cb25-19" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_y_continuous</span>(<span class="at">labels =</span> scales<span class="sc">::</span>percent) <span class="sc">+</span></span>
<span id="cb25-20"><a href="#cb25-20" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_fill_viridis_d</span>(<span class="at">option =</span> <span class="st">"cividis"</span>, </span>
<span id="cb25-21"><a href="#cb25-21" aria-hidden="true" tabindex="-1"></a>                       <span class="at">labels =</span> <span class="fu">c</span>(</span>
<span id="cb25-22"><a href="#cb25-22" aria-hidden="true" tabindex="-1"></a>                         <span class="st">"Passed.Filter"</span> <span class="ot">=</span> <span class="st">"Passed Filter"</span>,</span>
<span id="cb25-23"><a href="#cb25-23" aria-hidden="true" tabindex="-1"></a>                         <span class="st">"Low.Quality"</span> <span class="ot">=</span> <span class="st">"Low Quality"</span>,</span>
<span id="cb25-24"><a href="#cb25-24" aria-hidden="true" tabindex="-1"></a>                         <span class="st">"Too.Many.N"</span> <span class="ot">=</span> <span class="st">"Too Many N"</span>,</span>
<span id="cb25-25"><a href="#cb25-25" aria-hidden="true" tabindex="-1"></a>                         <span class="st">"Too.Short"</span> <span class="ot">=</span> <span class="st">"Too Short"</span></span>
<span id="cb25-26"><a href="#cb25-26" aria-hidden="true" tabindex="-1"></a>                         ))<span class="sc">+</span></span>
<span id="cb25-27"><a href="#cb25-27" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme_minimal</span>(<span class="at">base_size =</span> <span class="dv">14</span>) <span class="sc">+</span></span>
<span id="cb25-28"><a href="#cb25-28" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme</span>(</span>
<span id="cb25-29"><a href="#cb25-29" aria-hidden="true" tabindex="-1"></a>    <span class="at">axis.text.x =</span> <span class="fu">element_text</span>(<span class="at">angle =</span> <span class="dv">90</span>, <span class="at">hjust =</span> <span class="dv">1</span>, <span class="at">size =</span> <span class="dv">5</span>),</span>
<span id="cb25-30"><a href="#cb25-30" aria-hidden="true" tabindex="-1"></a>    <span class="at">axis.text.y =</span> <span class="fu">element_text</span>(<span class="at">angle =</span> <span class="dv">0</span>, <span class="at">hjust =</span> <span class="dv">1</span>, <span class="at">size =</span> <span class="dv">7</span>),</span>
<span id="cb25-31"><a href="#cb25-31" aria-hidden="true" tabindex="-1"></a>    <span class="at">axis.title =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">8</span>),</span>
<span id="cb25-32"><a href="#cb25-32" aria-hidden="true" tabindex="-1"></a>    <span class="at">plot.title =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">12</span>, <span class="at">face =</span> <span class="st">"bold"</span>),</span>
<span id="cb25-33"><a href="#cb25-33" aria-hidden="true" tabindex="-1"></a>    <span class="at">legend.title =</span> <span class="fu">element_blank</span>(),</span>
<span id="cb25-34"><a href="#cb25-34" aria-hidden="true" tabindex="-1"></a>    <span class="at">legend.text =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">6</span>),</span>
<span id="cb25-35"><a href="#cb25-35" aria-hidden="true" tabindex="-1"></a>    <span class="at">legend.position =</span> <span class="st">"bottom"</span>,</span>
<span id="cb25-36"><a href="#cb25-36" aria-hidden="true" tabindex="-1"></a>    <span class="at">legend.margin =</span> <span class="fu">margin</span>(<span class="at">t =</span> <span class="sc">-</span><span class="dv">20</span>),  <span class="co"># Ajusta el margen superior de la leyenda</span></span>
<span id="cb25-37"><a href="#cb25-37" aria-hidden="true" tabindex="-1"></a>  )</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<div id="fig-filter_reads" class="quarto-float quarto-figure quarto-figure-center anchored" data-fig-pos="H">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-filter_reads-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="Results_files/figure-html/fig-filter_reads-1.png" class="img-fluid figure-img" data-fig-pos="H" width="768">
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-filter_reads-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;1: Fastp: Filtered Reads
</figcaption>
</figure>
</div>
</div>
</div>
<p>The adapters used in library construction, specifically the <em>Illumina Universal Adapter</em> type, are optimized for the amplification and sequencing of diverse sample types. The adapter trimming was highly effective, achieving a mean success rate of 99.38% (±0.35) indicating a low level of adapter contamination in most samples, with a range from 96.96% to 99.62% (<a href="#tbl-metrics" class="quarto-xref">Table&nbsp;2</a>). In line with these findings, <a href="#fig-adaptercontent" class="quarto-xref">Figure&nbsp;2</a> illustrates the adapter content before and after processing with <em>fastp</em>, where the mean trimming percentage of 0.05 ± 0.06% for scWAT samples indicates that adapters were effectively removed from all samples. These results confirm the efficiency of the preprocessing steps, ensuring that high-quality reads were retained while effectively eliminating low-quality sequences and adapters.</p>
<div class="cell">
<div class="sourceCode cell-code" id="cb26"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb26-1"><a href="#cb26-1" aria-hidden="true" tabindex="-1"></a><span class="do">## FastQC Adapter Trimmed ##</span></span>
<span id="cb26-2"><a href="#cb26-2" aria-hidden="true" tabindex="-1"></a><span class="co"># Cargar los datos desde un archivo .tsv</span></span>
<span id="cb26-3"><a href="#cb26-3" aria-hidden="true" tabindex="-1"></a>file_path_trimed <span class="ot">&lt;-</span> <span class="st">"data/fastqc-1_adapter_content.tsv"</span>  <span class="co"># Cambia esto por la ruta real</span></span>
<span id="cb26-4"><a href="#cb26-4" aria-hidden="true" tabindex="-1"></a>data_adapter_trimmed <span class="ot">&lt;-</span> <span class="fu">read.delim</span>(file_path_trimed, <span class="at">header =</span> <span class="cn">TRUE</span>, <span class="at">sep =</span> <span class="st">"</span><span class="sc">\t</span><span class="st">"</span>)</span>
<span id="cb26-5"><a href="#cb26-5" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb26-6"><a href="#cb26-6" aria-hidden="true" tabindex="-1"></a><span class="co"># Renombrar las columnas</span></span>
<span id="cb26-7"><a href="#cb26-7" aria-hidden="true" tabindex="-1"></a><span class="fu">colnames</span>(data_adapter_trimmed) <span class="ot">&lt;-</span> <span class="fu">c</span>(<span class="st">"Sample"</span>, <span class="fu">as.character</span>(<span class="dv">1</span><span class="sc">:</span><span class="dv">39</span>))  <span class="co"># Asumiendo que tus columnas son del 1 al 39</span></span>
<span id="cb26-8"><a href="#cb26-8" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb26-9"><a href="#cb26-9" aria-hidden="true" tabindex="-1"></a><span class="co"># Transformar los datos de formato ancho a formato largo</span></span>
<span id="cb26-10"><a href="#cb26-10" aria-hidden="true" tabindex="-1"></a>data_long_adapter_trimmed <span class="ot">&lt;-</span> <span class="fu">pivot_longer</span>(data_adapter_trimmed, <span class="at">cols =</span> <span class="sc">-</span>Sample, <span class="at">names_to =</span> <span class="st">"bp"</span>, <span class="at">values_to =</span> <span class="st">"Contenido"</span>)</span>
<span id="cb26-11"><a href="#cb26-11" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb26-12"><a href="#cb26-12" aria-hidden="true" tabindex="-1"></a><span class="co"># Convertir el contenido a porcentajes</span></span>
<span id="cb26-13"><a href="#cb26-13" aria-hidden="true" tabindex="-1"></a>data_long_adapter_trimmed<span class="sc">$</span>Contenido <span class="ot">&lt;-</span> data_long_adapter_trimmed<span class="sc">$</span>Contenido  <span class="co"># Convertir a porcentaje</span></span>
<span id="cb26-14"><a href="#cb26-14" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb26-15"><a href="#cb26-15" aria-hidden="true" tabindex="-1"></a><span class="co"># Filtrar solo las posiciones de 1 a 30</span></span>
<span id="cb26-16"><a href="#cb26-16" aria-hidden="true" tabindex="-1"></a>data_long_adapter_trimmed <span class="ot">&lt;-</span>data_long_adapter_trimmed[data_long_adapter_trimmed<span class="sc">$</span>bp <span class="sc">%in%</span> <span class="fu">as.character</span>(<span class="dv">1</span><span class="sc">:</span><span class="dv">39</span>), ]</span>
<span id="cb26-17"><a href="#cb26-17" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb26-18"><a href="#cb26-18" aria-hidden="true" tabindex="-1"></a><span class="co">#mean(data_long_adapter_trimmed$Contenido)</span></span>
<span id="cb26-19"><a href="#cb26-19" aria-hidden="true" tabindex="-1"></a><span class="co">#sd(data_long_adapter_trimmed$Contenido)</span></span>
<span id="cb26-20"><a href="#cb26-20" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb26-21"><a href="#cb26-21" aria-hidden="true" tabindex="-1"></a><span class="co">#summary(data_long_adapter_trimmed$Contenido)</span></span>
<span id="cb26-22"><a href="#cb26-22" aria-hidden="true" tabindex="-1"></a><span class="do">################################################################################</span></span>
<span id="cb26-23"><a href="#cb26-23" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb26-24"><a href="#cb26-24" aria-hidden="true" tabindex="-1"></a><span class="do">### FastQC Adapter Raw </span><span class="al">###</span></span>
<span id="cb26-25"><a href="#cb26-25" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb26-26"><a href="#cb26-26" aria-hidden="true" tabindex="-1"></a><span class="co"># Cargar los datos desde un archivo .tsv</span></span>
<span id="cb26-27"><a href="#cb26-27" aria-hidden="true" tabindex="-1"></a>file_path_raw <span class="ot">&lt;-</span> <span class="st">"data/fastqc_adapter_content.tsv"</span>  <span class="co"># Cambia esto por la ruta real</span></span>
<span id="cb26-28"><a href="#cb26-28" aria-hidden="true" tabindex="-1"></a>data_adapter_raw <span class="ot">&lt;-</span> <span class="fu">read.delim</span>(file_path_raw, <span class="at">header =</span> <span class="cn">TRUE</span>, <span class="at">sep =</span> <span class="st">"</span><span class="sc">\t</span><span class="st">"</span>)</span>
<span id="cb26-29"><a href="#cb26-29" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb26-30"><a href="#cb26-30" aria-hidden="true" tabindex="-1"></a><span class="co"># Renombrar las columnas</span></span>
<span id="cb26-31"><a href="#cb26-31" aria-hidden="true" tabindex="-1"></a><span class="fu">colnames</span>(data_adapter_raw) <span class="ot">&lt;-</span> <span class="fu">c</span>(<span class="st">"Sample"</span>, <span class="fu">as.character</span>(<span class="dv">1</span><span class="sc">:</span><span class="dv">39</span>))  <span class="co"># Asumiendo que tus columnas son del 1 al 39</span></span>
<span id="cb26-32"><a href="#cb26-32" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb26-33"><a href="#cb26-33" aria-hidden="true" tabindex="-1"></a><span class="co"># Transformar los datos de formato ancho a formato largo</span></span>
<span id="cb26-34"><a href="#cb26-34" aria-hidden="true" tabindex="-1"></a>data_long_adapter_raw <span class="ot">&lt;-</span> <span class="fu">pivot_longer</span>(data_adapter_raw, <span class="at">cols =</span> <span class="sc">-</span>Sample, <span class="at">names_to =</span> <span class="st">"bp"</span>, <span class="at">values_to =</span> <span class="st">"Contenido"</span>)</span>
<span id="cb26-35"><a href="#cb26-35" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb26-36"><a href="#cb26-36" aria-hidden="true" tabindex="-1"></a><span class="co"># Convertir el contenido a porcentajes</span></span>
<span id="cb26-37"><a href="#cb26-37" aria-hidden="true" tabindex="-1"></a>data_long_adapter_raw<span class="sc">$</span>Contenido <span class="ot">&lt;-</span> data_long_adapter_raw<span class="sc">$</span>Contenido  <span class="co"># Convertir a porcentaje</span></span>
<span id="cb26-38"><a href="#cb26-38" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb26-39"><a href="#cb26-39" aria-hidden="true" tabindex="-1"></a><span class="co"># Filtrar solo las posiciones de 1 a 30</span></span>
<span id="cb26-40"><a href="#cb26-40" aria-hidden="true" tabindex="-1"></a>data_long_adapter_raw <span class="ot">&lt;-</span>data_long_adapter_raw[data_long_adapter_raw<span class="sc">$</span>bp <span class="sc">%in%</span> <span class="fu">as.character</span>(<span class="dv">1</span><span class="sc">:</span><span class="dv">39</span>), ]</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<div class="cell">
<div class="sourceCode cell-code" id="cb27"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb27-1"><a href="#cb27-1" aria-hidden="true" tabindex="-1"></a><span class="do">### FastQC: Adapter Content (raw)</span></span>
<span id="cb27-2"><a href="#cb27-2" aria-hidden="true" tabindex="-1"></a>p9 <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(data_long_adapter_raw, <span class="fu">aes</span>(<span class="at">x =</span> <span class="fu">as.numeric</span>(bp), <span class="at">y =</span> Contenido, <span class="at">color =</span> Sample, <span class="at">group =</span> Sample)) <span class="sc">+</span></span>
<span id="cb27-3"><a href="#cb27-3" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_line</span>() <span class="sc">+</span></span>
<span id="cb27-4"><a href="#cb27-4" aria-hidden="true" tabindex="-1"></a>  <span class="fu">labs</span>(<span class="at">title =</span> <span class="st">"A"</span>,</span>
<span id="cb27-5"><a href="#cb27-5" aria-hidden="true" tabindex="-1"></a>       <span class="at">subtitle =</span> <span class="st">"Raw"</span>,</span>
<span id="cb27-6"><a href="#cb27-6" aria-hidden="true" tabindex="-1"></a>       <span class="at">x =</span> <span class="st">"Position (bp)"</span>,</span>
<span id="cb27-7"><a href="#cb27-7" aria-hidden="true" tabindex="-1"></a>       <span class="at">y =</span> <span class="st">"Percentage of Sequences"</span>) <span class="sc">+</span></span>
<span id="cb27-8"><a href="#cb27-8" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme_minimal</span>() <span class="sc">+</span></span>
<span id="cb27-9"><a href="#cb27-9" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_color_viridis_d</span>(<span class="at">option =</span> <span class="st">"viridis"</span>) <span class="sc">+</span> </span>
<span id="cb27-10"><a href="#cb27-10" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb27-11"><a href="#cb27-11" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_y_continuous</span>(<span class="at">limits =</span> <span class="fu">c</span>(<span class="dv">0</span>, <span class="dv">100</span>), <span class="at">labels =</span> scales<span class="sc">::</span><span class="fu">percent_format</span>(<span class="at">scale =</span> <span class="dv">1</span>)) <span class="sc">+</span>  <span class="co"># Establecer límites del eje Y hasta 5%</span></span>
<span id="cb27-12"><a href="#cb27-12" aria-hidden="true" tabindex="-1"></a>    <span class="fu">theme</span>(</span>
<span id="cb27-13"><a href="#cb27-13" aria-hidden="true" tabindex="-1"></a>    <span class="at">axis.text.x =</span> <span class="fu">element_text</span>(<span class="at">angle =</span> <span class="dv">0</span>, <span class="at">hjust =</span> <span class="dv">1</span>, <span class="at">size =</span> <span class="dv">9</span>),</span>
<span id="cb27-14"><a href="#cb27-14" aria-hidden="true" tabindex="-1"></a>    <span class="at">axis.text.y =</span> <span class="fu">element_text</span>(<span class="at">angle =</span> <span class="dv">0</span>, <span class="at">hjust =</span> <span class="dv">1</span>, <span class="at">size =</span> <span class="dv">9</span>),</span>
<span id="cb27-15"><a href="#cb27-15" aria-hidden="true" tabindex="-1"></a>    <span class="at">axis.title =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">10</span>),</span>
<span id="cb27-16"><a href="#cb27-16" aria-hidden="true" tabindex="-1"></a>    <span class="at">plot.subtitle =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">10</span>, <span class="at">hjust =</span> <span class="fl">0.5</span> ),</span>
<span id="cb27-17"><a href="#cb27-17" aria-hidden="true" tabindex="-1"></a>    <span class="at">plot.title =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">12</span>, <span class="at">face =</span> <span class="st">"bold"</span>),</span>
<span id="cb27-18"><a href="#cb27-18" aria-hidden="true" tabindex="-1"></a>    <span class="at">plot.margin =</span> <span class="fu">margin</span>(<span class="at">t =</span> <span class="dv">10</span>, <span class="at">r =</span> <span class="dv">10</span>, <span class="at">b =</span> <span class="dv">30</span>, <span class="at">l =</span> <span class="dv">10</span>),</span>
<span id="cb27-19"><a href="#cb27-19" aria-hidden="true" tabindex="-1"></a>    <span class="at">legend.position =</span> <span class="st">"none"</span></span>
<span id="cb27-20"><a href="#cb27-20" aria-hidden="true" tabindex="-1"></a>  ) <span class="co"># Eliminar leyenda # Eliminar leyenda  # Eliminar leyenda</span></span>
<span id="cb27-21"><a href="#cb27-21" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb27-22"><a href="#cb27-22" aria-hidden="true" tabindex="-1"></a><span class="do">### FastQC: Adapter Content (trimmed)</span></span>
<span id="cb27-23"><a href="#cb27-23" aria-hidden="true" tabindex="-1"></a>p10 <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(data_long_adapter_trimmed, <span class="fu">aes</span>(<span class="at">x =</span> <span class="fu">as.numeric</span>(bp), <span class="at">y =</span> Contenido, <span class="at">color =</span> Sample, <span class="at">group =</span> Sample)) <span class="sc">+</span></span>
<span id="cb27-24"><a href="#cb27-24" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_line</span>() <span class="sc">+</span></span>
<span id="cb27-25"><a href="#cb27-25" aria-hidden="true" tabindex="-1"></a>  <span class="fu">labs</span>(<span class="at">title =</span> <span class="st">"B"</span>,</span>
<span id="cb27-26"><a href="#cb27-26" aria-hidden="true" tabindex="-1"></a>       <span class="at">subtitle =</span> <span class="st">"Trimmed"</span>,</span>
<span id="cb27-27"><a href="#cb27-27" aria-hidden="true" tabindex="-1"></a>       <span class="at">x =</span> <span class="st">"Position (bp)"</span>,</span>
<span id="cb27-28"><a href="#cb27-28" aria-hidden="true" tabindex="-1"></a>       <span class="at">y =</span> <span class="st">""</span>) <span class="sc">+</span></span>
<span id="cb27-29"><a href="#cb27-29" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme_minimal</span>() <span class="sc">+</span></span>
<span id="cb27-30"><a href="#cb27-30" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_color_viridis_d</span>(<span class="at">option =</span> <span class="st">""</span>) <span class="sc">+</span>  <span class="co"># Aplicar la paleta Cividis</span></span>
<span id="cb27-31"><a href="#cb27-31" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_y_continuous</span>(<span class="at">limits =</span> <span class="fu">c</span>(<span class="dv">0</span>, <span class="dv">2</span>), <span class="at">labels =</span> scales<span class="sc">::</span><span class="fu">percent_format</span>(<span class="at">scale =</span> <span class="dv">1</span>)) <span class="sc">+</span>  <span class="co"># Establecer límites del eje Y hasta 5%</span></span>
<span id="cb27-32"><a href="#cb27-32" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme</span>(</span>
<span id="cb27-33"><a href="#cb27-33" aria-hidden="true" tabindex="-1"></a>    <span class="at">axis.text.x =</span> <span class="fu">element_text</span>(<span class="at">angle =</span> <span class="dv">0</span>, <span class="at">hjust =</span> <span class="dv">1</span>, <span class="at">size =</span> <span class="dv">10</span>),</span>
<span id="cb27-34"><a href="#cb27-34" aria-hidden="true" tabindex="-1"></a>    <span class="at">axis.text.y =</span> <span class="fu">element_text</span>(<span class="at">angle =</span> <span class="dv">0</span>, <span class="at">hjust =</span> <span class="dv">1</span>, <span class="at">size =</span> <span class="dv">8</span>),</span>
<span id="cb27-35"><a href="#cb27-35" aria-hidden="true" tabindex="-1"></a>    <span class="at">axis.title =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">10</span>),</span>
<span id="cb27-36"><a href="#cb27-36" aria-hidden="true" tabindex="-1"></a>    <span class="at">plot.subtitle =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">10</span>, <span class="at">hjust =</span> <span class="fl">0.5</span> ),</span>
<span id="cb27-37"><a href="#cb27-37" aria-hidden="true" tabindex="-1"></a>    <span class="at">plot.title =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">12</span>, <span class="at">face =</span> <span class="st">"bold"</span>),</span>
<span id="cb27-38"><a href="#cb27-38" aria-hidden="true" tabindex="-1"></a>    <span class="at">plot.margin =</span> <span class="fu">margin</span>(<span class="at">t =</span> <span class="dv">10</span>, <span class="at">r =</span> <span class="dv">10</span>, <span class="at">b =</span> <span class="dv">30</span>, <span class="at">l =</span> <span class="dv">10</span>),</span>
<span id="cb27-39"><a href="#cb27-39" aria-hidden="true" tabindex="-1"></a>    <span class="at">legend.position =</span> <span class="st">"none"</span></span>
<span id="cb27-40"><a href="#cb27-40" aria-hidden="true" tabindex="-1"></a>  ) <span class="co"># Eliminar leyenda # Eliminar leyenda  # Eliminar leyenda</span></span>
<span id="cb27-41"><a href="#cb27-41" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb27-42"><a href="#cb27-42" aria-hidden="true" tabindex="-1"></a><span class="co"># Unir los gráficos en una fila</span></span>
<span id="cb27-43"><a href="#cb27-43" aria-hidden="true" tabindex="-1"></a><span class="fu">grid.arrange</span>(p9, p10, <span class="at">ncol =</span> <span class="dv">2</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>Warning in viridisLite::viridis(n, alpha, begin, end, direction, option):
Option '' does not exist. Defaulting to 'viridis'.</code></pre>
</div>
<div class="cell-output-display">
<div id="fig-adaptercontent" class="quarto-float quarto-figure quarto-figure-center anchored" data-fig-pos="H">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-adaptercontent-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="Results_files/figure-html/fig-adaptercontent-1.png" class="img-fluid figure-img" data-fig-pos="H" width="768">
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-adaptercontent-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;2: Adapters Content (%) across all bases before (A) and after (B) using <em>fastp</em> (version 0.23.4)
</figcaption>
</figure>
</div>
</div>
</div>
<p><a href="#fig-fstqc_quality_control" class="quarto-xref">Figure&nbsp;3</a> shows the analysis of the mean quality value of sequences across all bases, both before (<em>A</em>) and after (<em>B</em>) processing with <em>fastp</em>, it is evident from that the sequences from all samples consistently fall within an acceptable quality range, with mean Phred scores exceeding 30. This indicates a high base-calling accuracy, with an error probability of less than 0.1%, ensuring robust data integrity throughout the process. However, there is a noticeable drop in quality towards the extremes of the readings, particularly at position 31 bp. This decline can be attributed to the effects of adapter trimming in small RNA sequencing (smRNA-seq). Even though adapters have been removed, if they were present at the end of the sequence, their trimming may result in reduced quality for the subsequent bases. Bases near the adapter might have lower quality, leading to a set of bases that, while technically valid, originate from a lower-quality region. Additionally, it is common in smRNA-seq to encounter contaminants from various sources, such as rRNA or tRNA, which introduce background noise and can further diminish the quality of the readings obtained.</p>
<p>The <a href="#fig-fstqc_quality_control" class="quarto-xref">Figure&nbsp;3</a> illustrates the metric of the average quality value of each sequence across all samples. Before processing with <em>fastp</em> (<em>C</em>), the Phred scores consistently exceed 30, with most samples achieving scores around 35. This indicates that the average sequence quality is optimal, ensuring high reliability and low error rates. However, after trimming (<em>D</em>) a slight decrease in both Phred scores and read counts is observed. This reduction in the number of reads is expected after strict preprocessing with <em>fastp</em>, especially if contaminated or low-quality reads were present.</p>
<p>In <a href="#fig-fstqc_quality_control" class="quarto-xref">Figure&nbsp;3</a> (<em>E</em>), prior to trimming, the GC content distribution does not follow a normal pattern. Multiple peaks are observed, likely due to the presence of adapters, which typically have a fixed GC content that generates specific peaks. Additionally, since this is a <em>small RNA-seq</em> experiment, contamination from rRNA or tRNA is common, as these molecules often have a different GC content compared to other small RNAs. After processing with <em>fastp</em> (<em>F</em>), the GC content decreases, indicating that most of these contaminant sequences have been removed, leaving of reads with anomalies. This remaining may correspond to sequences that are difficult to classify or contaminants that are not easily eliminated by <em>fastp</em>.</p>
<p>Additionally, no issues with the presence of ambiguous bases (Ns) are detected in any of the samples before and after trimming. <a href="#fig-fstqc_quality_control" class="quarto-xref">Figure&nbsp;3</a> (<em>G</em> and <em>H</em>).</p>
<div class="cell">
<div class="sourceCode cell-code" id="cb29"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb29-1"><a href="#cb29-1" aria-hidden="true" tabindex="-1"></a><span class="do">### FastQC: Quality Histograms Raw</span></span>
<span id="cb29-2"><a href="#cb29-2" aria-hidden="true" tabindex="-1"></a><span class="co"># Cargar los datos desde un archivo .tsv</span></span>
<span id="cb29-3"><a href="#cb29-3" aria-hidden="true" tabindex="-1"></a>file_path_sq <span class="ot">&lt;-</span> <span class="st">"data/fastqc_per_base_sequence_quality_plot.tsv"</span>  <span class="co"># Cambia esto por la ruta real</span></span>
<span id="cb29-4"><a href="#cb29-4" aria-hidden="true" tabindex="-1"></a>data_sq <span class="ot">&lt;-</span> <span class="fu">read.delim</span>(file_path_sq, <span class="at">header =</span> <span class="cn">TRUE</span>, <span class="at">sep =</span> <span class="st">"</span><span class="sc">\t</span><span class="st">"</span>)</span>
<span id="cb29-5"><a href="#cb29-5" aria-hidden="true" tabindex="-1"></a><span class="co"># Renombrar las columnas</span></span>
<span id="cb29-6"><a href="#cb29-6" aria-hidden="true" tabindex="-1"></a><span class="fu">colnames</span>(data_sq) <span class="ot">&lt;-</span> <span class="fu">c</span>(<span class="st">"Sample"</span>, <span class="fu">as.character</span>(<span class="dv">1</span><span class="sc">:</span>(<span class="fu">ncol</span>(data_sq) <span class="sc">-</span> <span class="dv">1</span>))) </span>
<span id="cb29-7"><a href="#cb29-7" aria-hidden="true" tabindex="-1"></a><span class="co"># Transformar los datos de formato ancho a formato largo</span></span>
<span id="cb29-8"><a href="#cb29-8" aria-hidden="true" tabindex="-1"></a>data_long_sq<span class="ot">&lt;-</span> <span class="fu">pivot_longer</span>(data_sq, <span class="at">cols =</span> <span class="sc">-</span>Sample, <span class="at">names_to =</span> <span class="st">"bp"</span>, <span class="at">values_to =</span> <span class="st">"Contenido"</span>)</span>
<span id="cb29-9"><a href="#cb29-9" aria-hidden="true" tabindex="-1"></a><span class="co"># Convertir el contenido a porcentajes</span></span>
<span id="cb29-10"><a href="#cb29-10" aria-hidden="true" tabindex="-1"></a>data_long_sq<span class="sc">$</span>Contenido <span class="ot">&lt;-</span> data_long_sq<span class="sc">$</span>Contenido  <span class="co"># Convertir a porcentaje</span></span>
<span id="cb29-11"><a href="#cb29-11" aria-hidden="true" tabindex="-1"></a><span class="co"># Filtrar solo las posiciones de 1 a 30</span></span>
<span id="cb29-12"><a href="#cb29-12" aria-hidden="true" tabindex="-1"></a>data_long_sq <span class="ot">&lt;-</span> data_long_sq[data_long_sq<span class="sc">$</span>bp <span class="sc">%in%</span> <span class="fu">as.character</span>(<span class="dv">1</span><span class="sc">:</span><span class="dv">49</span>), ]</span>
<span id="cb29-13"><a href="#cb29-13" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb29-14"><a href="#cb29-14" aria-hidden="true" tabindex="-1"></a><span class="do">################################################################################</span></span>
<span id="cb29-15"><a href="#cb29-15" aria-hidden="true" tabindex="-1"></a><span class="do">### FastQC: Quality Histograms Trimmed</span></span>
<span id="cb29-16"><a href="#cb29-16" aria-hidden="true" tabindex="-1"></a><span class="co"># Cargar los datos desde un archivo .tsv</span></span>
<span id="cb29-17"><a href="#cb29-17" aria-hidden="true" tabindex="-1"></a>file_path_sqt <span class="ot">&lt;-</span> <span class="st">"data/fastqc-1_per_base_sequence_quality_plot.tsv"</span>  <span class="co"># Cambia esto por la ruta real</span></span>
<span id="cb29-18"><a href="#cb29-18" aria-hidden="true" tabindex="-1"></a>data_sqt <span class="ot">&lt;-</span> <span class="fu">read.delim</span>(file_path_sqt, <span class="at">header =</span> <span class="cn">TRUE</span>, <span class="at">sep =</span> <span class="st">"</span><span class="sc">\t</span><span class="st">"</span>)</span>
<span id="cb29-19"><a href="#cb29-19" aria-hidden="true" tabindex="-1"></a><span class="co"># Renombrar las columnas</span></span>
<span id="cb29-20"><a href="#cb29-20" aria-hidden="true" tabindex="-1"></a><span class="fu">colnames</span>(data_sqt) <span class="ot">&lt;-</span> <span class="fu">c</span>(<span class="st">"Sample"</span>, <span class="fu">as.character</span>(<span class="dv">1</span><span class="sc">:</span>(<span class="fu">ncol</span>(data_sqt) <span class="sc">-</span> <span class="dv">1</span>))) </span>
<span id="cb29-21"><a href="#cb29-21" aria-hidden="true" tabindex="-1"></a><span class="co"># Transformar los datos de formato ancho a formato largo</span></span>
<span id="cb29-22"><a href="#cb29-22" aria-hidden="true" tabindex="-1"></a>data_long_sqt <span class="ot">&lt;-</span> <span class="fu">pivot_longer</span>(data_sqt, <span class="at">cols =</span> <span class="sc">-</span>Sample, <span class="at">names_to =</span> <span class="st">"bp"</span>, <span class="at">values_to =</span> <span class="st">"Contenido"</span>)</span>
<span id="cb29-23"><a href="#cb29-23" aria-hidden="true" tabindex="-1"></a><span class="co"># Convertir el contenido a porcentajes</span></span>
<span id="cb29-24"><a href="#cb29-24" aria-hidden="true" tabindex="-1"></a>data_long_sq<span class="sc">$</span>Contenido <span class="ot">&lt;-</span> data_long_sq<span class="sc">$</span>Contenido  <span class="co"># Convertir a porcentaje</span></span>
<span id="cb29-25"><a href="#cb29-25" aria-hidden="true" tabindex="-1"></a><span class="co"># Filtrar solo las posiciones de 1 a 30</span></span>
<span id="cb29-26"><a href="#cb29-26" aria-hidden="true" tabindex="-1"></a>data_long_sqt <span class="ot">&lt;-</span> data_long_sqt[data_long_sqt<span class="sc">$</span>bp <span class="sc">%in%</span> <span class="fu">as.character</span>(<span class="dv">1</span><span class="sc">:</span><span class="dv">49</span>), ]</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<div class="cell">
<div class="sourceCode cell-code" id="cb30"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb30-1"><a href="#cb30-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Primer gráfico: FastQC: Quality Histograms Raw</span></span>
<span id="cb30-2"><a href="#cb30-2" aria-hidden="true" tabindex="-1"></a>p1 <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(data_long_sq, <span class="fu">aes</span>(<span class="at">x =</span> <span class="fu">as.numeric</span>(bp), <span class="at">y =</span> Contenido, <span class="at">color =</span> Sample, <span class="at">group =</span> Sample)) <span class="sc">+</span></span>
<span id="cb30-3"><a href="#cb30-3" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_hline</span>(<span class="at">yintercept =</span> <span class="dv">0</span>, <span class="at">linetype =</span> <span class="st">"dashed"</span>, <span class="at">color =</span> <span class="st">"red"</span>, <span class="at">alpha =</span> <span class="fl">0.8</span>) <span class="sc">+</span></span>
<span id="cb30-4"><a href="#cb30-4" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_hline</span>(<span class="at">yintercept =</span> <span class="dv">20</span>, <span class="at">linetype =</span> <span class="st">"dashed"</span>, <span class="at">color =</span> <span class="st">"orange"</span>, <span class="at">alpha =</span> <span class="fl">0.8</span>) <span class="sc">+</span></span>
<span id="cb30-5"><a href="#cb30-5" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_hline</span>(<span class="at">yintercept =</span> <span class="dv">28</span>, <span class="at">linetype =</span> <span class="st">"dashed"</span>, <span class="at">color =</span> <span class="st">"green"</span>, <span class="at">alpha =</span> <span class="fl">0.8</span>) <span class="sc">+</span></span>
<span id="cb30-6"><a href="#cb30-6" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_hline</span>(<span class="at">yintercept =</span> <span class="dv">38</span>, <span class="at">linetype =</span> <span class="st">"dashed"</span>, <span class="at">color =</span> <span class="st">"green"</span>, <span class="at">alpha =</span> <span class="fl">0.8</span>) <span class="sc">+</span></span>
<span id="cb30-7"><a href="#cb30-7" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_line</span>() <span class="sc">+</span></span>
<span id="cb30-8"><a href="#cb30-8" aria-hidden="true" tabindex="-1"></a>  <span class="fu">labs</span>(<span class="at">title =</span> <span class="st">"A"</span>,</span>
<span id="cb30-9"><a href="#cb30-9" aria-hidden="true" tabindex="-1"></a>       <span class="at">subtitle =</span> <span class="st">"Raw"</span>,</span>
<span id="cb30-10"><a href="#cb30-10" aria-hidden="true" tabindex="-1"></a>       <span class="at">x =</span> <span class="st">"Position (bp)"</span>,</span>
<span id="cb30-11"><a href="#cb30-11" aria-hidden="true" tabindex="-1"></a>       <span class="at">y =</span> <span class="st">"Phred Score"</span>) <span class="sc">+</span></span>
<span id="cb30-12"><a href="#cb30-12" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme_minimal</span>() <span class="sc">+</span></span>
<span id="cb30-13"><a href="#cb30-13" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_y_continuous</span>(<span class="at">limits =</span> <span class="fu">c</span>(<span class="dv">0</span>, <span class="dv">38</span>)) <span class="sc">+</span></span>
<span id="cb30-14"><a href="#cb30-14" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_color_viridis_d</span>(<span class="at">option =</span> <span class="st">"Cividis"</span>) <span class="sc">+</span>  <span class="co"># Aplicar la paleta Cividis</span></span>
<span id="cb30-15"><a href="#cb30-15" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme_minimal</span>(<span class="at">base_size =</span> <span class="dv">14</span>) <span class="sc">+</span></span>
<span id="cb30-16"><a href="#cb30-16" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme</span>(</span>
<span id="cb30-17"><a href="#cb30-17" aria-hidden="true" tabindex="-1"></a>    <span class="at">axis.text.x =</span> <span class="fu">element_text</span>(<span class="at">angle =</span> <span class="dv">0</span>, <span class="at">hjust =</span> <span class="dv">1</span>, <span class="at">size =</span> <span class="dv">9</span>),</span>
<span id="cb30-18"><a href="#cb30-18" aria-hidden="true" tabindex="-1"></a>    <span class="at">axis.text.y =</span> <span class="fu">element_text</span>(<span class="at">angle =</span> <span class="dv">0</span>, <span class="at">hjust =</span> <span class="dv">1</span>, <span class="at">size =</span> <span class="dv">9</span>),</span>
<span id="cb30-19"><a href="#cb30-19" aria-hidden="true" tabindex="-1"></a>    <span class="at">axis.title =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">10</span>),</span>
<span id="cb30-20"><a href="#cb30-20" aria-hidden="true" tabindex="-1"></a>    <span class="at">plot.subtitle =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">10</span>, <span class="at">hjust =</span> <span class="fl">0.5</span> ),</span>
<span id="cb30-21"><a href="#cb30-21" aria-hidden="true" tabindex="-1"></a>    <span class="at">plot.title =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">12</span>, <span class="at">face =</span> <span class="st">"bold"</span>),</span>
<span id="cb30-22"><a href="#cb30-22" aria-hidden="true" tabindex="-1"></a><span class="co">#    plot.margin = margin(t = 10, r = 10, b = 30, l = 10),</span></span>
<span id="cb30-23"><a href="#cb30-23" aria-hidden="true" tabindex="-1"></a>    <span class="at">legend.position =</span> <span class="st">"none"</span></span>
<span id="cb30-24"><a href="#cb30-24" aria-hidden="true" tabindex="-1"></a>  )</span>
<span id="cb30-25"><a href="#cb30-25" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb30-26"><a href="#cb30-26" aria-hidden="true" tabindex="-1"></a><span class="co"># Segundo gráfico: FastQC: Quality Histograms (trimmed)</span></span>
<span id="cb30-27"><a href="#cb30-27" aria-hidden="true" tabindex="-1"></a>p2 <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(data_long_sqt, <span class="fu">aes</span>(<span class="at">x =</span> <span class="fu">as.numeric</span>(bp), <span class="at">y =</span> Contenido, <span class="at">color =</span> Sample, <span class="at">group =</span> Sample)) <span class="sc">+</span></span>
<span id="cb30-28"><a href="#cb30-28" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_hline</span>(<span class="at">yintercept =</span> <span class="dv">0</span>, <span class="at">linetype =</span> <span class="st">"dashed"</span>, <span class="at">color =</span> <span class="st">"red"</span>, <span class="at">alpha =</span> <span class="fl">0.8</span>) <span class="sc">+</span></span>
<span id="cb30-29"><a href="#cb30-29" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_hline</span>(<span class="at">yintercept =</span> <span class="dv">20</span>, <span class="at">linetype =</span> <span class="st">"dashed"</span>, <span class="at">color =</span> <span class="st">"orange"</span>, <span class="at">alpha =</span> <span class="fl">0.8</span>) <span class="sc">+</span></span>
<span id="cb30-30"><a href="#cb30-30" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_hline</span>(<span class="at">yintercept =</span> <span class="dv">28</span>, <span class="at">linetype =</span> <span class="st">"dashed"</span>, <span class="at">color =</span> <span class="st">"green"</span>, <span class="at">alpha =</span> <span class="fl">0.8</span>) <span class="sc">+</span></span>
<span id="cb30-31"><a href="#cb30-31" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_hline</span>(<span class="at">yintercept =</span> <span class="dv">38</span>, <span class="at">linetype =</span> <span class="st">"dashed"</span>, <span class="at">color =</span> <span class="st">"green"</span>, <span class="at">alpha =</span> <span class="fl">0.8</span>) <span class="sc">+</span></span>
<span id="cb30-32"><a href="#cb30-32" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_line</span>() <span class="sc">+</span></span>
<span id="cb30-33"><a href="#cb30-33" aria-hidden="true" tabindex="-1"></a>  <span class="fu">labs</span>(<span class="at">title =</span> <span class="st">"B"</span>,</span>
<span id="cb30-34"><a href="#cb30-34" aria-hidden="true" tabindex="-1"></a>       <span class="at">subtitle =</span> <span class="st">"Trimmed"</span>,</span>
<span id="cb30-35"><a href="#cb30-35" aria-hidden="true" tabindex="-1"></a>       <span class="at">x =</span> <span class="st">"Position (bp)"</span>,</span>
<span id="cb30-36"><a href="#cb30-36" aria-hidden="true" tabindex="-1"></a>       <span class="at">y =</span> <span class="st">""</span>) <span class="sc">+</span></span>
<span id="cb30-37"><a href="#cb30-37" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme_minimal</span>() <span class="sc">+</span></span>
<span id="cb30-38"><a href="#cb30-38" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_y_continuous</span>(<span class="at">limits =</span> <span class="fu">c</span>(<span class="dv">0</span>, <span class="dv">38</span>)) <span class="sc">+</span></span>
<span id="cb30-39"><a href="#cb30-39" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_color_viridis_d</span>(<span class="at">option =</span> <span class="st">"Cividis"</span>) <span class="sc">+</span>  <span class="co"># Aplicar la paleta Cividis</span></span>
<span id="cb30-40"><a href="#cb30-40" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme</span>(</span>
<span id="cb30-41"><a href="#cb30-41" aria-hidden="true" tabindex="-1"></a>    <span class="at">axis.text.x =</span> <span class="fu">element_text</span>(<span class="at">angle =</span> <span class="dv">0</span>, <span class="at">hjust =</span> <span class="dv">1</span>, <span class="at">size =</span> <span class="dv">9</span>),</span>
<span id="cb30-42"><a href="#cb30-42" aria-hidden="true" tabindex="-1"></a>    <span class="at">axis.text.y =</span> <span class="fu">element_text</span>(<span class="at">angle =</span> <span class="dv">0</span>, <span class="at">hjust =</span> <span class="dv">1</span>, <span class="at">size =</span> <span class="dv">9</span>),</span>
<span id="cb30-43"><a href="#cb30-43" aria-hidden="true" tabindex="-1"></a>    <span class="at">axis.title =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">10</span>),</span>
<span id="cb30-44"><a href="#cb30-44" aria-hidden="true" tabindex="-1"></a>    <span class="at">plot.subtitle =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">10</span>, <span class="at">hjust =</span> <span class="fl">0.5</span> ),</span>
<span id="cb30-45"><a href="#cb30-45" aria-hidden="true" tabindex="-1"></a>    <span class="at">plot.title =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">12</span>, <span class="at">face =</span> <span class="st">"bold"</span>),</span>
<span id="cb30-46"><a href="#cb30-46" aria-hidden="true" tabindex="-1"></a><span class="co">#    plot.margin = margin(t = 10, r = 10, b = 30, l = 10),</span></span>
<span id="cb30-47"><a href="#cb30-47" aria-hidden="true" tabindex="-1"></a>    <span class="at">legend.position =</span> <span class="st">"none"</span></span>
<span id="cb30-48"><a href="#cb30-48" aria-hidden="true" tabindex="-1"></a>  )</span>
<span id="cb30-49"><a href="#cb30-49" aria-hidden="true" tabindex="-1"></a><span class="co"># Unir los gráficos en una fila</span></span>
<span id="cb30-50"><a href="#cb30-50" aria-hidden="true" tabindex="-1"></a><span class="co">#grid.arrange(p1, p2, ncol = 2)</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<div class="cell">
<div class="sourceCode cell-code" id="cb31"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb31-1"><a href="#cb31-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Cargar los datos desde un archivo .tsv</span></span>
<span id="cb31-2"><a href="#cb31-2" aria-hidden="true" tabindex="-1"></a>file_path_psq <span class="ot">&lt;-</span> <span class="st">"data/fastqc_per_sequence_quality_scores_plot.txt"</span>  <span class="co"># Cambia esto por la ruta real</span></span>
<span id="cb31-3"><a href="#cb31-3" aria-hidden="true" tabindex="-1"></a>data_psq <span class="ot">&lt;-</span> <span class="fu">read.delim</span>(file_path_psq, <span class="at">header =</span> <span class="cn">TRUE</span>, <span class="at">sep =</span> <span class="st">"</span><span class="sc">\t</span><span class="st">"</span>)</span>
<span id="cb31-4"><a href="#cb31-4" aria-hidden="true" tabindex="-1"></a>  </span>
<span id="cb31-5"><a href="#cb31-5" aria-hidden="true" tabindex="-1"></a>file_path_psqs <span class="ot">&lt;-</span> <span class="st">"data/fastqc-1_per_sequence_quality_scores_plot.txt"</span>  <span class="co"># Cambia esto por la ruta real</span></span>
<span id="cb31-6"><a href="#cb31-6" aria-hidden="true" tabindex="-1"></a>data_psqs <span class="ot">&lt;-</span> <span class="fu">read.delim</span>(file_path_psqs, <span class="at">header =</span> <span class="cn">TRUE</span>, <span class="at">sep =</span> <span class="st">"</span><span class="sc">\t</span><span class="st">"</span>)</span>
<span id="cb31-7"><a href="#cb31-7" aria-hidden="true" tabindex="-1"></a>  </span>
<span id="cb31-8"><a href="#cb31-8" aria-hidden="true" tabindex="-1"></a><span class="do">################################################################################</span></span>
<span id="cb31-9"><a href="#cb31-9" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb31-10"><a href="#cb31-10" aria-hidden="true" tabindex="-1"></a>long_data_psq <span class="ot">&lt;-</span> data_psq <span class="sc">%&gt;%</span></span>
<span id="cb31-11"><a href="#cb31-11" aria-hidden="true" tabindex="-1"></a><span class="fu">pivot_longer</span>(<span class="at">cols =</span> <span class="sc">-</span>Sample, <span class="at">names_to =</span> <span class="st">"Index"</span>, <span class="at">values_to =</span> <span class="st">"Pair"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb31-12"><a href="#cb31-12" aria-hidden="true" tabindex="-1"></a><span class="fu">mutate</span>(<span class="at">Pair =</span> <span class="fu">str_remove_all</span>(Pair, <span class="st">"[()]"</span>)) <span class="sc">%&gt;%</span> <span class="co"># Eliminar paréntesis</span></span>
<span id="cb31-13"><a href="#cb31-13" aria-hidden="true" tabindex="-1"></a><span class="fu">separate</span>(Pair, <span class="at">into =</span> <span class="fu">c</span>(<span class="st">"X"</span>, <span class="st">"Y"</span>), <span class="at">sep =</span> <span class="st">","</span>, <span class="at">convert =</span> <span class="cn">TRUE</span>) <span class="co"># Dividir en X e Y</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>Warning: Expected 2 pieces. Missing pieces filled with `NA` in 40 rows [21, 22, 23, 24,
48, 72, 118, 119, 120, 192, 215, 216, 261, 262, 263, 264, 312, 333, 334, 335,
...].</code></pre>
</div>
<div class="sourceCode cell-code" id="cb33"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb33-1"><a href="#cb33-1" aria-hidden="true" tabindex="-1"></a>long_data_psqs <span class="ot">&lt;-</span> data_psqs <span class="sc">%&gt;%</span></span>
<span id="cb33-2"><a href="#cb33-2" aria-hidden="true" tabindex="-1"></a><span class="fu">pivot_longer</span>(<span class="at">cols =</span> <span class="sc">-</span>Sample, <span class="at">names_to =</span> <span class="st">"Index"</span>, <span class="at">values_to =</span> <span class="st">"Pair"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb33-3"><a href="#cb33-3" aria-hidden="true" tabindex="-1"></a><span class="fu">mutate</span>(<span class="at">Pair =</span> <span class="fu">str_remove_all</span>(Pair, <span class="st">"[()]"</span>)) <span class="sc">%&gt;%</span> <span class="co"># Eliminar paréntesis</span></span>
<span id="cb33-4"><a href="#cb33-4" aria-hidden="true" tabindex="-1"></a><span class="fu">separate</span>(Pair, <span class="at">into =</span> <span class="fu">c</span>(<span class="st">"X"</span>, <span class="st">"Y"</span>), <span class="at">sep =</span> <span class="st">","</span>, <span class="at">convert =</span> <span class="cn">TRUE</span>) <span class="co"># Dividir en X e Y</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>Warning: Expected 2 pieces. Missing pieces filled with `NA` in 116 rows [17, 18, 36, 54,
71, 72, 90, 107, 108, 125, 126, 144, 161, 162, 179, 180, 197, 198, 215, 216,
...].</code></pre>
</div>
</div>
<div class="cell">
<div class="sourceCode cell-code" id="cb35"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb35-1"><a href="#cb35-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Primer gráfico</span></span>
<span id="cb35-2"><a href="#cb35-2" aria-hidden="true" tabindex="-1"></a>p3 <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(long_data_psq, <span class="fu">aes</span>(<span class="at">x =</span> <span class="fu">as.numeric</span>(X), <span class="at">y =</span> <span class="fu">as.numeric</span>(Y), <span class="at">color =</span> Sample)) <span class="sc">+</span></span>
<span id="cb35-3"><a href="#cb35-3" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_vline</span>(<span class="at">xintercept =</span> <span class="dv">0</span>, <span class="at">linetype =</span> <span class="st">"dashed"</span>, <span class="at">color =</span> <span class="st">"red"</span>, <span class="at">alpha =</span> <span class="fl">0.8</span>) <span class="sc">+</span></span>
<span id="cb35-4"><a href="#cb35-4" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_vline</span>(<span class="at">xintercept =</span> <span class="dv">20</span>, <span class="at">linetype =</span> <span class="st">"dashed"</span>, <span class="at">color =</span> <span class="st">"orange"</span>, <span class="at">alpha =</span> <span class="fl">0.8</span>) <span class="sc">+</span></span>
<span id="cb35-5"><a href="#cb35-5" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_vline</span>(<span class="at">xintercept =</span> <span class="dv">27</span>, <span class="at">linetype =</span> <span class="st">"dashed"</span>, <span class="at">color =</span> <span class="st">"green"</span>, <span class="at">alpha =</span> <span class="fl">0.8</span>) <span class="sc">+</span></span>
<span id="cb35-6"><a href="#cb35-6" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_line</span>() <span class="sc">+</span></span>
<span id="cb35-7"><a href="#cb35-7" aria-hidden="true" tabindex="-1"></a>  <span class="fu">labs</span>(<span class="at">title =</span> <span class="st">"C"</span>,</span>
<span id="cb35-8"><a href="#cb35-8" aria-hidden="true" tabindex="-1"></a>       <span class="at">subtitle =</span> <span class="st">"Raw"</span>,</span>
<span id="cb35-9"><a href="#cb35-9" aria-hidden="true" tabindex="-1"></a>       <span class="at">x =</span> <span class="st">"Mean Sequence Quality (Phred Score)"</span>,</span>
<span id="cb35-10"><a href="#cb35-10" aria-hidden="true" tabindex="-1"></a>       <span class="at">y =</span> <span class="st">"Count"</span>) <span class="sc">+</span></span>
<span id="cb35-11"><a href="#cb35-11" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_y_continuous</span>(<span class="at">labels =</span> <span class="fu">label_number</span>(<span class="at">scale =</span> <span class="fl">1e-6</span>, <span class="at">suffix =</span> <span class="st">"M"</span>)) <span class="sc">+</span> <span class="co"># Escalar a millones</span></span>
<span id="cb35-12"><a href="#cb35-12" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme_minimal</span>() <span class="sc">+</span></span>
<span id="cb35-13"><a href="#cb35-13" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_color_viridis_d</span>(<span class="at">option =</span> <span class="st">"Cividis"</span>) <span class="sc">+</span>  <span class="co"># Aplicar la paleta Cividis</span></span>
<span id="cb35-14"><a href="#cb35-14" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme</span>(</span>
<span id="cb35-15"><a href="#cb35-15" aria-hidden="true" tabindex="-1"></a>    <span class="at">axis.text.x =</span> <span class="fu">element_text</span>(<span class="at">angle =</span> <span class="dv">0</span>, <span class="at">hjust =</span> <span class="dv">1</span>, <span class="at">size =</span> <span class="dv">9</span>),</span>
<span id="cb35-16"><a href="#cb35-16" aria-hidden="true" tabindex="-1"></a>    <span class="at">axis.text.y =</span> <span class="fu">element_text</span>(<span class="at">angle =</span> <span class="dv">0</span>, <span class="at">hjust =</span> <span class="dv">1</span>, <span class="at">size =</span> <span class="dv">9</span>),</span>
<span id="cb35-17"><a href="#cb35-17" aria-hidden="true" tabindex="-1"></a>    <span class="at">axis.title =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">10</span>),</span>
<span id="cb35-18"><a href="#cb35-18" aria-hidden="true" tabindex="-1"></a>    <span class="at">plot.subtitle =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">10</span>, <span class="at">hjust =</span> <span class="fl">0.5</span> ),</span>
<span id="cb35-19"><a href="#cb35-19" aria-hidden="true" tabindex="-1"></a>    <span class="at">plot.title =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">12</span>, <span class="at">face =</span> <span class="st">"bold"</span>),</span>
<span id="cb35-20"><a href="#cb35-20" aria-hidden="true" tabindex="-1"></a><span class="co">#    plot.margin = margin(t = 10, r = 10, b = 30, l = 10),</span></span>
<span id="cb35-21"><a href="#cb35-21" aria-hidden="true" tabindex="-1"></a>    <span class="at">legend.position =</span> <span class="st">"none"</span></span>
<span id="cb35-22"><a href="#cb35-22" aria-hidden="true" tabindex="-1"></a>  )</span>
<span id="cb35-23"><a href="#cb35-23" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-24"><a href="#cb35-24" aria-hidden="true" tabindex="-1"></a><span class="co"># Segundo gráfico</span></span>
<span id="cb35-25"><a href="#cb35-25" aria-hidden="true" tabindex="-1"></a>p4 <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(long_data_psqs, <span class="fu">aes</span>(<span class="at">x =</span> <span class="fu">as.numeric</span>(X), <span class="at">y =</span> <span class="fu">as.numeric</span>(Y), <span class="at">color =</span> Sample)) <span class="sc">+</span></span>
<span id="cb35-26"><a href="#cb35-26" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_vline</span>(<span class="at">xintercept =</span> <span class="dv">0</span>, <span class="at">linetype =</span> <span class="st">"dashed"</span>, <span class="at">color =</span> <span class="st">"red"</span>, <span class="at">alpha =</span> <span class="fl">0.8</span>) <span class="sc">+</span></span>
<span id="cb35-27"><a href="#cb35-27" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_vline</span>(<span class="at">xintercept =</span> <span class="dv">20</span>, <span class="at">linetype =</span> <span class="st">"dashed"</span>, <span class="at">color =</span> <span class="st">"orange"</span>, <span class="at">alpha =</span> <span class="fl">0.8</span>) <span class="sc">+</span></span>
<span id="cb35-28"><a href="#cb35-28" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_vline</span>(<span class="at">xintercept =</span> <span class="dv">27</span>, <span class="at">linetype =</span> <span class="st">"dashed"</span>, <span class="at">color =</span> <span class="st">"green"</span>, <span class="at">alpha =</span> <span class="fl">0.8</span>) <span class="sc">+</span></span>
<span id="cb35-29"><a href="#cb35-29" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_line</span>() <span class="sc">+</span></span>
<span id="cb35-30"><a href="#cb35-30" aria-hidden="true" tabindex="-1"></a>  <span class="fu">labs</span>(<span class="at">title =</span> <span class="st">"D"</span>,</span>
<span id="cb35-31"><a href="#cb35-31" aria-hidden="true" tabindex="-1"></a>       <span class="at">subtitle =</span> <span class="st">"Trimmed"</span>,</span>
<span id="cb35-32"><a href="#cb35-32" aria-hidden="true" tabindex="-1"></a>       <span class="at">x =</span> <span class="st">"Mean Sequence Quality (Phred Score)"</span>,</span>
<span id="cb35-33"><a href="#cb35-33" aria-hidden="true" tabindex="-1"></a>       <span class="at">y =</span> <span class="st">""</span>) <span class="sc">+</span></span>
<span id="cb35-34"><a href="#cb35-34" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_y_continuous</span>(<span class="at">labels =</span> <span class="fu">label_number</span>(<span class="at">scale =</span> <span class="fl">1e-6</span>, <span class="at">suffix =</span> <span class="st">"M"</span>)) <span class="sc">+</span> <span class="co"># Escalar a millones</span></span>
<span id="cb35-35"><a href="#cb35-35" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme_minimal</span>() <span class="sc">+</span> </span>
<span id="cb35-36"><a href="#cb35-36" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_color_viridis_d</span>(<span class="at">option =</span> <span class="st">"Cividis"</span>) <span class="sc">+</span>  <span class="co"># Aplicar la paleta Cividis</span></span>
<span id="cb35-37"><a href="#cb35-37" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme</span>(</span>
<span id="cb35-38"><a href="#cb35-38" aria-hidden="true" tabindex="-1"></a>    <span class="at">axis.text.x =</span> <span class="fu">element_text</span>(<span class="at">angle =</span> <span class="dv">0</span>, <span class="at">hjust =</span> <span class="dv">1</span>, <span class="at">size =</span> <span class="dv">9</span>),</span>
<span id="cb35-39"><a href="#cb35-39" aria-hidden="true" tabindex="-1"></a>    <span class="at">axis.text.y =</span> <span class="fu">element_text</span>(<span class="at">angle =</span> <span class="dv">0</span>, <span class="at">hjust =</span> <span class="dv">1</span>, <span class="at">size =</span> <span class="dv">9</span>),</span>
<span id="cb35-40"><a href="#cb35-40" aria-hidden="true" tabindex="-1"></a>    <span class="at">axis.title =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">10</span>),</span>
<span id="cb35-41"><a href="#cb35-41" aria-hidden="true" tabindex="-1"></a>    <span class="at">plot.subtitle =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">10</span>, <span class="at">hjust =</span> <span class="fl">0.5</span> ),</span>
<span id="cb35-42"><a href="#cb35-42" aria-hidden="true" tabindex="-1"></a>    <span class="at">plot.title =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">12</span>, <span class="at">face =</span> <span class="st">"bold"</span>),</span>
<span id="cb35-43"><a href="#cb35-43" aria-hidden="true" tabindex="-1"></a><span class="co">#    plot.margin = margin(t = 10, r = 10, b = 30, l = 10),</span></span>
<span id="cb35-44"><a href="#cb35-44" aria-hidden="true" tabindex="-1"></a>    <span class="at">legend.position =</span> <span class="st">"none"</span></span>
<span id="cb35-45"><a href="#cb35-45" aria-hidden="true" tabindex="-1"></a>  )</span>
<span id="cb35-46"><a href="#cb35-46" aria-hidden="true" tabindex="-1"></a><span class="co"># Unir los gráficos en una fila</span></span>
<span id="cb35-47"><a href="#cb35-47" aria-hidden="true" tabindex="-1"></a><span class="co">#grid.arrange(p3, p4, ncol = 2)</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<div class="cell">
<div class="sourceCode cell-code" id="cb36"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb36-1"><a href="#cb36-1" aria-hidden="true" tabindex="-1"></a><span class="do">### FastQC: Per Sequence GC Content Raw</span></span>
<span id="cb36-2"><a href="#cb36-2" aria-hidden="true" tabindex="-1"></a><span class="co"># Cargar los datos desde un archivo .tsv</span></span>
<span id="cb36-3"><a href="#cb36-3" aria-hidden="true" tabindex="-1"></a>file_path_gc <span class="ot">&lt;-</span> <span class="st">"data/fastqc_per_sequence_gc_content_plot.tsv"</span>  <span class="co"># Cambia esto por la ruta real</span></span>
<span id="cb36-4"><a href="#cb36-4" aria-hidden="true" tabindex="-1"></a>data_gc <span class="ot">&lt;-</span> <span class="fu">read.delim</span>(file_path_gc, <span class="at">header =</span> <span class="cn">TRUE</span>, <span class="at">sep =</span> <span class="st">"</span><span class="sc">\t</span><span class="st">"</span>)</span>
<span id="cb36-5"><a href="#cb36-5" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb36-6"><a href="#cb36-6" aria-hidden="true" tabindex="-1"></a><span class="co"># Renombrar las columnas</span></span>
<span id="cb36-7"><a href="#cb36-7" aria-hidden="true" tabindex="-1"></a><span class="fu">colnames</span>(data_gc) <span class="ot">&lt;-</span> <span class="fu">c</span>(<span class="st">"Sample"</span>, <span class="fu">as.character</span>(<span class="dv">1</span><span class="sc">:</span>(<span class="fu">ncol</span>(data_gc) <span class="sc">-</span> <span class="dv">1</span>))) </span>
<span id="cb36-8"><a href="#cb36-8" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb36-9"><a href="#cb36-9" aria-hidden="true" tabindex="-1"></a><span class="co"># Transformar los datos de formato ancho a formato largo</span></span>
<span id="cb36-10"><a href="#cb36-10" aria-hidden="true" tabindex="-1"></a>data_long_gc <span class="ot">&lt;-</span> <span class="fu">pivot_longer</span>(data_gc, <span class="at">cols =</span> <span class="sc">-</span>Sample, <span class="at">names_to =</span> <span class="st">"bp"</span>, <span class="at">values_to =</span> <span class="st">"Contenido"</span>)</span>
<span id="cb36-11"><a href="#cb36-11" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb36-12"><a href="#cb36-12" aria-hidden="true" tabindex="-1"></a><span class="co"># Convertir el contenido a porcentajes</span></span>
<span id="cb36-13"><a href="#cb36-13" aria-hidden="true" tabindex="-1"></a>data_long_gc<span class="sc">$</span>Contenido <span class="ot">&lt;-</span> data_long_gc<span class="sc">$</span>Contenido  <span class="co"># Convertir a porcentaje</span></span>
<span id="cb36-14"><a href="#cb36-14" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb36-15"><a href="#cb36-15" aria-hidden="true" tabindex="-1"></a><span class="co"># Filtrar solo las posiciones de 1 a 30</span></span>
<span id="cb36-16"><a href="#cb36-16" aria-hidden="true" tabindex="-1"></a>data_long_gc <span class="ot">&lt;-</span> data_long_gc[data_long_gc<span class="sc">$</span>bp <span class="sc">%in%</span> <span class="fu">as.character</span>(<span class="dv">1</span><span class="sc">:</span><span class="dv">101</span>), ]</span>
<span id="cb36-17"><a href="#cb36-17" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb36-18"><a href="#cb36-18" aria-hidden="true" tabindex="-1"></a><span class="do">################################################################################</span></span>
<span id="cb36-19"><a href="#cb36-19" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb36-20"><a href="#cb36-20" aria-hidden="true" tabindex="-1"></a><span class="do">### FastQC: QPer Sequence GC Content Trimmed</span></span>
<span id="cb36-21"><a href="#cb36-21" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb36-22"><a href="#cb36-22" aria-hidden="true" tabindex="-1"></a><span class="co"># Cargar los datos desde un archivo .tsv</span></span>
<span id="cb36-23"><a href="#cb36-23" aria-hidden="true" tabindex="-1"></a>file_path_gct <span class="ot">&lt;-</span> <span class="st">"data/fastqc-1_per_sequence_gc_content_plot.tsv"</span>  <span class="co"># Cambia esto por la ruta real</span></span>
<span id="cb36-24"><a href="#cb36-24" aria-hidden="true" tabindex="-1"></a>data_gct <span class="ot">&lt;-</span> <span class="fu">read.delim</span>(file_path_gct, <span class="at">header =</span> <span class="cn">TRUE</span>, <span class="at">sep =</span> <span class="st">"</span><span class="sc">\t</span><span class="st">"</span>)</span>
<span id="cb36-25"><a href="#cb36-25" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb36-26"><a href="#cb36-26" aria-hidden="true" tabindex="-1"></a><span class="co"># Renombrar las columnas</span></span>
<span id="cb36-27"><a href="#cb36-27" aria-hidden="true" tabindex="-1"></a><span class="fu">colnames</span>(data_gct) <span class="ot">&lt;-</span> <span class="fu">c</span>(<span class="st">"Sample"</span>, <span class="fu">as.character</span>(<span class="dv">1</span><span class="sc">:</span>(<span class="fu">ncol</span>(data_gct) <span class="sc">-</span> <span class="dv">1</span>))) </span>
<span id="cb36-28"><a href="#cb36-28" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb36-29"><a href="#cb36-29" aria-hidden="true" tabindex="-1"></a><span class="co"># Transformar los datos de formato ancho a formato largo</span></span>
<span id="cb36-30"><a href="#cb36-30" aria-hidden="true" tabindex="-1"></a>data_long_gct <span class="ot">&lt;-</span> <span class="fu">pivot_longer</span>(data_gct, <span class="at">cols =</span> <span class="sc">-</span>Sample, <span class="at">names_to =</span> <span class="st">"bp"</span>, <span class="at">values_to =</span> <span class="st">"Contenido"</span>)</span>
<span id="cb36-31"><a href="#cb36-31" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb36-32"><a href="#cb36-32" aria-hidden="true" tabindex="-1"></a><span class="co"># Filtrar solo las posiciones de 1 a 30</span></span>
<span id="cb36-33"><a href="#cb36-33" aria-hidden="true" tabindex="-1"></a>data_long_gct <span class="ot">&lt;-</span> data_long_gct[data_long_gct<span class="sc">$</span>bp <span class="sc">%in%</span> <span class="fu">as.character</span>(<span class="dv">1</span><span class="sc">:</span><span class="dv">101</span>), ]</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<div class="cell">
<div class="sourceCode cell-code" id="cb37"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb37-1"><a href="#cb37-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Primer gráfico: FastQC: Quality Histograms Raw</span></span>
<span id="cb37-2"><a href="#cb37-2" aria-hidden="true" tabindex="-1"></a>p5 <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(data_long_gc, <span class="fu">aes</span>(<span class="at">x =</span> <span class="fu">as.numeric</span>(bp), <span class="at">y =</span> Contenido, <span class="at">color =</span> Sample, <span class="at">group =</span> Sample)) <span class="sc">+</span></span>
<span id="cb37-3"><a href="#cb37-3" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_line</span>() <span class="sc">+</span></span>
<span id="cb37-4"><a href="#cb37-4" aria-hidden="true" tabindex="-1"></a>  <span class="fu">labs</span>(<span class="at">title =</span> <span class="st">"E"</span>,</span>
<span id="cb37-5"><a href="#cb37-5" aria-hidden="true" tabindex="-1"></a>       <span class="at">subtitle =</span> <span class="st">"Raw"</span>,</span>
<span id="cb37-6"><a href="#cb37-6" aria-hidden="true" tabindex="-1"></a>       <span class="at">x =</span> <span class="st">"% GC"</span>,</span>
<span id="cb37-7"><a href="#cb37-7" aria-hidden="true" tabindex="-1"></a>       <span class="at">y =</span> <span class="st">"Percentage of counts"</span>) <span class="sc">+</span></span>
<span id="cb37-8"><a href="#cb37-8" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme_minimal</span>() <span class="sc">+</span></span>
<span id="cb37-9"><a href="#cb37-9" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_y_continuous</span>(<span class="at">limits =</span> <span class="fu">c</span>(<span class="dv">0</span>, <span class="dv">40</span>), <span class="at">labels =</span> scales<span class="sc">::</span><span class="fu">percent_format</span>(<span class="at">scale =</span> <span class="dv">1</span>)) <span class="sc">+</span>  <span class="co"># Mantén los límites sin formato de porcentaje</span></span>
<span id="cb37-10"><a href="#cb37-10" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_color_viridis_d</span>(<span class="at">option =</span> <span class="st">"Cividis"</span>) <span class="sc">+</span>  <span class="co"># Aplicar la paleta Cividis</span></span>
<span id="cb37-11"><a href="#cb37-11" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme</span>(</span>
<span id="cb37-12"><a href="#cb37-12" aria-hidden="true" tabindex="-1"></a>    <span class="at">axis.text.x =</span> <span class="fu">element_text</span>(<span class="at">angle =</span> <span class="dv">0</span>, <span class="at">hjust =</span> <span class="dv">1</span>, <span class="at">size =</span> <span class="dv">9</span>),</span>
<span id="cb37-13"><a href="#cb37-13" aria-hidden="true" tabindex="-1"></a>    <span class="at">axis.text.y =</span> <span class="fu">element_text</span>(<span class="at">angle =</span> <span class="dv">0</span>, <span class="at">hjust =</span> <span class="dv">1</span>, <span class="at">size =</span> <span class="dv">9</span>),</span>
<span id="cb37-14"><a href="#cb37-14" aria-hidden="true" tabindex="-1"></a>    <span class="at">axis.title =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">10</span>),</span>
<span id="cb37-15"><a href="#cb37-15" aria-hidden="true" tabindex="-1"></a>    <span class="at">plot.subtitle =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">10</span>, <span class="at">hjust =</span> <span class="fl">0.5</span> ),</span>
<span id="cb37-16"><a href="#cb37-16" aria-hidden="true" tabindex="-1"></a>    <span class="at">plot.title =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">12</span>, <span class="at">face =</span> <span class="st">"bold"</span>),</span>
<span id="cb37-17"><a href="#cb37-17" aria-hidden="true" tabindex="-1"></a><span class="co">#    plot.margin = margin(t = 10, r = 10, b = 30, l = 10),</span></span>
<span id="cb37-18"><a href="#cb37-18" aria-hidden="true" tabindex="-1"></a>    <span class="at">legend.position =</span> <span class="st">"none"</span></span>
<span id="cb37-19"><a href="#cb37-19" aria-hidden="true" tabindex="-1"></a>  )</span>
<span id="cb37-20"><a href="#cb37-20" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb37-21"><a href="#cb37-21" aria-hidden="true" tabindex="-1"></a><span class="co"># Segundo gráfico: FastQC: Quality Histograms (trimmed)</span></span>
<span id="cb37-22"><a href="#cb37-22" aria-hidden="true" tabindex="-1"></a>p6 <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(data_long_gct, <span class="fu">aes</span>(<span class="at">x =</span> <span class="fu">as.numeric</span>(bp), <span class="at">y =</span> Contenido, <span class="at">color =</span> Sample, <span class="at">group =</span> Sample)) <span class="sc">+</span></span>
<span id="cb37-23"><a href="#cb37-23" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_line</span>() <span class="sc">+</span></span>
<span id="cb37-24"><a href="#cb37-24" aria-hidden="true" tabindex="-1"></a>  <span class="fu">labs</span>(<span class="at">title =</span> <span class="st">"F"</span>,</span>
<span id="cb37-25"><a href="#cb37-25" aria-hidden="true" tabindex="-1"></a>       <span class="at">subtitle =</span> <span class="st">"Trimmed"</span>,</span>
<span id="cb37-26"><a href="#cb37-26" aria-hidden="true" tabindex="-1"></a>       <span class="at">x =</span> <span class="st">"% GC"</span>,</span>
<span id="cb37-27"><a href="#cb37-27" aria-hidden="true" tabindex="-1"></a>       <span class="at">y =</span> <span class="st">""</span>) <span class="sc">+</span></span>
<span id="cb37-28"><a href="#cb37-28" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme_minimal</span>() <span class="sc">+</span></span>
<span id="cb37-29"><a href="#cb37-29" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_y_continuous</span>(<span class="at">limits =</span> <span class="fu">c</span>(<span class="dv">0</span>, <span class="dv">40</span>), <span class="at">labels =</span> scales<span class="sc">::</span><span class="fu">percent_format</span>(<span class="at">scale =</span> <span class="dv">1</span>)) <span class="sc">+</span>  <span class="co"># Establecer límites del eje Y hasta 5%</span></span>
<span id="cb37-30"><a href="#cb37-30" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_color_viridis_d</span>(<span class="at">option =</span> <span class="st">"Cividis"</span>) <span class="sc">+</span>  <span class="co"># Aplicar la paleta Cividis</span></span>
<span id="cb37-31"><a href="#cb37-31" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme</span>(</span>
<span id="cb37-32"><a href="#cb37-32" aria-hidden="true" tabindex="-1"></a>    <span class="at">axis.text.x =</span> <span class="fu">element_text</span>(<span class="at">angle =</span> <span class="dv">0</span>, <span class="at">hjust =</span> <span class="dv">1</span>, <span class="at">size =</span> <span class="dv">9</span>),</span>
<span id="cb37-33"><a href="#cb37-33" aria-hidden="true" tabindex="-1"></a>    <span class="at">axis.text.y =</span> <span class="fu">element_text</span>(<span class="at">angle =</span> <span class="dv">0</span>, <span class="at">hjust =</span> <span class="dv">1</span>, <span class="at">size =</span> <span class="dv">9</span>),</span>
<span id="cb37-34"><a href="#cb37-34" aria-hidden="true" tabindex="-1"></a>    <span class="at">axis.title =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">10</span>),</span>
<span id="cb37-35"><a href="#cb37-35" aria-hidden="true" tabindex="-1"></a>    <span class="at">plot.subtitle =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">10</span>, <span class="at">hjust =</span> <span class="fl">0.5</span> ),</span>
<span id="cb37-36"><a href="#cb37-36" aria-hidden="true" tabindex="-1"></a>    <span class="at">plot.title =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">12</span>, <span class="at">face =</span> <span class="st">"bold"</span>),</span>
<span id="cb37-37"><a href="#cb37-37" aria-hidden="true" tabindex="-1"></a><span class="co">#    plot.margin = margin(t = 10, r = 10, b = 30, l = 10),</span></span>
<span id="cb37-38"><a href="#cb37-38" aria-hidden="true" tabindex="-1"></a>    <span class="at">legend.position =</span> <span class="st">"none"</span></span>
<span id="cb37-39"><a href="#cb37-39" aria-hidden="true" tabindex="-1"></a>  )</span>
<span id="cb37-40"><a href="#cb37-40" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb37-41"><a href="#cb37-41" aria-hidden="true" tabindex="-1"></a><span class="co"># Unir los gráficos en una fila</span></span>
<span id="cb37-42"><a href="#cb37-42" aria-hidden="true" tabindex="-1"></a><span class="co"># grid.arrange(p5, p6, ncol = 2)</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<div class="cell">
<div class="sourceCode cell-code" id="cb38"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb38-1"><a href="#cb38-1" aria-hidden="true" tabindex="-1"></a><span class="do">### FastQC: Per Sequence GC Content Raw</span></span>
<span id="cb38-2"><a href="#cb38-2" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb38-3"><a href="#cb38-3" aria-hidden="true" tabindex="-1"></a><span class="co"># Cargar los datos desde un archivo .tsv</span></span>
<span id="cb38-4"><a href="#cb38-4" aria-hidden="true" tabindex="-1"></a>file_path_n <span class="ot">&lt;-</span> <span class="st">"data/fastqc_per_base_n_content_plot.tsv"</span>  <span class="co"># Cambia esto por la ruta real</span></span>
<span id="cb38-5"><a href="#cb38-5" aria-hidden="true" tabindex="-1"></a>data_n <span class="ot">&lt;-</span> <span class="fu">read.delim</span>(file_path_n, <span class="at">header =</span> <span class="cn">TRUE</span>, <span class="at">sep =</span> <span class="st">"</span><span class="sc">\t</span><span class="st">"</span>)</span>
<span id="cb38-6"><a href="#cb38-6" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb38-7"><a href="#cb38-7" aria-hidden="true" tabindex="-1"></a><span class="co"># Renombrar las columnas</span></span>
<span id="cb38-8"><a href="#cb38-8" aria-hidden="true" tabindex="-1"></a><span class="fu">colnames</span>(data_n) <span class="ot">&lt;-</span> <span class="fu">c</span>(<span class="st">"Sample"</span>, <span class="fu">as.character</span>(<span class="dv">1</span><span class="sc">:</span>(<span class="fu">ncol</span>(data_n) <span class="sc">-</span> <span class="dv">1</span>))) </span>
<span id="cb38-9"><a href="#cb38-9" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb38-10"><a href="#cb38-10" aria-hidden="true" tabindex="-1"></a><span class="co"># Transformar los datos de formato ancho a formato largo</span></span>
<span id="cb38-11"><a href="#cb38-11" aria-hidden="true" tabindex="-1"></a>data_long_n <span class="ot">&lt;-</span> <span class="fu">pivot_longer</span>(data_n, <span class="at">cols =</span> <span class="sc">-</span>Sample, <span class="at">names_to =</span> <span class="st">"bp"</span>, <span class="at">values_to =</span> <span class="st">"Contenido"</span>)</span>
<span id="cb38-12"><a href="#cb38-12" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb38-13"><a href="#cb38-13" aria-hidden="true" tabindex="-1"></a><span class="co"># Convertir el contenido a porcentajes</span></span>
<span id="cb38-14"><a href="#cb38-14" aria-hidden="true" tabindex="-1"></a>data_long_n<span class="sc">$</span>Contenido <span class="ot">&lt;-</span> data_long_n<span class="sc">$</span>Contenido  <span class="co"># Convertir a porcentaje</span></span>
<span id="cb38-15"><a href="#cb38-15" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb38-16"><a href="#cb38-16" aria-hidden="true" tabindex="-1"></a><span class="co"># Filtrar solo las posiciones de 1 a 30</span></span>
<span id="cb38-17"><a href="#cb38-17" aria-hidden="true" tabindex="-1"></a>data_long_n <span class="ot">&lt;-</span> data_long_n[data_long_n<span class="sc">$</span>bp <span class="sc">%in%</span> <span class="fu">as.character</span>(<span class="dv">1</span><span class="sc">:</span><span class="dv">50</span>), ]</span>
<span id="cb38-18"><a href="#cb38-18" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb38-19"><a href="#cb38-19" aria-hidden="true" tabindex="-1"></a><span class="do">################################################################################</span></span>
<span id="cb38-20"><a href="#cb38-20" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb38-21"><a href="#cb38-21" aria-hidden="true" tabindex="-1"></a><span class="do">### FastQC: QPer Sequence GC Content Trimmed</span></span>
<span id="cb38-22"><a href="#cb38-22" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb38-23"><a href="#cb38-23" aria-hidden="true" tabindex="-1"></a><span class="co"># Cargar los datos desde un archivo .tsv</span></span>
<span id="cb38-24"><a href="#cb38-24" aria-hidden="true" tabindex="-1"></a>file_path_nt <span class="ot">&lt;-</span> <span class="st">"data/fastqc-1_per_base_n_content_plot.tsv"</span>  <span class="co"># Cambia esto por la ruta real</span></span>
<span id="cb38-25"><a href="#cb38-25" aria-hidden="true" tabindex="-1"></a>data_nt <span class="ot">&lt;-</span> <span class="fu">read.delim</span>(file_path_nt, <span class="at">header =</span> <span class="cn">TRUE</span>, <span class="at">sep =</span> <span class="st">"</span><span class="sc">\t</span><span class="st">"</span>)</span>
<span id="cb38-26"><a href="#cb38-26" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb38-27"><a href="#cb38-27" aria-hidden="true" tabindex="-1"></a><span class="co"># Renombrar las columnas</span></span>
<span id="cb38-28"><a href="#cb38-28" aria-hidden="true" tabindex="-1"></a><span class="fu">colnames</span>(data_nt) <span class="ot">&lt;-</span> <span class="fu">c</span>(<span class="st">"Sample"</span>, <span class="fu">as.character</span>(<span class="dv">1</span><span class="sc">:</span>(<span class="fu">ncol</span>(data_nt) <span class="sc">-</span> <span class="dv">1</span>))) </span>
<span id="cb38-29"><a href="#cb38-29" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb38-30"><a href="#cb38-30" aria-hidden="true" tabindex="-1"></a><span class="co"># Transformar los datos de formato ancho a formato largo</span></span>
<span id="cb38-31"><a href="#cb38-31" aria-hidden="true" tabindex="-1"></a>data_long_nt <span class="ot">&lt;-</span> <span class="fu">pivot_longer</span>(data_nt, <span class="at">cols =</span> <span class="sc">-</span>Sample, <span class="at">names_to =</span> <span class="st">"bp"</span>, <span class="at">values_to =</span> <span class="st">"Contenido"</span>)</span>
<span id="cb38-32"><a href="#cb38-32" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb38-33"><a href="#cb38-33" aria-hidden="true" tabindex="-1"></a><span class="co"># Filtrar solo las posiciones de 1 a 30</span></span>
<span id="cb38-34"><a href="#cb38-34" aria-hidden="true" tabindex="-1"></a>data_long_nt <span class="ot">&lt;-</span> data_long_nt[data_long_nt<span class="sc">$</span>bp <span class="sc">%in%</span> <span class="fu">as.character</span>(<span class="dv">1</span><span class="sc">:</span><span class="dv">50</span>), ]</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<div class="cell">
<div class="sourceCode cell-code" id="cb39"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb39-1"><a href="#cb39-1" aria-hidden="true" tabindex="-1"></a>p7 <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(data_long_n, <span class="fu">aes</span>(<span class="at">x =</span> <span class="fu">as.numeric</span>(bp), <span class="at">y =</span> Contenido, <span class="at">color =</span> Sample, <span class="at">group =</span> Sample)) <span class="sc">+</span></span>
<span id="cb39-2"><a href="#cb39-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_line</span>() <span class="sc">+</span></span>
<span id="cb39-3"><a href="#cb39-3" aria-hidden="true" tabindex="-1"></a>  <span class="fu">labs</span>(<span class="at">title =</span> <span class="st">"G"</span>,</span>
<span id="cb39-4"><a href="#cb39-4" aria-hidden="true" tabindex="-1"></a>       <span class="at">subtitle =</span> <span class="st">"Raw"</span>,</span>
<span id="cb39-5"><a href="#cb39-5" aria-hidden="true" tabindex="-1"></a>       <span class="at">x =</span> <span class="st">"Position in read (bp)"</span>,</span>
<span id="cb39-6"><a href="#cb39-6" aria-hidden="true" tabindex="-1"></a>       <span class="at">y =</span> <span class="st">"Percentage N-Count"</span>) <span class="sc">+</span></span>
<span id="cb39-7"><a href="#cb39-7" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme_minimal</span>() <span class="sc">+</span></span>
<span id="cb39-8"><a href="#cb39-8" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_y_continuous</span>(<span class="at">limits =</span> <span class="fu">c</span>(<span class="dv">0</span>, <span class="dv">5</span>), <span class="at">labels =</span> scales<span class="sc">::</span><span class="fu">percent_format</span>(<span class="at">scale =</span> <span class="dv">1</span>)) <span class="sc">+</span>  <span class="co"># Mantén los límites sin formato de porcentaje</span></span>
<span id="cb39-9"><a href="#cb39-9" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_color_viridis_d</span>(<span class="at">option =</span> <span class="st">"Cividis"</span>) <span class="sc">+</span>  <span class="co"># Aplicar la paleta Cividis</span></span>
<span id="cb39-10"><a href="#cb39-10" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme</span>(</span>
<span id="cb39-11"><a href="#cb39-11" aria-hidden="true" tabindex="-1"></a>    <span class="at">axis.text.x =</span> <span class="fu">element_text</span>(<span class="at">angle =</span> <span class="dv">0</span>, <span class="at">hjust =</span> <span class="dv">1</span>, <span class="at">size =</span> <span class="dv">9</span>),</span>
<span id="cb39-12"><a href="#cb39-12" aria-hidden="true" tabindex="-1"></a>    <span class="at">axis.text.y =</span> <span class="fu">element_text</span>(<span class="at">angle =</span> <span class="dv">0</span>, <span class="at">hjust =</span> <span class="dv">1</span>, <span class="at">size =</span> <span class="dv">9</span>),</span>
<span id="cb39-13"><a href="#cb39-13" aria-hidden="true" tabindex="-1"></a>    <span class="at">axis.title =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">10</span>),</span>
<span id="cb39-14"><a href="#cb39-14" aria-hidden="true" tabindex="-1"></a>    <span class="at">plot.subtitle =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">10</span>, <span class="at">hjust =</span> <span class="fl">0.5</span> ),</span>
<span id="cb39-15"><a href="#cb39-15" aria-hidden="true" tabindex="-1"></a>    <span class="at">plot.title =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">12</span>, <span class="at">face =</span> <span class="st">"bold"</span>),</span>
<span id="cb39-16"><a href="#cb39-16" aria-hidden="true" tabindex="-1"></a><span class="co">#    plot.margin = margin(t = 10, r = 10, b = 30, l = 10),</span></span>
<span id="cb39-17"><a href="#cb39-17" aria-hidden="true" tabindex="-1"></a>    <span class="at">legend.position =</span> <span class="st">"none"</span></span>
<span id="cb39-18"><a href="#cb39-18" aria-hidden="true" tabindex="-1"></a>  )</span>
<span id="cb39-19"><a href="#cb39-19" aria-hidden="true" tabindex="-1"></a><span class="co"># Segundo gráfico: FastQC: Quality Histograms (trimmed)</span></span>
<span id="cb39-20"><a href="#cb39-20" aria-hidden="true" tabindex="-1"></a>p8 <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(data_long_nt, <span class="fu">aes</span>(<span class="at">x =</span> <span class="fu">as.numeric</span>(bp), <span class="at">y =</span> Contenido, <span class="at">color =</span> Sample, <span class="at">group =</span> Sample)) <span class="sc">+</span></span>
<span id="cb39-21"><a href="#cb39-21" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_line</span>() <span class="sc">+</span></span>
<span id="cb39-22"><a href="#cb39-22" aria-hidden="true" tabindex="-1"></a>  <span class="fu">labs</span>(<span class="at">title =</span> <span class="st">"H"</span>,</span>
<span id="cb39-23"><a href="#cb39-23" aria-hidden="true" tabindex="-1"></a>       <span class="at">subtitle =</span> <span class="st">"Trimmed"</span>,</span>
<span id="cb39-24"><a href="#cb39-24" aria-hidden="true" tabindex="-1"></a>       <span class="at">x =</span> <span class="st">"Position in read (bp)"</span>,</span>
<span id="cb39-25"><a href="#cb39-25" aria-hidden="true" tabindex="-1"></a>       <span class="at">y =</span> <span class="st">""</span>) <span class="sc">+</span></span>
<span id="cb39-26"><a href="#cb39-26" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme_minimal</span>() <span class="sc">+</span></span>
<span id="cb39-27"><a href="#cb39-27" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_y_continuous</span>(<span class="at">limits =</span> <span class="fu">c</span>(<span class="dv">0</span>, <span class="dv">5</span>), <span class="at">labels =</span> scales<span class="sc">::</span><span class="fu">percent_format</span>(<span class="at">scale =</span> <span class="dv">1</span>)) <span class="sc">+</span>  <span class="co"># Establecer límites del eje Y hasta 5%</span></span>
<span id="cb39-28"><a href="#cb39-28" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_color_viridis_d</span>(<span class="at">option =</span> <span class="st">"Cividis"</span>) <span class="sc">+</span>  <span class="co"># Aplicar la paleta Cividis</span></span>
<span id="cb39-29"><a href="#cb39-29" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme</span>(</span>
<span id="cb39-30"><a href="#cb39-30" aria-hidden="true" tabindex="-1"></a>    <span class="at">axis.text.x =</span> <span class="fu">element_text</span>(<span class="at">angle =</span> <span class="dv">0</span>, <span class="at">hjust =</span> <span class="dv">1</span>, <span class="at">size =</span> <span class="dv">9</span>),</span>
<span id="cb39-31"><a href="#cb39-31" aria-hidden="true" tabindex="-1"></a>    <span class="at">axis.text.y =</span> <span class="fu">element_text</span>(<span class="at">angle =</span> <span class="dv">0</span>, <span class="at">hjust =</span> <span class="dv">1</span>, <span class="at">size =</span> <span class="dv">9</span>),</span>
<span id="cb39-32"><a href="#cb39-32" aria-hidden="true" tabindex="-1"></a>    <span class="at">axis.title =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">10</span>),</span>
<span id="cb39-33"><a href="#cb39-33" aria-hidden="true" tabindex="-1"></a>    <span class="at">plot.subtitle =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">10</span>, <span class="at">hjust =</span> <span class="fl">0.5</span> ),</span>
<span id="cb39-34"><a href="#cb39-34" aria-hidden="true" tabindex="-1"></a>    <span class="at">plot.title =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">12</span>, <span class="at">face =</span> <span class="st">"bold"</span>),</span>
<span id="cb39-35"><a href="#cb39-35" aria-hidden="true" tabindex="-1"></a><span class="co">#    plot.margin = margin(t = 10, r = 10, b = 30, l = 10),</span></span>
<span id="cb39-36"><a href="#cb39-36" aria-hidden="true" tabindex="-1"></a>    <span class="at">legend.position =</span> <span class="st">"none"</span></span>
<span id="cb39-37"><a href="#cb39-37" aria-hidden="true" tabindex="-1"></a>  )</span>
<span id="cb39-38"><a href="#cb39-38" aria-hidden="true" tabindex="-1"></a><span class="co"># Unir los gráficos en una fila</span></span>
<span id="cb39-39"><a href="#cb39-39" aria-hidden="true" tabindex="-1"></a><span class="co"># grid.arrange(p7, p8, ncol = 2)</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<div class="cell">
<div class="sourceCode cell-code" id="cb40"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb40-1"><a href="#cb40-1" aria-hidden="true" tabindex="-1"></a><span class="fu">grid.arrange</span>(p1, p2, p3, p4, p5, p6, p7, p8, </span>
<span id="cb40-2"><a href="#cb40-2" aria-hidden="true" tabindex="-1"></a>             <span class="at">ncol =</span> <span class="dv">2</span>, </span>
<span id="cb40-3"><a href="#cb40-3" aria-hidden="true" tabindex="-1"></a>             <span class="at">heights =</span> <span class="fu">rep</span>(<span class="dv">1</span>, <span class="dv">4</span>), <span class="co"># Espaciado uniforme entre filas</span></span>
<span id="cb40-4"><a href="#cb40-4" aria-hidden="true" tabindex="-1"></a>             <span class="at">widths =</span> <span class="fu">rep</span>(<span class="dv">1</span>, <span class="dv">2</span>)   <span class="co"># Espaciado uniforme entre columnas</span></span>
<span id="cb40-5"><a href="#cb40-5" aria-hidden="true" tabindex="-1"></a>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>Warning in viridisLite::viridis(n, alpha, begin, end, direction, option):
Option 'Cividis' does not exist. Defaulting to 'viridis'.
Warning in viridisLite::viridis(n, alpha, begin, end, direction, option):
Option 'Cividis' does not exist. Defaulting to 'viridis'.
Warning in viridisLite::viridis(n, alpha, begin, end, direction, option):
Option 'Cividis' does not exist. Defaulting to 'viridis'.</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>Warning: Removed 40 rows containing missing values or values outside the scale range
(`geom_line()`).</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>Warning in viridisLite::viridis(n, alpha, begin, end, direction, option):
Option 'Cividis' does not exist. Defaulting to 'viridis'.</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>Warning: Removed 116 rows containing missing values or values outside the scale range
(`geom_line()`).</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>Warning in viridisLite::viridis(n, alpha, begin, end, direction, option):
Option 'Cividis' does not exist. Defaulting to 'viridis'.
Warning in viridisLite::viridis(n, alpha, begin, end, direction, option):
Option 'Cividis' does not exist. Defaulting to 'viridis'.
Warning in viridisLite::viridis(n, alpha, begin, end, direction, option):
Option 'Cividis' does not exist. Defaulting to 'viridis'.
Warning in viridisLite::viridis(n, alpha, begin, end, direction, option):
Option 'Cividis' does not exist. Defaulting to 'viridis'.</code></pre>
</div>
<div class="cell-output-display">
<div id="fig-fstqc_quality_control" class="quarto-float quarto-figure quarto-figure-center anchored" data-fig-pos="H">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-fstqc_quality_control-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="Results_files/figure-html/fig-fstqc_quality_control-1.png" class="img-fluid figure-img" data-fig-pos="H" width="768">
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-fstqc_quality_control-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;3: Quality Control Analysis. <em>A</em> and <em>B</em>: Mean quality values of sequences across all bases after and before using fastp (v0.23.4); <em>C</em> and <em>D</em>: Per Sequence Quality Scores across all bases after and before using fastp; <em>E</em> and <em>F</em>: Per Sequence GC Content Raw after and before using fastp; <em>G</em> and <em>H</em>: Read N content after and before using fastp.
</figcaption>
</figure>
</div>
</div>
</div>
<p>However, the distribution of sequence lengths is irregular across all samples <a href="#fig-sld" class="quarto-xref">Figure&nbsp;4</a>. After trimming, the majority of sequences cluster around a length of 20-25 nucleotides, although a smaller subset of sequences with lengths between 29-32 nucleotides is also observed. This indicates the presence of different types of small RNAs. Generally, miRNAs are found in the range of 20-24 bp, with a peak at 22 bp potentially indicating an abundant population of miRNAs. In contrast, tRNAs can vary in length but are commonly found at lengths that may include higher peaks, such as at 31 bp. This may reflect the presence of tRNAs derived from the degradation of double-stranded RNA or transposons.</p>
<div class="cell">
<div class="sourceCode cell-code" id="cb46"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb46-1"><a href="#cb46-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Cargar librerías</span></span>
<span id="cb46-2"><a href="#cb46-2" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(ggplot2)</span>
<span id="cb46-3"><a href="#cb46-3" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(tidyr)</span>
<span id="cb46-4"><a href="#cb46-4" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(dplyr)</span>
<span id="cb46-5"><a href="#cb46-5" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb46-6"><a href="#cb46-6" aria-hidden="true" tabindex="-1"></a><span class="do">################################################################################</span></span>
<span id="cb46-7"><a href="#cb46-7" aria-hidden="true" tabindex="-1"></a><span class="do">### FastQC: Sequence Length Distribution</span></span>
<span id="cb46-8"><a href="#cb46-8" aria-hidden="true" tabindex="-1"></a><span class="co"># Cargar los datos desde un archivo .tsv</span></span>
<span id="cb46-9"><a href="#cb46-9" aria-hidden="true" tabindex="-1"></a>file_path_sld <span class="ot">&lt;-</span> <span class="st">"data/fastqc_sequence_length_distribution_plot.tsv"</span>  <span class="co"># Cambia esto por la ruta real</span></span>
<span id="cb46-10"><a href="#cb46-10" aria-hidden="true" tabindex="-1"></a>data_sld <span class="ot">&lt;-</span> <span class="fu">read.delim</span>(file_path_sld, <span class="at">header =</span> <span class="cn">TRUE</span>, <span class="at">sep =</span> <span class="st">"</span><span class="sc">\t</span><span class="st">"</span>)</span>
<span id="cb46-11"><a href="#cb46-11" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb46-12"><a href="#cb46-12" aria-hidden="true" tabindex="-1"></a><span class="co"># Renombrar las columnas</span></span>
<span id="cb46-13"><a href="#cb46-13" aria-hidden="true" tabindex="-1"></a><span class="fu">colnames</span>(data_sld) <span class="ot">&lt;-</span> <span class="fu">c</span>(<span class="st">"Sample"</span>, <span class="fu">as.character</span>(<span class="dv">17</span><span class="sc">:</span><span class="dv">50</span>)) </span>
<span id="cb46-14"><a href="#cb46-14" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb46-15"><a href="#cb46-15" aria-hidden="true" tabindex="-1"></a><span class="co"># Transformar los datos de formato ancho a formato largo</span></span>
<span id="cb46-16"><a href="#cb46-16" aria-hidden="true" tabindex="-1"></a>data_long_sld <span class="ot">&lt;-</span> <span class="fu">pivot_longer</span>(data_sld, <span class="at">cols =</span> <span class="sc">-</span>Sample, <span class="at">names_to =</span> <span class="st">"bp"</span>, <span class="at">values_to =</span> <span class="st">"Contenido"</span>)</span>
<span id="cb46-17"><a href="#cb46-17" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb46-18"><a href="#cb46-18" aria-hidden="true" tabindex="-1"></a><span class="co"># Filtrar solo las posiciones de 1 a 30</span></span>
<span id="cb46-19"><a href="#cb46-19" aria-hidden="true" tabindex="-1"></a>data_long_sld <span class="ot">&lt;-</span> data_long_sld[data_long_sld<span class="sc">$</span>bp <span class="sc">%in%</span> <span class="fu">as.character</span>(<span class="dv">1</span><span class="sc">:</span><span class="dv">50</span>), ]</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<div class="cell">
<div class="sourceCode cell-code" id="cb47"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb47-1"><a href="#cb47-1" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(scales)</span>
<span id="cb47-2"><a href="#cb47-2" aria-hidden="true" tabindex="-1"></a><span class="do">### FastQC: Quality Histograms Raw</span></span>
<span id="cb47-3"><a href="#cb47-3" aria-hidden="true" tabindex="-1"></a><span class="fu">ggplot</span>(data_long_sld, <span class="fu">aes</span>(<span class="at">x =</span> <span class="fu">as.numeric</span>(bp), <span class="at">y =</span> Contenido, <span class="at">color =</span> Sample, <span class="at">group =</span> Sample)) <span class="sc">+</span></span>
<span id="cb47-4"><a href="#cb47-4" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_line</span>() <span class="sc">+</span></span>
<span id="cb47-5"><a href="#cb47-5" aria-hidden="true" tabindex="-1"></a>  <span class="fu">labs</span>(<span class="at">x =</span> <span class="st">"Sequence Length (bp)"</span>,</span>
<span id="cb47-6"><a href="#cb47-6" aria-hidden="true" tabindex="-1"></a>       <span class="at">y =</span> <span class="st">"Read Count"</span>) <span class="sc">+</span></span>
<span id="cb47-7"><a href="#cb47-7" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme_minimal</span>() <span class="sc">+</span></span>
<span id="cb47-8"><a href="#cb47-8" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_color_viridis_d</span>(<span class="at">option =</span> <span class="st">"Cividis"</span>) <span class="sc">+</span>  <span class="co"># Aplicar la paleta Cividis</span></span>
<span id="cb47-9"><a href="#cb47-9" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_y_continuous</span>(<span class="at">labels =</span> <span class="fu">label_number</span>(<span class="at">scale =</span> <span class="fl">1e-6</span>, <span class="at">suffix =</span> <span class="st">"M"</span>)) <span class="sc">+</span> <span class="co"># Escalar a millones</span></span>
<span id="cb47-10"><a href="#cb47-10" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme</span>(</span>
<span id="cb47-11"><a href="#cb47-11" aria-hidden="true" tabindex="-1"></a>    <span class="at">axis.text.x =</span> <span class="fu">element_text</span>(<span class="at">angle =</span> <span class="dv">0</span>, <span class="at">hjust =</span> <span class="dv">1</span>, <span class="at">size =</span> <span class="dv">9</span>),</span>
<span id="cb47-12"><a href="#cb47-12" aria-hidden="true" tabindex="-1"></a>    <span class="at">axis.text.y =</span> <span class="fu">element_text</span>(<span class="at">angle =</span> <span class="dv">0</span>, <span class="at">hjust =</span> <span class="dv">1</span>, <span class="at">size =</span> <span class="dv">9</span>),</span>
<span id="cb47-13"><a href="#cb47-13" aria-hidden="true" tabindex="-1"></a>    <span class="at">axis.title =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">10</span>),</span>
<span id="cb47-14"><a href="#cb47-14" aria-hidden="true" tabindex="-1"></a>    <span class="at">plot.subtitle =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">10</span>, <span class="at">hjust =</span> <span class="fl">0.5</span> ),</span>
<span id="cb47-15"><a href="#cb47-15" aria-hidden="true" tabindex="-1"></a>    <span class="at">plot.title =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">12</span>, <span class="at">face =</span> <span class="st">"bold"</span>),</span>
<span id="cb47-16"><a href="#cb47-16" aria-hidden="true" tabindex="-1"></a>    <span class="at">plot.margin =</span> <span class="fu">margin</span>(<span class="at">t =</span> <span class="dv">10</span>, <span class="at">r =</span> <span class="dv">10</span>, <span class="at">b =</span> <span class="dv">30</span>, <span class="at">l =</span> <span class="dv">10</span>),</span>
<span id="cb47-17"><a href="#cb47-17" aria-hidden="true" tabindex="-1"></a>    <span class="at">legend.position =</span> <span class="st">"none"</span></span>
<span id="cb47-18"><a href="#cb47-18" aria-hidden="true" tabindex="-1"></a>  ) <span class="co"># Eliminar leyenda # Eliminar leyenda</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>Warning in viridisLite::viridis(n, alpha, begin, end, direction, option):
Option 'Cividis' does not exist. Defaulting to 'viridis'.</code></pre>
</div>
<div class="sourceCode cell-code" id="cb49"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb49-1"><a href="#cb49-1" aria-hidden="true" tabindex="-1"></a><span class="co">#ggsave("Sequence Length.png", plot = sld, device = NULL, path = "figures/", scale = 1, width = 4, height = 3, units = "in", dpi = 300, limitsize = TRUE, bg = NULL, create.dir = FALSE)</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<div id="fig-sld" class="quarto-float quarto-figure quarto-figure-center anchored" data-fig-pos="H">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-sld-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="Results_files/figure-html/fig-sld-1.png" class="img-fluid figure-img" data-fig-pos="H" width="768">
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-sld-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;4: FastQC: Sequence Length Distribution
</figcaption>
</figure>
</div>
</div>
</div>
<p>In the <a href="#fig-totalreads" class="quarto-xref">Figure&nbsp;5</a> shows the total number of reads per sample is around 25 million, both in Raw and Trimmed, with means of 25.47 and 25.23 million, respectively. The interquartile ranges (IQR) of these measurements are similar, approximately 4.2 million, reflecting a high consistency among the samples.</p>
<p>Regarding duplicate reads, they dominate the data, with means of 25.10 million in Raw and 24.95 million in Trimmed, and an IQR of about 4.2 million in both conditions. This indicates that more than 90% of the sequences are duplicated, a high value but expected in smRNAseq samples, given the nature of the short reads of 20 to 24 nucleotides.</p>
<p>In contrast, unique reads are significantly less frequent, with means of 0.38 million in Raw and 0.28 million in Trimmed. The interquartile ranges for these measurements are also low, around 0.13 million in Raw and 0.10 million in Trim.</p>
<div class="cell">
<div class="sourceCode cell-code" id="cb50"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb50-1"><a href="#cb50-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Cargar librerías</span></span>
<span id="cb50-2"><a href="#cb50-2" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(ggplot2)</span>
<span id="cb50-3"><a href="#cb50-3" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(tidyr)</span>
<span id="cb50-4"><a href="#cb50-4" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(dplyr)</span>
<span id="cb50-5"><a href="#cb50-5" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb50-6"><a href="#cb50-6" aria-hidden="true" tabindex="-1"></a><span class="do">################################################################################</span></span>
<span id="cb50-7"><a href="#cb50-7" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb50-8"><a href="#cb50-8" aria-hidden="true" tabindex="-1"></a><span class="do">### FastQC Total reads raw</span></span>
<span id="cb50-9"><a href="#cb50-9" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb50-10"><a href="#cb50-10" aria-hidden="true" tabindex="-1"></a><span class="co"># Cargar los datos desde un archivo .tsv</span></span>
<span id="cb50-11"><a href="#cb50-11" aria-hidden="true" tabindex="-1"></a>file_path_udr <span class="ot">&lt;-</span> <span class="st">"data/fastqc_sequence_counts_plot.tsv"</span>  <span class="co"># Cambia esto por la ruta real</span></span>
<span id="cb50-12"><a href="#cb50-12" aria-hidden="true" tabindex="-1"></a>data_udr <span class="ot">&lt;-</span> <span class="fu">read.delim</span>(file_path_udr, <span class="at">header =</span> <span class="cn">TRUE</span>, <span class="at">sep =</span> <span class="st">"</span><span class="sc">\t</span><span class="st">"</span>)</span>
<span id="cb50-13"><a href="#cb50-13" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb50-14"><a href="#cb50-14" aria-hidden="true" tabindex="-1"></a>data_udr <span class="ot">&lt;-</span> data_udr <span class="sc">|&gt;</span></span>
<span id="cb50-15"><a href="#cb50-15" aria-hidden="true" tabindex="-1"></a>  <span class="fu">rename</span>(</span>
<span id="cb50-16"><a href="#cb50-16" aria-hidden="true" tabindex="-1"></a>    <span class="at">Unique_reads =</span> Unique.Reads,   <span class="co"># Cambiar Unique.Reads a Unique_Reads</span></span>
<span id="cb50-17"><a href="#cb50-17" aria-hidden="true" tabindex="-1"></a>    <span class="at">Duplicate_reads =</span> Duplicate.Reads <span class="co"># Cambiar Duplicate.Reads a Duplicate_Reads</span></span>
<span id="cb50-18"><a href="#cb50-18" aria-hidden="true" tabindex="-1"></a>  ) <span class="sc">|&gt;</span></span>
<span id="cb50-19"><a href="#cb50-19" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="st">"Total reads"</span> <span class="ot">=</span> Unique_reads <span class="sc">+</span> Duplicate_reads,</span>
<span id="cb50-20"><a href="#cb50-20" aria-hidden="true" tabindex="-1"></a>         <span class="st">"Condition"</span> <span class="ot">=</span> <span class="st">"Raw"</span>)</span>
<span id="cb50-21"><a href="#cb50-21" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb50-22"><a href="#cb50-22" aria-hidden="true" tabindex="-1"></a><span class="co"># Transformar el dataframe</span></span>
<span id="cb50-23"><a href="#cb50-23" aria-hidden="true" tabindex="-1"></a>datos_longr <span class="ot">&lt;-</span> <span class="fu">pivot_longer</span>(data_udr, <span class="at">cols =</span> <span class="fu">c</span>(<span class="st">"Unique_reads"</span>, <span class="st">"Duplicate_reads"</span>,<span class="st">"Total reads"</span>), </span>
<span id="cb50-24"><a href="#cb50-24" aria-hidden="true" tabindex="-1"></a>                            <span class="at">names_to =</span> <span class="st">"Type"</span>, <span class="at">values_to =</span> <span class="st">"Reads"</span>)</span>
<span id="cb50-25"><a href="#cb50-25" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb50-26"><a href="#cb50-26" aria-hidden="true" tabindex="-1"></a><span class="do">################################################################################</span></span>
<span id="cb50-27"><a href="#cb50-27" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb50-28"><a href="#cb50-28" aria-hidden="true" tabindex="-1"></a><span class="do">### FastQC total reads after trimmming</span></span>
<span id="cb50-29"><a href="#cb50-29" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb50-30"><a href="#cb50-30" aria-hidden="true" tabindex="-1"></a><span class="co"># Cargar los datos desde un archivo .tsv</span></span>
<span id="cb50-31"><a href="#cb50-31" aria-hidden="true" tabindex="-1"></a>file_path_ud <span class="ot">&lt;-</span> <span class="st">"data/fastqc-1_sequence_counts_plot.tsv"</span>  <span class="co"># Cambia esto por la ruta real</span></span>
<span id="cb50-32"><a href="#cb50-32" aria-hidden="true" tabindex="-1"></a>data_ud <span class="ot">&lt;-</span> <span class="fu">read.delim</span>(file_path_ud, <span class="at">header =</span> <span class="cn">TRUE</span>, <span class="at">sep =</span> <span class="st">"</span><span class="sc">\t</span><span class="st">"</span>)</span>
<span id="cb50-33"><a href="#cb50-33" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb50-34"><a href="#cb50-34" aria-hidden="true" tabindex="-1"></a>data_ud <span class="ot">&lt;-</span> data_ud <span class="sc">|&gt;</span></span>
<span id="cb50-35"><a href="#cb50-35" aria-hidden="true" tabindex="-1"></a>  <span class="fu">rename</span>(</span>
<span id="cb50-36"><a href="#cb50-36" aria-hidden="true" tabindex="-1"></a>    <span class="at">Unique_reads =</span> Unique.Reads,   <span class="co"># Cambiar Unique.Reads a Unique_Reads</span></span>
<span id="cb50-37"><a href="#cb50-37" aria-hidden="true" tabindex="-1"></a>    <span class="at">Duplicate_reads =</span> Duplicate.Reads <span class="co"># Cambiar Duplicate.Reads a Duplicate_Reads</span></span>
<span id="cb50-38"><a href="#cb50-38" aria-hidden="true" tabindex="-1"></a>  )<span class="sc">|&gt;</span></span>
<span id="cb50-39"><a href="#cb50-39" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="st">"Total reads"</span> <span class="ot">=</span> Unique_reads <span class="sc">+</span> Duplicate_reads,</span>
<span id="cb50-40"><a href="#cb50-40" aria-hidden="true" tabindex="-1"></a>         <span class="st">"Condition"</span> <span class="ot">=</span> <span class="st">"Trimmed"</span>) </span>
<span id="cb50-41"><a href="#cb50-41" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb50-42"><a href="#cb50-42" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb50-43"><a href="#cb50-43" aria-hidden="true" tabindex="-1"></a><span class="co"># Transformar el dataframe</span></span>
<span id="cb50-44"><a href="#cb50-44" aria-hidden="true" tabindex="-1"></a>datos_long <span class="ot">&lt;-</span> <span class="fu">pivot_longer</span>(data_ud, <span class="at">cols =</span> <span class="fu">c</span>(<span class="st">"Unique_reads"</span>, <span class="st">"Duplicate_reads"</span>,<span class="st">"Total reads"</span>), </span>
<span id="cb50-45"><a href="#cb50-45" aria-hidden="true" tabindex="-1"></a>                           <span class="at">names_to =</span> <span class="st">"Type"</span>, <span class="at">values_to =</span> <span class="st">"Reads"</span>)</span>
<span id="cb50-46"><a href="#cb50-46" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb50-47"><a href="#cb50-47" aria-hidden="true" tabindex="-1"></a><span class="co"># Unir ambos dataframes</span></span>
<span id="cb50-48"><a href="#cb50-48" aria-hidden="true" tabindex="-1"></a>datos_combined <span class="ot">&lt;-</span> <span class="fu">bind_rows</span>(datos_longr, datos_long)</span>
<span id="cb50-49"><a href="#cb50-49" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb50-50"><a href="#cb50-50" aria-hidden="true" tabindex="-1"></a><span class="co"># Convertir los valores de Reads a millones</span></span>
<span id="cb50-51"><a href="#cb50-51" aria-hidden="true" tabindex="-1"></a>datos_combined <span class="ot">&lt;-</span> datos_combined <span class="sc">|&gt;</span></span>
<span id="cb50-52"><a href="#cb50-52" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">Reads =</span> Reads <span class="sc">/</span> <span class="fl">1e6</span>)  <span class="co"># Dividir por 1,000,000 para convertir a millones</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<div class="cell">
<div class="sourceCode cell-code" id="cb51"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb51-1"><a href="#cb51-1" aria-hidden="true" tabindex="-1"></a><span class="fu">ggplot</span>(datos_combined, <span class="fu">aes</span>(<span class="at">x =</span> Type, <span class="at">y =</span> Reads, <span class="at">fill =</span> Condition)) <span class="sc">+</span>  <span class="co"># Cambiar fill a Condition</span></span>
<span id="cb51-2"><a href="#cb51-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_boxplot</span>(<span class="at">position =</span> <span class="fu">position_dodge</span>(<span class="at">width =</span> <span class="fl">0.75</span>)) <span class="sc">+</span></span>
<span id="cb51-3"><a href="#cb51-3" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme_minimal</span>() <span class="sc">+</span></span>
<span id="cb51-4"><a href="#cb51-4" aria-hidden="true" tabindex="-1"></a>  <span class="fu">labs</span>(<span class="at">x =</span> <span class="st">""</span>,<span class="at">y =</span> <span class="st">"Number of reads (millions)"</span>) <span class="sc">+</span></span>
<span id="cb51-5"><a href="#cb51-5" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb51-6"><a href="#cb51-6" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_fill_viridis_d</span>(<span class="at">option =</span> <span class="st">"cividis"</span>) <span class="sc">+</span></span>
<span id="cb51-7"><a href="#cb51-7" aria-hidden="true" tabindex="-1"></a>  </span>
<span id="cb51-8"><a href="#cb51-8" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_x_discrete</span>(<span class="at">labels =</span> <span class="fu">c</span>(<span class="st">"Duplicate_reads"</span> <span class="ot">=</span> <span class="st">"Duplicate reads"</span>, </span>
<span id="cb51-9"><a href="#cb51-9" aria-hidden="true" tabindex="-1"></a>                               <span class="st">"Total Reads"</span> <span class="ot">=</span> <span class="st">"Total Reads"</span>, </span>
<span id="cb51-10"><a href="#cb51-10" aria-hidden="true" tabindex="-1"></a>                               <span class="st">"Unique_reads"</span> <span class="ot">=</span> <span class="st">"Unique reads"</span>)) <span class="sc">+</span>  <span class="co"># Cambiar los nombres aquí</span></span>
<span id="cb51-11"><a href="#cb51-11" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb51-12"><a href="#cb51-12" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme_minimal</span>(<span class="at">base_size =</span> <span class="dv">14</span>) <span class="sc">+</span></span>
<span id="cb51-13"><a href="#cb51-13" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme</span>(</span>
<span id="cb51-14"><a href="#cb51-14" aria-hidden="true" tabindex="-1"></a>    <span class="at">axis.text.x =</span> <span class="fu">element_text</span>(<span class="at">angle =</span> <span class="dv">0</span> , <span class="at">hjust =</span> <span class="dv">1</span>, <span class="at">size =</span> <span class="dv">9</span>),</span>
<span id="cb51-15"><a href="#cb51-15" aria-hidden="true" tabindex="-1"></a>    <span class="at">axis.text.y =</span> <span class="fu">element_text</span>(<span class="at">angle =</span> <span class="dv">0</span> , <span class="at">hjust =</span> <span class="dv">1</span>, <span class="at">size =</span> <span class="dv">9</span>),</span>
<span id="cb51-16"><a href="#cb51-16" aria-hidden="true" tabindex="-1"></a>    <span class="at">axis.title =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">10</span>),</span>
<span id="cb51-17"><a href="#cb51-17" aria-hidden="true" tabindex="-1"></a>    <span class="at">plot.title =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">12</span>, <span class="at">face =</span> <span class="st">"bold"</span>),</span>
<span id="cb51-18"><a href="#cb51-18" aria-hidden="true" tabindex="-1"></a>    <span class="at">legend.position =</span> <span class="st">"right"</span>,</span>
<span id="cb51-19"><a href="#cb51-19" aria-hidden="true" tabindex="-1"></a>    <span class="at">legend.title =</span> <span class="fu">element_blank</span>()</span>
<span id="cb51-20"><a href="#cb51-20" aria-hidden="true" tabindex="-1"></a>  )   <span class="co"># Eliminar título de leyenda</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<div id="fig-totalreads" class="quarto-float quarto-figure quarto-figure-center anchored" data-fig-pos="H">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-totalreads-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="Results_files/figure-html/fig-totalreads-1.png" class="img-fluid figure-img" data-fig-pos="H" width="576">
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-totalreads-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;5: Number of reads from small RNA-seq. Total reads before and after trimming of adapters
</figcaption>
</figure>
</div>
</div>
</div>
<div class="cell">
<div class="sourceCode cell-code" id="cb52"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb52-1"><a href="#cb52-1" aria-hidden="true" tabindex="-1"></a>resumen_stats <span class="ot">&lt;-</span> datos_combined <span class="sc">|&gt;</span></span>
<span id="cb52-2"><a href="#cb52-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">group_by</span>(Type, Condition) <span class="sc">|&gt;</span></span>
<span id="cb52-3"><a href="#cb52-3" aria-hidden="true" tabindex="-1"></a>  <span class="fu">summarize</span>(</span>
<span id="cb52-4"><a href="#cb52-4" aria-hidden="true" tabindex="-1"></a>    <span class="at">Media =</span> <span class="fu">mean</span>(Reads),</span>
<span id="cb52-5"><a href="#cb52-5" aria-hidden="true" tabindex="-1"></a>    <span class="at">IQR =</span> <span class="fu">IQR</span>(Reads),</span>
<span id="cb52-6"><a href="#cb52-6" aria-hidden="true" tabindex="-1"></a>    <span class="at">.groups =</span> <span class="st">"drop"</span></span>
<span id="cb52-7"><a href="#cb52-7" aria-hidden="true" tabindex="-1"></a>  )</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
</section>
<section id="mirna-quality-control" class="level2">
<h2 class="anchored" data-anchor-id="mirna-quality-control">miRNA Quality Control</h2>
<p>The <em>nf-core/smrnaseq</em> pipeline performs a quality analysis specific to smRNAseq data using <em>miRTrace</em>. This analysis assesses sequencing quality, identifies the presence of miRNA and undesired sequences from tRNA, rRNA, or Illumina artifact sequences, and identifies clade-specific miRNA profiles based on a comprehensive catalog of previously identified miRNA families.</p>
<p>In the annotation step <a href="#fig-mirtrace_RNA_Categories" class="quarto-xref">Figure&nbsp;6</a>, the mapped reads against reference databases revealed that a mean of 65.06% of analyzed sequences per sample corresponded to miRNA precursors, with a range of 22.51% to 91.77%. Other categories included 21.16% tRNA sequences, 9.16% unknown sequences, 4.55% rRNA sequences, and 0.05% artifacts.</p>
<div class="cell">
<div class="sourceCode cell-code" id="cb53"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb53-1"><a href="#cb53-1" aria-hidden="true" tabindex="-1"></a><span class="do">##| </span></span>
<span id="cb53-2"><a href="#cb53-2" aria-hidden="true" tabindex="-1"></a><span class="co"># Gráfico 2: Tipos de RNA</span></span>
<span id="cb53-3"><a href="#cb53-3" aria-hidden="true" tabindex="-1"></a>tipos_rna <span class="ot">&lt;-</span> <span class="fu">read.table</span>(<span class="st">"data/mirtrace_rna_categories.txt"</span>, <span class="at">header =</span> <span class="cn">TRUE</span>, <span class="at">sep =</span> <span class="st">"</span><span class="sc">\t</span><span class="st">"</span>)</span>
<span id="cb53-4"><a href="#cb53-4" aria-hidden="true" tabindex="-1"></a>tipos_rna_long <span class="ot">&lt;-</span> tipos_rna <span class="sc">%&gt;%</span></span>
<span id="cb53-5"><a href="#cb53-5" aria-hidden="true" tabindex="-1"></a>  <span class="fu">pivot_longer</span>(<span class="at">cols =</span> <span class="fu">c</span>(miRNA, rRNA, tRNA, Artifact, Unknown), </span>
<span id="cb53-6"><a href="#cb53-6" aria-hidden="true" tabindex="-1"></a>               <span class="at">names_to =</span> <span class="st">"Tipo_RNA"</span>, </span>
<span id="cb53-7"><a href="#cb53-7" aria-hidden="true" tabindex="-1"></a>               <span class="at">values_to =</span> <span class="st">"Conteo"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb53-8"><a href="#cb53-8" aria-hidden="true" tabindex="-1"></a>  <span class="fu">group_by</span>(Sample) <span class="sc">%&gt;%</span></span>
<span id="cb53-9"><a href="#cb53-9" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">Proporcion =</span> Conteo <span class="sc">/</span> <span class="fu">sum</span>(Conteo)) <span class="sc">%&gt;%</span> </span>
<span id="cb53-10"><a href="#cb53-10" aria-hidden="true" tabindex="-1"></a>  <span class="fu">ungroup</span>()</span>
<span id="cb53-11"><a href="#cb53-11" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb53-12"><a href="#cb53-12" aria-hidden="true" tabindex="-1"></a><span class="co">#Plot</span></span>
<span id="cb53-13"><a href="#cb53-13" aria-hidden="true" tabindex="-1"></a>m2 <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(tipos_rna_long, <span class="fu">aes</span>(<span class="at">x =</span> Sample, <span class="at">y =</span> Proporcion, <span class="at">fill =</span> Tipo_RNA)) <span class="sc">+</span></span>
<span id="cb53-14"><a href="#cb53-14" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_bar</span>(<span class="at">stat =</span> <span class="st">"identity"</span>, <span class="at">position =</span> <span class="st">"stack"</span>, <span class="at">alpha =</span> <span class="fl">0.9</span>) <span class="sc">+</span></span>
<span id="cb53-15"><a href="#cb53-15" aria-hidden="true" tabindex="-1"></a>  <span class="fu">labs</span>(</span>
<span id="cb53-16"><a href="#cb53-16" aria-hidden="true" tabindex="-1"></a>    <span class="at">title =</span> <span class="st">""</span>,</span>
<span id="cb53-17"><a href="#cb53-17" aria-hidden="true" tabindex="-1"></a>    <span class="at">x =</span> <span class="st">""</span>,</span>
<span id="cb53-18"><a href="#cb53-18" aria-hidden="true" tabindex="-1"></a>    <span class="at">y =</span> <span class="st">"Quality filtered reads (%)"</span></span>
<span id="cb53-19"><a href="#cb53-19" aria-hidden="true" tabindex="-1"></a>  ) <span class="sc">+</span></span>
<span id="cb53-20"><a href="#cb53-20" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_y_continuous</span>(<span class="at">labels =</span> scales<span class="sc">::</span>percent) <span class="sc">+</span></span>
<span id="cb53-21"><a href="#cb53-21" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_fill_viridis_d</span>(<span class="at">option =</span> <span class="st">"cividis"</span>) <span class="sc">+</span></span>
<span id="cb53-22"><a href="#cb53-22" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme_minimal</span>(<span class="at">base_size =</span> <span class="dv">14</span>) <span class="sc">+</span></span>
<span id="cb53-23"><a href="#cb53-23" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme</span>(</span>
<span id="cb53-24"><a href="#cb53-24" aria-hidden="true" tabindex="-1"></a>    <span class="at">axis.text.x =</span> <span class="fu">element_text</span>(<span class="at">angle =</span> <span class="dv">90</span> , <span class="at">hjust =</span> <span class="dv">1</span>, <span class="at">size =</span> <span class="dv">5</span>),</span>
<span id="cb53-25"><a href="#cb53-25" aria-hidden="true" tabindex="-1"></a>    <span class="at">axis.text.y =</span> <span class="fu">element_text</span>(<span class="at">angle =</span> <span class="dv">0</span> , <span class="at">hjust =</span> <span class="dv">1</span>, <span class="at">size =</span> <span class="dv">7</span>),</span>
<span id="cb53-26"><a href="#cb53-26" aria-hidden="true" tabindex="-1"></a>    <span class="at">axis.title =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">8</span>),</span>
<span id="cb53-27"><a href="#cb53-27" aria-hidden="true" tabindex="-1"></a>    <span class="at">plot.title =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">12</span>, <span class="at">face =</span> <span class="st">"bold"</span>),</span>
<span id="cb53-28"><a href="#cb53-28" aria-hidden="true" tabindex="-1"></a>    <span class="at">legend.title =</span> <span class="fu">element_blank</span>(),</span>
<span id="cb53-29"><a href="#cb53-29" aria-hidden="true" tabindex="-1"></a>    <span class="at">legend.text =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">6</span>),</span>
<span id="cb53-30"><a href="#cb53-30" aria-hidden="true" tabindex="-1"></a>    <span class="at">legend.position =</span> <span class="st">"bottom"</span>,</span>
<span id="cb53-31"><a href="#cb53-31" aria-hidden="true" tabindex="-1"></a>    <span class="at">legend.margin =</span> <span class="fu">margin</span>(<span class="at">t =</span> <span class="sc">-</span><span class="dv">20</span>),  <span class="co"># Ajusta el margen superior de la leyenda</span></span>
<span id="cb53-32"><a href="#cb53-32" aria-hidden="true" tabindex="-1"></a><span class="co">#    plot.margin = margin(t = 10, r = 10, b = 30, l = 10)</span></span>
<span id="cb53-33"><a href="#cb53-33" aria-hidden="true" tabindex="-1"></a>  )</span>
<span id="cb53-34"><a href="#cb53-34" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb53-35"><a href="#cb53-35" aria-hidden="true" tabindex="-1"></a><span class="fu">print</span>(m2)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<div id="fig-mirtrace_RNA_Categories" class="quarto-float quarto-figure quarto-figure-center anchored" data-fig-pos="H">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-mirtrace_RNA_Categories-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="Results_files/figure-html/fig-mirtrace_RNA_Categories-1.png" class="img-fluid figure-img" data-fig-pos="H" width="768">
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-mirtrace_RNA_Categories-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;6: <em>miRTrace</em> (v1.0.1) Analysis: RNA Categories
</figcaption>
</figure>
</div>
</div>
</div>
<p>During the contamination assessment step <a href="#fig-mirtrace_Contamination_Check" class="quarto-xref">Figure&nbsp;7</a>, mapping miRNA precursor sequences against the clade-specific miRNA catalog showed that a mean of 94.14% of analyzed sequences belonged to the human category, with a range of 67.21% to 99.94%. Minor contributions from other clades, such as Rodentia (2.07%), Dicots (2.73%), Insects (0.58%), and Monocots (0.38%), were detected at low proportions. These identifications could result from contamination, such as incorrect index assignment during sample demultiplexing, or may have a biological origin.</p>
<div class="cell">
<div class="sourceCode cell-code" id="cb54"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb54-1"><a href="#cb54-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Gráfico 3: Contaminación</span></span>
<span id="cb54-2"><a href="#cb54-2" aria-hidden="true" tabindex="-1"></a>mirtrace_data <span class="ot">&lt;-</span> <span class="fu">read.table</span>(<span class="st">"data/mirtrace_contamination_check_plot.tsv"</span>, <span class="at">header =</span> <span class="cn">TRUE</span>, <span class="at">sep =</span> <span class="st">"</span><span class="sc">\t</span><span class="st">"</span>)</span>
<span id="cb54-3"><a href="#cb54-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb54-4"><a href="#cb54-4" aria-hidden="true" tabindex="-1"></a><span class="co"># Transformación de los datos</span></span>
<span id="cb54-5"><a href="#cb54-5" aria-hidden="true" tabindex="-1"></a>data_long <span class="ot">&lt;-</span> mirtrace_data <span class="sc">%&gt;%</span></span>
<span id="cb54-6"><a href="#cb54-6" aria-hidden="true" tabindex="-1"></a>  <span class="fu">pivot_longer</span>(<span class="at">cols =</span> <span class="fu">c</span>(<span class="st">"gymnosperms"</span>, <span class="st">"monocots"</span>, <span class="st">"dicots"</span>, <span class="st">"nematode"</span>, </span>
<span id="cb54-7"><a href="#cb54-7" aria-hidden="true" tabindex="-1"></a>                        <span class="st">"insects"</span>, <span class="st">"lophotrochozoa"</span>, <span class="st">"fish"</span>, <span class="st">"rodents"</span>, </span>
<span id="cb54-8"><a href="#cb54-8" aria-hidden="true" tabindex="-1"></a>                        <span class="st">"primates"</span>, <span class="st">"bryophytes"</span>, <span class="st">"birds_reptiles"</span>), </span>
<span id="cb54-9"><a href="#cb54-9" aria-hidden="true" tabindex="-1"></a>               <span class="at">names_to =</span> <span class="st">"Group"</span>, </span>
<span id="cb54-10"><a href="#cb54-10" aria-hidden="true" tabindex="-1"></a>               <span class="at">values_to =</span> <span class="st">"Conteo"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb54-11"><a href="#cb54-11" aria-hidden="true" tabindex="-1"></a>  <span class="fu">group_by</span>(Sample) <span class="sc">%&gt;%</span></span>
<span id="cb54-12"><a href="#cb54-12" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">Proporcion =</span> Conteo <span class="sc">/</span> <span class="fu">sum</span>(Conteo)) <span class="sc">%&gt;%</span> </span>
<span id="cb54-13"><a href="#cb54-13" aria-hidden="true" tabindex="-1"></a>  <span class="fu">ungroup</span>()</span>
<span id="cb54-14"><a href="#cb54-14" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb54-15"><a href="#cb54-15" aria-hidden="true" tabindex="-1"></a><span class="co"># Gráfico de barras apiladas al 100%</span></span>
<span id="cb54-16"><a href="#cb54-16" aria-hidden="true" tabindex="-1"></a>m3 <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(data_long, <span class="fu">aes</span>(<span class="at">x =</span> Sample, <span class="at">y =</span> Proporcion, <span class="at">fill =</span> Group)) <span class="sc">+</span></span>
<span id="cb54-17"><a href="#cb54-17" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_bar</span>(<span class="at">stat =</span> <span class="st">"identity"</span>, <span class="at">position =</span> <span class="st">"stack"</span>, <span class="at">alpha =</span> <span class="fl">0.9</span>) <span class="sc">+</span></span>
<span id="cb54-18"><a href="#cb54-18" aria-hidden="true" tabindex="-1"></a>  <span class="fu">labs</span>(</span>
<span id="cb54-19"><a href="#cb54-19" aria-hidden="true" tabindex="-1"></a>    <span class="at">title =</span> <span class="st">""</span>,</span>
<span id="cb54-20"><a href="#cb54-20" aria-hidden="true" tabindex="-1"></a>    <span class="at">x =</span> <span class="st">""</span>,</span>
<span id="cb54-21"><a href="#cb54-21" aria-hidden="true" tabindex="-1"></a>    <span class="at">y =</span> <span class="st">"Clade-specific miRNAs (%)"</span></span>
<span id="cb54-22"><a href="#cb54-22" aria-hidden="true" tabindex="-1"></a>  ) <span class="sc">+</span></span>
<span id="cb54-23"><a href="#cb54-23" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_y_continuous</span>(<span class="at">labels =</span> scales<span class="sc">::</span>percent) <span class="sc">+</span></span>
<span id="cb54-24"><a href="#cb54-24" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_fill_viridis_d</span>(<span class="at">option =</span> <span class="st">"cividis"</span>, </span>
<span id="cb54-25"><a href="#cb54-25" aria-hidden="true" tabindex="-1"></a>                       <span class="at">labels =</span> <span class="fu">c</span>(</span>
<span id="cb54-26"><a href="#cb54-26" aria-hidden="true" tabindex="-1"></a>      <span class="st">"gymnosperms"</span> <span class="ot">=</span> <span class="st">"Gymnosperms"</span>,</span>
<span id="cb54-27"><a href="#cb54-27" aria-hidden="true" tabindex="-1"></a>      <span class="st">"monocots"</span> <span class="ot">=</span> <span class="st">"Monocots"</span>,</span>
<span id="cb54-28"><a href="#cb54-28" aria-hidden="true" tabindex="-1"></a>      <span class="st">"dicots"</span> <span class="ot">=</span> <span class="st">"Dicots"</span>,</span>
<span id="cb54-29"><a href="#cb54-29" aria-hidden="true" tabindex="-1"></a>      <span class="st">"nematode"</span> <span class="ot">=</span> <span class="st">"Nematode"</span>,</span>
<span id="cb54-30"><a href="#cb54-30" aria-hidden="true" tabindex="-1"></a>      <span class="st">"insects"</span> <span class="ot">=</span> <span class="st">"Insects"</span>,</span>
<span id="cb54-31"><a href="#cb54-31" aria-hidden="true" tabindex="-1"></a>      <span class="st">"lophotrochozoa"</span> <span class="ot">=</span> <span class="st">"Lophotrochozoa"</span>,</span>
<span id="cb54-32"><a href="#cb54-32" aria-hidden="true" tabindex="-1"></a>      <span class="st">"fish"</span> <span class="ot">=</span> <span class="st">"Fish"</span>,</span>
<span id="cb54-33"><a href="#cb54-33" aria-hidden="true" tabindex="-1"></a>      <span class="st">"rodents"</span> <span class="ot">=</span> <span class="st">"Rodents"</span>,</span>
<span id="cb54-34"><a href="#cb54-34" aria-hidden="true" tabindex="-1"></a>      <span class="st">"primates"</span> <span class="ot">=</span> <span class="st">"Primates"</span>,</span>
<span id="cb54-35"><a href="#cb54-35" aria-hidden="true" tabindex="-1"></a>      <span class="st">"bryophytes"</span> <span class="ot">=</span> <span class="st">"Bryophytes"</span>,</span>
<span id="cb54-36"><a href="#cb54-36" aria-hidden="true" tabindex="-1"></a>      <span class="st">"birds_reptiles"</span> <span class="ot">=</span> <span class="st">"Birds &amp; Reptiles"</span></span>
<span id="cb54-37"><a href="#cb54-37" aria-hidden="true" tabindex="-1"></a>    )) <span class="sc">+</span></span>
<span id="cb54-38"><a href="#cb54-38" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme_minimal</span>(<span class="at">base_size =</span> <span class="dv">14</span>) <span class="sc">+</span></span>
<span id="cb54-39"><a href="#cb54-39" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme</span>(</span>
<span id="cb54-40"><a href="#cb54-40" aria-hidden="true" tabindex="-1"></a>    <span class="at">axis.text.x =</span> <span class="fu">element_text</span>(<span class="at">angle =</span> <span class="dv">90</span>, <span class="at">hjust =</span> <span class="dv">1</span>, <span class="at">size =</span> <span class="dv">5</span>),</span>
<span id="cb54-41"><a href="#cb54-41" aria-hidden="true" tabindex="-1"></a>    <span class="at">axis.text.y =</span> <span class="fu">element_text</span>(<span class="at">angle =</span> <span class="dv">0</span>, <span class="at">hjust =</span> <span class="dv">1</span>, <span class="at">size =</span> <span class="dv">7</span>),</span>
<span id="cb54-42"><a href="#cb54-42" aria-hidden="true" tabindex="-1"></a>    <span class="at">axis.title =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">8</span>),</span>
<span id="cb54-43"><a href="#cb54-43" aria-hidden="true" tabindex="-1"></a>    <span class="at">plot.title =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">12</span>, <span class="at">face =</span> <span class="st">"bold"</span>),</span>
<span id="cb54-44"><a href="#cb54-44" aria-hidden="true" tabindex="-1"></a>    <span class="at">legend.title =</span> <span class="fu">element_blank</span>(),</span>
<span id="cb54-45"><a href="#cb54-45" aria-hidden="true" tabindex="-1"></a>    <span class="at">legend.text =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">6</span>),</span>
<span id="cb54-46"><a href="#cb54-46" aria-hidden="true" tabindex="-1"></a>    <span class="at">legend.position =</span> <span class="st">"bottom"</span>,</span>
<span id="cb54-47"><a href="#cb54-47" aria-hidden="true" tabindex="-1"></a>    <span class="at">legend.margin =</span> <span class="fu">margin</span>(<span class="at">t =</span> <span class="sc">-</span><span class="dv">20</span>),  <span class="co"># Ajusta el margen superior de la leyenda</span></span>
<span id="cb54-48"><a href="#cb54-48" aria-hidden="true" tabindex="-1"></a>    <span class="at">legend.spacing.x =</span> <span class="fu">unit</span>(<span class="fl">0.3</span>, <span class="st">"cm"</span>),</span>
<span id="cb54-49"><a href="#cb54-49" aria-hidden="true" tabindex="-1"></a><span class="co">#    plot.margin = margin(t = 10, r = 10, b = 30, l = 10)</span></span>
<span id="cb54-50"><a href="#cb54-50" aria-hidden="true" tabindex="-1"></a>  ) <span class="sc">+</span></span>
<span id="cb54-51"><a href="#cb54-51" aria-hidden="true" tabindex="-1"></a>  <span class="fu">guides</span>(<span class="at">fill =</span> <span class="fu">guide_legend</span>(<span class="at">nrow =</span> <span class="dv">2</span>))  <span class="co"># Forzar que la leyenda tenga 2 filas</span></span>
<span id="cb54-52"><a href="#cb54-52" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb54-53"><a href="#cb54-53" aria-hidden="true" tabindex="-1"></a><span class="fu">print</span>(m3)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<div id="fig-mirtrace_Contamination_Check" class="quarto-float quarto-figure quarto-figure-center anchored" data-fig-pos="H">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-mirtrace_Contamination_Check-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="Results_files/figure-html/fig-mirtrace_Contamination_Check-1.png" class="img-fluid figure-img" data-fig-pos="H" width="768">
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-mirtrace_Contamination_Check-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;7: <em>miRTrace</em> (v1.0.1) Analysis: Contamination Check
</figcaption>
</figure>
</div>
</div>
</div>
<div class="cell">
<div class="sourceCode cell-code" id="cb55"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb55-1"><a href="#cb55-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Cargar datos</span></span>
<span id="cb55-2"><a href="#cb55-2" aria-hidden="true" tabindex="-1"></a>qc_mirna <span class="ot">&lt;-</span> <span class="fu">read.table</span>(<span class="st">"data/mirtrace_qc_plot.tsv"</span>, <span class="at">header =</span> <span class="cn">TRUE</span>, <span class="at">sep =</span> <span class="st">"</span><span class="sc">\t</span><span class="st">"</span>)</span>
<span id="cb55-3"><a href="#cb55-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb55-4"><a href="#cb55-4" aria-hidden="true" tabindex="-1"></a><span class="co"># Transformación de los datos</span></span>
<span id="cb55-5"><a href="#cb55-5" aria-hidden="true" tabindex="-1"></a>data_long <span class="ot">&lt;-</span> qc_mirna <span class="sc">%&gt;%</span></span>
<span id="cb55-6"><a href="#cb55-6" aria-hidden="true" tabindex="-1"></a>  <span class="fu">pivot_longer</span>(<span class="at">cols =</span> <span class="sc">-</span>Sample, <span class="at">names_to =</span> <span class="st">"Category"</span>, <span class="at">values_to =</span> <span class="st">"Reads"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb55-7"><a href="#cb55-7" aria-hidden="true" tabindex="-1"></a>  <span class="fu">group_by</span>(Sample) <span class="sc">%&gt;%</span></span>
<span id="cb55-8"><a href="#cb55-8" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">Proportion =</span> Reads <span class="sc">/</span> <span class="fu">sum</span>(Reads)) <span class="sc">%&gt;%</span>  <span class="co"># Calcular proporciones por muestra</span></span>
<span id="cb55-9"><a href="#cb55-9" aria-hidden="true" tabindex="-1"></a>  <span class="fu">ungroup</span>()</span>
<span id="cb55-10"><a href="#cb55-10" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb55-11"><a href="#cb55-11" aria-hidden="true" tabindex="-1"></a><span class="co"># Gráfico de barras apiladas al 100%</span></span>
<span id="cb55-12"><a href="#cb55-12" aria-hidden="true" tabindex="-1"></a>m1 <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(data_long, <span class="fu">aes</span>(<span class="at">x =</span> Sample, <span class="at">y =</span> Proportion, <span class="at">fill =</span> Category)) <span class="sc">+</span></span>
<span id="cb55-13"><a href="#cb55-13" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_bar</span>(<span class="at">stat =</span> <span class="st">"identity"</span>, <span class="at">position =</span> <span class="st">"stack"</span>, <span class="at">alpha =</span> <span class="fl">0.9</span>) <span class="sc">+</span></span>
<span id="cb55-14"><a href="#cb55-14" aria-hidden="true" tabindex="-1"></a>  <span class="fu">labs</span>(</span>
<span id="cb55-15"><a href="#cb55-15" aria-hidden="true" tabindex="-1"></a>    <span class="at">title =</span> <span class="st">"A"</span>,</span>
<span id="cb55-16"><a href="#cb55-16" aria-hidden="true" tabindex="-1"></a>    <span class="at">x =</span> <span class="st">""</span>,</span>
<span id="cb55-17"><a href="#cb55-17" aria-hidden="true" tabindex="-1"></a>    <span class="at">y =</span> <span class="st">"Reads (%)"</span></span>
<span id="cb55-18"><a href="#cb55-18" aria-hidden="true" tabindex="-1"></a>  ) <span class="sc">+</span></span>
<span id="cb55-19"><a href="#cb55-19" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_y_continuous</span>(<span class="at">labels =</span> scales<span class="sc">::</span>percent) <span class="sc">+</span></span>
<span id="cb55-20"><a href="#cb55-20" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_fill_viridis_d</span>(<span class="at">option =</span> <span class="st">"cividis"</span>, </span>
<span id="cb55-21"><a href="#cb55-21" aria-hidden="true" tabindex="-1"></a>                       <span class="at">labels =</span> <span class="fu">c</span>(</span>
<span id="cb55-22"><a href="#cb55-22" aria-hidden="true" tabindex="-1"></a>                         <span class="st">"Reads...18.nt.after.adapter.removal"</span> <span class="ot">=</span> <span class="st">"Adapter detected, insert &gt;= 18 nt"</span>,</span>
<span id="cb55-23"><a href="#cb55-23" aria-hidden="true" tabindex="-1"></a>                         <span class="st">"Reads.without.adapter"</span> <span class="ot">=</span> <span class="st">"Adapter not detected"</span>,</span>
<span id="cb55-24"><a href="#cb55-24" aria-hidden="true" tabindex="-1"></a>                         <span class="st">"Reads...18.nt.after.adapter.removal.1"</span> <span class="ot">=</span> <span class="st">"Length &lt; 18 nt"</span>,</span>
<span id="cb55-25"><a href="#cb55-25" aria-hidden="true" tabindex="-1"></a>                         <span class="st">"Reads.with.low.complexity"</span> <span class="ot">=</span> <span class="st">"Low complexity"</span>,</span>
<span id="cb55-26"><a href="#cb55-26" aria-hidden="true" tabindex="-1"></a>                         <span class="st">"Reads.with.low.PHRED.score"</span> <span class="ot">=</span> <span class="st">"Low PHRED score"</span></span>
<span id="cb55-27"><a href="#cb55-27" aria-hidden="true" tabindex="-1"></a>                       ))<span class="sc">+</span></span>
<span id="cb55-28"><a href="#cb55-28" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme_minimal</span>(<span class="at">base_size =</span> <span class="dv">14</span>) <span class="sc">+</span></span>
<span id="cb55-29"><a href="#cb55-29" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme</span>(</span>
<span id="cb55-30"><a href="#cb55-30" aria-hidden="true" tabindex="-1"></a>    <span class="at">axis.text.x =</span> <span class="fu">element_text</span>(<span class="at">angle =</span> <span class="dv">90</span>, <span class="at">hjust =</span> <span class="dv">1</span>, <span class="at">size =</span> <span class="dv">5</span>),</span>
<span id="cb55-31"><a href="#cb55-31" aria-hidden="true" tabindex="-1"></a>    <span class="at">axis.text.y =</span> <span class="fu">element_text</span>(<span class="at">angle =</span> <span class="dv">0</span>, <span class="at">hjust =</span> <span class="dv">1</span>, <span class="at">size =</span> <span class="dv">7</span>),</span>
<span id="cb55-32"><a href="#cb55-32" aria-hidden="true" tabindex="-1"></a>    <span class="at">axis.title =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">8</span>),</span>
<span id="cb55-33"><a href="#cb55-33" aria-hidden="true" tabindex="-1"></a>    <span class="at">plot.title =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">12</span>, <span class="at">face =</span> <span class="st">"bold"</span>),</span>
<span id="cb55-34"><a href="#cb55-34" aria-hidden="true" tabindex="-1"></a>    <span class="at">legend.title =</span> <span class="fu">element_blank</span>(),</span>
<span id="cb55-35"><a href="#cb55-35" aria-hidden="true" tabindex="-1"></a>    <span class="at">legend.text =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">6</span>),</span>
<span id="cb55-36"><a href="#cb55-36" aria-hidden="true" tabindex="-1"></a>    <span class="at">legend.position =</span> <span class="st">"bottom"</span>,</span>
<span id="cb55-37"><a href="#cb55-37" aria-hidden="true" tabindex="-1"></a>    <span class="at">legend.margin =</span> <span class="fu">margin</span>(<span class="at">t =</span> <span class="sc">-</span><span class="dv">20</span>),  <span class="co"># Ajusta el margen superior de la leyenda</span></span>
<span id="cb55-38"><a href="#cb55-38" aria-hidden="true" tabindex="-1"></a>  )</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<div class="cell">
<div class="sourceCode cell-code" id="cb56"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb56-1"><a href="#cb56-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Gráfico 2: Tipos de RNA</span></span>
<span id="cb56-2"><a href="#cb56-2" aria-hidden="true" tabindex="-1"></a>tipos_rna <span class="ot">&lt;-</span> <span class="fu">read.table</span>(<span class="st">"data/mirtrace_rna_categories.txt"</span>, <span class="at">header =</span> <span class="cn">TRUE</span>, <span class="at">sep =</span> <span class="st">"</span><span class="sc">\t</span><span class="st">"</span>)</span>
<span id="cb56-3"><a href="#cb56-3" aria-hidden="true" tabindex="-1"></a>tipos_rna_long <span class="ot">&lt;-</span> tipos_rna <span class="sc">%&gt;%</span></span>
<span id="cb56-4"><a href="#cb56-4" aria-hidden="true" tabindex="-1"></a>  <span class="fu">pivot_longer</span>(<span class="at">cols =</span> <span class="fu">c</span>(miRNA, rRNA, tRNA, Artifact, Unknown), </span>
<span id="cb56-5"><a href="#cb56-5" aria-hidden="true" tabindex="-1"></a>               <span class="at">names_to =</span> <span class="st">"Tipo_RNA"</span>, </span>
<span id="cb56-6"><a href="#cb56-6" aria-hidden="true" tabindex="-1"></a>               <span class="at">values_to =</span> <span class="st">"Conteo"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb56-7"><a href="#cb56-7" aria-hidden="true" tabindex="-1"></a>  <span class="fu">group_by</span>(Sample) <span class="sc">%&gt;%</span></span>
<span id="cb56-8"><a href="#cb56-8" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">Proporcion =</span> Conteo <span class="sc">/</span> <span class="fu">sum</span>(Conteo)) <span class="sc">%&gt;%</span> </span>
<span id="cb56-9"><a href="#cb56-9" aria-hidden="true" tabindex="-1"></a>  <span class="fu">ungroup</span>()</span>
<span id="cb56-10"><a href="#cb56-10" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb56-11"><a href="#cb56-11" aria-hidden="true" tabindex="-1"></a><span class="co">#Plot</span></span>
<span id="cb56-12"><a href="#cb56-12" aria-hidden="true" tabindex="-1"></a>m2 <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(tipos_rna_long, <span class="fu">aes</span>(<span class="at">x =</span> Sample, <span class="at">y =</span> Proporcion, <span class="at">fill =</span> Tipo_RNA)) <span class="sc">+</span></span>
<span id="cb56-13"><a href="#cb56-13" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_bar</span>(<span class="at">stat =</span> <span class="st">"identity"</span>, <span class="at">position =</span> <span class="st">"stack"</span>, <span class="at">alpha =</span> <span class="fl">0.9</span>) <span class="sc">+</span></span>
<span id="cb56-14"><a href="#cb56-14" aria-hidden="true" tabindex="-1"></a>  <span class="fu">labs</span>(</span>
<span id="cb56-15"><a href="#cb56-15" aria-hidden="true" tabindex="-1"></a>    <span class="at">title =</span> <span class="st">"A"</span>,</span>
<span id="cb56-16"><a href="#cb56-16" aria-hidden="true" tabindex="-1"></a>    <span class="at">x =</span> <span class="st">""</span>,</span>
<span id="cb56-17"><a href="#cb56-17" aria-hidden="true" tabindex="-1"></a>    <span class="at">y =</span> <span class="st">"Quality filtered reads (%)"</span></span>
<span id="cb56-18"><a href="#cb56-18" aria-hidden="true" tabindex="-1"></a>  ) <span class="sc">+</span></span>
<span id="cb56-19"><a href="#cb56-19" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_y_continuous</span>(<span class="at">labels =</span> scales<span class="sc">::</span>percent) <span class="sc">+</span></span>
<span id="cb56-20"><a href="#cb56-20" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_fill_viridis_d</span>(<span class="at">option =</span> <span class="st">"cividis"</span>) <span class="sc">+</span></span>
<span id="cb56-21"><a href="#cb56-21" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme_minimal</span>(<span class="at">base_size =</span> <span class="dv">14</span>) <span class="sc">+</span></span>
<span id="cb56-22"><a href="#cb56-22" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme</span>(</span>
<span id="cb56-23"><a href="#cb56-23" aria-hidden="true" tabindex="-1"></a>    <span class="at">axis.text.x =</span> <span class="fu">element_text</span>(<span class="at">angle =</span> <span class="dv">90</span> , <span class="at">hjust =</span> <span class="dv">1</span>, <span class="at">size =</span> <span class="dv">5</span>),</span>
<span id="cb56-24"><a href="#cb56-24" aria-hidden="true" tabindex="-1"></a>    <span class="at">axis.text.y =</span> <span class="fu">element_text</span>(<span class="at">angle =</span> <span class="dv">0</span> , <span class="at">hjust =</span> <span class="dv">1</span>, <span class="at">size =</span> <span class="dv">7</span>),</span>
<span id="cb56-25"><a href="#cb56-25" aria-hidden="true" tabindex="-1"></a>    <span class="at">axis.title =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">8</span>),</span>
<span id="cb56-26"><a href="#cb56-26" aria-hidden="true" tabindex="-1"></a>    <span class="at">plot.title =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">12</span>, <span class="at">face =</span> <span class="st">"bold"</span>),</span>
<span id="cb56-27"><a href="#cb56-27" aria-hidden="true" tabindex="-1"></a>    <span class="at">legend.title =</span> <span class="fu">element_blank</span>(),</span>
<span id="cb56-28"><a href="#cb56-28" aria-hidden="true" tabindex="-1"></a>    <span class="at">legend.text =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">6</span>),</span>
<span id="cb56-29"><a href="#cb56-29" aria-hidden="true" tabindex="-1"></a>    <span class="at">legend.position =</span> <span class="st">"bottom"</span>,</span>
<span id="cb56-30"><a href="#cb56-30" aria-hidden="true" tabindex="-1"></a>    <span class="at">legend.margin =</span> <span class="fu">margin</span>(<span class="at">t =</span> <span class="sc">-</span><span class="dv">20</span>),  <span class="co"># Ajusta el margen superior de la leyenda</span></span>
<span id="cb56-31"><a href="#cb56-31" aria-hidden="true" tabindex="-1"></a><span class="co">#    plot.margin = margin(t = 10, r = 10, b = 30, l = 10)</span></span>
<span id="cb56-32"><a href="#cb56-32" aria-hidden="true" tabindex="-1"></a>  )</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<div class="cell">
<div class="sourceCode cell-code" id="cb57"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb57-1"><a href="#cb57-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Gráfico 3: Contaminación</span></span>
<span id="cb57-2"><a href="#cb57-2" aria-hidden="true" tabindex="-1"></a>mirtrace_data <span class="ot">&lt;-</span> <span class="fu">read.table</span>(<span class="st">"data/mirtrace_contamination_check_plot.tsv"</span>, <span class="at">header =</span> <span class="cn">TRUE</span>, <span class="at">sep =</span> <span class="st">"</span><span class="sc">\t</span><span class="st">"</span>)</span>
<span id="cb57-3"><a href="#cb57-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb57-4"><a href="#cb57-4" aria-hidden="true" tabindex="-1"></a><span class="co"># Transformación de los datos</span></span>
<span id="cb57-5"><a href="#cb57-5" aria-hidden="true" tabindex="-1"></a>data_long <span class="ot">&lt;-</span> mirtrace_data <span class="sc">%&gt;%</span></span>
<span id="cb57-6"><a href="#cb57-6" aria-hidden="true" tabindex="-1"></a>  <span class="fu">pivot_longer</span>(<span class="at">cols =</span> <span class="fu">c</span>(<span class="st">"gymnosperms"</span>, <span class="st">"monocots"</span>, <span class="st">"dicots"</span>, <span class="st">"nematode"</span>, </span>
<span id="cb57-7"><a href="#cb57-7" aria-hidden="true" tabindex="-1"></a>                        <span class="st">"insects"</span>, <span class="st">"lophotrochozoa"</span>, <span class="st">"fish"</span>, <span class="st">"rodents"</span>, </span>
<span id="cb57-8"><a href="#cb57-8" aria-hidden="true" tabindex="-1"></a>                        <span class="st">"primates"</span>, <span class="st">"bryophytes"</span>, <span class="st">"birds_reptiles"</span>), </span>
<span id="cb57-9"><a href="#cb57-9" aria-hidden="true" tabindex="-1"></a>               <span class="at">names_to =</span> <span class="st">"Group"</span>, </span>
<span id="cb57-10"><a href="#cb57-10" aria-hidden="true" tabindex="-1"></a>               <span class="at">values_to =</span> <span class="st">"Conteo"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb57-11"><a href="#cb57-11" aria-hidden="true" tabindex="-1"></a>  <span class="fu">group_by</span>(Sample) <span class="sc">%&gt;%</span></span>
<span id="cb57-12"><a href="#cb57-12" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">Proporcion =</span> Conteo <span class="sc">/</span> <span class="fu">sum</span>(Conteo)) <span class="sc">%&gt;%</span> </span>
<span id="cb57-13"><a href="#cb57-13" aria-hidden="true" tabindex="-1"></a>  <span class="fu">ungroup</span>()</span>
<span id="cb57-14"><a href="#cb57-14" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb57-15"><a href="#cb57-15" aria-hidden="true" tabindex="-1"></a><span class="co"># Gráfico de barras apiladas al 100%</span></span>
<span id="cb57-16"><a href="#cb57-16" aria-hidden="true" tabindex="-1"></a>m3 <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(data_long, <span class="fu">aes</span>(<span class="at">x =</span> Sample, <span class="at">y =</span> Proporcion, <span class="at">fill =</span> Group)) <span class="sc">+</span></span>
<span id="cb57-17"><a href="#cb57-17" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_bar</span>(<span class="at">stat =</span> <span class="st">"identity"</span>, <span class="at">position =</span> <span class="st">"stack"</span>, <span class="at">alpha =</span> <span class="fl">0.9</span>) <span class="sc">+</span></span>
<span id="cb57-18"><a href="#cb57-18" aria-hidden="true" tabindex="-1"></a>  <span class="fu">labs</span>(</span>
<span id="cb57-19"><a href="#cb57-19" aria-hidden="true" tabindex="-1"></a>    <span class="at">title =</span> <span class="st">"B"</span>,</span>
<span id="cb57-20"><a href="#cb57-20" aria-hidden="true" tabindex="-1"></a>    <span class="at">x =</span> <span class="st">""</span>,</span>
<span id="cb57-21"><a href="#cb57-21" aria-hidden="true" tabindex="-1"></a>    <span class="at">y =</span> <span class="st">"Clade-specific miRNAs (%)"</span></span>
<span id="cb57-22"><a href="#cb57-22" aria-hidden="true" tabindex="-1"></a>  ) <span class="sc">+</span></span>
<span id="cb57-23"><a href="#cb57-23" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_y_continuous</span>(<span class="at">labels =</span> scales<span class="sc">::</span>percent) <span class="sc">+</span></span>
<span id="cb57-24"><a href="#cb57-24" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_fill_viridis_d</span>(<span class="at">option =</span> <span class="st">"cividis"</span>, </span>
<span id="cb57-25"><a href="#cb57-25" aria-hidden="true" tabindex="-1"></a>                       <span class="at">labels =</span> <span class="fu">c</span>(</span>
<span id="cb57-26"><a href="#cb57-26" aria-hidden="true" tabindex="-1"></a>      <span class="st">"gymnosperms"</span> <span class="ot">=</span> <span class="st">"Gymnosperms"</span>,</span>
<span id="cb57-27"><a href="#cb57-27" aria-hidden="true" tabindex="-1"></a>      <span class="st">"monocots"</span> <span class="ot">=</span> <span class="st">"Monocots"</span>,</span>
<span id="cb57-28"><a href="#cb57-28" aria-hidden="true" tabindex="-1"></a>      <span class="st">"dicots"</span> <span class="ot">=</span> <span class="st">"Dicots"</span>,</span>
<span id="cb57-29"><a href="#cb57-29" aria-hidden="true" tabindex="-1"></a>      <span class="st">"nematode"</span> <span class="ot">=</span> <span class="st">"Nematode"</span>,</span>
<span id="cb57-30"><a href="#cb57-30" aria-hidden="true" tabindex="-1"></a>      <span class="st">"insects"</span> <span class="ot">=</span> <span class="st">"Insects"</span>,</span>
<span id="cb57-31"><a href="#cb57-31" aria-hidden="true" tabindex="-1"></a>      <span class="st">"lophotrochozoa"</span> <span class="ot">=</span> <span class="st">"Lophotrochozoa"</span>,</span>
<span id="cb57-32"><a href="#cb57-32" aria-hidden="true" tabindex="-1"></a>      <span class="st">"fish"</span> <span class="ot">=</span> <span class="st">"Fish"</span>,</span>
<span id="cb57-33"><a href="#cb57-33" aria-hidden="true" tabindex="-1"></a>      <span class="st">"rodents"</span> <span class="ot">=</span> <span class="st">"Rodents"</span>,</span>
<span id="cb57-34"><a href="#cb57-34" aria-hidden="true" tabindex="-1"></a>      <span class="st">"primates"</span> <span class="ot">=</span> <span class="st">"Primates"</span>,</span>
<span id="cb57-35"><a href="#cb57-35" aria-hidden="true" tabindex="-1"></a>      <span class="st">"bryophytes"</span> <span class="ot">=</span> <span class="st">"Bryophytes"</span>,</span>
<span id="cb57-36"><a href="#cb57-36" aria-hidden="true" tabindex="-1"></a>      <span class="st">"birds_reptiles"</span> <span class="ot">=</span> <span class="st">"Birds &amp; Reptiles"</span></span>
<span id="cb57-37"><a href="#cb57-37" aria-hidden="true" tabindex="-1"></a>    )) <span class="sc">+</span></span>
<span id="cb57-38"><a href="#cb57-38" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme_minimal</span>(<span class="at">base_size =</span> <span class="dv">14</span>) <span class="sc">+</span></span>
<span id="cb57-39"><a href="#cb57-39" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme</span>(</span>
<span id="cb57-40"><a href="#cb57-40" aria-hidden="true" tabindex="-1"></a>    <span class="at">axis.text.x =</span> <span class="fu">element_text</span>(<span class="at">angle =</span> <span class="dv">90</span>, <span class="at">hjust =</span> <span class="dv">1</span>, <span class="at">size =</span> <span class="dv">5</span>),</span>
<span id="cb57-41"><a href="#cb57-41" aria-hidden="true" tabindex="-1"></a>    <span class="at">axis.text.y =</span> <span class="fu">element_text</span>(<span class="at">angle =</span> <span class="dv">0</span>, <span class="at">hjust =</span> <span class="dv">1</span>, <span class="at">size =</span> <span class="dv">7</span>),</span>
<span id="cb57-42"><a href="#cb57-42" aria-hidden="true" tabindex="-1"></a>    <span class="at">axis.title =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">8</span>),</span>
<span id="cb57-43"><a href="#cb57-43" aria-hidden="true" tabindex="-1"></a>    <span class="at">plot.title =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">12</span>, <span class="at">face =</span> <span class="st">"bold"</span>),</span>
<span id="cb57-44"><a href="#cb57-44" aria-hidden="true" tabindex="-1"></a>    <span class="at">legend.title =</span> <span class="fu">element_blank</span>(),</span>
<span id="cb57-45"><a href="#cb57-45" aria-hidden="true" tabindex="-1"></a>    <span class="at">legend.text =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">6</span>),</span>
<span id="cb57-46"><a href="#cb57-46" aria-hidden="true" tabindex="-1"></a>    <span class="at">legend.position =</span> <span class="st">"bottom"</span>,</span>
<span id="cb57-47"><a href="#cb57-47" aria-hidden="true" tabindex="-1"></a>    <span class="at">legend.margin =</span> <span class="fu">margin</span>(<span class="at">t =</span> <span class="sc">-</span><span class="dv">20</span>),  <span class="co"># Ajusta el margen superior de la leyenda</span></span>
<span id="cb57-48"><a href="#cb57-48" aria-hidden="true" tabindex="-1"></a>    <span class="at">legend.spacing.x =</span> <span class="fu">unit</span>(<span class="fl">0.3</span>, <span class="st">"cm"</span>),</span>
<span id="cb57-49"><a href="#cb57-49" aria-hidden="true" tabindex="-1"></a><span class="co">#    plot.margin = margin(t = 10, r = 10, b = 30, l = 10)</span></span>
<span id="cb57-50"><a href="#cb57-50" aria-hidden="true" tabindex="-1"></a>  ) <span class="sc">+</span></span>
<span id="cb57-51"><a href="#cb57-51" aria-hidden="true" tabindex="-1"></a>  <span class="fu">guides</span>(<span class="at">fill =</span> <span class="fu">guide_legend</span>(<span class="at">nrow =</span> <span class="dv">2</span>))  <span class="co"># Forzar que la leyenda tenga 2 filas</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<div class="cell">
<div class="sourceCode cell-code" id="cb58"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb58-1"><a href="#cb58-1" aria-hidden="true" tabindex="-1"></a><span class="do">##| label: fig-mirtrace</span></span>
<span id="cb58-2"><a href="#cb58-2" aria-hidden="true" tabindex="-1"></a><span class="do">##| fig-cap: "*miRTrace* (v1.0.1) Analysis. *A*: RNA Categories; *B*: Contamination Check"</span></span>
<span id="cb58-3"><a href="#cb58-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb58-4"><a href="#cb58-4" aria-hidden="true" tabindex="-1"></a><span class="co">#grid.arrange(m2, m3,nrow = 2,  heights = rep(1, 2))</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<div class="cell">
<div class="sourceCode cell-code" id="cb59"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb59-1"><a href="#cb59-1" aria-hidden="true" tabindex="-1"></a><span class="do">##| tbl-cap: "Descriptive statistics of Isomir Read counts "</span></span>
<span id="cb59-2"><a href="#cb59-2" aria-hidden="true" tabindex="-1"></a><span class="do">##| label: "tbl-isomir_read_counts"</span></span>
<span id="cb59-3"><a href="#cb59-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb59-4"><a href="#cb59-4" aria-hidden="true" tabindex="-1"></a>tipos_rna <span class="ot">&lt;-</span> <span class="fu">read.table</span>(<span class="st">"data/mirtrace_rna_categories.txt"</span>, <span class="at">header =</span> <span class="cn">TRUE</span>, <span class="at">sep =</span> <span class="st">"</span><span class="sc">\t</span><span class="st">"</span>)</span>
<span id="cb59-5"><a href="#cb59-5" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb59-6"><a href="#cb59-6" aria-hidden="true" tabindex="-1"></a>tipos_rna <span class="ot">&lt;-</span> tipos_rna <span class="sc">%&gt;%</span></span>
<span id="cb59-7"><a href="#cb59-7" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">Total =</span> (miRNA<span class="sc">+</span> tRNA <span class="sc">+</span> rRNA <span class="sc">+</span>  <span class="sc">+</span> Artifact <span class="sc">+</span> Unknown),</span>
<span id="cb59-8"><a href="#cb59-8" aria-hidden="true" tabindex="-1"></a>         <span class="at">miRNA_percentage =</span> (miRNA<span class="sc">/</span>Total)<span class="sc">*</span> <span class="dv">100</span>,</span>
<span id="cb59-9"><a href="#cb59-9" aria-hidden="true" tabindex="-1"></a>         <span class="at">tRNA_percentage =</span> (tRNA<span class="sc">/</span>Total)<span class="sc">*</span> <span class="dv">100</span>,</span>
<span id="cb59-10"><a href="#cb59-10" aria-hidden="true" tabindex="-1"></a>         <span class="at">rRNA_percentgae =</span> (rRNA<span class="sc">/</span>Total)<span class="sc">*</span> <span class="dv">100</span>,</span>
<span id="cb59-11"><a href="#cb59-11" aria-hidden="true" tabindex="-1"></a>         <span class="at">Artifact_percentage =</span> (Artifact<span class="sc">/</span>Total)<span class="sc">*</span> <span class="dv">100</span>,</span>
<span id="cb59-12"><a href="#cb59-12" aria-hidden="true" tabindex="-1"></a>         <span class="at">Unknown_percentage =</span> (Unknown<span class="sc">/</span>Total)<span class="sc">*</span> <span class="dv">100</span>)</span>
<span id="cb59-13"><a href="#cb59-13" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb59-14"><a href="#cb59-14" aria-hidden="true" tabindex="-1"></a>stats_tipos_rna <span class="ot">&lt;-</span> tipos_rna <span class="sc">%&gt;%</span></span>
<span id="cb59-15"><a href="#cb59-15" aria-hidden="true" tabindex="-1"></a>  <span class="fu">summarise</span>(</span>
<span id="cb59-16"><a href="#cb59-16" aria-hidden="true" tabindex="-1"></a>    <span class="at">Mean_miRNA_percentage =</span> <span class="fu">mean</span>(miRNA_percentage, <span class="at">na.rm =</span> <span class="cn">TRUE</span>),</span>
<span id="cb59-17"><a href="#cb59-17" aria-hidden="true" tabindex="-1"></a>    <span class="at">Max_miRNA_percentage =</span> <span class="fu">max</span>(miRNA_percentage, <span class="at">na.rm =</span> <span class="cn">TRUE</span>),</span>
<span id="cb59-18"><a href="#cb59-18" aria-hidden="true" tabindex="-1"></a>    <span class="at">Min_miRNA_percentage =</span> <span class="fu">min</span>(miRNA_percentage, <span class="at">na.rm =</span> <span class="cn">TRUE</span>),</span>
<span id="cb59-19"><a href="#cb59-19" aria-hidden="true" tabindex="-1"></a>    <span class="at">Mean_tRNA_percentage =</span> <span class="fu">mean</span>(tRNA_percentage, <span class="at">na.rm =</span> <span class="cn">TRUE</span>),</span>
<span id="cb59-20"><a href="#cb59-20" aria-hidden="true" tabindex="-1"></a>    <span class="at">Max_tRNA_percentage =</span> <span class="fu">max</span>(tRNA_percentage, <span class="at">na.rm =</span> <span class="cn">TRUE</span>),</span>
<span id="cb59-21"><a href="#cb59-21" aria-hidden="true" tabindex="-1"></a>    <span class="at">Min_tRNA_percentage =</span> <span class="fu">min</span>(tRNA_percentage, <span class="at">na.rm =</span> <span class="cn">TRUE</span>),</span>
<span id="cb59-22"><a href="#cb59-22" aria-hidden="true" tabindex="-1"></a>    <span class="at">Mean_rRNA_percentgae =</span> <span class="fu">mean</span>(rRNA_percentgae, <span class="at">na.rm =</span> <span class="cn">TRUE</span>),</span>
<span id="cb59-23"><a href="#cb59-23" aria-hidden="true" tabindex="-1"></a>    <span class="at">Max_rRNA_percentgae =</span> <span class="fu">max</span>(rRNA_percentgae, <span class="at">na.rm =</span> <span class="cn">TRUE</span>),</span>
<span id="cb59-24"><a href="#cb59-24" aria-hidden="true" tabindex="-1"></a>    <span class="at">Min_rRNA_percentgae =</span> <span class="fu">min</span>(rRNA_percentgae, <span class="at">na.rm =</span> <span class="cn">TRUE</span>), </span>
<span id="cb59-25"><a href="#cb59-25" aria-hidden="true" tabindex="-1"></a>    <span class="at">Mean_Artifact_percentage =</span> <span class="fu">mean</span>(Artifact_percentage, <span class="at">na.rm =</span> <span class="cn">TRUE</span>),</span>
<span id="cb59-26"><a href="#cb59-26" aria-hidden="true" tabindex="-1"></a>    <span class="at">Max_Artifact_percentage =</span> <span class="fu">max</span>(Artifact_percentage, <span class="at">na.rm =</span> <span class="cn">TRUE</span>),</span>
<span id="cb59-27"><a href="#cb59-27" aria-hidden="true" tabindex="-1"></a>    <span class="at">Min_Artifact_percentage =</span> <span class="fu">min</span>(Artifact_percentage, <span class="at">na.rm =</span> <span class="cn">TRUE</span>),</span>
<span id="cb59-28"><a href="#cb59-28" aria-hidden="true" tabindex="-1"></a>    <span class="at">Mean_Unknown_percentage =</span> <span class="fu">mean</span>(Unknown_percentage, <span class="at">na.rm =</span> <span class="cn">TRUE</span>),</span>
<span id="cb59-29"><a href="#cb59-29" aria-hidden="true" tabindex="-1"></a>    <span class="at">Max_Unknown_percentage =</span> <span class="fu">max</span>(Unknown_percentage, <span class="at">na.rm =</span> <span class="cn">TRUE</span>),</span>
<span id="cb59-30"><a href="#cb59-30" aria-hidden="true" tabindex="-1"></a>    <span class="at">Min_Unknown_percentage =</span> <span class="fu">min</span>(Unknown_percentage, <span class="at">na.rm =</span> <span class="cn">TRUE</span>)</span>
<span id="cb59-31"><a href="#cb59-31" aria-hidden="true" tabindex="-1"></a>    ) </span>
<span id="cb59-32"><a href="#cb59-32" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb59-33"><a href="#cb59-33" aria-hidden="true" tabindex="-1"></a><span class="co">#knitr::kable(stats_by_group,</span></span>
<span id="cb59-34"><a href="#cb59-34" aria-hidden="true" tabindex="-1"></a><span class="co">#             col.names = c("Group", </span></span>
<span id="cb59-35"><a href="#cb59-35" aria-hidden="true" tabindex="-1"></a><span class="co">#                           "TM", </span></span>
<span id="cb59-36"><a href="#cb59-36" aria-hidden="true" tabindex="-1"></a><span class="co">#                           "TU", </span></span>
<span id="cb59-37"><a href="#cb59-37" aria-hidden="true" tabindex="-1"></a><span class="co">#                           "Mean M", </span></span>
<span id="cb59-38"><a href="#cb59-38" aria-hidden="true" tabindex="-1"></a><span class="co">#                           "Max M", </span></span>
<span id="cb59-39"><a href="#cb59-39" aria-hidden="true" tabindex="-1"></a><span class="co">#                           "Min M")) %&gt;%</span></span>
<span id="cb59-40"><a href="#cb59-40" aria-hidden="true" tabindex="-1"></a><span class="co">#  kable_styling(</span></span>
<span id="cb59-41"><a href="#cb59-41" aria-hidden="true" tabindex="-1"></a><span class="co">#    latex_options = c("hold_position", "scale_down"), # Escala automáticamente</span></span>
<span id="cb59-42"><a href="#cb59-42" aria-hidden="true" tabindex="-1"></a><span class="co">#    font_size = 10.5   # Ajustar tamaño de fuente</span></span>
<span id="cb59-43"><a href="#cb59-43" aria-hidden="true" tabindex="-1"></a><span class="co">#  ) %&gt;%</span></span>
<span id="cb59-44"><a href="#cb59-44" aria-hidden="true" tabindex="-1"></a><span class="co">#  row_spec(0, bold = TRUE) # Encabezado en negrita</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<div class="cell">
<div class="sourceCode cell-code" id="cb60"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb60-1"><a href="#cb60-1" aria-hidden="true" tabindex="-1"></a><span class="do">##| tbl-cap: "Descriptive statistics of Isomir Read counts "</span></span>
<span id="cb60-2"><a href="#cb60-2" aria-hidden="true" tabindex="-1"></a><span class="do">##| label: "tbl-isomir_read_counts"</span></span>
<span id="cb60-3"><a href="#cb60-3" aria-hidden="true" tabindex="-1"></a>mirtrace_data <span class="ot">&lt;-</span> <span class="fu">read.table</span>(<span class="st">"data/mirtrace_contamination_check_plot.tsv"</span>, <span class="at">header =</span> <span class="cn">TRUE</span>, <span class="at">sep =</span> <span class="st">"</span><span class="sc">\t</span><span class="st">"</span>)</span>
<span id="cb60-4"><a href="#cb60-4" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb60-5"><a href="#cb60-5" aria-hidden="true" tabindex="-1"></a>mirtrace_data <span class="ot">&lt;-</span> mirtrace_data <span class="sc">%&gt;%</span></span>
<span id="cb60-6"><a href="#cb60-6" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">Total =</span> (gymnosperms<span class="sc">+</span> monocots <span class="sc">+</span> dicots <span class="sc">+</span> nematode <span class="sc">+</span> insects <span class="sc">+</span> lophotrochozoa <span class="sc">+</span> fish <span class="sc">+</span> rodents <span class="sc">+</span> primates <span class="sc">+</span> bryophytes <span class="sc">+</span> birds_reptiles),</span>
<span id="cb60-7"><a href="#cb60-7" aria-hidden="true" tabindex="-1"></a>         <span class="at">gymnosperms_percentage =</span> (gymnosperms<span class="sc">/</span>Total)<span class="sc">*</span> <span class="dv">100</span>,</span>
<span id="cb60-8"><a href="#cb60-8" aria-hidden="true" tabindex="-1"></a>         <span class="at">monocots_percentage =</span> (monocots<span class="sc">/</span>Total)<span class="sc">*</span> <span class="dv">100</span>,</span>
<span id="cb60-9"><a href="#cb60-9" aria-hidden="true" tabindex="-1"></a>         <span class="at">dicots_percentgae =</span> (dicots<span class="sc">/</span>Total)<span class="sc">*</span> <span class="dv">100</span>,</span>
<span id="cb60-10"><a href="#cb60-10" aria-hidden="true" tabindex="-1"></a>         <span class="at">nematode_percentage =</span> (nematode<span class="sc">/</span>Total)<span class="sc">*</span> <span class="dv">100</span>,</span>
<span id="cb60-11"><a href="#cb60-11" aria-hidden="true" tabindex="-1"></a>         <span class="at">insects_percentage =</span> (insects<span class="sc">/</span>Total)<span class="sc">*</span> <span class="dv">100</span>,</span>
<span id="cb60-12"><a href="#cb60-12" aria-hidden="true" tabindex="-1"></a>         <span class="at">lophotrochozoa_percentage =</span> (lophotrochozoa<span class="sc">/</span>Total)<span class="sc">*</span> <span class="dv">100</span>,</span>
<span id="cb60-13"><a href="#cb60-13" aria-hidden="true" tabindex="-1"></a>         <span class="at">fish_percentage =</span> (fish<span class="sc">/</span>Total)<span class="sc">*</span> <span class="dv">100</span>,</span>
<span id="cb60-14"><a href="#cb60-14" aria-hidden="true" tabindex="-1"></a>         <span class="at">rodents_percentage =</span> (rodents<span class="sc">/</span>Total)<span class="sc">*</span> <span class="dv">100</span>,</span>
<span id="cb60-15"><a href="#cb60-15" aria-hidden="true" tabindex="-1"></a>         <span class="at">primates_percentage =</span> (primates<span class="sc">/</span>Total)<span class="sc">*</span> <span class="dv">100</span>,</span>
<span id="cb60-16"><a href="#cb60-16" aria-hidden="true" tabindex="-1"></a>         <span class="at">bryophytes_percentage =</span> (bryophytes<span class="sc">/</span>Total)<span class="sc">*</span> <span class="dv">100</span>,</span>
<span id="cb60-17"><a href="#cb60-17" aria-hidden="true" tabindex="-1"></a>         <span class="at">birds_reptiles_percentage =</span> (birds_reptiles<span class="sc">/</span>Total)<span class="sc">*</span> <span class="dv">100</span></span>
<span id="cb60-18"><a href="#cb60-18" aria-hidden="true" tabindex="-1"></a>         )</span>
<span id="cb60-19"><a href="#cb60-19" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb60-20"><a href="#cb60-20" aria-hidden="true" tabindex="-1"></a>stats_mirtrace_data <span class="ot">&lt;-</span> mirtrace_data <span class="sc">%&gt;%</span></span>
<span id="cb60-21"><a href="#cb60-21" aria-hidden="true" tabindex="-1"></a>  <span class="fu">summarise</span>(</span>
<span id="cb60-22"><a href="#cb60-22" aria-hidden="true" tabindex="-1"></a>    <span class="at">Mean_primates_percentage =</span> <span class="fu">mean</span>( primates_percentage, <span class="at">na.rm =</span> <span class="cn">TRUE</span>),</span>
<span id="cb60-23"><a href="#cb60-23" aria-hidden="true" tabindex="-1"></a>    <span class="at">Max_primates_percentage =</span> <span class="fu">max</span>( primates_percentage, <span class="at">na.rm =</span> <span class="cn">TRUE</span>),</span>
<span id="cb60-24"><a href="#cb60-24" aria-hidden="true" tabindex="-1"></a>    <span class="at">Min_primates_percentage =</span> <span class="fu">min</span>( primates_percentage, <span class="at">na.rm =</span> <span class="cn">TRUE</span>),</span>
<span id="cb60-25"><a href="#cb60-25" aria-hidden="true" tabindex="-1"></a>    <span class="at">Mean_birds_reptiles_percentage =</span> <span class="fu">mean</span>(birds_reptiles_percentage, <span class="at">na.rm =</span> <span class="cn">TRUE</span>),</span>
<span id="cb60-26"><a href="#cb60-26" aria-hidden="true" tabindex="-1"></a>    <span class="at">Mean_gymnosperms_percentgae =</span> <span class="fu">mean</span>(gymnosperms_percentage, <span class="at">na.rm =</span> <span class="cn">TRUE</span>),</span>
<span id="cb60-27"><a href="#cb60-27" aria-hidden="true" tabindex="-1"></a>    <span class="at">Mean_monocots_percentgae =</span> <span class="fu">mean</span>(monocots_percentage, <span class="at">na.rm =</span> <span class="cn">TRUE</span>),</span>
<span id="cb60-28"><a href="#cb60-28" aria-hidden="true" tabindex="-1"></a>    <span class="at">Mean_dicots_percentgae =</span> <span class="fu">mean</span>(dicots_percentgae, <span class="at">na.rm =</span> <span class="cn">TRUE</span>),</span>
<span id="cb60-29"><a href="#cb60-29" aria-hidden="true" tabindex="-1"></a>    <span class="at">Mean_nematode_percentgae =</span> <span class="fu">mean</span>(nematode_percentage, <span class="at">na.rm =</span> <span class="cn">TRUE</span>),</span>
<span id="cb60-30"><a href="#cb60-30" aria-hidden="true" tabindex="-1"></a>    <span class="at">Mean_insects_percentgae =</span> <span class="fu">mean</span>(insects_percentage, <span class="at">na.rm =</span> <span class="cn">TRUE</span>),</span>
<span id="cb60-31"><a href="#cb60-31" aria-hidden="true" tabindex="-1"></a>    <span class="at">Mean_lophotrochozoa_percentgae =</span> <span class="fu">mean</span>(lophotrochozoa_percentage, <span class="at">na.rm =</span> <span class="cn">TRUE</span>),</span>
<span id="cb60-32"><a href="#cb60-32" aria-hidden="true" tabindex="-1"></a>    <span class="at">Mean_fish_percentgae =</span> <span class="fu">mean</span>(fish_percentage, <span class="at">na.rm =</span> <span class="cn">TRUE</span>),</span>
<span id="cb60-33"><a href="#cb60-33" aria-hidden="true" tabindex="-1"></a>    <span class="at">Mean_rodents_percentgae =</span> <span class="fu">mean</span>(rodents_percentage, <span class="at">na.rm =</span> <span class="cn">TRUE</span>),</span>
<span id="cb60-34"><a href="#cb60-34" aria-hidden="true" tabindex="-1"></a>    <span class="at">Mean_bryophytes_percentgae =</span> <span class="fu">mean</span>(bryophytes_percentage, <span class="at">na.rm =</span> <span class="cn">TRUE</span>),</span>
<span id="cb60-35"><a href="#cb60-35" aria-hidden="true" tabindex="-1"></a>    ) </span>
<span id="cb60-36"><a href="#cb60-36" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb60-37"><a href="#cb60-37" aria-hidden="true" tabindex="-1"></a><span class="co">#knitr::kable(stats_by_group,</span></span>
<span id="cb60-38"><a href="#cb60-38" aria-hidden="true" tabindex="-1"></a><span class="co">#             col.names = c("Group", </span></span>
<span id="cb60-39"><a href="#cb60-39" aria-hidden="true" tabindex="-1"></a><span class="co">#                           "TM", </span></span>
<span id="cb60-40"><a href="#cb60-40" aria-hidden="true" tabindex="-1"></a><span class="co">#                           "TU", </span></span>
<span id="cb60-41"><a href="#cb60-41" aria-hidden="true" tabindex="-1"></a><span class="co">#                           "Mean M", </span></span>
<span id="cb60-42"><a href="#cb60-42" aria-hidden="true" tabindex="-1"></a><span class="co">#                           "Max M", </span></span>
<span id="cb60-43"><a href="#cb60-43" aria-hidden="true" tabindex="-1"></a><span class="co">#                           "Min M")) %&gt;%</span></span>
<span id="cb60-44"><a href="#cb60-44" aria-hidden="true" tabindex="-1"></a><span class="co">#  kable_styling(</span></span>
<span id="cb60-45"><a href="#cb60-45" aria-hidden="true" tabindex="-1"></a><span class="co">#    latex_options = c("hold_position", "scale_down"), # Escala automáticamente</span></span>
<span id="cb60-46"><a href="#cb60-46" aria-hidden="true" tabindex="-1"></a><span class="co">#    font_size = 10.5   # Ajustar tamaño de fuente</span></span>
<span id="cb60-47"><a href="#cb60-47" aria-hidden="true" tabindex="-1"></a><span class="co">#  ) %&gt;%</span></span>
<span id="cb60-48"><a href="#cb60-48" aria-hidden="true" tabindex="-1"></a><span class="co">#  row_spec(0, bold = TRUE) # Encabezado en negrita</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
</section>
<section id="mirna-quantification" class="level2">
<h2 class="anchored" data-anchor-id="mirna-quantification">miRNA Quantification</h2>
<p>In this step of the <em>nf-core/smrnaseq pipeline</em>, the alignment of reads is conducted sequentially against databases of mature miRNAs, precursor miRNAs, and a combined database of both. These alignments enable the identification and quantification of miRNAs. The statistics obtained for mature miRNAs, precursor miRNAs, and the combination of both against the reference genome are presented in <a href="#tbl-samtools_overall" class="quarto-xref">Table&nbsp;3</a>. On average, 52.07% of the mature miRNAs were mapped, with a range between 17.72% and 79.74%.</p>
<p>For precursor miRNAs, an average of 32.33% of sequences was identified, ranging from 6.31% to 68.83%. Subsequently, the reads of mature and precursor miRNAs were aligned against a reference genome, not for miRNA identification and quantification but as a quality control for the sequences. In this regard, the mean percentage of aligned reads was around 43% of the total reads. Across the sample set, minimum values of 10.27% and maximum values of 73.31% were observed. For more details, the metrics for each sample can be found in <a href="https://github.com/joshoandres13/miRNAs" class="uri">https://github.com/joshoandres13/miRNAs</a>.</p>
<div class="cell">
<div class="sourceCode cell-code" id="cb61"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb61-1"><a href="#cb61-1" aria-hidden="true" tabindex="-1"></a>file_path_sam <span class="ot">&lt;-</span> <span class="st">"data/samtools_alignment_plot.tsv"</span>  <span class="co"># Cambia esto por la ruta real</span></span>
<span id="cb61-2"><a href="#cb61-2" aria-hidden="true" tabindex="-1"></a>data_Stats_samtools <span class="ot">&lt;-</span> <span class="fu">read.table</span>(file_path_sam, <span class="at">header =</span> <span class="cn">TRUE</span>, <span class="at">sep =</span> <span class="st">"</span><span class="sc">\t</span><span class="st">"</span>)</span>
<span id="cb61-3"><a href="#cb61-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb61-4"><a href="#cb61-4" aria-hidden="true" tabindex="-1"></a>data_Stats_samtools <span class="ot">&lt;-</span> data_Stats_samtools <span class="sc">%&gt;%</span></span>
<span id="cb61-5"><a href="#cb61-5" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">Percentage_Mapped =</span> (Mapped..with.MQ.<span class="fl">0.</span> <span class="sc">/</span> (Mapped..with.MQ.<span class="fl">0.</span> <span class="sc">+</span> Unmapped)) <span class="sc">*</span> <span class="dv">100</span>,</span>
<span id="cb61-6"><a href="#cb61-6" aria-hidden="true" tabindex="-1"></a>         <span class="at">Percentage_Unmapped =</span> (Unmapped <span class="sc">/</span> (Mapped..with.MQ.<span class="fl">0.</span> <span class="sc">+</span> Unmapped)) <span class="sc">*</span> <span class="dv">100</span>)</span>
<span id="cb61-7"><a href="#cb61-7" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb61-8"><a href="#cb61-8" aria-hidden="true" tabindex="-1"></a>data_Stats_samtools<span class="sc">$</span>Group <span class="ot">&lt;-</span> <span class="fu">sub</span>(<span class="st">".*_(mature_hairpin_genome|mature_hairpin|mature)$"</span>, <span class="st">"</span><span class="sc">\\</span><span class="st">1"</span>, data_Stats_samtools<span class="sc">$</span>Sample)</span>
<span id="cb61-9"><a href="#cb61-9" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb61-10"><a href="#cb61-10" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb61-11"><a href="#cb61-11" aria-hidden="true" tabindex="-1"></a><span class="co"># Agrupar por el grupo y calcular las estadísticas</span></span>
<span id="cb61-12"><a href="#cb61-12" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(dplyr)</span>
<span id="cb61-13"><a href="#cb61-13" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb61-14"><a href="#cb61-14" aria-hidden="true" tabindex="-1"></a>stats_by_group <span class="ot">&lt;-</span> data_Stats_samtools <span class="sc">%&gt;%</span></span>
<span id="cb61-15"><a href="#cb61-15" aria-hidden="true" tabindex="-1"></a>  <span class="fu">group_by</span>(Group) <span class="sc">%&gt;%</span></span>
<span id="cb61-16"><a href="#cb61-16" aria-hidden="true" tabindex="-1"></a>  <span class="fu">summarise</span>(</span>
<span id="cb61-17"><a href="#cb61-17" aria-hidden="true" tabindex="-1"></a>    <span class="at">Total_Mapped =</span> <span class="fu">mean</span>(Mapped..with.MQ.<span class="fl">0.</span>, <span class="at">na.rm =</span> <span class="cn">TRUE</span>),</span>
<span id="cb61-18"><a href="#cb61-18" aria-hidden="true" tabindex="-1"></a>    <span class="at">Total_Unmapped =</span> <span class="fu">mean</span>(Unmapped, <span class="at">na.rm =</span> <span class="cn">TRUE</span>),</span>
<span id="cb61-19"><a href="#cb61-19" aria-hidden="true" tabindex="-1"></a>    <span class="at">Avg_Percentage_Mapped =</span> <span class="fu">mean</span>(Percentage_Mapped, <span class="at">na.rm =</span> <span class="cn">TRUE</span>),</span>
<span id="cb61-20"><a href="#cb61-20" aria-hidden="true" tabindex="-1"></a>    <span class="at">Avg_Percentage_Unmapped =</span> <span class="fu">mean</span>(Percentage_Unmapped, <span class="at">na.rm =</span> <span class="cn">TRUE</span>),</span>
<span id="cb61-21"><a href="#cb61-21" aria-hidden="true" tabindex="-1"></a>    <span class="at">Max_Percentage_Mapped =</span> <span class="fu">max</span>(Percentage_Mapped, <span class="at">na.rm =</span> <span class="cn">TRUE</span>),</span>
<span id="cb61-22"><a href="#cb61-22" aria-hidden="true" tabindex="-1"></a>    <span class="at">Min_Percentage_Mapped =</span> <span class="fu">min</span>(Percentage_Mapped, <span class="at">na.rm =</span> <span class="cn">TRUE</span>)</span>
<span id="cb61-23"><a href="#cb61-23" aria-hidden="true" tabindex="-1"></a>  ) <span class="sc">%&gt;%</span></span>
<span id="cb61-24"><a href="#cb61-24" aria-hidden="true" tabindex="-1"></a>  <span class="fu">select</span>(<span class="sc">-</span> Avg_Percentage_Unmapped)<span class="sc">%&gt;%</span></span>
<span id="cb61-25"><a href="#cb61-25" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(</span>
<span id="cb61-26"><a href="#cb61-26" aria-hidden="true" tabindex="-1"></a>    <span class="at">Total_Mapped =</span> <span class="fu">round</span>(Total_Mapped, <span class="dv">2</span>),</span>
<span id="cb61-27"><a href="#cb61-27" aria-hidden="true" tabindex="-1"></a>    <span class="at">Total_Unmapped =</span> <span class="fu">round</span>(Total_Unmapped, <span class="dv">2</span>),</span>
<span id="cb61-28"><a href="#cb61-28" aria-hidden="true" tabindex="-1"></a>    <span class="at">Avg_Percentage_Mapped =</span> <span class="fu">round</span>(Avg_Percentage_Mapped, <span class="dv">2</span>),</span>
<span id="cb61-29"><a href="#cb61-29" aria-hidden="true" tabindex="-1"></a>    <span class="at">Max_Percentage_Mapped =</span> <span class="fu">round</span>(Max_Percentage_Mapped, <span class="dv">2</span>),</span>
<span id="cb61-30"><a href="#cb61-30" aria-hidden="true" tabindex="-1"></a>    <span class="at">Min_Percentage_Mapped =</span> <span class="fu">round</span>(Min_Percentage_Mapped, <span class="dv">2</span>)</span>
<span id="cb61-31"><a href="#cb61-31" aria-hidden="true" tabindex="-1"></a>  )</span>
<span id="cb61-32"><a href="#cb61-32" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb61-33"><a href="#cb61-33" aria-hidden="true" tabindex="-1"></a>knitr<span class="sc">::</span><span class="fu">kable</span>(stats_by_group,</span>
<span id="cb61-34"><a href="#cb61-34" aria-hidden="true" tabindex="-1"></a>             <span class="at">col.names =</span> <span class="fu">c</span>(<span class="st">"Group"</span>, </span>
<span id="cb61-35"><a href="#cb61-35" aria-hidden="true" tabindex="-1"></a>                           <span class="st">"TM"</span>, </span>
<span id="cb61-36"><a href="#cb61-36" aria-hidden="true" tabindex="-1"></a>                           <span class="st">"TU"</span>, </span>
<span id="cb61-37"><a href="#cb61-37" aria-hidden="true" tabindex="-1"></a>                           <span class="st">"Mean M"</span>, </span>
<span id="cb61-38"><a href="#cb61-38" aria-hidden="true" tabindex="-1"></a>                           <span class="st">"Max M"</span>, </span>
<span id="cb61-39"><a href="#cb61-39" aria-hidden="true" tabindex="-1"></a>                           <span class="st">"Min M"</span>)) <span class="sc">%&gt;%</span></span>
<span id="cb61-40"><a href="#cb61-40" aria-hidden="true" tabindex="-1"></a>  <span class="fu">kable_styling</span>(</span>
<span id="cb61-41"><a href="#cb61-41" aria-hidden="true" tabindex="-1"></a>    <span class="at">latex_options =</span> <span class="fu">c</span>(<span class="st">"hold_position"</span>, <span class="st">"scale_down"</span>), <span class="co"># Escala automáticamente</span></span>
<span id="cb61-42"><a href="#cb61-42" aria-hidden="true" tabindex="-1"></a>    <span class="at">font_size =</span> <span class="dv">11</span>   <span class="co"># Ajustar tamaño de fuente</span></span>
<span id="cb61-43"><a href="#cb61-43" aria-hidden="true" tabindex="-1"></a>  ) <span class="sc">%&gt;%</span></span>
<span id="cb61-44"><a href="#cb61-44" aria-hidden="true" tabindex="-1"></a>  <span class="fu">row_spec</span>(<span class="dv">0</span>, <span class="at">bold =</span> <span class="cn">TRUE</span>) <span class="co"># Encabezado en negrita</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div id="tbl-samtools_overall" class="cell quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-tbl figure">
<figcaption class="quarto-float-caption-top quarto-float-caption quarto-float-tbl" id="tbl-samtools_overall-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Table&nbsp;3: Descriptive statistics of alignment with samtools of all samples. <em>TM</em>: Mean of Total Mapped (reads); <em>TU</em>: Mean of Total Unmapped (reads): <em>Mean M</em>: Mean Mapped (%); <em>Max M</em>: Max Mapped (%); <em>Min M</em>: Min Mapped (%)
</figcaption>
<div aria-describedby="tbl-samtools_overall-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<div class="cell-output-display">
<table class="table do-not-create-environment cell caption-top table-sm table-striped small" data-quarto-postprocess="true">
<thead>
<tr class="header">
<th data-quarto-table-cell-role="th" style="text-align: left; font-weight: bold;">Group</th>
<th data-quarto-table-cell-role="th" style="text-align: right; font-weight: bold;">TM</th>
<th data-quarto-table-cell-role="th" style="text-align: right; font-weight: bold;">TU</th>
<th data-quarto-table-cell-role="th" style="text-align: right; font-weight: bold;">Mean M</th>
<th data-quarto-table-cell-role="th" style="text-align: right; font-weight: bold;">Max M</th>
<th data-quarto-table-cell-role="th" style="text-align: right; font-weight: bold;">Min M</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">mature</td>
<td style="text-align: right;">13166950</td>
<td style="text-align: right;">12194855</td>
<td style="text-align: right;">52.07</td>
<td style="text-align: right;">79.75</td>
<td style="text-align: right;">17.72</td>
</tr>
<tr class="even">
<td style="text-align: left;">mature_hairpin</td>
<td style="text-align: right;">3644435</td>
<td style="text-align: right;">8836242</td>
<td style="text-align: right;">32.33</td>
<td style="text-align: right;">68.83</td>
<td style="text-align: right;">6.31</td>
</tr>
<tr class="odd">
<td style="text-align: left;">mature_hairpin_genome</td>
<td style="text-align: right;">4334118</td>
<td style="text-align: right;">4565712</td>
<td style="text-align: right;">43.87</td>
<td style="text-align: right;">73.31</td>
<td style="text-align: right;">10.28</td>
</tr>
</tbody>
</table>


</div>
</div>
</figure>
</div>
</div>
</section>
<section id="isomir-annotation" class="level2">
<h2 class="anchored" data-anchor-id="isomir-annotation">IsomiR Annotation</h2>
<p><em>Mirtop</em> (v0.4.28) was used for the annotation of miRNAs and isomiRs. In figure <a href="#fig-mirtop_mean_isomir_read_counts" class="quarto-xref">Figure&nbsp;8</a>, the <em>mean isomiR read counts</em> are presented, which refer to an average calculation that helps describe how the reads of isomiRs are distributed within a dataset. Among the annotated isomiR sequences per sample, the <em>reference miRNA</em> accounts for 96.51%, with ranges varying from 88.70% to 98.13%. On the other hand, the distributions of isomiR variants are as follows: <em>3’ Isoform</em> (1.36%), <em>3’ Addition</em> (1.23%), <em>5’ Isoform</em> (0.64%), <em>SNVs in the Central Offset Region</em> (0.05%), <em>SNVs in the Central Region</em> (0.06%), <em>Supported SNVs in the Central Region</em> (0.05%), and <em>SNV in Seed Region</em> (0.05%), all of which are below 1.40%.</p>
<div class="cell">
<div class="sourceCode cell-code" id="cb62"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb62-1"><a href="#cb62-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Mean isomiR read counts</span></span>
<span id="cb62-2"><a href="#cb62-2" aria-hidden="true" tabindex="-1"></a>mirtop_mean_isomir_read_counts_data <span class="ot">&lt;-</span> <span class="fu">read.table</span>(<span class="st">"data/mirtop_mean_read_count_plot.tsv"</span>, <span class="at">header =</span> <span class="cn">TRUE</span>, <span class="at">sep =</span> <span class="st">"</span><span class="sc">\t</span><span class="st">"</span>)</span>
<span id="cb62-3"><a href="#cb62-3" aria-hidden="true" tabindex="-1"></a>mirtop_mean_isomir_read_counts_data <span class="ot">&lt;-</span> mirtop_mean_isomir_read_counts_data <span class="sc">%&gt;%</span></span>
<span id="cb62-4"><a href="#cb62-4" aria-hidden="true" tabindex="-1"></a>  <span class="fu">pivot_longer</span>(<span class="at">cols =</span> <span class="fu">c</span>(ref_miRNA,iso_3p,iso_5p,iso_add3p,iso_snv_seed,iso_snv_central_offset,iso_snv_central,iso_snv_central_supp), </span>
<span id="cb62-5"><a href="#cb62-5" aria-hidden="true" tabindex="-1"></a>               <span class="at">names_to =</span> <span class="st">"Isomirs"</span>, </span>
<span id="cb62-6"><a href="#cb62-6" aria-hidden="true" tabindex="-1"></a>               <span class="at">values_to =</span> <span class="st">"Conteo"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb62-7"><a href="#cb62-7" aria-hidden="true" tabindex="-1"></a>  <span class="fu">group_by</span>(Sample) <span class="sc">%&gt;%</span></span>
<span id="cb62-8"><a href="#cb62-8" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">Proporcion =</span> Conteo <span class="sc">/</span> <span class="fu">sum</span>(Conteo)) <span class="sc">%&gt;%</span> </span>
<span id="cb62-9"><a href="#cb62-9" aria-hidden="true" tabindex="-1"></a>  <span class="fu">ungroup</span>()</span>
<span id="cb62-10"><a href="#cb62-10" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb62-11"><a href="#cb62-11" aria-hidden="true" tabindex="-1"></a><span class="co">#Plot</span></span>
<span id="cb62-12"><a href="#cb62-12" aria-hidden="true" tabindex="-1"></a>mirtop3 <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(mirtop_mean_isomir_read_counts_data, <span class="fu">aes</span>(<span class="at">x =</span> Sample, <span class="at">y =</span> Proporcion, <span class="at">fill =</span> Isomirs)) <span class="sc">+</span></span>
<span id="cb62-13"><a href="#cb62-13" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_bar</span>(<span class="at">stat =</span> <span class="st">"identity"</span>, <span class="at">position =</span> <span class="st">"stack"</span>, <span class="at">alpha =</span> <span class="fl">0.9</span>) <span class="sc">+</span></span>
<span id="cb62-14"><a href="#cb62-14" aria-hidden="true" tabindex="-1"></a>  <span class="fu">labs</span>(</span>
<span id="cb62-15"><a href="#cb62-15" aria-hidden="true" tabindex="-1"></a>    <span class="at">title =</span> <span class="st">""</span>,</span>
<span id="cb62-16"><a href="#cb62-16" aria-hidden="true" tabindex="-1"></a>    <span class="at">x =</span> <span class="st">""</span>,</span>
<span id="cb62-17"><a href="#cb62-17" aria-hidden="true" tabindex="-1"></a>    <span class="at">y =</span> <span class="st">"Means"</span></span>
<span id="cb62-18"><a href="#cb62-18" aria-hidden="true" tabindex="-1"></a>  ) <span class="sc">+</span></span>
<span id="cb62-19"><a href="#cb62-19" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_y_continuous</span>(<span class="at">labels =</span> scales<span class="sc">::</span>percent) <span class="sc">+</span></span>
<span id="cb62-20"><a href="#cb62-20" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_fill_viridis_d</span>(<span class="at">option =</span> <span class="st">"cividis"</span>, </span>
<span id="cb62-21"><a href="#cb62-21" aria-hidden="true" tabindex="-1"></a>                       <span class="at">labels =</span> <span class="fu">c</span>(</span>
<span id="cb62-22"><a href="#cb62-22" aria-hidden="true" tabindex="-1"></a>      <span class="st">"ref_miRNA"</span> <span class="ot">=</span> <span class="st">"Reference miRNA"</span>,</span>
<span id="cb62-23"><a href="#cb62-23" aria-hidden="true" tabindex="-1"></a>      <span class="st">"iso_3p"</span> <span class="ot">=</span> <span class="st">"3' Isoform"</span>,</span>
<span id="cb62-24"><a href="#cb62-24" aria-hidden="true" tabindex="-1"></a>      <span class="st">"iso_5p"</span> <span class="ot">=</span> <span class="st">"5' Isoform"</span>,</span>
<span id="cb62-25"><a href="#cb62-25" aria-hidden="true" tabindex="-1"></a>      <span class="st">"iso_add3p"</span> <span class="ot">=</span> <span class="st">"3' Addition"</span>,</span>
<span id="cb62-26"><a href="#cb62-26" aria-hidden="true" tabindex="-1"></a>      <span class="st">"iso_snv_seed"</span> <span class="ot">=</span> <span class="st">"SNV in Seed Region"</span>,</span>
<span id="cb62-27"><a href="#cb62-27" aria-hidden="true" tabindex="-1"></a>      <span class="st">"iso_snv_central_offset"</span> <span class="ot">=</span> <span class="st">"SNV in Central Offset Region"</span>,</span>
<span id="cb62-28"><a href="#cb62-28" aria-hidden="true" tabindex="-1"></a>      <span class="st">"iso_snv_central"</span> <span class="ot">=</span> <span class="st">"SNV in Central Region"</span>,</span>
<span id="cb62-29"><a href="#cb62-29" aria-hidden="true" tabindex="-1"></a>      <span class="st">"iso_snv_central_supp"</span> <span class="ot">=</span> <span class="st">"Supported SNV in Central Region"</span></span>
<span id="cb62-30"><a href="#cb62-30" aria-hidden="true" tabindex="-1"></a>    )) <span class="sc">+</span></span>
<span id="cb62-31"><a href="#cb62-31" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme_minimal</span>(<span class="at">base_size =</span> <span class="dv">14</span>) <span class="sc">+</span></span>
<span id="cb62-32"><a href="#cb62-32" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme</span>(</span>
<span id="cb62-33"><a href="#cb62-33" aria-hidden="true" tabindex="-1"></a>    <span class="at">axis.text.x =</span> <span class="fu">element_text</span>(<span class="at">angle =</span> <span class="dv">90</span> , <span class="at">hjust =</span> <span class="dv">1</span>, <span class="at">size =</span> <span class="dv">5</span>),</span>
<span id="cb62-34"><a href="#cb62-34" aria-hidden="true" tabindex="-1"></a>    <span class="at">axis.text.y =</span> <span class="fu">element_text</span>(<span class="at">angle =</span> <span class="dv">0</span> , <span class="at">hjust =</span> <span class="dv">1</span>, <span class="at">size =</span> <span class="dv">7</span>),</span>
<span id="cb62-35"><a href="#cb62-35" aria-hidden="true" tabindex="-1"></a>    <span class="at">axis.title =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">8</span>),</span>
<span id="cb62-36"><a href="#cb62-36" aria-hidden="true" tabindex="-1"></a>    <span class="at">plot.title =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">12</span>, <span class="at">face =</span> <span class="st">"bold"</span>),</span>
<span id="cb62-37"><a href="#cb62-37" aria-hidden="true" tabindex="-1"></a>    <span class="at">legend.title =</span> <span class="fu">element_blank</span>(),</span>
<span id="cb62-38"><a href="#cb62-38" aria-hidden="true" tabindex="-1"></a>    <span class="at">legend.text =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">6</span>),</span>
<span id="cb62-39"><a href="#cb62-39" aria-hidden="true" tabindex="-1"></a>    <span class="at">legend.position =</span> <span class="st">"bottom"</span>,</span>
<span id="cb62-40"><a href="#cb62-40" aria-hidden="true" tabindex="-1"></a>    <span class="at">legend.margin =</span> <span class="fu">margin</span>(<span class="at">t =</span> <span class="sc">-</span><span class="dv">20</span>),  <span class="co"># Ajusta el margen superior de la leyenda</span></span>
<span id="cb62-41"><a href="#cb62-41" aria-hidden="true" tabindex="-1"></a><span class="co">#    plot.margin = margin(t = 10, r = 10, b = 30, l = 10)</span></span>
<span id="cb62-42"><a href="#cb62-42" aria-hidden="true" tabindex="-1"></a>  )</span>
<span id="cb62-43"><a href="#cb62-43" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb62-44"><a href="#cb62-44" aria-hidden="true" tabindex="-1"></a><span class="fu">print</span>(mirtop3)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<div id="fig-mirtop_mean_isomir_read_counts" class="quarto-float quarto-figure quarto-figure-center anchored" data-fig-pos="H">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-mirtop_mean_isomir_read_counts-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="Results_files/figure-html/fig-mirtop_mean_isomir_read_counts-1.png" class="img-fluid figure-img" data-fig-pos="H" width="768">
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-mirtop_mean_isomir_read_counts-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;8: Annotation of miRNAs and isomiRs with <em>mirtop</em> (v0.4.28): Mean isomiR read counts
</figcaption>
</figure>
</div>
</div>
</div>
<div class="cell">
<div class="sourceCode cell-code" id="cb63"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb63-1"><a href="#cb63-1" aria-hidden="true" tabindex="-1"></a><span class="co"># IsomiR read counts</span></span>
<span id="cb63-2"><a href="#cb63-2" aria-hidden="true" tabindex="-1"></a>mirtop_read_count_data <span class="ot">&lt;-</span> <span class="fu">read.table</span>(<span class="st">"data/mirtop_read_count_plot.tsv"</span>, <span class="at">header =</span> <span class="cn">TRUE</span>, <span class="at">sep =</span> <span class="st">"</span><span class="sc">\t</span><span class="st">"</span>)</span>
<span id="cb63-3"><a href="#cb63-3" aria-hidden="true" tabindex="-1"></a>mirtop_read_count_data <span class="ot">&lt;-</span> mirtop_read_count_data <span class="sc">%&gt;%</span></span>
<span id="cb63-4"><a href="#cb63-4" aria-hidden="true" tabindex="-1"></a>  <span class="fu">pivot_longer</span>(<span class="at">cols =</span> <span class="fu">c</span>(ref_miRNA,iso_3p,iso_5p,iso_add3p,iso_snv_seed,iso_snv_central_offset,iso_snv_central,iso_snv_central_supp), </span>
<span id="cb63-5"><a href="#cb63-5" aria-hidden="true" tabindex="-1"></a>               <span class="at">names_to =</span> <span class="st">"Isomirs"</span>, </span>
<span id="cb63-6"><a href="#cb63-6" aria-hidden="true" tabindex="-1"></a>               <span class="at">values_to =</span> <span class="st">"Conteo"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb63-7"><a href="#cb63-7" aria-hidden="true" tabindex="-1"></a>  <span class="fu">group_by</span>(Sample) <span class="sc">%&gt;%</span></span>
<span id="cb63-8"><a href="#cb63-8" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">Proporcion =</span> Conteo <span class="sc">/</span> <span class="fu">sum</span>(Conteo)) <span class="sc">%&gt;%</span> </span>
<span id="cb63-9"><a href="#cb63-9" aria-hidden="true" tabindex="-1"></a>  <span class="fu">ungroup</span>()</span>
<span id="cb63-10"><a href="#cb63-10" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb63-11"><a href="#cb63-11" aria-hidden="true" tabindex="-1"></a><span class="co">#Plot</span></span>
<span id="cb63-12"><a href="#cb63-12" aria-hidden="true" tabindex="-1"></a>mirtop1 <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(mirtop_read_count_data, <span class="fu">aes</span>(<span class="at">x =</span> Sample, <span class="at">y =</span> Proporcion, <span class="at">fill =</span> Isomirs)) <span class="sc">+</span></span>
<span id="cb63-13"><a href="#cb63-13" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_bar</span>(<span class="at">stat =</span> <span class="st">"identity"</span>, <span class="at">position =</span> <span class="st">"stack"</span>, <span class="at">alpha =</span> <span class="fl">0.9</span>) <span class="sc">+</span></span>
<span id="cb63-14"><a href="#cb63-14" aria-hidden="true" tabindex="-1"></a>  <span class="fu">labs</span>(</span>
<span id="cb63-15"><a href="#cb63-15" aria-hidden="true" tabindex="-1"></a>    <span class="at">title =</span> <span class="st">"A"</span>,</span>
<span id="cb63-16"><a href="#cb63-16" aria-hidden="true" tabindex="-1"></a>    <span class="at">x =</span> <span class="st">""</span>,</span>
<span id="cb63-17"><a href="#cb63-17" aria-hidden="true" tabindex="-1"></a>    <span class="at">y =</span> <span class="st">"Read counts"</span></span>
<span id="cb63-18"><a href="#cb63-18" aria-hidden="true" tabindex="-1"></a>  ) <span class="sc">+</span></span>
<span id="cb63-19"><a href="#cb63-19" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_y_continuous</span>(<span class="at">labels =</span> scales<span class="sc">::</span>percent) <span class="sc">+</span></span>
<span id="cb63-20"><a href="#cb63-20" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_fill_viridis_d</span>(<span class="at">option =</span> <span class="st">"cividis"</span>, </span>
<span id="cb63-21"><a href="#cb63-21" aria-hidden="true" tabindex="-1"></a>                       <span class="at">labels =</span> <span class="fu">c</span>(</span>
<span id="cb63-22"><a href="#cb63-22" aria-hidden="true" tabindex="-1"></a>      <span class="st">"ref_miRNA"</span> <span class="ot">=</span> <span class="st">"Reference miRNA"</span>,</span>
<span id="cb63-23"><a href="#cb63-23" aria-hidden="true" tabindex="-1"></a>      <span class="st">"iso_3p"</span> <span class="ot">=</span> <span class="st">"3' Isoform"</span>,</span>
<span id="cb63-24"><a href="#cb63-24" aria-hidden="true" tabindex="-1"></a>      <span class="st">"iso_5p"</span> <span class="ot">=</span> <span class="st">"5' Isoform"</span>,</span>
<span id="cb63-25"><a href="#cb63-25" aria-hidden="true" tabindex="-1"></a>      <span class="st">"iso_add3p"</span> <span class="ot">=</span> <span class="st">"3' Addition"</span>,</span>
<span id="cb63-26"><a href="#cb63-26" aria-hidden="true" tabindex="-1"></a>      <span class="st">"iso_snv_seed"</span> <span class="ot">=</span> <span class="st">"SNV in Seed Region"</span>,</span>
<span id="cb63-27"><a href="#cb63-27" aria-hidden="true" tabindex="-1"></a>      <span class="st">"iso_snv_central_offset"</span> <span class="ot">=</span> <span class="st">"SNV in Central Offset Region"</span>,</span>
<span id="cb63-28"><a href="#cb63-28" aria-hidden="true" tabindex="-1"></a>      <span class="st">"iso_snv_central"</span> <span class="ot">=</span> <span class="st">"SNV in Central Region"</span>,</span>
<span id="cb63-29"><a href="#cb63-29" aria-hidden="true" tabindex="-1"></a>      <span class="st">"iso_snv_central_supp"</span> <span class="ot">=</span> <span class="st">"Supported SNV in Central Region"</span></span>
<span id="cb63-30"><a href="#cb63-30" aria-hidden="true" tabindex="-1"></a>    )) <span class="sc">+</span></span>
<span id="cb63-31"><a href="#cb63-31" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme_minimal</span>(<span class="at">base_size =</span> <span class="dv">14</span>) <span class="sc">+</span></span>
<span id="cb63-32"><a href="#cb63-32" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme</span>(</span>
<span id="cb63-33"><a href="#cb63-33" aria-hidden="true" tabindex="-1"></a>    <span class="at">axis.text.x =</span> <span class="fu">element_text</span>(<span class="at">angle =</span> <span class="dv">90</span> , <span class="at">hjust =</span> <span class="dv">1</span>, <span class="at">size =</span> <span class="dv">5</span>),</span>
<span id="cb63-34"><a href="#cb63-34" aria-hidden="true" tabindex="-1"></a>    <span class="at">axis.text.y =</span> <span class="fu">element_text</span>(<span class="at">angle =</span> <span class="dv">0</span> , <span class="at">hjust =</span> <span class="dv">1</span>, <span class="at">size =</span> <span class="dv">7</span>),</span>
<span id="cb63-35"><a href="#cb63-35" aria-hidden="true" tabindex="-1"></a>    <span class="at">axis.title =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">8</span>),</span>
<span id="cb63-36"><a href="#cb63-36" aria-hidden="true" tabindex="-1"></a>    <span class="at">plot.title =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">12</span>, <span class="at">face =</span> <span class="st">"bold"</span>),</span>
<span id="cb63-37"><a href="#cb63-37" aria-hidden="true" tabindex="-1"></a>    <span class="at">legend.title =</span> <span class="fu">element_blank</span>(),</span>
<span id="cb63-38"><a href="#cb63-38" aria-hidden="true" tabindex="-1"></a>    <span class="at">legend.text =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">6</span>),</span>
<span id="cb63-39"><a href="#cb63-39" aria-hidden="true" tabindex="-1"></a>    <span class="at">legend.position =</span> <span class="st">"bottom"</span>,</span>
<span id="cb63-40"><a href="#cb63-40" aria-hidden="true" tabindex="-1"></a>    <span class="at">legend.margin =</span> <span class="fu">margin</span>(<span class="at">t =</span> <span class="sc">-</span><span class="dv">20</span>),  <span class="co"># Ajusta el margen superior de la leyenda</span></span>
<span id="cb63-41"><a href="#cb63-41" aria-hidden="true" tabindex="-1"></a><span class="co">#    plot.margin = margin(t = 10, r = 10, b = 30, l = 10)</span></span>
<span id="cb63-42"><a href="#cb63-42" aria-hidden="true" tabindex="-1"></a>  )</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<div class="cell">
<div class="sourceCode cell-code" id="cb64"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb64-1"><a href="#cb64-1" aria-hidden="true" tabindex="-1"></a><span class="co"># IsomiR unique read counts</span></span>
<span id="cb64-2"><a href="#cb64-2" aria-hidden="true" tabindex="-1"></a>mirtop_unique_read_count_data <span class="ot">&lt;-</span> <span class="fu">read.table</span>(<span class="st">"data/mirtop_unique_read_count_plot.tsv"</span>, <span class="at">header =</span> <span class="cn">TRUE</span>, <span class="at">sep =</span> <span class="st">"</span><span class="sc">\t</span><span class="st">"</span>)</span>
<span id="cb64-3"><a href="#cb64-3" aria-hidden="true" tabindex="-1"></a>mirtop_unique_read_count_data <span class="ot">&lt;-</span> mirtop_unique_read_count_data <span class="sc">%&gt;%</span></span>
<span id="cb64-4"><a href="#cb64-4" aria-hidden="true" tabindex="-1"></a>  <span class="fu">pivot_longer</span>(<span class="at">cols =</span> <span class="fu">c</span>(ref_miRNA,iso_3p,iso_5p,iso_add3p,iso_snv_seed,iso_snv_central_offset,iso_snv_central,iso_snv_central_supp), </span>
<span id="cb64-5"><a href="#cb64-5" aria-hidden="true" tabindex="-1"></a>               <span class="at">names_to =</span> <span class="st">"Isomirs"</span>, </span>
<span id="cb64-6"><a href="#cb64-6" aria-hidden="true" tabindex="-1"></a>               <span class="at">values_to =</span> <span class="st">"Conteo"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb64-7"><a href="#cb64-7" aria-hidden="true" tabindex="-1"></a>  <span class="fu">group_by</span>(Sample) <span class="sc">%&gt;%</span></span>
<span id="cb64-8"><a href="#cb64-8" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">Proporcion =</span> Conteo <span class="sc">/</span> <span class="fu">sum</span>(Conteo)) <span class="sc">%&gt;%</span> </span>
<span id="cb64-9"><a href="#cb64-9" aria-hidden="true" tabindex="-1"></a>  <span class="fu">ungroup</span>()</span>
<span id="cb64-10"><a href="#cb64-10" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb64-11"><a href="#cb64-11" aria-hidden="true" tabindex="-1"></a><span class="co">#Plot</span></span>
<span id="cb64-12"><a href="#cb64-12" aria-hidden="true" tabindex="-1"></a>mirtop2 <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(mirtop_unique_read_count_data, <span class="fu">aes</span>(<span class="at">x =</span> Sample, <span class="at">y =</span> Proporcion, <span class="at">fill =</span> Isomirs)) <span class="sc">+</span></span>
<span id="cb64-13"><a href="#cb64-13" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_bar</span>(<span class="at">stat =</span> <span class="st">"identity"</span>, <span class="at">position =</span> <span class="st">"stack"</span>, <span class="at">alpha =</span> <span class="fl">0.9</span>) <span class="sc">+</span></span>
<span id="cb64-14"><a href="#cb64-14" aria-hidden="true" tabindex="-1"></a>  <span class="fu">labs</span>(</span>
<span id="cb64-15"><a href="#cb64-15" aria-hidden="true" tabindex="-1"></a>    <span class="at">title =</span> <span class="st">"B"</span>,</span>
<span id="cb64-16"><a href="#cb64-16" aria-hidden="true" tabindex="-1"></a>    <span class="at">x =</span> <span class="st">""</span>,</span>
<span id="cb64-17"><a href="#cb64-17" aria-hidden="true" tabindex="-1"></a>    <span class="at">y =</span> <span class="st">"Unique sequences"</span></span>
<span id="cb64-18"><a href="#cb64-18" aria-hidden="true" tabindex="-1"></a>  ) <span class="sc">+</span></span>
<span id="cb64-19"><a href="#cb64-19" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_y_continuous</span>(<span class="at">labels =</span> scales<span class="sc">::</span>percent) <span class="sc">+</span></span>
<span id="cb64-20"><a href="#cb64-20" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_fill_viridis_d</span>(<span class="at">option =</span> <span class="st">"cividis"</span>, </span>
<span id="cb64-21"><a href="#cb64-21" aria-hidden="true" tabindex="-1"></a>                       <span class="at">labels =</span> <span class="fu">c</span>(</span>
<span id="cb64-22"><a href="#cb64-22" aria-hidden="true" tabindex="-1"></a>      <span class="st">"ref_miRNA"</span> <span class="ot">=</span> <span class="st">"Reference miRNA"</span>,</span>
<span id="cb64-23"><a href="#cb64-23" aria-hidden="true" tabindex="-1"></a>      <span class="st">"iso_3p"</span> <span class="ot">=</span> <span class="st">"3' Isoform"</span>,</span>
<span id="cb64-24"><a href="#cb64-24" aria-hidden="true" tabindex="-1"></a>      <span class="st">"iso_5p"</span> <span class="ot">=</span> <span class="st">"5' Isoform"</span>,</span>
<span id="cb64-25"><a href="#cb64-25" aria-hidden="true" tabindex="-1"></a>      <span class="st">"iso_add3p"</span> <span class="ot">=</span> <span class="st">"3' Addition"</span>,</span>
<span id="cb64-26"><a href="#cb64-26" aria-hidden="true" tabindex="-1"></a>      <span class="st">"iso_snv_seed"</span> <span class="ot">=</span> <span class="st">"SNV in Seed Region"</span>,</span>
<span id="cb64-27"><a href="#cb64-27" aria-hidden="true" tabindex="-1"></a>      <span class="st">"iso_snv_central_offset"</span> <span class="ot">=</span> <span class="st">"SNV in Central Offset Region"</span>,</span>
<span id="cb64-28"><a href="#cb64-28" aria-hidden="true" tabindex="-1"></a>      <span class="st">"iso_snv_central"</span> <span class="ot">=</span> <span class="st">"SNV in Central Region"</span>,</span>
<span id="cb64-29"><a href="#cb64-29" aria-hidden="true" tabindex="-1"></a>      <span class="st">"iso_snv_central_supp"</span> <span class="ot">=</span> <span class="st">"Supported SNV in Central Region"</span></span>
<span id="cb64-30"><a href="#cb64-30" aria-hidden="true" tabindex="-1"></a>    )) <span class="sc">+</span></span>
<span id="cb64-31"><a href="#cb64-31" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme_minimal</span>(<span class="at">base_size =</span> <span class="dv">14</span>) <span class="sc">+</span></span>
<span id="cb64-32"><a href="#cb64-32" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme</span>(</span>
<span id="cb64-33"><a href="#cb64-33" aria-hidden="true" tabindex="-1"></a>    <span class="at">axis.text.x =</span> <span class="fu">element_text</span>(<span class="at">angle =</span> <span class="dv">90</span> , <span class="at">hjust =</span> <span class="dv">1</span>, <span class="at">size =</span> <span class="dv">5</span>),</span>
<span id="cb64-34"><a href="#cb64-34" aria-hidden="true" tabindex="-1"></a>    <span class="at">axis.text.y =</span> <span class="fu">element_text</span>(<span class="at">angle =</span> <span class="dv">0</span> , <span class="at">hjust =</span> <span class="dv">1</span>, <span class="at">size =</span> <span class="dv">7</span>),</span>
<span id="cb64-35"><a href="#cb64-35" aria-hidden="true" tabindex="-1"></a>    <span class="at">axis.title =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">8</span>),</span>
<span id="cb64-36"><a href="#cb64-36" aria-hidden="true" tabindex="-1"></a>    <span class="at">plot.title =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">12</span>, <span class="at">face =</span> <span class="st">"bold"</span>),</span>
<span id="cb64-37"><a href="#cb64-37" aria-hidden="true" tabindex="-1"></a>    <span class="at">legend.title =</span> <span class="fu">element_blank</span>(),</span>
<span id="cb64-38"><a href="#cb64-38" aria-hidden="true" tabindex="-1"></a>    <span class="at">legend.text =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">6</span>),</span>
<span id="cb64-39"><a href="#cb64-39" aria-hidden="true" tabindex="-1"></a>    <span class="at">legend.position =</span> <span class="st">"bottom"</span>,</span>
<span id="cb64-40"><a href="#cb64-40" aria-hidden="true" tabindex="-1"></a>    <span class="at">legend.margin =</span> <span class="fu">margin</span>(<span class="at">t =</span> <span class="sc">-</span><span class="dv">20</span>),  <span class="co"># Ajusta el margen superior de la leyenda</span></span>
<span id="cb64-41"><a href="#cb64-41" aria-hidden="true" tabindex="-1"></a><span class="co">#    plot.margin = margin(t = 10, r = 10, b = 30, l = 10)</span></span>
<span id="cb64-42"><a href="#cb64-42" aria-hidden="true" tabindex="-1"></a>  )</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<div class="cell">
<div class="sourceCode cell-code" id="cb65"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb65-1"><a href="#cb65-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Mean isomiR read counts</span></span>
<span id="cb65-2"><a href="#cb65-2" aria-hidden="true" tabindex="-1"></a>mirtop_mean_isomir_read_counts_data <span class="ot">&lt;-</span> <span class="fu">read.table</span>(<span class="st">"data/mirtop_mean_read_count_plot.tsv"</span>, <span class="at">header =</span> <span class="cn">TRUE</span>, <span class="at">sep =</span> <span class="st">"</span><span class="sc">\t</span><span class="st">"</span>)</span>
<span id="cb65-3"><a href="#cb65-3" aria-hidden="true" tabindex="-1"></a>mirtop_mean_isomir_read_counts_data <span class="ot">&lt;-</span> mirtop_mean_isomir_read_counts_data <span class="sc">%&gt;%</span></span>
<span id="cb65-4"><a href="#cb65-4" aria-hidden="true" tabindex="-1"></a>  <span class="fu">pivot_longer</span>(<span class="at">cols =</span> <span class="fu">c</span>(ref_miRNA,iso_3p,iso_5p,iso_add3p,iso_snv_seed,iso_snv_central_offset,iso_snv_central,iso_snv_central_supp), </span>
<span id="cb65-5"><a href="#cb65-5" aria-hidden="true" tabindex="-1"></a>               <span class="at">names_to =</span> <span class="st">"Isomirs"</span>, </span>
<span id="cb65-6"><a href="#cb65-6" aria-hidden="true" tabindex="-1"></a>               <span class="at">values_to =</span> <span class="st">"Conteo"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb65-7"><a href="#cb65-7" aria-hidden="true" tabindex="-1"></a>  <span class="fu">group_by</span>(Sample) <span class="sc">%&gt;%</span></span>
<span id="cb65-8"><a href="#cb65-8" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">Proporcion =</span> Conteo <span class="sc">/</span> <span class="fu">sum</span>(Conteo)) <span class="sc">%&gt;%</span> </span>
<span id="cb65-9"><a href="#cb65-9" aria-hidden="true" tabindex="-1"></a>  <span class="fu">ungroup</span>()</span>
<span id="cb65-10"><a href="#cb65-10" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb65-11"><a href="#cb65-11" aria-hidden="true" tabindex="-1"></a><span class="co">#Plot</span></span>
<span id="cb65-12"><a href="#cb65-12" aria-hidden="true" tabindex="-1"></a>mirtop3 <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(mirtop_mean_isomir_read_counts_data, <span class="fu">aes</span>(<span class="at">x =</span> Sample, <span class="at">y =</span> Proporcion, <span class="at">fill =</span> Isomirs)) <span class="sc">+</span></span>
<span id="cb65-13"><a href="#cb65-13" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_bar</span>(<span class="at">stat =</span> <span class="st">"identity"</span>, <span class="at">position =</span> <span class="st">"stack"</span>, <span class="at">alpha =</span> <span class="fl">0.9</span>) <span class="sc">+</span></span>
<span id="cb65-14"><a href="#cb65-14" aria-hidden="true" tabindex="-1"></a>  <span class="fu">labs</span>(</span>
<span id="cb65-15"><a href="#cb65-15" aria-hidden="true" tabindex="-1"></a>    <span class="at">title =</span> <span class="st">"C"</span>,</span>
<span id="cb65-16"><a href="#cb65-16" aria-hidden="true" tabindex="-1"></a>    <span class="at">x =</span> <span class="st">""</span>,</span>
<span id="cb65-17"><a href="#cb65-17" aria-hidden="true" tabindex="-1"></a>    <span class="at">y =</span> <span class="st">"Means"</span></span>
<span id="cb65-18"><a href="#cb65-18" aria-hidden="true" tabindex="-1"></a>  ) <span class="sc">+</span></span>
<span id="cb65-19"><a href="#cb65-19" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_y_continuous</span>(<span class="at">labels =</span> scales<span class="sc">::</span>percent) <span class="sc">+</span></span>
<span id="cb65-20"><a href="#cb65-20" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_fill_viridis_d</span>(<span class="at">option =</span> <span class="st">"cividis"</span>, </span>
<span id="cb65-21"><a href="#cb65-21" aria-hidden="true" tabindex="-1"></a>                       <span class="at">labels =</span> <span class="fu">c</span>(</span>
<span id="cb65-22"><a href="#cb65-22" aria-hidden="true" tabindex="-1"></a>      <span class="st">"ref_miRNA"</span> <span class="ot">=</span> <span class="st">"Reference miRNA"</span>,</span>
<span id="cb65-23"><a href="#cb65-23" aria-hidden="true" tabindex="-1"></a>      <span class="st">"iso_3p"</span> <span class="ot">=</span> <span class="st">"3' Isoform"</span>,</span>
<span id="cb65-24"><a href="#cb65-24" aria-hidden="true" tabindex="-1"></a>      <span class="st">"iso_5p"</span> <span class="ot">=</span> <span class="st">"5' Isoform"</span>,</span>
<span id="cb65-25"><a href="#cb65-25" aria-hidden="true" tabindex="-1"></a>      <span class="st">"iso_add3p"</span> <span class="ot">=</span> <span class="st">"3' Addition"</span>,</span>
<span id="cb65-26"><a href="#cb65-26" aria-hidden="true" tabindex="-1"></a>      <span class="st">"iso_snv_seed"</span> <span class="ot">=</span> <span class="st">"SNV in Seed Region"</span>,</span>
<span id="cb65-27"><a href="#cb65-27" aria-hidden="true" tabindex="-1"></a>      <span class="st">"iso_snv_central_offset"</span> <span class="ot">=</span> <span class="st">"SNV in Central Offset Region"</span>,</span>
<span id="cb65-28"><a href="#cb65-28" aria-hidden="true" tabindex="-1"></a>      <span class="st">"iso_snv_central"</span> <span class="ot">=</span> <span class="st">"SNV in Central Region"</span>,</span>
<span id="cb65-29"><a href="#cb65-29" aria-hidden="true" tabindex="-1"></a>      <span class="st">"iso_snv_central_supp"</span> <span class="ot">=</span> <span class="st">"Supported SNV in Central Region"</span></span>
<span id="cb65-30"><a href="#cb65-30" aria-hidden="true" tabindex="-1"></a>    )) <span class="sc">+</span></span>
<span id="cb65-31"><a href="#cb65-31" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme_minimal</span>(<span class="at">base_size =</span> <span class="dv">14</span>) <span class="sc">+</span></span>
<span id="cb65-32"><a href="#cb65-32" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme</span>(</span>
<span id="cb65-33"><a href="#cb65-33" aria-hidden="true" tabindex="-1"></a>    <span class="at">axis.text.x =</span> <span class="fu">element_text</span>(<span class="at">angle =</span> <span class="dv">90</span> , <span class="at">hjust =</span> <span class="dv">1</span>, <span class="at">size =</span> <span class="dv">5</span>),</span>
<span id="cb65-34"><a href="#cb65-34" aria-hidden="true" tabindex="-1"></a>    <span class="at">axis.text.y =</span> <span class="fu">element_text</span>(<span class="at">angle =</span> <span class="dv">0</span> , <span class="at">hjust =</span> <span class="dv">1</span>, <span class="at">size =</span> <span class="dv">7</span>),</span>
<span id="cb65-35"><a href="#cb65-35" aria-hidden="true" tabindex="-1"></a>    <span class="at">axis.title =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">8</span>),</span>
<span id="cb65-36"><a href="#cb65-36" aria-hidden="true" tabindex="-1"></a>    <span class="at">plot.title =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">12</span>, <span class="at">face =</span> <span class="st">"bold"</span>),</span>
<span id="cb65-37"><a href="#cb65-37" aria-hidden="true" tabindex="-1"></a>    <span class="at">legend.title =</span> <span class="fu">element_blank</span>(),</span>
<span id="cb65-38"><a href="#cb65-38" aria-hidden="true" tabindex="-1"></a>    <span class="at">legend.text =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">6</span>),</span>
<span id="cb65-39"><a href="#cb65-39" aria-hidden="true" tabindex="-1"></a>    <span class="at">legend.position =</span> <span class="st">"bottom"</span>,</span>
<span id="cb65-40"><a href="#cb65-40" aria-hidden="true" tabindex="-1"></a>    <span class="at">legend.margin =</span> <span class="fu">margin</span>(<span class="at">t =</span> <span class="sc">-</span><span class="dv">20</span>),  <span class="co"># Ajusta el margen superior de la leyenda</span></span>
<span id="cb65-41"><a href="#cb65-41" aria-hidden="true" tabindex="-1"></a><span class="co">#    plot.margin = margin(t = 10, r = 10, b = 30, l = 10)</span></span>
<span id="cb65-42"><a href="#cb65-42" aria-hidden="true" tabindex="-1"></a>  )</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<div class="cell">
<div class="sourceCode cell-code" id="cb66"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb66-1"><a href="#cb66-1" aria-hidden="true" tabindex="-1"></a><span class="do">##| label: fig-mirtop</span></span>
<span id="cb66-2"><a href="#cb66-2" aria-hidden="true" tabindex="-1"></a><span class="do">##| fig-cap: "Annotation of miRNAs and isomiRs with mirtop (v0.4.28). *A*: IsomiR read counts; *B*: IsomiR unique read counts; *C*: Mean isomiR read counts"</span></span>
<span id="cb66-3"><a href="#cb66-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb66-4"><a href="#cb66-4" aria-hidden="true" tabindex="-1"></a><span class="co">#grid.arrange(mirtop1, mirtop2, mirtop3, </span></span>
<span id="cb66-5"><a href="#cb66-5" aria-hidden="true" tabindex="-1"></a><span class="co">#             nrow = 3, </span></span>
<span id="cb66-6"><a href="#cb66-6" aria-hidden="true" tabindex="-1"></a><span class="co">#             heights = rep(1, 3) # Espaciado uniforme entre filas</span></span>
<span id="cb66-7"><a href="#cb66-7" aria-hidden="true" tabindex="-1"></a><span class="co">#)</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<div class="cell">
<div class="sourceCode cell-code" id="cb67"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb67-1"><a href="#cb67-1" aria-hidden="true" tabindex="-1"></a><span class="do">##| tbl-cap: "Descriptive statistics of Isomir Read counts "</span></span>
<span id="cb67-2"><a href="#cb67-2" aria-hidden="true" tabindex="-1"></a><span class="do">##| label: "tbl-isomir_read_counts"</span></span>
<span id="cb67-3"><a href="#cb67-3" aria-hidden="true" tabindex="-1"></a>file <span class="ot">&lt;-</span> <span class="st">"data/mirtop_read_count_plot.tsv"</span>  </span>
<span id="cb67-4"><a href="#cb67-4" aria-hidden="true" tabindex="-1"></a>data_mirtop1 <span class="ot">&lt;-</span> <span class="fu">read.table</span>(file, <span class="at">header =</span> <span class="cn">TRUE</span>, <span class="at">sep =</span> <span class="st">"</span><span class="sc">\t</span><span class="st">"</span>)</span>
<span id="cb67-5"><a href="#cb67-5" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb67-6"><a href="#cb67-6" aria-hidden="true" tabindex="-1"></a>data_mirtop1 <span class="ot">&lt;-</span> data_mirtop1 <span class="sc">%&gt;%</span></span>
<span id="cb67-7"><a href="#cb67-7" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">Total =</span> (ref_miRNA <span class="sc">+</span> iso_3p <span class="sc">+</span> iso_5p <span class="sc">+</span> iso_add3p <span class="sc">+</span> iso_snv_seed <span class="sc">+</span> iso_snv_central_offset <span class="sc">+</span> iso_snv_central <span class="sc">+</span>iso_snv_central_supp),</span>
<span id="cb67-8"><a href="#cb67-8" aria-hidden="true" tabindex="-1"></a>         <span class="at">ref_miRNA_percentage =</span> (ref_miRNA<span class="sc">/</span>Total)<span class="sc">*</span> <span class="dv">100</span>,</span>
<span id="cb67-9"><a href="#cb67-9" aria-hidden="true" tabindex="-1"></a>         <span class="at">iso_3p_percentage =</span> (iso_3p<span class="sc">/</span>Total)<span class="sc">*</span> <span class="dv">100</span>,</span>
<span id="cb67-10"><a href="#cb67-10" aria-hidden="true" tabindex="-1"></a>         <span class="at">iso_add3p_percentgae =</span> (iso_add3p<span class="sc">/</span>Total)<span class="sc">*</span> <span class="dv">100</span>,</span>
<span id="cb67-11"><a href="#cb67-11" aria-hidden="true" tabindex="-1"></a>         <span class="at">iso_5p_percentage =</span> (iso_5p<span class="sc">/</span>Total)<span class="sc">*</span> <span class="dv">100</span>,</span>
<span id="cb67-12"><a href="#cb67-12" aria-hidden="true" tabindex="-1"></a>         <span class="at">iso_snv_seed_percentage =</span> (iso_snv_seed<span class="sc">/</span>Total)<span class="sc">*</span> <span class="dv">100</span>)</span>
<span id="cb67-13"><a href="#cb67-13" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb67-14"><a href="#cb67-14" aria-hidden="true" tabindex="-1"></a>stats <span class="ot">&lt;-</span> data_mirtop1 <span class="sc">%&gt;%</span></span>
<span id="cb67-15"><a href="#cb67-15" aria-hidden="true" tabindex="-1"></a>  <span class="fu">summarise</span>(</span>
<span id="cb67-16"><a href="#cb67-16" aria-hidden="true" tabindex="-1"></a>    <span class="at">Mean_ref_miRNA_percentage =</span> <span class="fu">mean</span>(ref_miRNA_percentage, <span class="at">na.rm =</span> <span class="cn">TRUE</span>),</span>
<span id="cb67-17"><a href="#cb67-17" aria-hidden="true" tabindex="-1"></a>    <span class="at">Mean_iso_3p_percentage =</span> <span class="fu">mean</span>(iso_3p_percentage, <span class="at">na.rm =</span> <span class="cn">TRUE</span>),</span>
<span id="cb67-18"><a href="#cb67-18" aria-hidden="true" tabindex="-1"></a>    <span class="at">Max_ref_miRNA_percentage =</span> <span class="fu">max</span>(ref_miRNA_percentage, <span class="at">na.rm =</span> <span class="cn">TRUE</span>),</span>
<span id="cb67-19"><a href="#cb67-19" aria-hidden="true" tabindex="-1"></a>    <span class="at">Min_ref_miRNA_percentage =</span> <span class="fu">min</span>(ref_miRNA_percentage, <span class="at">na.rm =</span> <span class="cn">TRUE</span>),</span>
<span id="cb67-20"><a href="#cb67-20" aria-hidden="true" tabindex="-1"></a>    <span class="at">Max_iso_3p_percentage =</span> <span class="fu">max</span>(iso_3p_percentage, <span class="at">na.rm =</span> <span class="cn">TRUE</span>),</span>
<span id="cb67-21"><a href="#cb67-21" aria-hidden="true" tabindex="-1"></a>    <span class="at">Min_iso_3p_percentage =</span> <span class="fu">min</span>(iso_3p_percentage, <span class="at">na.rm =</span> <span class="cn">TRUE</span>),</span>
<span id="cb67-22"><a href="#cb67-22" aria-hidden="true" tabindex="-1"></a>    <span class="at">Mean_iso_add3p_percentgae =</span> <span class="fu">mean</span>(iso_add3p_percentgae, <span class="at">na.rm =</span> <span class="cn">TRUE</span>),</span>
<span id="cb67-23"><a href="#cb67-23" aria-hidden="true" tabindex="-1"></a>    <span class="at">Max_iso_add3p_percentgae =</span> <span class="fu">max</span>(iso_add3p_percentgae, <span class="at">na.rm =</span> <span class="cn">TRUE</span>),</span>
<span id="cb67-24"><a href="#cb67-24" aria-hidden="true" tabindex="-1"></a>    <span class="at">Min_iso_add3p_percentgae =</span> <span class="fu">min</span>(iso_add3p_percentgae, <span class="at">na.rm =</span> <span class="cn">TRUE</span>), </span>
<span id="cb67-25"><a href="#cb67-25" aria-hidden="true" tabindex="-1"></a>    <span class="at">Mean_iso_5p_percentage =</span> <span class="fu">mean</span>(iso_5p_percentage, <span class="at">na.rm =</span> <span class="cn">TRUE</span>),</span>
<span id="cb67-26"><a href="#cb67-26" aria-hidden="true" tabindex="-1"></a>    <span class="at">Max_iso_5p_percentage =</span> <span class="fu">max</span>(iso_5p_percentage, <span class="at">na.rm =</span> <span class="cn">TRUE</span>),</span>
<span id="cb67-27"><a href="#cb67-27" aria-hidden="true" tabindex="-1"></a>    <span class="at">Min_iso_5p_percentage =</span> <span class="fu">min</span>(iso_5p_percentage, <span class="at">na.rm =</span> <span class="cn">TRUE</span>),</span>
<span id="cb67-28"><a href="#cb67-28" aria-hidden="true" tabindex="-1"></a>    <span class="at">Mean_iso_snv_seed_percentage =</span> <span class="fu">mean</span>(iso_snv_seed_percentage, <span class="at">na.rm =</span> <span class="cn">TRUE</span>)</span>
<span id="cb67-29"><a href="#cb67-29" aria-hidden="true" tabindex="-1"></a>    ) </span>
<span id="cb67-30"><a href="#cb67-30" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb67-31"><a href="#cb67-31" aria-hidden="true" tabindex="-1"></a><span class="co">#knitr::kable(stats_by_group,</span></span>
<span id="cb67-32"><a href="#cb67-32" aria-hidden="true" tabindex="-1"></a><span class="co">#             col.names = c("Group", </span></span>
<span id="cb67-33"><a href="#cb67-33" aria-hidden="true" tabindex="-1"></a><span class="co">#                           "TM", </span></span>
<span id="cb67-34"><a href="#cb67-34" aria-hidden="true" tabindex="-1"></a><span class="co">#                           "TU", </span></span>
<span id="cb67-35"><a href="#cb67-35" aria-hidden="true" tabindex="-1"></a><span class="co">#                           "Mean M", </span></span>
<span id="cb67-36"><a href="#cb67-36" aria-hidden="true" tabindex="-1"></a><span class="co">#                           "Max M", </span></span>
<span id="cb67-37"><a href="#cb67-37" aria-hidden="true" tabindex="-1"></a><span class="co">#                           "Min M")) %&gt;%</span></span>
<span id="cb67-38"><a href="#cb67-38" aria-hidden="true" tabindex="-1"></a><span class="co">#  kable_styling(</span></span>
<span id="cb67-39"><a href="#cb67-39" aria-hidden="true" tabindex="-1"></a><span class="co">#    latex_options = c("hold_position", "scale_down"), # Escala automáticamente</span></span>
<span id="cb67-40"><a href="#cb67-40" aria-hidden="true" tabindex="-1"></a><span class="co">#    font_size = 10.5   # Ajustar tamaño de fuente</span></span>
<span id="cb67-41"><a href="#cb67-41" aria-hidden="true" tabindex="-1"></a><span class="co">#  ) %&gt;%</span></span>
<span id="cb67-42"><a href="#cb67-42" aria-hidden="true" tabindex="-1"></a><span class="co">#  row_spec(0, bold = TRUE) # Encabezado en negrita</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<div class="cell">
<div class="sourceCode cell-code" id="cb68"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb68-1"><a href="#cb68-1" aria-hidden="true" tabindex="-1"></a><span class="do">##| tbl-cap: "Descriptive statistics of Isomir Read counts "</span></span>
<span id="cb68-2"><a href="#cb68-2" aria-hidden="true" tabindex="-1"></a><span class="do">##| label: "tbl-isomir_read_counts"</span></span>
<span id="cb68-3"><a href="#cb68-3" aria-hidden="true" tabindex="-1"></a>file <span class="ot">&lt;-</span> <span class="st">"data/mirtop_unique_read_count_plot.tsv"</span>  </span>
<span id="cb68-4"><a href="#cb68-4" aria-hidden="true" tabindex="-1"></a>data_mirtop2 <span class="ot">&lt;-</span> <span class="fu">read.table</span>(file, <span class="at">header =</span> <span class="cn">TRUE</span>, <span class="at">sep =</span> <span class="st">"</span><span class="sc">\t</span><span class="st">"</span>)</span>
<span id="cb68-5"><a href="#cb68-5" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb68-6"><a href="#cb68-6" aria-hidden="true" tabindex="-1"></a>data_mirtop2 <span class="ot">&lt;-</span> data_mirtop2 <span class="sc">%&gt;%</span></span>
<span id="cb68-7"><a href="#cb68-7" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">Total =</span> (ref_miRNA <span class="sc">+</span> iso_3p <span class="sc">+</span> iso_5p <span class="sc">+</span> iso_add3p <span class="sc">+</span> iso_snv_seed <span class="sc">+</span> iso_snv_central_offset <span class="sc">+</span> iso_snv_central <span class="sc">+</span>iso_snv_central_supp),</span>
<span id="cb68-8"><a href="#cb68-8" aria-hidden="true" tabindex="-1"></a>         <span class="at">ref_miRNA_percentage =</span> (ref_miRNA<span class="sc">/</span>Total)<span class="sc">*</span> <span class="dv">100</span>,</span>
<span id="cb68-9"><a href="#cb68-9" aria-hidden="true" tabindex="-1"></a>         <span class="at">iso_3p_percentage =</span> (iso_3p<span class="sc">/</span>Total)<span class="sc">*</span> <span class="dv">100</span>,</span>
<span id="cb68-10"><a href="#cb68-10" aria-hidden="true" tabindex="-1"></a>         <span class="at">iso_add3p_percentgae =</span> (iso_add3p<span class="sc">/</span>Total)<span class="sc">*</span> <span class="dv">100</span>,</span>
<span id="cb68-11"><a href="#cb68-11" aria-hidden="true" tabindex="-1"></a>         <span class="at">iso_5p_percentage =</span> (iso_5p<span class="sc">/</span>Total)<span class="sc">*</span> <span class="dv">100</span>,</span>
<span id="cb68-12"><a href="#cb68-12" aria-hidden="true" tabindex="-1"></a>         <span class="at">iso_snv_central_offset_percentage =</span> (iso_snv_central_offset<span class="sc">/</span>Total)<span class="sc">*</span> <span class="dv">100</span>,</span>
<span id="cb68-13"><a href="#cb68-13" aria-hidden="true" tabindex="-1"></a>         <span class="at">iso_snv_central_percentage =</span> (iso_snv_central<span class="sc">/</span>Total)<span class="sc">*</span> <span class="dv">100</span>,</span>
<span id="cb68-14"><a href="#cb68-14" aria-hidden="true" tabindex="-1"></a>         <span class="at">iso_snv_central_supp_percentage =</span> (iso_snv_central_supp<span class="sc">/</span>Total)<span class="sc">*</span> <span class="dv">100</span>,</span>
<span id="cb68-15"><a href="#cb68-15" aria-hidden="true" tabindex="-1"></a>         <span class="at">iso_snv_seed_percentage =</span> (iso_snv_seed<span class="sc">/</span>Total)<span class="sc">*</span> <span class="dv">100</span>)</span>
<span id="cb68-16"><a href="#cb68-16" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb68-17"><a href="#cb68-17" aria-hidden="true" tabindex="-1"></a>stats2 <span class="ot">&lt;-</span> data_mirtop2 <span class="sc">%&gt;%</span></span>
<span id="cb68-18"><a href="#cb68-18" aria-hidden="true" tabindex="-1"></a>  <span class="fu">summarise</span>(</span>
<span id="cb68-19"><a href="#cb68-19" aria-hidden="true" tabindex="-1"></a>    <span class="at">Mean_ref_miRNA_percentage =</span> <span class="fu">mean</span>(ref_miRNA_percentage, <span class="at">na.rm =</span> <span class="cn">TRUE</span>),</span>
<span id="cb68-20"><a href="#cb68-20" aria-hidden="true" tabindex="-1"></a>    <span class="at">Max_ref_miRNA_percentage =</span> <span class="fu">max</span>(ref_miRNA_percentage, <span class="at">na.rm =</span> <span class="cn">TRUE</span>),</span>
<span id="cb68-21"><a href="#cb68-21" aria-hidden="true" tabindex="-1"></a>    <span class="at">Min_ref_miRNA_percentage =</span> <span class="fu">min</span>(ref_miRNA_percentage, <span class="at">na.rm =</span> <span class="cn">TRUE</span>),</span>
<span id="cb68-22"><a href="#cb68-22" aria-hidden="true" tabindex="-1"></a>    <span class="at">Mean_iso_3p_percentage =</span> <span class="fu">mean</span>(iso_3p_percentage, <span class="at">na.rm =</span> <span class="cn">TRUE</span>),</span>
<span id="cb68-23"><a href="#cb68-23" aria-hidden="true" tabindex="-1"></a>    <span class="at">Max_iso_3p_percentage =</span> <span class="fu">max</span>(iso_3p_percentage, <span class="at">na.rm =</span> <span class="cn">TRUE</span>),</span>
<span id="cb68-24"><a href="#cb68-24" aria-hidden="true" tabindex="-1"></a>    <span class="at">Min_iso_3p_percentage =</span> <span class="fu">min</span>(iso_3p_percentage, <span class="at">na.rm =</span> <span class="cn">TRUE</span>),</span>
<span id="cb68-25"><a href="#cb68-25" aria-hidden="true" tabindex="-1"></a>    <span class="at">Mean_iso_add3p_percentgae =</span> <span class="fu">mean</span>(iso_add3p_percentgae, <span class="at">na.rm =</span> <span class="cn">TRUE</span>),</span>
<span id="cb68-26"><a href="#cb68-26" aria-hidden="true" tabindex="-1"></a>    <span class="at">Max_iso_add3p_percentgae =</span> <span class="fu">max</span>(iso_add3p_percentgae, <span class="at">na.rm =</span> <span class="cn">TRUE</span>),</span>
<span id="cb68-27"><a href="#cb68-27" aria-hidden="true" tabindex="-1"></a>    <span class="at">Min_iso_add3p_percentgae =</span> <span class="fu">min</span>(iso_add3p_percentgae, <span class="at">na.rm =</span> <span class="cn">TRUE</span>), </span>
<span id="cb68-28"><a href="#cb68-28" aria-hidden="true" tabindex="-1"></a>    <span class="at">Mean_iso_5p_percentage =</span> <span class="fu">mean</span>(iso_5p_percentage, <span class="at">na.rm =</span> <span class="cn">TRUE</span>),</span>
<span id="cb68-29"><a href="#cb68-29" aria-hidden="true" tabindex="-1"></a>    <span class="at">Max_iso_5p_percentage =</span> <span class="fu">max</span>(iso_5p_percentage, <span class="at">na.rm =</span> <span class="cn">TRUE</span>),</span>
<span id="cb68-30"><a href="#cb68-30" aria-hidden="true" tabindex="-1"></a>    <span class="at">Min_iso_5p_percentage =</span> <span class="fu">min</span>(iso_5p_percentage, <span class="at">na.rm =</span> <span class="cn">TRUE</span>),</span>
<span id="cb68-31"><a href="#cb68-31" aria-hidden="true" tabindex="-1"></a>    <span class="at">Mean_iso_snv_central_offset_percentage =</span> <span class="fu">mean</span>(iso_snv_central_offset_percentage, <span class="at">na.rm =</span> <span class="cn">TRUE</span>),</span>
<span id="cb68-32"><a href="#cb68-32" aria-hidden="true" tabindex="-1"></a>    <span class="at">Mean_iso_snv_central_percentage =</span> <span class="fu">mean</span>(iso_snv_central_percentage, <span class="at">na.rm =</span> <span class="cn">TRUE</span>),</span>
<span id="cb68-33"><a href="#cb68-33" aria-hidden="true" tabindex="-1"></a>    <span class="at">Mean_iso_snv_central_supp_percentage =</span> <span class="fu">mean</span>(iso_snv_central_supp_percentage, <span class="at">na.rm =</span> <span class="cn">TRUE</span>),</span>
<span id="cb68-34"><a href="#cb68-34" aria-hidden="true" tabindex="-1"></a>    <span class="at">Mean_iso_snv_seed_percentage =</span> <span class="fu">mean</span>(iso_snv_seed_percentage, <span class="at">na.rm =</span> <span class="cn">TRUE</span>),</span>
<span id="cb68-35"><a href="#cb68-35" aria-hidden="true" tabindex="-1"></a>    <span class="at">mean_ref_miRNA =</span> <span class="fu">mean</span>(ref_miRNA, <span class="at">na.rm =</span> <span class="cn">TRUE</span>),</span>
<span id="cb68-36"><a href="#cb68-36" aria-hidden="true" tabindex="-1"></a>    <span class="at">max_ref_mirna =</span> <span class="fu">max</span>(ref_miRNA, <span class="at">na.rm =</span> <span class="cn">TRUE</span>),</span>
<span id="cb68-37"><a href="#cb68-37" aria-hidden="true" tabindex="-1"></a>    <span class="at">min_ref_mirna =</span> <span class="fu">min</span>(ref_miRNA, <span class="at">na.rm =</span> <span class="cn">TRUE</span>)</span>
<span id="cb68-38"><a href="#cb68-38" aria-hidden="true" tabindex="-1"></a>    ) </span>
<span id="cb68-39"><a href="#cb68-39" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb68-40"><a href="#cb68-40" aria-hidden="true" tabindex="-1"></a><span class="co">#knitr::kable(stats_by_group,</span></span>
<span id="cb68-41"><a href="#cb68-41" aria-hidden="true" tabindex="-1"></a><span class="co">#             col.names = c("Group", </span></span>
<span id="cb68-42"><a href="#cb68-42" aria-hidden="true" tabindex="-1"></a><span class="co">#                           "TM", </span></span>
<span id="cb68-43"><a href="#cb68-43" aria-hidden="true" tabindex="-1"></a><span class="co">#                           "TU", </span></span>
<span id="cb68-44"><a href="#cb68-44" aria-hidden="true" tabindex="-1"></a><span class="co">#                           "Mean M", </span></span>
<span id="cb68-45"><a href="#cb68-45" aria-hidden="true" tabindex="-1"></a><span class="co">#                           "Max M", </span></span>
<span id="cb68-46"><a href="#cb68-46" aria-hidden="true" tabindex="-1"></a><span class="co">#                           "Min M")) %&gt;%</span></span>
<span id="cb68-47"><a href="#cb68-47" aria-hidden="true" tabindex="-1"></a><span class="co">#  kable_styling(</span></span>
<span id="cb68-48"><a href="#cb68-48" aria-hidden="true" tabindex="-1"></a><span class="co">#    latex_options = c("hold_position", "scale_down"), # Escala automáticamente</span></span>
<span id="cb68-49"><a href="#cb68-49" aria-hidden="true" tabindex="-1"></a><span class="co">#    font_size = 10.5   # Ajustar tamaño de fuente</span></span>
<span id="cb68-50"><a href="#cb68-50" aria-hidden="true" tabindex="-1"></a><span class="co">#  ) %&gt;%</span></span>
<span id="cb68-51"><a href="#cb68-51" aria-hidden="true" tabindex="-1"></a><span class="co">#  row_spec(0, bold = TRUE) # Encabezado en negrita</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<div class="cell">
<div class="sourceCode cell-code" id="cb69"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb69-1"><a href="#cb69-1" aria-hidden="true" tabindex="-1"></a><span class="do">##| tbl-cap: "Descriptive statistics of Isomir Read counts "</span></span>
<span id="cb69-2"><a href="#cb69-2" aria-hidden="true" tabindex="-1"></a><span class="do">##| label: "tbl-isomir_read_counts"</span></span>
<span id="cb69-3"><a href="#cb69-3" aria-hidden="true" tabindex="-1"></a>file <span class="ot">&lt;-</span> <span class="st">"data/mirtop_mean_read_count_plot.tsv"</span>  </span>
<span id="cb69-4"><a href="#cb69-4" aria-hidden="true" tabindex="-1"></a>data_mirtop3 <span class="ot">&lt;-</span> <span class="fu">read.table</span>(file, <span class="at">header =</span> <span class="cn">TRUE</span>, <span class="at">sep =</span> <span class="st">"</span><span class="sc">\t</span><span class="st">"</span>)</span>
<span id="cb69-5"><a href="#cb69-5" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb69-6"><a href="#cb69-6" aria-hidden="true" tabindex="-1"></a>data_mirtop3 <span class="ot">&lt;-</span> data_mirtop3 <span class="sc">%&gt;%</span></span>
<span id="cb69-7"><a href="#cb69-7" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">Total =</span> (ref_miRNA <span class="sc">+</span> iso_3p <span class="sc">+</span> iso_5p <span class="sc">+</span> iso_add3p <span class="sc">+</span> iso_snv_seed <span class="sc">+</span> iso_snv_central_offset <span class="sc">+</span> iso_snv_central <span class="sc">+</span>iso_snv_central_supp),</span>
<span id="cb69-8"><a href="#cb69-8" aria-hidden="true" tabindex="-1"></a>         <span class="at">ref_miRNA_percentage =</span> (ref_miRNA<span class="sc">/</span>Total)<span class="sc">*</span> <span class="dv">100</span>,</span>
<span id="cb69-9"><a href="#cb69-9" aria-hidden="true" tabindex="-1"></a>         <span class="at">iso_3p_percentage =</span> (iso_3p<span class="sc">/</span>Total)<span class="sc">*</span> <span class="dv">100</span>,</span>
<span id="cb69-10"><a href="#cb69-10" aria-hidden="true" tabindex="-1"></a>         <span class="at">iso_add3p_percentgae =</span> (iso_add3p<span class="sc">/</span>Total)<span class="sc">*</span> <span class="dv">100</span>,</span>
<span id="cb69-11"><a href="#cb69-11" aria-hidden="true" tabindex="-1"></a>         <span class="at">iso_5p_percentage =</span> (iso_5p<span class="sc">/</span>Total)<span class="sc">*</span> <span class="dv">100</span>,</span>
<span id="cb69-12"><a href="#cb69-12" aria-hidden="true" tabindex="-1"></a>         <span class="at">iso_snv_central_offset_percentage =</span> (iso_snv_central_offset<span class="sc">/</span>Total)<span class="sc">*</span> <span class="dv">100</span>,</span>
<span id="cb69-13"><a href="#cb69-13" aria-hidden="true" tabindex="-1"></a>         <span class="at">iso_snv_central_percentage =</span> (iso_snv_central<span class="sc">/</span>Total)<span class="sc">*</span> <span class="dv">100</span>,</span>
<span id="cb69-14"><a href="#cb69-14" aria-hidden="true" tabindex="-1"></a>         <span class="at">iso_snv_central_supp_percentage =</span> (iso_snv_central_supp<span class="sc">/</span>Total)<span class="sc">*</span> <span class="dv">100</span>,</span>
<span id="cb69-15"><a href="#cb69-15" aria-hidden="true" tabindex="-1"></a>         <span class="at">iso_snv_seed_percentage =</span> (iso_snv_seed<span class="sc">/</span>Total)<span class="sc">*</span> <span class="dv">100</span>)</span>
<span id="cb69-16"><a href="#cb69-16" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb69-17"><a href="#cb69-17" aria-hidden="true" tabindex="-1"></a>stats3 <span class="ot">&lt;-</span> data_mirtop3 <span class="sc">%&gt;%</span></span>
<span id="cb69-18"><a href="#cb69-18" aria-hidden="true" tabindex="-1"></a>  <span class="fu">summarise</span>(</span>
<span id="cb69-19"><a href="#cb69-19" aria-hidden="true" tabindex="-1"></a>    <span class="at">Mean_ref_miRNA_percentage =</span> <span class="fu">mean</span>(ref_miRNA_percentage, <span class="at">na.rm =</span> <span class="cn">TRUE</span>),</span>
<span id="cb69-20"><a href="#cb69-20" aria-hidden="true" tabindex="-1"></a>    <span class="at">Max_ref_miRNA_percentage =</span> <span class="fu">max</span>(ref_miRNA_percentage, <span class="at">na.rm =</span> <span class="cn">TRUE</span>),</span>
<span id="cb69-21"><a href="#cb69-21" aria-hidden="true" tabindex="-1"></a>    <span class="at">Min_ref_miRNA_percentage =</span> <span class="fu">min</span>(ref_miRNA_percentage, <span class="at">na.rm =</span> <span class="cn">TRUE</span>),</span>
<span id="cb69-22"><a href="#cb69-22" aria-hidden="true" tabindex="-1"></a>    <span class="at">Mean_iso_3p_percentage =</span> <span class="fu">mean</span>(iso_3p_percentage, <span class="at">na.rm =</span> <span class="cn">TRUE</span>),</span>
<span id="cb69-23"><a href="#cb69-23" aria-hidden="true" tabindex="-1"></a>    <span class="at">Max_iso_3p_percentage =</span> <span class="fu">max</span>(iso_3p_percentage, <span class="at">na.rm =</span> <span class="cn">TRUE</span>),</span>
<span id="cb69-24"><a href="#cb69-24" aria-hidden="true" tabindex="-1"></a>    <span class="at">Min_iso_3p_percentage =</span> <span class="fu">min</span>(iso_3p_percentage, <span class="at">na.rm =</span> <span class="cn">TRUE</span>),</span>
<span id="cb69-25"><a href="#cb69-25" aria-hidden="true" tabindex="-1"></a>    <span class="at">Mean_iso_add3p_percentgae =</span> <span class="fu">mean</span>(iso_add3p_percentgae, <span class="at">na.rm =</span> <span class="cn">TRUE</span>),</span>
<span id="cb69-26"><a href="#cb69-26" aria-hidden="true" tabindex="-1"></a>    <span class="at">Max_iso_add3p_percentgae =</span> <span class="fu">max</span>(iso_add3p_percentgae, <span class="at">na.rm =</span> <span class="cn">TRUE</span>),</span>
<span id="cb69-27"><a href="#cb69-27" aria-hidden="true" tabindex="-1"></a>    <span class="at">Min_iso_add3p_percentgae =</span> <span class="fu">min</span>(iso_add3p_percentgae, <span class="at">na.rm =</span> <span class="cn">TRUE</span>), </span>
<span id="cb69-28"><a href="#cb69-28" aria-hidden="true" tabindex="-1"></a>    <span class="at">Mean_iso_5p_percentage =</span> <span class="fu">mean</span>(iso_5p_percentage, <span class="at">na.rm =</span> <span class="cn">TRUE</span>),</span>
<span id="cb69-29"><a href="#cb69-29" aria-hidden="true" tabindex="-1"></a>    <span class="at">Max_iso_5p_percentage =</span> <span class="fu">max</span>(iso_5p_percentage, <span class="at">na.rm =</span> <span class="cn">TRUE</span>),</span>
<span id="cb69-30"><a href="#cb69-30" aria-hidden="true" tabindex="-1"></a>    <span class="at">Min_iso_5p_percentage =</span> <span class="fu">min</span>(iso_5p_percentage, <span class="at">na.rm =</span> <span class="cn">TRUE</span>),</span>
<span id="cb69-31"><a href="#cb69-31" aria-hidden="true" tabindex="-1"></a>    <span class="at">Mean_iso_snv_central_offset_percentage =</span> <span class="fu">mean</span>(iso_snv_central_offset_percentage, <span class="at">na.rm =</span> <span class="cn">TRUE</span>),</span>
<span id="cb69-32"><a href="#cb69-32" aria-hidden="true" tabindex="-1"></a>    <span class="at">Mean_iso_snv_central_percentage =</span> <span class="fu">mean</span>(iso_snv_central_percentage, <span class="at">na.rm =</span> <span class="cn">TRUE</span>),</span>
<span id="cb69-33"><a href="#cb69-33" aria-hidden="true" tabindex="-1"></a>    <span class="at">Mean_iso_snv_central_supp_percentage =</span> <span class="fu">mean</span>(iso_snv_central_supp_percentage, <span class="at">na.rm =</span> <span class="cn">TRUE</span>),</span>
<span id="cb69-34"><a href="#cb69-34" aria-hidden="true" tabindex="-1"></a>    <span class="at">Mean_iso_snv_seed_percentage =</span> <span class="fu">mean</span>(iso_snv_seed_percentage, <span class="at">na.rm =</span> <span class="cn">TRUE</span>)</span>
<span id="cb69-35"><a href="#cb69-35" aria-hidden="true" tabindex="-1"></a>    </span>
<span id="cb69-36"><a href="#cb69-36" aria-hidden="true" tabindex="-1"></a>    ) </span>
<span id="cb69-37"><a href="#cb69-37" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb69-38"><a href="#cb69-38" aria-hidden="true" tabindex="-1"></a><span class="co">#knitr::kable(stats_by_group,</span></span>
<span id="cb69-39"><a href="#cb69-39" aria-hidden="true" tabindex="-1"></a><span class="co">#             col.names = c("Group", </span></span>
<span id="cb69-40"><a href="#cb69-40" aria-hidden="true" tabindex="-1"></a><span class="co">#                           "TM", </span></span>
<span id="cb69-41"><a href="#cb69-41" aria-hidden="true" tabindex="-1"></a><span class="co">#                           "TU", </span></span>
<span id="cb69-42"><a href="#cb69-42" aria-hidden="true" tabindex="-1"></a><span class="co">#                           "Mean M", </span></span>
<span id="cb69-43"><a href="#cb69-43" aria-hidden="true" tabindex="-1"></a><span class="co">#                           "Max M", </span></span>
<span id="cb69-44"><a href="#cb69-44" aria-hidden="true" tabindex="-1"></a><span class="co">#                           "Min M")) %&gt;%</span></span>
<span id="cb69-45"><a href="#cb69-45" aria-hidden="true" tabindex="-1"></a><span class="co">#  kable_styling(</span></span>
<span id="cb69-46"><a href="#cb69-46" aria-hidden="true" tabindex="-1"></a><span class="co">#    latex_options = c("hold_position", "scale_down"), # Escala automáticamente</span></span>
<span id="cb69-47"><a href="#cb69-47" aria-hidden="true" tabindex="-1"></a><span class="co">#    font_size = 10.5   # Ajustar tamaño de fuente</span></span>
<span id="cb69-48"><a href="#cb69-48" aria-hidden="true" tabindex="-1"></a><span class="co">#  ) %&gt;%</span></span>
<span id="cb69-49"><a href="#cb69-49" aria-hidden="true" tabindex="-1"></a><span class="co">#  row_spec(0, bold = TRUE) # Encabezado en negrita</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
</section>
<section id="differential-expression-analysis" class="level2">
<h2 class="anchored" data-anchor-id="differential-expression-analysis">Differential expression analysis</h2>
<div class="cell">
<div class="sourceCode cell-code" id="cb70"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb70-1"><a href="#cb70-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Cuentas con los miRNA de referencia</span></span>
<span id="cb70-2"><a href="#cb70-2" aria-hidden="true" tabindex="-1"></a>ids_counts <span class="ot">&lt;-</span> <span class="fu">isoCounts</span>(mirna_data_steatosis, <span class="at">minc =</span> <span class="dv">20</span>, <span class="at">mins =</span> <span class="dv">40</span>)</span>
<span id="cb70-3"><a href="#cb70-3" aria-hidden="true" tabindex="-1"></a><span class="co"># Plot</span></span>
<span id="cb70-4"><a href="#cb70-4" aria-hidden="true" tabindex="-1"></a><span class="do">##| fig-cap: "fig-isomiRsAbundance"</span></span>
<span id="cb70-5"><a href="#cb70-5" aria-hidden="true" tabindex="-1"></a><span class="do">##| out-width: "15cm"</span></span>
<span id="cb70-6"><a href="#cb70-6" aria-hidden="true" tabindex="-1"></a><span class="co">#isoPlot(ids_counts,type="all")</span></span>
<span id="cb70-7"><a href="#cb70-7" aria-hidden="true" tabindex="-1"></a><span class="co">#title(xlab = "Unique Identifier", ylab = "Percentage of Relative Abundance")</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<p>Differential expression analysis is a bioinformatics and statistical technique used to identify genes, proteins, or other biomolecules that exhibit significant differences in expression levels between two or more biological conditions. For this analysis, the sequences of miRNAs of reference were utilized, as annotated in the previous analysis.</p>
<p>Under the hood of <em>DESeq2</em>, we applied the <em>regularized logarithm</em> (<em>rlog</em>) transformation for normalization, which incorporates a prior on the sample differences <span class="citation" data-cites="love2014moderated">[@love2014moderated]</span>. This step is crucial because the standard logarithmic transformation can be sensitive to low expression values, whereas <em>rlog</em> addresses this issue by introducing regularization that helps stabilize variability between samples. This method facilitates seamless integration into subsequent analyses, as demonstrated in <a href="#fig-heatmap" class="quarto-xref">Figure&nbsp;9</a>.The figure shows a total of 374 miRNAs were found to be expressed differentially in the overall cohort when patients (samples) were categorized by sex and steatosis.</p>
<div class="cell">
<div class="sourceCode cell-code" id="cb71"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb71-1"><a href="#cb71-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Visualizar resultados</span></span>
<span id="cb71-2"><a href="#cb71-2" aria-hidden="true" tabindex="-1"></a><span class="co">#| label: fig-heatmap</span></span>
<span id="cb71-3"><a href="#cb71-3" aria-hidden="true" tabindex="-1"></a><span class="co">#| fig-cap: "The heatmap displays all miRNA data (374) across all samples. In the heatmap, shades of red represent increased miRNA expression, while shades of blue signify reduced or absent miRNA expression. Despite the identification of numerous miRNAs, no clear grouping patterns were observed across the samples."</span></span>
<span id="cb71-4"><a href="#cb71-4" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(pheatmap)</span>
<span id="cb71-5"><a href="#cb71-5" aria-hidden="true" tabindex="-1"></a>ids <span class="ot">=</span> <span class="fu">isoNorm</span>(ids_counts, <span class="at">formula =</span>  <span class="sc">~</span> sexo <span class="sc">+</span> Esteatosis)</span>
<span id="cb71-6"><a href="#cb71-6" aria-hidden="true" tabindex="-1"></a><span class="fu">pheatmap</span>(<span class="fu">counts</span>(ids, <span class="at">norm=</span><span class="cn">TRUE</span>)[<span class="dv">1</span><span class="sc">:</span><span class="dv">374</span>,], </span>
<span id="cb71-7"><a href="#cb71-7" aria-hidden="true" tabindex="-1"></a>         <span class="at">annotation_col =</span> <span class="fu">data.frame</span>(<span class="fu">colData</span>(ids)[,<span class="fu">c</span>(<span class="dv">1</span><span class="sc">:</span><span class="dv">2</span>),<span class="at">drop=</span><span class="cn">FALSE</span>]),</span>
<span id="cb71-8"><a href="#cb71-8" aria-hidden="true" tabindex="-1"></a>         <span class="at">show_rownames =</span> <span class="cn">FALSE</span>, <span class="at">scale=</span><span class="st">"row"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<div id="fig-heatmap" class="quarto-float quarto-figure quarto-figure-center anchored" data-fig-pos="H">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-heatmap-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="Results_files/figure-html/fig-heatmap-1.png" class="img-fluid figure-img" data-fig-pos="H" width="1152">
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-heatmap-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;9: The heatmap illustrates the expression of 374 individual miRNA sequences across the analyzed samples. In this visualization, shades of red indicate increased miRNA expression, whereas shades of blue denote reduced or absent miRNA expression. Although a substantial number of miRNAs were identified, no distinct grouping patterns emerged among the analyzed samples, suggesting heterogeneity in miRNA expression profiles across the dataset.
</figcaption>
</figure>
</div>
</div>
</div>
<p>Since we were primarily interested in the transcriptional changes that may precede the initiation of the steatosis process, we focused on contrasting individuals across different levels of steatosis. In this experiment, the goal is to identify miRNAs that are expressed differentially at various levels of steatosis. In our differential expression analysis using <em>DESeq2</em> with the LRT model, we initially identified 169 miRNAs as upregulated and 205 downregulated in the subcutaneous white adipose tissue (scWAT) of individuals according their steatosis degree. Comparing these four groups and applying a false discovery rate (FDR) cutoff of less than 0.05, the number of statistically siginificant differentially expressed miRNAs was greatly reduced to 2 (<a href="#fig-volcan" class="quarto-xref">Figure&nbsp;10</a>). Of these, we found <em>hsa-miR-372-3p</em> is upregulated and <em>hsa-miR-144-3p</em> is downregulated (<a href="#tbl-mirnas_selected" class="quarto-xref">Table&nbsp;4</a>).</p>
<div class="cell">
<div class="sourceCode cell-code" id="cb72"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb72-1"><a href="#cb72-1" aria-hidden="true" tabindex="-1"></a><span class="do">## DESIGNS</span></span>
<span id="cb72-2"><a href="#cb72-2" aria-hidden="true" tabindex="-1"></a>  <span class="co"># Reduced</span></span>
<span id="cb72-3"><a href="#cb72-3" aria-hidden="true" tabindex="-1"></a>reduced_design<span class="ot">=</span><span class="fu">model.matrix</span>( <span class="sc">~</span> sexo, <span class="at">data=</span>mdata_steatosis)</span>
<span id="cb72-4"><a href="#cb72-4" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb72-5"><a href="#cb72-5" aria-hidden="true" tabindex="-1"></a>  <span class="co"># Complete</span></span>
<span id="cb72-6"><a href="#cb72-6" aria-hidden="true" tabindex="-1"></a>complete_design<span class="ot">=</span><span class="fu">model.matrix</span>( <span class="sc">~</span> sexo <span class="sc">+</span> Esteatosis,<span class="at">data=</span>mdata_steatosis)</span>
<span id="cb72-7"><a href="#cb72-7" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb72-8"><a href="#cb72-8" aria-hidden="true" tabindex="-1"></a><span class="co"># Ajustar el Modelo Completo</span></span>
<span id="cb72-9"><a href="#cb72-9" aria-hidden="true" tabindex="-1"></a>dds <span class="ot">&lt;-</span> <span class="fu">DESeqDataSetFromMatrix</span>(<span class="at">countData =</span> <span class="fu">counts</span>(ids_counts),</span>
<span id="cb72-10"><a href="#cb72-10" aria-hidden="true" tabindex="-1"></a>                              <span class="at">colData =</span> <span class="fu">colData</span>(ids_counts),</span>
<span id="cb72-11"><a href="#cb72-11" aria-hidden="true" tabindex="-1"></a>                              <span class="at">design =</span> complete_design)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>converting counts to integer mode</code></pre>
</div>
<div class="sourceCode cell-code" id="cb74"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb74-1"><a href="#cb74-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Ajustar modelo reducido (sin 'steatosis')</span></span>
<span id="cb74-2"><a href="#cb74-2" aria-hidden="true" tabindex="-1"></a>dds_1 <span class="ot">&lt;-</span> <span class="fu">DESeq</span>(dds, <span class="at">test =</span> <span class="st">"LRT"</span>, <span class="at">reduced =</span> reduced_design)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>using supplied model matrix</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>estimating size factors</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>estimating dispersions</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>gene-wise dispersion estimates</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>mean-dispersion relationship</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>final dispersion estimates</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>fitting model and testing</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>-- replacing outliers and refitting for 4 genes
-- DESeq argument 'minReplicatesForReplace' = 7 
-- original counts are preserved in counts(dds)</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>estimating dispersions</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>fitting model and testing</code></pre>
</div>
<div class="sourceCode cell-code" id="cb85"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb85-1"><a href="#cb85-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Obtener los Resultados</span></span>
<span id="cb85-2"><a href="#cb85-2" aria-hidden="true" tabindex="-1"></a>resultsLRT <span class="ot">&lt;-</span> <span class="fu">results</span>(dds_1, <span class="at">pAdjustMethod =</span> <span class="st">"fdr"</span>, <span class="at">independentFiltering =</span> <span class="cn">TRUE</span>)</span>
<span id="cb85-3"><a href="#cb85-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb85-4"><a href="#cb85-4" aria-hidden="true" tabindex="-1"></a><span class="co"># Apply the filter of 0.05 (FDR)</span></span>
<span id="cb85-5"><a href="#cb85-5" aria-hidden="true" tabindex="-1"></a>resultsLRT_df <span class="ot">&lt;-</span> resultsLRT <span class="sc">|&gt;</span></span>
<span id="cb85-6"><a href="#cb85-6" aria-hidden="true" tabindex="-1"></a>  <span class="fu">as.data.frame</span>()</span>
<span id="cb85-7"><a href="#cb85-7" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb85-8"><a href="#cb85-8" aria-hidden="true" tabindex="-1"></a>r_sc <span class="ot">&lt;-</span> resultsLRT_df <span class="sc">|&gt;</span> </span>
<span id="cb85-9"><a href="#cb85-9" aria-hidden="true" tabindex="-1"></a>  <span class="fu">filter</span>(padj <span class="sc">&lt;</span> <span class="fl">0.05</span>)</span>
<span id="cb85-10"><a href="#cb85-10" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb85-11"><a href="#cb85-11" aria-hidden="true" tabindex="-1"></a><span class="co"># Count up and down regulated</span></span>
<span id="cb85-12"><a href="#cb85-12" aria-hidden="true" tabindex="-1"></a>up <span class="ot">&lt;-</span> <span class="fu">sum</span>(resultsLRT_df<span class="sc">$</span>log2FoldChange <span class="sc">&gt;</span> <span class="dv">0</span>, <span class="at">na.rm =</span> <span class="cn">TRUE</span>)</span>
<span id="cb85-13"><a href="#cb85-13" aria-hidden="true" tabindex="-1"></a>dowm <span class="ot">&lt;-</span> <span class="fu">sum</span>(resultsLRT_df<span class="sc">$</span>log2FoldChange <span class="sc">&lt;</span> <span class="dv">0</span>, <span class="at">na.rm =</span> <span class="cn">TRUE</span>) </span>
<span id="cb85-14"><a href="#cb85-14" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb85-15"><a href="#cb85-15" aria-hidden="true" tabindex="-1"></a><span class="co"># Filtrar resultados significativos</span></span>
<span id="cb85-16"><a href="#cb85-16" aria-hidden="true" tabindex="-1"></a>significant_isomiRs <span class="ot">&lt;-</span> resultsLRT[resultsLRT<span class="sc">$</span>padj <span class="sc">&lt;</span> <span class="fl">0.05</span>, ]  <span class="co"># Estricto</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<div class="cell">
<div class="sourceCode cell-code" id="cb86"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb86-1"><a href="#cb86-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Subset the LRT results to return genes with padj &lt; 0.05</span></span>
<span id="cb86-2"><a href="#cb86-2" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb86-3"><a href="#cb86-3" aria-hidden="true" tabindex="-1"></a>table_mirnas <span class="ot">&lt;-</span> resultsLRT_df <span class="sc">%&gt;%</span></span>
<span id="cb86-4"><a href="#cb86-4" aria-hidden="true" tabindex="-1"></a>  <span class="fu">data.frame</span>() <span class="sc">%&gt;%</span></span>
<span id="cb86-5"><a href="#cb86-5" aria-hidden="true" tabindex="-1"></a>  <span class="fu">rownames_to_column</span>(<span class="at">var=</span><span class="st">"row"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb86-6"><a href="#cb86-6" aria-hidden="true" tabindex="-1"></a>  <span class="fu">as_tibble</span>() <span class="sc">%&gt;%</span> </span>
<span id="cb86-7"><a href="#cb86-7" aria-hidden="true" tabindex="-1"></a>  <span class="fu">filter</span>(padj <span class="sc">&lt;</span> <span class="fl">0.05</span>) <span class="sc">%&gt;%</span></span>
<span id="cb86-8"><a href="#cb86-8" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(</span>
<span id="cb86-9"><a href="#cb86-9" aria-hidden="true" tabindex="-1"></a>    <span class="at">baseMean =</span> <span class="fu">round</span>(baseMean, <span class="dv">3</span>),</span>
<span id="cb86-10"><a href="#cb86-10" aria-hidden="true" tabindex="-1"></a>    <span class="at">log2FoldChange =</span> <span class="fu">round</span>(log2FoldChange, <span class="dv">3</span>),</span>
<span id="cb86-11"><a href="#cb86-11" aria-hidden="true" tabindex="-1"></a>    <span class="at">lfcSE =</span> <span class="fu">round</span>(lfcSE, <span class="dv">3</span>),</span>
<span id="cb86-12"><a href="#cb86-12" aria-hidden="true" tabindex="-1"></a>    <span class="at">stat =</span> <span class="fu">round</span>(stat, <span class="dv">3</span>),</span>
<span id="cb86-13"><a href="#cb86-13" aria-hidden="true" tabindex="-1"></a>    <span class="at">pvalue =</span> <span class="fu">case_when</span>(</span>
<span id="cb86-14"><a href="#cb86-14" aria-hidden="true" tabindex="-1"></a>      pvalue <span class="sc">&lt;</span> <span class="fl">0.000215</span> <span class="sc">~</span> <span class="st">"p&lt; 0.0002"</span>,</span>
<span id="cb86-15"><a href="#cb86-15" aria-hidden="true" tabindex="-1"></a>      <span class="cn">TRUE</span> <span class="sc">~</span> <span class="fu">as.character</span>(pvalue)),</span>
<span id="cb86-16"><a href="#cb86-16" aria-hidden="true" tabindex="-1"></a>    <span class="at">padj =</span> <span class="fu">round</span>(padj,<span class="dv">3</span>)</span>
<span id="cb86-17"><a href="#cb86-17" aria-hidden="true" tabindex="-1"></a>  ) <span class="sc">%&gt;%</span> </span>
<span id="cb86-18"><a href="#cb86-18" aria-hidden="true" tabindex="-1"></a>  <span class="fu">select</span>(<span class="st">"row"</span>,<span class="st">"log2FoldChange"</span>,<span class="st">"pvalue"</span>, <span class="st">"padj"</span>)</span>
<span id="cb86-19"><a href="#cb86-19" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb86-20"><a href="#cb86-20" aria-hidden="true" tabindex="-1"></a>knitr<span class="sc">::</span><span class="fu">kable</span>(table_mirnas,</span>
<span id="cb86-21"><a href="#cb86-21" aria-hidden="true" tabindex="-1"></a>             <span class="at">col.names =</span> <span class="fu">c</span>(<span class="st">""</span>, </span>
<span id="cb86-22"><a href="#cb86-22" aria-hidden="true" tabindex="-1"></a>                              <span class="st">"log2FoldChange"</span>,</span>
<span id="cb86-23"><a href="#cb86-23" aria-hidden="true" tabindex="-1"></a>                           <span class="st">"pvalue"</span>, <span class="st">"FDR"</span>),)<span class="sc">%&gt;%</span></span>
<span id="cb86-24"><a href="#cb86-24" aria-hidden="true" tabindex="-1"></a>  <span class="fu">kable_styling</span>(</span>
<span id="cb86-25"><a href="#cb86-25" aria-hidden="true" tabindex="-1"></a>    <span class="at">latex_options =</span> <span class="fu">c</span>(<span class="st">"hold_position"</span>, <span class="st">"scale_down"</span>), <span class="co"># Escala automáticamente</span></span>
<span id="cb86-26"><a href="#cb86-26" aria-hidden="true" tabindex="-1"></a>    <span class="at">font_size =</span> <span class="dv">11</span>   <span class="co"># Ajustar tamaño de fuente</span></span>
<span id="cb86-27"><a href="#cb86-27" aria-hidden="true" tabindex="-1"></a>  ) <span class="sc">%&gt;%</span></span>
<span id="cb86-28"><a href="#cb86-28" aria-hidden="true" tabindex="-1"></a>  <span class="fu">row_spec</span>(<span class="dv">0</span>, <span class="at">bold =</span> <span class="cn">TRUE</span>) <span class="co"># Encabezado en negrita</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div id="tbl-mirnas_selected" class="cell quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-tbl figure">
<figcaption class="quarto-float-caption-top quarto-float-caption quarto-float-tbl" id="tbl-mirnas_selected-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Table&nbsp;4: Differentially expressed miRNAs in subcutaneous white adipose tissue (scWAT). <em>FDR</em> : False Discovery Rate
</figcaption>
<div aria-describedby="tbl-mirnas_selected-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<div class="cell-output-display">
<table class="table do-not-create-environment cell caption-top table-sm table-striped small" data-quarto-postprocess="true">
<thead>
<tr class="header">
<th data-quarto-table-cell-role="th" style="text-align: left; font-weight: bold;"></th>
<th data-quarto-table-cell-role="th" style="text-align: right; font-weight: bold;">log2FoldChange</th>
<th data-quarto-table-cell-role="th" style="text-align: left; font-weight: bold;">pvalue</th>
<th data-quarto-table-cell-role="th" style="text-align: right; font-weight: bold;">FDR</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">hsa-miR-144-3p</td>
<td style="text-align: right;">-3.319</td>
<td style="text-align: left;">p&lt; 0.0002</td>
<td style="text-align: right;">0.04</td>
</tr>
<tr class="even">
<td style="text-align: left;">hsa-miR-372-3p</td>
<td style="text-align: right;">1.085</td>
<td style="text-align: left;">p&lt; 0.0002</td>
<td style="text-align: right;">0.04</td>
</tr>
</tbody>
</table>


</div>
</div>
</figure>
</div>
</div>
<div class="cell">
<div class="sourceCode cell-code" id="cb87"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb87-1"><a href="#cb87-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Visualizar resultados</span></span>
<span id="cb87-2"><a href="#cb87-2" aria-hidden="true" tabindex="-1"></a><span class="co">#| label: fig-volcan</span></span>
<span id="cb87-3"><a href="#cb87-3" aria-hidden="true" tabindex="-1"></a><span class="co">#| fig-cap: "Plot showing differentially expressed miRNAs (red) in subcutaneous white adipose tissue (scWAT) according to the four groups of steatosis"</span></span>
<span id="cb87-4"><a href="#cb87-4" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb87-5"><a href="#cb87-5" aria-hidden="true" tabindex="-1"></a>resultsLRT <span class="ot">&lt;-</span> <span class="fu">results</span>(dds_1, <span class="at">tidy =</span> <span class="cn">TRUE</span>)</span>
<span id="cb87-6"><a href="#cb87-6" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb87-7"><a href="#cb87-7" aria-hidden="true" tabindex="-1"></a>data <span class="ot">&lt;-</span> <span class="fu">data.frame</span>(</span>
<span id="cb87-8"><a href="#cb87-8" aria-hidden="true" tabindex="-1"></a>  <span class="at">miRNA=</span> resultsLRT<span class="sc">$</span>row,</span>
<span id="cb87-9"><a href="#cb87-9" aria-hidden="true" tabindex="-1"></a>  <span class="at">log2FoldChange =</span> resultsLRT<span class="sc">$</span>log2FoldChange,</span>
<span id="cb87-10"><a href="#cb87-10" aria-hidden="true" tabindex="-1"></a>  <span class="at">pvalue =</span> resultsLRT<span class="sc">$</span>pvalue,</span>
<span id="cb87-11"><a href="#cb87-11" aria-hidden="true" tabindex="-1"></a>  <span class="at">padj =</span> resultsLRT<span class="sc">$</span>padj</span>
<span id="cb87-12"><a href="#cb87-12" aria-hidden="true" tabindex="-1"></a>)</span>
<span id="cb87-13"><a href="#cb87-13" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb87-14"><a href="#cb87-14" aria-hidden="true" tabindex="-1"></a><span class="co"># Crear la visualización</span></span>
<span id="cb87-15"><a href="#cb87-15" aria-hidden="true" tabindex="-1"></a><span class="fu">ggplot</span>(data, <span class="fu">aes</span>(<span class="at">x =</span> log2FoldChange, <span class="at">y =</span> <span class="sc">-</span><span class="fu">log10</span>(pvalue))) <span class="sc">+</span></span>
<span id="cb87-16"><a href="#cb87-16" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_point</span>(<span class="fu">aes</span>(<span class="at">color =</span> padj <span class="sc">&lt;</span> <span class="fl">0.05</span>), <span class="at">size =</span> <span class="dv">2</span>) <span class="sc">+</span>  <span class="co"># Puntos</span></span>
<span id="cb87-17"><a href="#cb87-17" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_text_repel</span>(</span>
<span id="cb87-18"><a href="#cb87-18" aria-hidden="true" tabindex="-1"></a>    <span class="at">data =</span> <span class="fu">subset</span>(data, padj <span class="sc">&lt;</span> <span class="fl">0.05</span>),               <span class="co"># Filtrar puntos significativos</span></span>
<span id="cb87-19"><a href="#cb87-19" aria-hidden="true" tabindex="-1"></a>    <span class="fu">aes</span>(<span class="at">label =</span> miRNA),                             <span class="co"># Etiquetas con nombres de miRNA</span></span>
<span id="cb87-20"><a href="#cb87-20" aria-hidden="true" tabindex="-1"></a>    <span class="at">box.padding =</span> <span class="fl">0.35</span>,                             <span class="co"># Espaciado alrededor de las etiquetas</span></span>
<span id="cb87-21"><a href="#cb87-21" aria-hidden="true" tabindex="-1"></a>    <span class="at">point.padding =</span> <span class="fl">0.3</span>,                            <span class="co"># Espaciado respecto al punto</span></span>
<span id="cb87-22"><a href="#cb87-22" aria-hidden="true" tabindex="-1"></a>    <span class="at">segment.color =</span> <span class="st">"gray"</span>,                         <span class="co"># Color de las líneas</span></span>
<span id="cb87-23"><a href="#cb87-23" aria-hidden="true" tabindex="-1"></a>    <span class="at">max.overlaps =</span> <span class="cn">Inf</span>                              <span class="co"># Mostrar todas las etiquetas</span></span>
<span id="cb87-24"><a href="#cb87-24" aria-hidden="true" tabindex="-1"></a>  ) <span class="sc">+</span></span>
<span id="cb87-25"><a href="#cb87-25" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_color_manual</span>(<span class="at">values =</span> <span class="fu">c</span>(<span class="st">"TRUE"</span> <span class="ot">=</span> <span class="st">"darkred"</span>, <span class="st">"FALSE"</span> <span class="ot">=</span> <span class="st">"darkgreen"</span>)) <span class="sc">+</span></span>
<span id="cb87-26"><a href="#cb87-26" aria-hidden="true" tabindex="-1"></a>  <span class="fu">labs</span>(</span>
<span id="cb87-27"><a href="#cb87-27" aria-hidden="true" tabindex="-1"></a>    <span class="at">x =</span> <span class="st">"Log2 Fold Change"</span>,</span>
<span id="cb87-28"><a href="#cb87-28" aria-hidden="true" tabindex="-1"></a>    <span class="at">y =</span> <span class="st">"-Log10 FDR"</span>,</span>
<span id="cb87-29"><a href="#cb87-29" aria-hidden="true" tabindex="-1"></a>    <span class="at">color =</span> <span class="st">"Significant"</span></span>
<span id="cb87-30"><a href="#cb87-30" aria-hidden="true" tabindex="-1"></a>  ) <span class="sc">+</span></span>
<span id="cb87-31"><a href="#cb87-31" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme_minimal</span>(<span class="at">base_size =</span> <span class="dv">14</span>) <span class="sc">+</span></span>
<span id="cb87-32"><a href="#cb87-32" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme</span>(</span>
<span id="cb87-33"><a href="#cb87-33" aria-hidden="true" tabindex="-1"></a>    <span class="at">axis.text.x =</span> <span class="fu">element_text</span>(<span class="at">angle =</span> <span class="dv">90</span> , <span class="at">hjust =</span> <span class="dv">1</span>, <span class="at">size =</span> <span class="dv">5</span>),</span>
<span id="cb87-34"><a href="#cb87-34" aria-hidden="true" tabindex="-1"></a>    <span class="at">axis.text.y =</span> <span class="fu">element_text</span>(<span class="at">angle =</span> <span class="dv">0</span> , <span class="at">hjust =</span> <span class="dv">1</span>, <span class="at">size =</span> <span class="dv">7</span>),</span>
<span id="cb87-35"><a href="#cb87-35" aria-hidden="true" tabindex="-1"></a>    <span class="at">axis.title =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">8</span>),</span>
<span id="cb87-36"><a href="#cb87-36" aria-hidden="true" tabindex="-1"></a>    <span class="at">plot.title =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">12</span>, <span class="at">face =</span> <span class="st">"bold"</span>),</span>
<span id="cb87-37"><a href="#cb87-37" aria-hidden="true" tabindex="-1"></a>    <span class="at">legend.title =</span> <span class="fu">element_blank</span>(),</span>
<span id="cb87-38"><a href="#cb87-38" aria-hidden="true" tabindex="-1"></a>    <span class="at">legend.text =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">6</span>),</span>
<span id="cb87-39"><a href="#cb87-39" aria-hidden="true" tabindex="-1"></a>    <span class="at">legend.position =</span> <span class="st">"none"</span>,</span>
<span id="cb87-40"><a href="#cb87-40" aria-hidden="true" tabindex="-1"></a>    <span class="at">legend.margin =</span> <span class="fu">margin</span>(<span class="at">t =</span> <span class="sc">-</span><span class="dv">20</span>),  <span class="co"># Ajusta el margen superior de la leyenda</span></span>
<span id="cb87-41"><a href="#cb87-41" aria-hidden="true" tabindex="-1"></a><span class="co">#    plot.margin = margin(t = 10, r = 10, b = 30, l = 10)</span></span>
<span id="cb87-42"><a href="#cb87-42" aria-hidden="true" tabindex="-1"></a>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<div id="fig-volcan" class="quarto-float quarto-figure quarto-figure-center anchored" data-fig-pos="H">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-volcan-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="Results_files/figure-html/fig-volcan-1.png" class="img-fluid figure-img" data-fig-pos="H" width="768">
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-volcan-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;10: Plot showing differentially expressed miRNAs (red) in subcutaneous white adipose tissue (scWAT) according to the four groups of steatosis
</figcaption>
</figure>
</div>
</div>
</div>
<div class="cell">
<div class="sourceCode cell-code" id="cb88"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb88-1"><a href="#cb88-1" aria-hidden="true" tabindex="-1"></a><span class="do">##| fig-cap: "Estimation of the dispersion "</span></span>
<span id="cb88-2"><a href="#cb88-2" aria-hidden="true" tabindex="-1"></a><span class="do">##| label: fig-disp</span></span>
<span id="cb88-3"><a href="#cb88-3" aria-hidden="true" tabindex="-1"></a><span class="co">#dds_est &lt;- estimateDispersions(dds_1)</span></span>
<span id="cb88-4"><a href="#cb88-4" aria-hidden="true" tabindex="-1"></a><span class="co">#plotDispEsts(dds_est)</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<p>Following their identification, we further analyzed the expression patterns of these two differentially expressed miRNAs by representing their normalized counts in boxplots across the four steatosis groups <a href="#fig-mirnasselected" class="quarto-xref">Figure&nbsp;11</a>. This approach allowed us to observe how the expression of these significant miRNAs varies across different degrees of steatosis. The boxplots highlight the consistency of expression changes across groups, reinforcing the relevance of these miRNAs in the context of steatosis severity. The miRNA <em>hsa-miR-144-3p</em> exhibits a decrease in expression as the level of steatosis increases, whereas <em>hsa-miR-372-3p</em> shows a trend of increasing expression across the different levels of steatosis.</p>
<div class="cell">
<div class="sourceCode cell-code" id="cb89"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb89-1"><a href="#cb89-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Visualizar resultados</span></span>
<span id="cb89-2"><a href="#cb89-2" aria-hidden="true" tabindex="-1"></a><span class="co">#| label: fig-mirnasselected</span></span>
<span id="cb89-3"><a href="#cb89-3" aria-hidden="true" tabindex="-1"></a><span class="co">#| fig-cap: "Boxplots of the differentially expressed miRNAs in subcutaneous white adipose tissue (scWAT) according to the four groups of steatosis. Each box represents the interquartile range (IQR) of the normalized counts, with the line inside the box indicating the median. The whiskers extend to show the range of the data, excluding outliers, which are displayed as individual points."</span></span>
<span id="cb89-4"><a href="#cb89-4" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb89-5"><a href="#cb89-5" aria-hidden="true" tabindex="-1"></a><span class="co"># Normalize the counts for scWAT data and transpose the matrix7</span></span>
<span id="cb89-6"><a href="#cb89-6" aria-hidden="true" tabindex="-1"></a>dds_1 <span class="ot">&lt;-</span> <span class="fu">DESeq</span>(dds, <span class="at">test =</span> <span class="st">"LRT"</span>, <span class="at">reduced =</span> reduced_design)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>using supplied model matrix</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>estimating size factors</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>estimating dispersions</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>gene-wise dispersion estimates</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>mean-dispersion relationship</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>final dispersion estimates</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>fitting model and testing</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>-- replacing outliers and refitting for 4 genes
-- DESeq argument 'minReplicatesForReplace' = 7 
-- original counts are preserved in counts(dds)</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>estimating dispersions</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>fitting model and testing</code></pre>
</div>
<div class="sourceCode cell-code" id="cb100"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb100-1"><a href="#cb100-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Normalize the metadata for scWAT to match the normalized counts</span></span>
<span id="cb100-2"><a href="#cb100-2" aria-hidden="true" tabindex="-1"></a>counts_norm_sc <span class="ot">&lt;-</span> <span class="fu">t</span>(<span class="fu">counts</span>(dds_1, <span class="at">norm=</span>T))</span>
<span id="cb100-3"><a href="#cb100-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb100-4"><a href="#cb100-4" aria-hidden="true" tabindex="-1"></a><span class="co"># Subset the metadata for scWAT to match the normalized counts</span></span>
<span id="cb100-5"><a href="#cb100-5" aria-hidden="true" tabindex="-1"></a>meta_data_sc <span class="ot">&lt;-</span> mdata_steatosis[<span class="fu">rownames</span>(counts_norm_sc), <span class="fu">c</span>(<span class="st">"sexo"</span>, <span class="st">"Esteatosis"</span>)]</span>
<span id="cb100-6"><a href="#cb100-6" aria-hidden="true" tabindex="-1"></a>meta_data_sc <span class="ot">&lt;-</span> meta_data_sc <span class="sc">%&gt;%</span></span>
<span id="cb100-7"><a href="#cb100-7" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(</span>
<span id="cb100-8"><a href="#cb100-8" aria-hidden="true" tabindex="-1"></a>    <span class="at">Esteatosis =</span> <span class="fu">case_when</span>(</span>
<span id="cb100-9"><a href="#cb100-9" aria-hidden="true" tabindex="-1"></a>      Esteatosis <span class="sc">==</span> <span class="dv">0</span> <span class="sc">~</span> <span class="st">"&lt;5%"</span>,</span>
<span id="cb100-10"><a href="#cb100-10" aria-hidden="true" tabindex="-1"></a>      Esteatosis <span class="sc">==</span> <span class="dv">1</span> <span class="sc">~</span> <span class="st">"5–33%"</span>,</span>
<span id="cb100-11"><a href="#cb100-11" aria-hidden="true" tabindex="-1"></a>      Esteatosis <span class="sc">==</span> <span class="dv">2</span> <span class="sc">~</span> <span class="st">"33–66%"</span>,</span>
<span id="cb100-12"><a href="#cb100-12" aria-hidden="true" tabindex="-1"></a>      Esteatosis <span class="sc">==</span> <span class="dv">3</span> <span class="sc">~</span> <span class="st">"&gt;66%"</span>,</span>
<span id="cb100-13"><a href="#cb100-13" aria-hidden="true" tabindex="-1"></a>      <span class="cn">TRUE</span> <span class="sc">~</span> <span class="fu">as.character</span>(Esteatosis) <span class="co"># Para manejar otros casos</span></span>
<span id="cb100-14"><a href="#cb100-14" aria-hidden="true" tabindex="-1"></a>    )</span>
<span id="cb100-15"><a href="#cb100-15" aria-hidden="true" tabindex="-1"></a>  )</span>
<span id="cb100-16"><a href="#cb100-16" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb100-17"><a href="#cb100-17" aria-hidden="true" tabindex="-1"></a>meta_data_sc <span class="ot">&lt;-</span> meta_data_sc <span class="sc">%&gt;%</span></span>
<span id="cb100-18"><a href="#cb100-18" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(</span>
<span id="cb100-19"><a href="#cb100-19" aria-hidden="true" tabindex="-1"></a>    <span class="at">Esteatosis =</span> <span class="fu">factor</span>(Esteatosis, <span class="at">levels =</span> <span class="fu">c</span>(<span class="st">"&lt;5%"</span>, <span class="st">"5–33%"</span>, <span class="st">"33–66%"</span>, <span class="st">"&gt;66%"</span>))</span>
<span id="cb100-20"><a href="#cb100-20" aria-hidden="true" tabindex="-1"></a>  )</span>
<span id="cb100-21"><a href="#cb100-21" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb100-22"><a href="#cb100-22" aria-hidden="true" tabindex="-1"></a>    <span class="co"># Combining normaized data with metadata</span></span>
<span id="cb100-23"><a href="#cb100-23" aria-hidden="true" tabindex="-1"></a>sc <span class="ot">&lt;-</span> <span class="fu">as.data.frame</span>(counts_norm_sc) <span class="sc">|&gt;</span></span>
<span id="cb100-24"><a href="#cb100-24" aria-hidden="true" tabindex="-1"></a>  <span class="co"># Select columns corresponding to the miRNAs identified in scWAT</span></span>
<span id="cb100-25"><a href="#cb100-25" aria-hidden="true" tabindex="-1"></a>  <span class="fu">select</span>(<span class="fu">rownames</span>(r_sc)) <span class="sc">|&gt;</span></span>
<span id="cb100-26"><a href="#cb100-26" aria-hidden="true" tabindex="-1"></a>  <span class="co"># Bind the selected miRNAs with the corresponding metadata</span></span>
<span id="cb100-27"><a href="#cb100-27" aria-hidden="true" tabindex="-1"></a>  <span class="fu">bind_cols</span>(meta_data_sc) <span class="sc">|&gt;</span></span>
<span id="cb100-28"><a href="#cb100-28" aria-hidden="true" tabindex="-1"></a>  <span class="co"># Reshape the data to long format for ggplot, with miRNA and Normalized Counts</span></span>
<span id="cb100-29"><a href="#cb100-29" aria-hidden="true" tabindex="-1"></a>  <span class="fu">pivot_longer</span>(<span class="dv">1</span><span class="sc">:</span><span class="fu">nrow</span>(r_sc), <span class="at">names_to =</span> <span class="st">"miRNA"</span>, <span class="at">values_to =</span> <span class="st">"Normalized Counts"</span>)<span class="sc">|&gt;</span></span>
<span id="cb100-30"><a href="#cb100-30" aria-hidden="true" tabindex="-1"></a>  <span class="co"># Add 1 to avoid infinity in the log scale</span></span>
<span id="cb100-31"><a href="#cb100-31" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="st">`</span><span class="at">Normalized Counts</span><span class="st">`</span><span class="ot">=</span><span class="st">`</span><span class="at">Normalized Counts</span><span class="st">`</span> <span class="sc">+</span><span class="dv">1</span>)</span>
<span id="cb100-32"><a href="#cb100-32" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb100-33"><a href="#cb100-33" aria-hidden="true" tabindex="-1"></a>gg_miRNA <span class="ot">&lt;-</span> <span class="cf">function</span>() {</span>
<span id="cb100-34"><a href="#cb100-34" aria-hidden="true" tabindex="-1"></a>  <span class="fu">list</span>(</span>
<span id="cb100-35"><a href="#cb100-35" aria-hidden="true" tabindex="-1"></a>    <span class="fu">scale_y_continuous</span>(<span class="at">trans =</span> <span class="st">"log2"</span>),</span>
<span id="cb100-36"><a href="#cb100-36" aria-hidden="true" tabindex="-1"></a>    <span class="fu">labs</span>(<span class="at">x=</span><span class="st">"</span><span class="sc">\n</span><span class="st">Steatosis"</span>),</span>
<span id="cb100-37"><a href="#cb100-37" aria-hidden="true" tabindex="-1"></a>    <span class="fu">geom_boxplot</span>(<span class="at">width=</span> <span class="fl">0.5</span>, <span class="at">outliers =</span> <span class="cn">FALSE</span>),</span>
<span id="cb100-38"><a href="#cb100-38" aria-hidden="true" tabindex="-1"></a>    <span class="fu">geom_jitter</span>(<span class="at">width =</span> <span class="fl">0.25</span>, <span class="at">size=</span> <span class="dv">2</span>, <span class="at">alpha =</span> <span class="fl">0.3</span>),</span>
<span id="cb100-39"><a href="#cb100-39" aria-hidden="true" tabindex="-1"></a>    <span class="fu">geom_smooth</span>(<span class="fu">aes</span>(<span class="at">group =</span> <span class="dv">1</span>), <span class="at">method =</span> <span class="st">"lm"</span>, <span class="at">color =</span> <span class="st">"#8c8c8c"</span>, <span class="at">se =</span> <span class="cn">FALSE</span>), </span>
<span id="cb100-40"><a href="#cb100-40" aria-hidden="true" tabindex="-1"></a>    <span class="fu">stat_compare_means</span>(<span class="fu">aes</span>(<span class="at">label=</span> <span class="fu">paste0</span>(<span class="st">"p= "</span>, ..p.format..)),</span>
<span id="cb100-41"><a href="#cb100-41" aria-hidden="true" tabindex="-1"></a>                       <span class="at">method =</span> <span class="st">"kruskal.test"</span>,</span>
<span id="cb100-42"><a href="#cb100-42" aria-hidden="true" tabindex="-1"></a>                       <span class="at">label.x.npc =</span> <span class="fl">0.3</span>,</span>
<span id="cb100-43"><a href="#cb100-43" aria-hidden="true" tabindex="-1"></a>                       <span class="at">label.y.npc =</span> <span class="fl">0.8</span>,</span>
<span id="cb100-44"><a href="#cb100-44" aria-hidden="true" tabindex="-1"></a>                       <span class="at">size =</span> <span class="dv">4</span>),</span>
<span id="cb100-45"><a href="#cb100-45" aria-hidden="true" tabindex="-1"></a>    <span class="fu">theme_minimal</span>(),</span>
<span id="cb100-46"><a href="#cb100-46" aria-hidden="true" tabindex="-1"></a>    <span class="fu">scale_fill_viridis_d</span>(),  <span class="co"># Para los rellenos</span></span>
<span id="cb100-47"><a href="#cb100-47" aria-hidden="true" tabindex="-1"></a>    <span class="fu">facet_wrap</span>(<span class="at">facets =</span> <span class="sc">~</span>miRNA, <span class="at">scales =</span> <span class="st">"free_y"</span>, <span class="at">ncol=</span><span class="dv">4</span>),</span>
<span id="cb100-48"><a href="#cb100-48" aria-hidden="true" tabindex="-1"></a>    <span class="fu">theme</span>(<span class="at">legend.position =</span> <span class="st">"none"</span>,</span>
<span id="cb100-49"><a href="#cb100-49" aria-hidden="true" tabindex="-1"></a>          <span class="at">strip.text =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">13</span>),</span>
<span id="cb100-50"><a href="#cb100-50" aria-hidden="true" tabindex="-1"></a>          <span class="at">axis.title =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">13</span>),</span>
<span id="cb100-51"><a href="#cb100-51" aria-hidden="true" tabindex="-1"></a>          <span class="at">axis.text.y =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">11</span>),</span>
<span id="cb100-52"><a href="#cb100-52" aria-hidden="true" tabindex="-1"></a>          <span class="at">axis.text.x =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">12</span>),</span>
<span id="cb100-53"><a href="#cb100-53" aria-hidden="true" tabindex="-1"></a>          <span class="at">strip.text.x =</span> <span class="fu">element_text</span>(<span class="at">face =</span> <span class="st">"italic"</span>)</span>
<span id="cb100-54"><a href="#cb100-54" aria-hidden="true" tabindex="-1"></a>          )</span>
<span id="cb100-55"><a href="#cb100-55" aria-hidden="true" tabindex="-1"></a>  )</span>
<span id="cb100-56"><a href="#cb100-56" aria-hidden="true" tabindex="-1"></a>}</span>
<span id="cb100-57"><a href="#cb100-57" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb100-58"><a href="#cb100-58" aria-hidden="true" tabindex="-1"></a><span class="fu">ggplot</span>(sc, <span class="fu">aes</span>(<span class="at">x=</span> Esteatosis, <span class="at">y =</span><span class="st">`</span><span class="at">Normalized Counts</span><span class="st">`</span>, <span class="at">fill=</span> Esteatosis)) <span class="sc">+</span> <span class="fu">gg_miRNA</span>()</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>`geom_smooth()` using formula = 'y ~ x'</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>Warning: The following aesthetics were dropped during statistical transformation: fill.
ℹ This can happen when ggplot fails to infer the correct grouping structure in
  the data.
ℹ Did you forget to specify a `group` aesthetic or to convert a numerical
  variable into a factor?
The following aesthetics were dropped during statistical transformation: fill.
ℹ This can happen when ggplot fails to infer the correct grouping structure in
  the data.
ℹ Did you forget to specify a `group` aesthetic or to convert a numerical
  variable into a factor?</code></pre>
</div>
<div class="cell-output-display">
<div id="fig-mirnasselected" class="quarto-float quarto-figure quarto-figure-center anchored" data-fig-pos="H">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-mirnasselected-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="Results_files/figure-html/fig-mirnasselected-1.png" class="img-fluid figure-img" data-fig-pos="H" width="768">
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-mirnasselected-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;11: Boxplots of the differentially expressed miRNAs in subcutaneous white adipose tissue (scWAT) according to the four groups of steatosis. Each box represents the interquartile range (IQR) of the normalized counts, with the line inside the box indicating the median. The whiskers extend to show the range of the data, excluding outliers, which are displayed as individual points. p: p-value fot the Kruskal–Wallis test for the comparison between groups.
</figcaption>
</figure>
</div>
</div>
</div>
</section>
<section id="target-mrna-selection-and-validation" class="level2">
<h2 class="anchored" data-anchor-id="target-mrna-selection-and-validation">Target mRNA Selection and Validation</h2>
<section id="hsa-mir-372-3p" class="level3">
<h3 class="anchored" data-anchor-id="hsa-mir-372-3p"><em>hsa-miR-372-3p</em></h3>
<p>Potential target genes of the selected miRNAs were identified using the <em>multiMiR</em> package. This analysis included only validated interaction data, ensuring the biological relevance of the identified mRNAs. A subsequent filtering process was applied to select targets based on the type of experimental validation (e.g., luciferase assays, Western blot, or qRT-PCR) and functional support. As shown in <a href="#tbl-hsa-miR-372-3p" class="quarto-xref">Table&nbsp;5</a>, 24 target genes were identified after filtering.</p>
<div class="cell">
<div class="sourceCode cell-code" id="cb103"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb103-1"><a href="#cb103-1" aria-hidden="true" tabindex="-1"></a><span class="co">#val_hsa_miR_372_3p &lt;- get_multimir(org = "hsa", mirna = 'hsa-miR-372-3p', table = "validated", add.link = TRUE, use.tibble = TRUE, summary = TRUE)</span></span>
<span id="cb103-2"><a href="#cb103-2" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb103-3"><a href="#cb103-3" aria-hidden="true" tabindex="-1"></a><span class="co"># Check which types of associations were returned</span></span>
<span id="cb103-4"><a href="#cb103-4" aria-hidden="true" tabindex="-1"></a><span class="co">#table(val_hsa_miR_372_3p@data$type)</span></span>
<span id="cb103-5"><a href="#cb103-5" aria-hidden="true" tabindex="-1"></a><span class="co">#dim(val_hsa_miR_372_3p@data)</span></span>
<span id="cb103-6"><a href="#cb103-6" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb103-7"><a href="#cb103-7" aria-hidden="true" tabindex="-1"></a><span class="co"># Detailed information of the validated miRNA-target interaction</span></span>
<span id="cb103-8"><a href="#cb103-8" aria-hidden="true" tabindex="-1"></a><span class="co">#head(val_hsa_miR_372_3p@data)</span></span>
<span id="cb103-9"><a href="#cb103-9" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb103-10"><a href="#cb103-10" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb103-11"><a href="#cb103-11" aria-hidden="true" tabindex="-1"></a><span class="co"># Extraer los datos como un data.frame o tibble</span></span>
<span id="cb103-12"><a href="#cb103-12" aria-hidden="true" tabindex="-1"></a><span class="co">#val_hsa_miR_372_3p_df &lt;- as_tibble(val_hsa_miR_372_3p@data)</span></span>
<span id="cb103-13"><a href="#cb103-13" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb103-14"><a href="#cb103-14" aria-hidden="true" tabindex="-1"></a><span class="co"># Guardar el data.frame en un archivo .RData</span></span>
<span id="cb103-15"><a href="#cb103-15" aria-hidden="true" tabindex="-1"></a><span class="co">#save(val_hsa_miR_372_3p_df, file = "data/val_hsa_miR_372_3p.RData")</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<div class="cell">
<div class="sourceCode cell-code" id="cb104"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb104-1"><a href="#cb104-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Load the target genes for hsa-miR-372-3p</span></span>
<span id="cb104-2"><a href="#cb104-2" aria-hidden="true" tabindex="-1"></a><span class="fu">load</span>(<span class="st">"/home/joshoacr13/Documentos/TFM/mirna_analysis/miRNA/TFM/data/val_hsa_miR_372_3p.RData"</span>)</span>
<span id="cb104-3"><a href="#cb104-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb104-4"><a href="#cb104-4" aria-hidden="true" tabindex="-1"></a><span class="co"># Filtrar los datos con las condiciones especificadas</span></span>
<span id="cb104-5"><a href="#cb104-5" aria-hidden="true" tabindex="-1"></a>filter_val_hsa_miR_372_3p <span class="ot">&lt;-</span> val_hsa_miR_372_3p_df[</span>
<span id="cb104-6"><a href="#cb104-6" aria-hidden="true" tabindex="-1"></a>  val_hsa_miR_372_3p_df[[<span class="st">"database"</span>]] <span class="sc">%in%</span> <span class="fu">c</span>(<span class="st">"mirtarbase"</span>, <span class="st">"tarbase"</span>) <span class="sc">&amp;</span> </span>
<span id="cb104-7"><a href="#cb104-7" aria-hidden="true" tabindex="-1"></a>  <span class="fu">grepl</span>(<span class="st">"Luciferase|Western blot|qRT-PCR"</span>, val_hsa_miR_372_3p_df[[<span class="st">"experiment"</span>]], <span class="at">ignore.case =</span> <span class="cn">TRUE</span>) <span class="sc">&amp;</span>  <span class="co"># Filtro por experimentos específicos</span></span>
<span id="cb104-8"><a href="#cb104-8" aria-hidden="true" tabindex="-1"></a>  val_hsa_miR_372_3p_df[[<span class="st">"support_type"</span>]] <span class="sc">%in%</span> <span class="fu">c</span>(<span class="st">"Functional MTI"</span>,<span class="st">"Functional MTI (Weak)"</span>,<span class="st">""</span>) <span class="sc">&amp;</span>  <span class="co"># Filtro por soporte funcional</span></span>
<span id="cb104-9"><a href="#cb104-9" aria-hidden="true" tabindex="-1"></a>  val_hsa_miR_372_3p_df[[<span class="st">"type"</span>]] <span class="sc">==</span> <span class="st">"validated"</span>, <span class="co"># Filtro por tipo validado</span></span>
<span id="cb104-10"><a href="#cb104-10" aria-hidden="true" tabindex="-1"></a>]</span>
<span id="cb104-11"><a href="#cb104-11" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb104-12"><a href="#cb104-12" aria-hidden="true" tabindex="-1"></a><span class="co"># Eliminar algunas columnas</span></span>
<span id="cb104-13"><a href="#cb104-13" aria-hidden="true" tabindex="-1"></a>filter_val_hsa_miR_372_3p_edit <span class="ot">&lt;-</span> <span class="fu">as.data.frame</span>(filter_val_hsa_miR_372_3p) <span class="sc">%&gt;%</span></span>
<span id="cb104-14"><a href="#cb104-14" aria-hidden="true" tabindex="-1"></a>  dplyr<span class="sc">::</span><span class="fu">select</span>(<span class="sc">-</span>database,<span class="sc">-</span>mature_mirna_id,<span class="sc">-</span>pubmed_id,<span class="sc">-</span>mature_mirna_acc,<span class="sc">-</span>DB.link,<span class="sc">-</span>target_entrez,<span class="sc">-</span>experiment,<span class="sc">-</span>support_type) </span>
<span id="cb104-15"><a href="#cb104-15" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb104-16"><a href="#cb104-16" aria-hidden="true" tabindex="-1"></a><span class="co"># Eliminar algunas filas</span></span>
<span id="cb104-17"><a href="#cb104-17" aria-hidden="true" tabindex="-1"></a>filter_val_hsa_miR_372_3p_edit <span class="ot">&lt;-</span> filter_val_hsa_miR_372_3p_edit[<span class="sc">-</span><span class="fu">c</span>(<span class="dv">2</span>, <span class="dv">3</span>, <span class="dv">4</span>, <span class="dv">5</span>, <span class="dv">6</span>, <span class="dv">7</span>, <span class="dv">8</span>, <span class="dv">9</span>, <span class="dv">10</span>, <span class="dv">11</span>, <span class="dv">13</span>, <span class="dv">14</span>, <span class="dv">15</span>, <span class="dv">17</span>, <span class="dv">18</span>, <span class="dv">20</span>, <span class="dv">21</span>, <span class="dv">22</span>, <span class="dv">24</span>, <span class="dv">26</span>, <span class="dv">28</span>, <span class="dv">29</span>, <span class="dv">31</span>, <span class="dv">32</span>, <span class="dv">34</span>, <span class="dv">35</span>, <span class="dv">36</span>, <span class="dv">38</span>, <span class="dv">40</span>, <span class="dv">42</span>, <span class="dv">43</span>, <span class="dv">44</span>, <span class="dv">46</span>, <span class="dv">47</span>, <span class="dv">49</span>, <span class="dv">50</span>, <span class="dv">52</span>, <span class="dv">53</span>, <span class="dv">55</span>, <span class="dv">57</span>, <span class="dv">59</span>, <span class="dv">61</span>, <span class="dv">62</span>, <span class="dv">65</span>, <span class="dv">68</span>, <span class="dv">69</span>, <span class="dv">70</span>, <span class="dv">71</span>, <span class="dv">72</span>, <span class="dv">74</span>, <span class="dv">75</span>), ]</span>
<span id="cb104-18"><a href="#cb104-18" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb104-19"><a href="#cb104-19" aria-hidden="true" tabindex="-1"></a><span class="do">### TABLE</span></span>
<span id="cb104-20"><a href="#cb104-20" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb104-21"><a href="#cb104-21" aria-hidden="true" tabindex="-1"></a>knitr<span class="sc">::</span><span class="fu">kable</span>(filter_val_hsa_miR_372_3p_edit,</span>
<span id="cb104-22"><a href="#cb104-22" aria-hidden="true" tabindex="-1"></a>             <span class="at">col.names =</span> <span class="fu">c</span>(<span class="st">"Gene"</span>, <span class="st">"Ensembl Identifier"</span>, <span class="st">"Type"</span>),</span>
<span id="cb104-23"><a href="#cb104-23" aria-hidden="true" tabindex="-1"></a>             <span class="at">row.names =</span> <span class="cn">FALSE</span>) <span class="sc">%&gt;%</span></span>
<span id="cb104-24"><a href="#cb104-24" aria-hidden="true" tabindex="-1"></a>  <span class="fu">kable_styling</span>(</span>
<span id="cb104-25"><a href="#cb104-25" aria-hidden="true" tabindex="-1"></a>    <span class="at">latex_options =</span> <span class="fu">c</span>(<span class="st">"hold_position"</span>, <span class="st">"scale_down"</span>), <span class="co"># Escala automáticamente</span></span>
<span id="cb104-26"><a href="#cb104-26" aria-hidden="true" tabindex="-1"></a>    <span class="at">font_size =</span> <span class="dv">11</span>   <span class="co"># Ajustar tamaño de fuente</span></span>
<span id="cb104-27"><a href="#cb104-27" aria-hidden="true" tabindex="-1"></a>  ) <span class="sc">%&gt;%</span></span>
<span id="cb104-28"><a href="#cb104-28" aria-hidden="true" tabindex="-1"></a>  <span class="fu">row_spec</span>(<span class="dv">0</span>, <span class="at">bold =</span> <span class="cn">TRUE</span>) <span class="co"># Encabezado en negrita</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div id="tbl-hsa-miR-372-3p" class="cell quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-tbl figure">
<figcaption class="quarto-float-caption-top quarto-float-caption quarto-float-tbl" id="tbl-hsa-miR-372-3p-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Table&nbsp;5: Selected interactions after filtering by database, experiment type (including luciferase assays, Western blot, or qRT-PCR), functional support (Functional MTI) and validated type for <em>hsa-miR-372-3p</em>.
</figcaption>
<div aria-describedby="tbl-hsa-miR-372-3p-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<div class="cell-output-display">
<table class="table do-not-create-environment cell caption-top table-sm table-striped small" data-quarto-postprocess="true">
<thead>
<tr class="header">
<th data-quarto-table-cell-role="th" style="text-align: left; font-weight: bold;">Gene</th>
<th data-quarto-table-cell-role="th" style="text-align: left; font-weight: bold;">Ensembl Identifier</th>
<th data-quarto-table-cell-role="th" style="text-align: left; font-weight: bold;">Type</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">LATS2</td>
<td style="text-align: left;">ENSG00000150457</td>
<td style="text-align: left;">validated</td>
</tr>
<tr class="even">
<td style="text-align: left;">TGFBR2</td>
<td style="text-align: left;">ENSG00000163513</td>
<td style="text-align: left;">validated</td>
</tr>
<tr class="odd">
<td style="text-align: left;">NFIB</td>
<td style="text-align: left;">ENSG00000147862</td>
<td style="text-align: left;">validated</td>
</tr>
<tr class="even">
<td style="text-align: left;">CDKN1A</td>
<td style="text-align: left;">ENSG00000124762</td>
<td style="text-align: left;">validated</td>
</tr>
<tr class="odd">
<td style="text-align: left;">VEGFA</td>
<td style="text-align: left;">ENSG00000112715</td>
<td style="text-align: left;">validated</td>
</tr>
<tr class="even">
<td style="text-align: left;">TNFAIP1</td>
<td style="text-align: left;">ENSG00000109079</td>
<td style="text-align: left;">validated</td>
</tr>
<tr class="odd">
<td style="text-align: left;">TRPS1</td>
<td style="text-align: left;">ENSG00000104447</td>
<td style="text-align: left;">validated</td>
</tr>
<tr class="even">
<td style="text-align: left;">MBNL2</td>
<td style="text-align: left;">ENSG00000139793</td>
<td style="text-align: left;">validated</td>
</tr>
<tr class="odd">
<td style="text-align: left;">RHOC</td>
<td style="text-align: left;">ENSG00000155366</td>
<td style="text-align: left;">validated</td>
</tr>
<tr class="even">
<td style="text-align: left;">NR4A2</td>
<td style="text-align: left;">ENSG00000153234</td>
<td style="text-align: left;">validated</td>
</tr>
<tr class="odd">
<td style="text-align: left;">ERBB4</td>
<td style="text-align: left;">ENSG00000178568</td>
<td style="text-align: left;">validated</td>
</tr>
<tr class="even">
<td style="text-align: left;">CDK2</td>
<td style="text-align: left;">ENSG00000123374</td>
<td style="text-align: left;">validated</td>
</tr>
<tr class="odd">
<td style="text-align: left;">LEFTY1</td>
<td style="text-align: left;">ENSG00000243709</td>
<td style="text-align: left;">validated</td>
</tr>
<tr class="even">
<td style="text-align: left;">BTG1</td>
<td style="text-align: left;">ENSG00000133639</td>
<td style="text-align: left;">validated</td>
</tr>
<tr class="odd">
<td style="text-align: left;">WEE1</td>
<td style="text-align: left;">ENSG00000166483</td>
<td style="text-align: left;">validated</td>
</tr>
<tr class="even">
<td style="text-align: left;">CCNA1</td>
<td style="text-align: left;">ENSG00000133101</td>
<td style="text-align: left;">validated</td>
</tr>
<tr class="odd">
<td style="text-align: left;">DKK1</td>
<td style="text-align: left;">ENSG00000107984</td>
<td style="text-align: left;">validated</td>
</tr>
<tr class="even">
<td style="text-align: left;">PHLPP2</td>
<td style="text-align: left;">ENSG00000040199</td>
<td style="text-align: left;">validated</td>
</tr>
<tr class="odd">
<td style="text-align: left;">ATAD2</td>
<td style="text-align: left;">ENSG00000156802</td>
<td style="text-align: left;">validated</td>
</tr>
<tr class="even">
<td style="text-align: left;">ADAMTS9</td>
<td style="text-align: left;">ENSG00000163638</td>
<td style="text-align: left;">validated</td>
</tr>
<tr class="odd">
<td style="text-align: left;">CADM2</td>
<td style="text-align: left;">ENSG00000175161</td>
<td style="text-align: left;">validated</td>
</tr>
<tr class="even">
<td style="text-align: left;">ZBTB7A</td>
<td style="text-align: left;">ENSG00000178951</td>
<td style="text-align: left;">validated</td>
</tr>
<tr class="odd">
<td style="text-align: left;">TXNIP</td>
<td style="text-align: left;">ENSG00000265972</td>
<td style="text-align: left;">validated</td>
</tr>
<tr class="even">
<td style="text-align: left;">KLF13</td>
<td style="text-align: left;">ENSG00000275746</td>
<td style="text-align: left;">validated</td>
</tr>
</tbody>
</table>


</div>
</div>
</figure>
</div>
</div>
<p>The target genes identified through multiMiR were subjected to functional enrichment analysis using KEGG (Kyoto Encyclopedia of Genes and Genomes) pathways <span class="citation" data-cites="KEGG">[@KEGG]</span>. The KEGG pathway enrichment analysis identified five significantly enriched pathways (p &lt; 0.05) (<a href="#fig-enrichment_hsamiR_372_3p" class="quarto-xref">Figure&nbsp;12</a>). The most prominent pathways include:</p>
<ul>
<li><em>Cellular senescence</em> (KEGG ID: hsa04218): associated with cellular aging and the permanent arrest of the cell cycle in response to genetic damage, oxidative stress, or oncogenic signals.</li>
<li><em>Cell cycle</em> (KEGG ID: hsa04110): involved in the regulation and progression of the cell cycle, including critical checkpoints that ensure the fidelity of cell division.</li>
<li><em>Hepatitis B</em> (KEGG ID: hsa05161): related to infection by the hepatitis B virus and the inflammatory and carcinogenic processes it can trigger in the liver.</li>
<li><em>PI3K-Akt signaling pathway</em> (KEGG ID: hsa04151): a key pathway regulating cellular processes such as proliferation, survival, and metabolism, frequently deregulated in cancer.</li>
<li><em>Pancreatic cancer</em> (KEGG ID: hsa05212): associated with molecular and cellular alterations characteristic of this cancer type, including aberrant signaling and apoptosis resistance.</li>
</ul>
<p>These pathways are primarily linked to essential processes such as cell cycle regulation, intracellular signaling in cancer, and cellular responses to damage and infection.</p>
<div class="cell">
<div class="sourceCode cell-code" id="cb105"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb105-1"><a href="#cb105-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Visualizar resultados</span></span>
<span id="cb105-2"><a href="#cb105-2" aria-hidden="true" tabindex="-1"></a><span class="co">#| label: fig-enrichment_hsamiR_372_3p</span></span>
<span id="cb105-3"><a href="#cb105-3" aria-hidden="true" tabindex="-1"></a><span class="co">#| fig-cap: "Top 5 Validated Enriched Biological Processes of *hsa-miR-372-3p* "</span></span>
<span id="cb105-4"><a href="#cb105-4" aria-hidden="true" tabindex="-1"></a><span class="co"># t_372 &lt;- enrich_result_hsa_miR_372_3p@result</span></span>
<span id="cb105-5"><a href="#cb105-5" aria-hidden="true" tabindex="-1"></a><span class="fu">load</span>(<span class="st">"data/KEGG_hsa_miR_372_3p.RData"</span>)</span>
<span id="cb105-6"><a href="#cb105-6" aria-hidden="true" tabindex="-1"></a><span class="fu">barplot</span>(enrich_result_hsa_miR_372_3p, <span class="at">showCategory =</span> <span class="dv">5</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<div id="fig-enrichment_hsamiR_372_3p" class="quarto-float quarto-figure quarto-figure-center anchored" data-fig-pos="H">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-enrichment_hsamiR_372_3p-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="Results_files/figure-html/fig-enrichment_hsamiR_372_3p-1.png" class="img-fluid figure-img" data-fig-pos="H" width="768">
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-enrichment_hsamiR_372_3p-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;12: Top 5 Significant KEGG Pathways of <em>hsa-miR-372-3p</em>
</figcaption>
</figure>
</div>
</div>
</div>
</section>
<section id="hsa-mir-144-3p" class="level3">
<h3 class="anchored" data-anchor-id="hsa-mir-144-3p"><em>hsa-miR-144-3p</em></h3>
<p><a href="#tbl-hsa-miR-144-3p" class="quarto-xref">Table&nbsp;6</a> shows 22 target genes were identified for <em>hsa-miR-372-3p</em>, filtered by validated interactions and experimental evidence.</p>
<div class="cell">
<div id="tbl-hsa-miR-144-3p" class="cell quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-tbl figure">
<figcaption class="quarto-float-caption-top quarto-float-caption quarto-float-tbl" id="tbl-hsa-miR-144-3p-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Table&nbsp;6: Selected interactions after filtering by database, experiment type (including luciferase assays, Western blot, or qRT-PCR), functional support (Functional MTI) and validated type for <em>hsa-miR-144-3p</em>.
</figcaption>
<div aria-describedby="tbl-hsa-miR-144-3p-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<div class="cell-output-display">
<table class="table do-not-create-environment cell caption-top table-sm table-striped small" data-quarto-postprocess="true">
<thead>
<tr class="header">
<th data-quarto-table-cell-role="th" style="text-align: left; font-weight: bold;">Gene</th>
<th data-quarto-table-cell-role="th" style="text-align: left; font-weight: bold;">Ensembl Identifier</th>
<th data-quarto-table-cell-role="th" style="text-align: left; font-weight: bold;">Type</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">NOTCH1</td>
<td style="text-align: left;">ENSG00000148400</td>
<td style="text-align: left;">validated</td>
</tr>
<tr class="even">
<td style="text-align: left;">PLAG1</td>
<td style="text-align: left;">ENSG00000181690</td>
<td style="text-align: left;">validated</td>
</tr>
<tr class="odd">
<td style="text-align: left;">ZEB1</td>
<td style="text-align: left;">ENSG00000148516</td>
<td style="text-align: left;">validated</td>
</tr>
<tr class="even">
<td style="text-align: left;">ZEB2</td>
<td style="text-align: left;">ENSG00000169554</td>
<td style="text-align: left;">validated</td>
</tr>
<tr class="odd">
<td style="text-align: left;">IRS1</td>
<td style="text-align: left;">ENSG00000169047</td>
<td style="text-align: left;">validated</td>
</tr>
<tr class="even">
<td style="text-align: left;">MAP3K8</td>
<td style="text-align: left;">ENSG00000107968</td>
<td style="text-align: left;">validated</td>
</tr>
<tr class="odd">
<td style="text-align: left;">EZH2</td>
<td style="text-align: left;">ENSG00000106462</td>
<td style="text-align: left;">validated</td>
</tr>
<tr class="even">
<td style="text-align: left;">APP</td>
<td style="text-align: left;">ENSG00000142192</td>
<td style="text-align: left;">validated</td>
</tr>
<tr class="odd">
<td style="text-align: left;">PTGS2</td>
<td style="text-align: left;">ENSG00000073756</td>
<td style="text-align: left;">validated</td>
</tr>
<tr class="even">
<td style="text-align: left;">MET</td>
<td style="text-align: left;">ENSG00000105976</td>
<td style="text-align: left;">validated</td>
</tr>
<tr class="odd">
<td style="text-align: left;">ETS1</td>
<td style="text-align: left;">ENSG00000134954</td>
<td style="text-align: left;">validated</td>
</tr>
<tr class="even">
<td style="text-align: left;">TGFB1</td>
<td style="text-align: left;">ENSG00000105329</td>
<td style="text-align: left;">validated</td>
</tr>
<tr class="odd">
<td style="text-align: left;">CFTR</td>
<td style="text-align: left;">ENSG00000001626</td>
<td style="text-align: left;">validated</td>
</tr>
<tr class="even">
<td style="text-align: left;">FGG</td>
<td style="text-align: left;">ENSG00000171557</td>
<td style="text-align: left;">validated</td>
</tr>
<tr class="odd">
<td style="text-align: left;">MTOR</td>
<td style="text-align: left;">ENSG00000198793</td>
<td style="text-align: left;">validated</td>
</tr>
<tr class="even">
<td style="text-align: left;">SMAD4</td>
<td style="text-align: left;">ENSG00000141646</td>
<td style="text-align: left;">validated</td>
</tr>
<tr class="odd">
<td style="text-align: left;">NFE2L2</td>
<td style="text-align: left;">ENSG00000116044</td>
<td style="text-align: left;">validated</td>
</tr>
<tr class="even">
<td style="text-align: left;">PBX3</td>
<td style="text-align: left;">ENSG00000167081</td>
<td style="text-align: left;">validated</td>
</tr>
<tr class="odd">
<td style="text-align: left;">TTN</td>
<td style="text-align: left;">ENSG00000155657</td>
<td style="text-align: left;">validated</td>
</tr>
<tr class="even">
<td style="text-align: left;">TUG1</td>
<td style="text-align: left;">ENSG00000253352</td>
<td style="text-align: left;">validated</td>
</tr>
<tr class="odd">
<td style="text-align: left;">PTEN</td>
<td style="text-align: left;">ENSG00000284792</td>
<td style="text-align: left;">validated</td>
</tr>
<tr class="even">
<td style="text-align: left;">XIST</td>
<td style="text-align: left;">ENSG00000229807</td>
<td style="text-align: left;">validated</td>
</tr>
</tbody>
</table>


</div>
</div>
</figure>
</div>
</div>
<p>The target genes identified through multiMiR were subjected to functional enrichment analysis using KEGG where five significantly enriched pathways (p &lt; 0.05) (<a href="#fig-enrichment_hsamiR_144_3p" class="quarto-xref">Figure&nbsp;13</a>). The most prominent pathways include:</p>
<ul>
<li><em>MicroRNAs in cancer</em> (KEGG ID: hsa05206): Highlighting the role of miRNAs in the regulation of gene expression, particularly in pathways associated with tumorigenesis and cancer progression.<br>
</li>
<li><em>Hepatocellular carcinoma</em> (KEGG ID: hsa05225): Specifically linked to liver cancer and the molecular mechanisms underlying its development and progression.<br>
</li>
<li><em>FoxO signaling pathway</em> (KEGG ID: hsa04068): A critical pathway regulating oxidative stress response, apoptosis, and metabolism, which plays a significant role in cancer and aging.<br>
</li>
<li><em>Gastric cancer</em> (KEGG ID: hsa05226): Associated with molecular and cellular alterations characteristic of gastric tumorigenesis.<br>
</li>
<li><em>Cellular senescence</em> (KEGG ID: hsa04218): Related to aging and the permanent arrest of the cell cycle due to stress signals or damage.</li>
</ul>
<div class="cell">
<div class="sourceCode cell-code" id="cb106"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb106-1"><a href="#cb106-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Gene Ontology (GO) para todos los genes combinados</span></span>
<span id="cb106-2"><a href="#cb106-2" aria-hidden="true" tabindex="-1"></a>gene_symbols_miR_144_3p_validated <span class="ot">&lt;-</span> filter_val_hsa_miR_144_3p<span class="sc">$</span>target_entrez <span class="sc">%&gt;%</span> <span class="fu">unique</span>()</span>
<span id="cb106-3"><a href="#cb106-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb106-4"><a href="#cb106-4" aria-hidden="true" tabindex="-1"></a><span class="co"># Realizar análisis de enriquecimiento</span></span>
<span id="cb106-5"><a href="#cb106-5" aria-hidden="true" tabindex="-1"></a><span class="co">#enrich_result_hsa_miR_144_3p &lt;- enrichKEGG(gene = gene_symbols_miR_144_3p_validated, organism = "hsa", pvalueCutoff = 0.05)</span></span>
<span id="cb106-6"><a href="#cb106-6" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb106-7"><a href="#cb106-7" aria-hidden="true" tabindex="-1"></a><span class="co"># Guardar el data.frame en un archivo .RData</span></span>
<span id="cb106-8"><a href="#cb106-8" aria-hidden="true" tabindex="-1"></a><span class="co">#save(enrich_result_hsa_miR_144_3p, file = "data/KEGG_hsa_miR_144_3p.RData")</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
<div class="cell">
<div class="sourceCode cell-code" id="cb107"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb107-1"><a href="#cb107-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Visualizar resultados</span></span>
<span id="cb107-2"><a href="#cb107-2" aria-hidden="true" tabindex="-1"></a><span class="co">#| label: fig-enrichment_hsamiR_144_3p</span></span>
<span id="cb107-3"><a href="#cb107-3" aria-hidden="true" tabindex="-1"></a><span class="co">#| fig-cap: "Top 5 Validated Enriched Biological Processes of *hsa-miR-144-3p* "</span></span>
<span id="cb107-4"><a href="#cb107-4" aria-hidden="true" tabindex="-1"></a><span class="co">#| </span></span>
<span id="cb107-5"><a href="#cb107-5" aria-hidden="true" tabindex="-1"></a><span class="fu">load</span>(<span class="st">"data/KEGG_hsa_miR_144_3p.RData"</span>)</span>
<span id="cb107-6"><a href="#cb107-6" aria-hidden="true" tabindex="-1"></a><span class="fu">barplot</span>(enrich_result_hsa_miR_144_3p, <span class="at">showCategory =</span> <span class="dv">5</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<div id="fig-enrichment_hsamiR_144_3p" class="quarto-float quarto-figure quarto-figure-center anchored" data-fig-pos="H">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-enrichment_hsamiR_144_3p-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<img src="Results_files/figure-html/fig-enrichment_hsamiR_144_3p-1.png" class="img-fluid figure-img" data-fig-pos="H" width="768">
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-enrichment_hsamiR_144_3p-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;13: Top 5 Significant KEGG Pathways of <em>hsa-miR-144-3p</em>
</figcaption>
</figure>
</div>
</div>
</div>
<p>These pathways reflect key biological processes, including tumorigenesis, stress response, and cell cycle regulation, underscoring the importance of miRNAs in these mechanisms. The findings provide valuable insights into potential molecular interactions and pathways relevant to the studied system.</p>
</section>
</section>
</section>

</main>
<!-- /main column -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const onCopySuccess = function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  }
  const getTextToCopy = function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button:not([data-in-quarto-modal])', {
    text: getTextToCopy
  });
  clipboard.on('success', onCopySuccess);
  if (window.document.getElementById('quarto-embedded-source-code-modal')) {
    const clipboardModal = new window.ClipboardJS('.code-copy-button[data-in-quarto-modal]', {
      text: getTextToCopy,
      container: window.document.getElementById('quarto-embedded-source-code-modal')
    });
    clipboardModal.on('success', onCopySuccess);
  }
    var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
    var mailtoRegex = new RegExp(/^mailto:/);
      var filterRegex = new RegExp('/' + window.location.host + '/');
    var isInternal = (href) => {
        return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
    }
    // Inspect non-navigation links and adorn them if external
 	var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool):not(.about-link)');
    for (var i=0; i<links.length; i++) {
      const link = links[i];
      if (!isInternal(link.href)) {
        // undo the damage that might have been done by quarto-nav.js in the case of
        // links that we want to consider external
        if (link.dataset.originalHref !== undefined) {
          link.href = link.dataset.originalHref;
        }
      }
    }
  function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
    const config = {
      allowHTML: true,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start',
    };
    if (contentFn) {
      config.content = contentFn;
    }
    if (onTriggerFn) {
      config.onTrigger = onTriggerFn;
    }
    if (onUntriggerFn) {
      config.onUntrigger = onUntriggerFn;
    }
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      if (note) {
        return note.innerHTML;
      } else {
        return "";
      }
    });
  }
  const xrefs = window.document.querySelectorAll('a.quarto-xref');
  const processXRef = (id, note) => {
    // Strip column container classes
    const stripColumnClz = (el) => {
      el.classList.remove("page-full", "page-columns");
      if (el.children) {
        for (const child of el.children) {
          stripColumnClz(child);
        }
      }
    }
    stripColumnClz(note)
    if (id === null || id.startsWith('sec-')) {
      // Special case sections, only their first couple elements
      const container = document.createElement("div");
      if (note.children && note.children.length > 2) {
        container.appendChild(note.children[0].cloneNode(true));
        for (let i = 1; i < note.children.length; i++) {
          const child = note.children[i];
          if (child.tagName === "P" && child.innerText === "") {
            continue;
          } else {
            container.appendChild(child.cloneNode(true));
            break;
          }
        }
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(container);
        }
        return container.innerHTML
      } else {
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(note);
        }
        return note.innerHTML;
      }
    } else {
      // Remove any anchor links if they are present
      const anchorLink = note.querySelector('a.anchorjs-link');
      if (anchorLink) {
        anchorLink.remove();
      }
      if (window.Quarto?.typesetMath) {
        window.Quarto.typesetMath(note);
      }
      if (note.classList.contains("callout")) {
        return note.outerHTML;
      } else {
        return note.innerHTML;
      }
    }
  }
  for (var i=0; i<xrefs.length; i++) {
    const xref = xrefs[i];
    tippyHover(xref, undefined, function(instance) {
      instance.disable();
      let url = xref.getAttribute('href');
      let hash = undefined; 
      if (url.startsWith('#')) {
        hash = url;
      } else {
        try { hash = new URL(url).hash; } catch {}
      }
      if (hash) {
        const id = hash.replace(/^#\/?/, "");
        const note = window.document.getElementById(id);
        if (note !== null) {
          try {
            const html = processXRef(id, note.cloneNode(true));
            instance.setContent(html);
          } finally {
            instance.enable();
            instance.show();
          }
        } else {
          // See if we can fetch this
          fetch(url.split('#')[0])
          .then(res => res.text())
          .then(html => {
            const parser = new DOMParser();
            const htmlDoc = parser.parseFromString(html, "text/html");
            const note = htmlDoc.getElementById(id);
            if (note !== null) {
              const html = processXRef(id, note);
              instance.setContent(html);
            } 
          }).finally(() => {
            instance.enable();
            instance.show();
          });
        }
      } else {
        // See if we can fetch a full url (with no hash to target)
        // This is a special case and we should probably do some content thinning / targeting
        fetch(url)
        .then(res => res.text())
        .then(html => {
          const parser = new DOMParser();
          const htmlDoc = parser.parseFromString(html, "text/html");
          const note = htmlDoc.querySelector('main.content');
          if (note !== null) {
            // This should only happen for chapter cross references
            // (since there is no id in the URL)
            // remove the first header
            if (note.children.length > 0 && note.children[0].tagName === "HEADER") {
              note.children[0].remove();
            }
            const html = processXRef(null, note);
            instance.setContent(html);
          } 
        }).finally(() => {
          instance.enable();
          instance.show();
        });
      }
    }, function(instance) {
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            div.style.left = 0;
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
        // Handle positioning of the toggle
    window.addEventListener(
      "resize",
      throttle(() => {
        elRect = undefined;
        if (selectedAnnoteEl) {
          selectCodeLines(selectedAnnoteEl);
        }
      }, 10)
    );
    function throttle(fn, ms) {
    let throttle = false;
    let timer;
      return (...args) => {
        if(!throttle) { // first call gets through
            fn.apply(this, args);
            throttle = true;
        } else { // all the others get throttled
            if(timer) clearTimeout(timer); // cancel #2
            timer = setTimeout(() => {
              fn.apply(this, args);
              timer = throttle = false;
            }, ms);
        }
      };
    }
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
</div> <!-- /content -->




</body></html>